window._articleFullText = window._articleFullText || {};
window._articleFullText[1795] = "<div class=\"col-lg-12 col-md-12 col-sm-12 col-xs-12\">\n\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t<div  style=\"width:100%; float:left;  padding-bottom:0px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:100%; float:left; min-height:45px; text-align:justify\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<a href=\"\" name=\"summary_en\"></a>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<!--<h2>Summary</h2>-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<p class=\"yazi\">\r\nIn this report it was presented two members of a family with non-syndromic high frequency sensorineural hearing loss who are homozygous for a novel pathogenic SYNE4 variant c.129-1G>T. This case report provides supportive evidence for the causative role of SYNE4 variants in hearing loss by presenting an additional family with a novel DNA variant.\r\n</p>\r\n\r\n\r\n<h4>Abstract</h4>\r\n<p><strong>Background:</strong> Hearing loss is the most common sensory deficit with many genetic and environmental underpinnings. While causative DNA variants have been identified in over 100 genes, most deafness-causing variants are rare, apart from a few exceptions. A single <em>SYNE4</em> variant co-segregating with hearing loss has recently been reported in two Middle-Eastern families. <br /><strong>Case Report:</strong> In this report we present two members of a family with non-syndromic high frequency sensorineural hearing loss who are homozygous for a novel pathogenic <em>SYNE4</em> variant c.129-1G&gt;T. <br /><strong>Conclusion:</strong> This case report provides supportive evidence for the causative role of <em>SYNE4</em> variants in hearing loss by presenting an additional family with a novel DNA variant.</p>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<!--\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t<a class=\"selected_menu\" href=\"#summary_en\">Summary</a><a class=\"menu\" href=\"#introduction\">Introduction</a><a class=\"menu\" href=\"#methods\">Methods</a><a class=\"menu\" href=\"#results\">Results</a><a class=\"menu\" href=\"#reference\">Reference</a>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t<div  style=\"width:100%; float:left;  padding-bottom:0px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:100%; float:left; min-height:45px; text-align:justify\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<a href=\"\" name=\"introduction\"></a>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<!--<h2>Introduction</h2>-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<hr color=\"#f2f2f2\" size=\"1\">\r\n<p>Hearing loss (HL) is the most frequently occurring of all congenital sensory deficits; displaying an incidence of 0.1-0.2% in the newborn population (1). While HL has many known genetic and environmental causes, genetic aetiologies predominate in the developed world (2). Genetic or hereditary HL can present as part of a larger syndrome or have a non-syndromic phenotype where HL is the only clinical symptom. Previous studies have elucidated that the genetic transmission of non-syndromic HL is autosomal recessive in 75-85% of all cases, and autosomal dominant in 15-25% of cases, with a small proportion of cases showing X-linked or mitochondrial inheritance (1-2%) (3).</p>\r\n<p>More than 140 loci have been found to be associated with non-syndromic HL. There have also been over 400 HL syndromes described in the Online Mendelian Inheritance in Man database which show a distinct constellation of clinical features including HL (4,5).</p>\r\n<p>Despite this large number of identified mutations, roughly half of all cases of non-syndromic congenital deafness in certain populations can be traced to a single gene: <em>GJB2</em>. The remaining deafness-causing mutations are rare and many are typically found in only a single, or very few families (6). The goal of this report is to identify the genetic aetiology of deafness in individuals with rare forms of non-syndromic sensorineural HL. A novel variant in a rare deafness-causing gene (<em>SYNE4</em>; MIM: 615535) was identified and helps to clarify the genotypic profile of this uncommon form of sensorineural HL.</p>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<!--\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t<a class=\"menu\" href=\"#summary_en\">Summary</a><a class=\"selected_menu\" href=\"#introduction\">Introduction</a><a class=\"menu\" href=\"#methods\">Methods</a><a class=\"menu\" href=\"#results\">Results</a><a class=\"menu\" href=\"#reference\">Reference</a>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t<div  style=\"width:100%; float:left;  padding-bottom:0px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:100%; float:left; min-height:45px; text-align:justify\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<a href=\"\" name=\"methods\"></a>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<!--<h2>Methods</h2>-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<h3>CASE PRESENTATION </h3>\r\n<p>The study was approved by the IRB at the University of Miami and the Ethics Committee of Ankara University. Blood samples were collected after informed consent was obtained. We evaluated a family (family 951) consisting of a brother and sister (from the same mother and father) with HL born to Turkish parents who are first cousins.</p>\r\n<p>Individual II:1, the proband, is a 13-year-old female born after an uncomplicated pre- and neonatal period. She has bilateral post-lingual HL that was diagnosed three months prior to examination. She had no earache, ear discharge, or ear congestion, and no sense of pressure in the ear or dizziness. An audiogram performed at the time showed moderate bilateral sensorineural HL with a significant loss of higher frequencies (&gt;1 kHz; <a href=\"javascript:openWin('uploads/grafik/figure_BMJ_1795_0.jpg', 640, 480)\"><b>Figure 1a</b></a>). Inner ear and temporal magnetic resonance imaging (MRI) studies were normal. An electrocardiogram (EKG) and echocardiogram were unremarkable. A thorough physical exam including an ophthalmoscopy revealed no abnormalities. </p>\r\n<p>Individual II:2, the 11-year-old brother of II:1, was also noted to have HL two months prior to examination. He complained of hearing impairment in high-pitched sounds. He denied any earache, discharge, or congestion, and there was no sense of pressure in the ear or dizziness. An audiogram performed at the time showed a moderate bilateral sensorineural HL with sudden decreases in 1 kHz and 2 kHz. Inner ear and temporal MRI, echocardiogram and EKG were unremarkable.</p>\r\n<h4>Whole exome sequencing</h4>\r\n<p>Whole exome sequencing (WES), variant filtering/interpretation and Copy Number Variant analysis were performed in the proband to identify potential causative variants using previously published protocols (6). Sanger sequencing was performed for the confirmation and co-segregation of candidate variants.</p>\r\n<p>WES revealed that the proband of family 951 was homozygous for the <em>SYNE4</em> NM-001039876.1:c.129-1G&gt;T variant (<a href=\"javascript:openWin('uploads/grafik/table_BMJ_1795_0.jpg', 640, 480)\"><b>Table 1</b></a>). This variant was confirmed by Sanger sequencing (<a href=\"javascript:openWin('uploads/grafik/figure_BMJ_1795_0.jpg', 640, 480)\"><b>Figure 1b</b></a>). The affected brother was also shown to be homozygous and both parents were heterozygous for the variant (<a href=\"javascript:openWin('uploads/grafik/figure_BMJ_1795_0.jpg', 640, 480)\"><b>Figure 1c</b></a>).</p>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<!--\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t<a class=\"menu\" href=\"#summary_en\">Summary</a><a class=\"menu\" href=\"#introduction\">Introduction</a><a class=\"selected_menu\" href=\"#methods\">Methods</a><a class=\"menu\" href=\"#results\">Results</a><a class=\"menu\" href=\"#reference\">Reference</a>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t<div  style=\"width:100%; float:left;  padding-bottom:0px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:100%; float:left; min-height:45px; text-align:justify\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<a href=\"\" name=\"results\"></a>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<!--<h2>Results</h2>-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<h3>DISCUSSION</h3>\r\n<p>HL is the most common sensory disorder which displays extreme genetic heterogeneity. In this case study, we identified a novel variant in <em>SYNE4</em> causing HL.</p>\r\n<p><em>SYNE4</em> encodes nesprin-4 (Nesp4), which localises to the outer nuclear membrane and is part of the linker of nucleoskeleton and cytoskeleton (LINC) complex in the nuclear envelope. On one end, Nesp4 attaches to the protein Sun1, another member of the LINC complex, which is anchored beneath the inner nuclear membrane and spans the perinuclear space. On the opposite end, Nesp4 attaches kinesin in the cytoplasm, effectively forming a bridge between the nuclear membrane and the cytoskeleton of the cell, allowing the nucleus to be securely anchored within the cytosol. This complex plays an essential role in allowing cell nuclei to localise to a specific region within the cytosol, and to relocate within the cell as necessary, which is essential for cell function (7).</p>\r\n<p>Human outer hair cells, which have a columnar morphology, show preferential nuclear localisation to the basal part of the cell. These cells are extremely sensitive to environmental stressors and lack the ability to regenerate and their degeneration results in permanent HL that cannot be recovered. Proper function of outer hair cells is intrinsically tied to their motile nature and ability to change length upon membrane hyperpolarisation and depolarisation (8). It is therefore reasonable to predict that a compromised connection between the nucleus and cytoskeleton would be detrimental to the proper transmission of mechanical signals from the cell to the nucleus and therefore impact proper cell function.</p>\r\n<p>In a 2013 publication, Horn et al. (9) identified progressive high-frequency HL (MIM: 615540) in two consanguineous families of Iraqi Jewish ancestry with a homozygous deletion mutation, c.228delAT, in <em>SYNE4</em>, presumably rendering the product protein non-functioning. As part of the same publication, knockout mice of <em>SYNE4</em> showed apical -as opposed to the normal basal- localisation of the nuclei and degeneration of outer hair cells as hearing matured, leading to progressive HL. To affirm that a wholly intact LINC complex is essential to outer heir cell survival, Horn et al. (9) also created a SUN1 knockout mouse model, removing the protein to which Nesp4 connects. These mice demonstrated an identical phenotype to the <em>SYNE4</em> knockout mice (9).</p>\r\n<p>Affected members of family 951 mirrored this clinical presentation of progressive, high-frequency HL, lending further support to the association of deafness with the <em>SYNE4</em> variant. Additionally, the novel variant identified in family 951 is a splice-site mutation. According to the 2015 The American College of Medical Genetics and Genomics (ACMG) Standards and Guidelines, this mutation has very strong evidence of pathogenicity as it is located at the canonical-1 splice site, where a loss of function is a known mechanism of disease (10), as reported by Horn et al. (9). The ACMG also recommends the use of multiple computational predictive programs to assess the pathogenicity of missense variants. The variant in family 951 was predicted to be disease causing by both Mutation Taster and CADD with a score of 24.3 (<a href=\"javascript:openWin('uploads/grafik/table_BMJ_1795_0.jpg', 640, 480)\"><b>Table 1</b></a>). Furthermore, when compared to the deletion mutation identified by Horn et al. (9), the mutation in family 951 provides genotypic heterogeneity within the similar clinical presentation shown by <em>SYNE4</em> dysfunction in these families.</p>\r\n<p>Identification of this novel variant in <em>SYNE4</em> further elucidates the genotypic profile of the rare deafness phenotype with which it is associated. Its identification also lends further support to the pathogenic role of <em>SYNE4</em> and fortifies the already growing list of causative variants of hereditary HL. Continual expansion of this database will allow physicians to more speedily and accurately identify culprit gene mutation variants in patients presenting with less common HL phenotypes.</p>\r\n<p><strong>Financial Disclosure:</strong> This work was supported by the National Institutes of Health grant R01DC009645 to M.T.</br><strong>Conflict of Interest:</strong> No conflict of interest was declared by the authors.</p>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<!--\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t<a class=\"menu\" href=\"#summary_en\">Summary</a><a class=\"menu\" href=\"#introduction\">Introduction</a><a class=\"menu\" href=\"#methods\">Methods</a><a class=\"selected_menu\" href=\"#results\">Results</a><a class=\"menu\" href=\"#reference\">Reference</a>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t<div  style=\"width:100%; float:left;  padding-bottom:0px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:100%; float:left; min-height:45px; text-align:justify\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<a href=\"\" name=\"reference\"></a>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<!--<h2>Reference</h2>-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<h3>REFERENCES</h3>\r\n<p>1. Kemperman MH, Hoefsloot LH, Cremers CW. Hearing loss and connexin 26. J R Soc Med 2002;95:171-7. <br />2. Morton CC, Nance WE. Newborn hearing screening a silent revolution. N Engl J Med 2006;354:2151-64. <br />3. Vozzi D, Morgan A, Vuckovic D, D'Eustacchio A, Abdulhadi K, Rubinato E, et al. Hereditary hearing loss: a 96 gene targeted sequencing protocol reveals novel alleles in a series of Italian and Qatari patients. Gene 2014;542:209-16. <br />4. Finsterer J, Fellinger J. Nuclear and mitochondrial genes mutated in nonsyndromic impaired hearing. Int J Pediatr Otorhinolaryngol 2005;69:621-47.<br />5. Atik T, Bademci G, Diaz-Horta O, Blanton SH, Tekin M. Whole-exome sequencing and its impact in hereditary hearing loss. Genet Res (Camb) 2015;97:e4.<br />6. Bademci G, Foster J, Mahdieh N, Bonyadi M, Duman D, Cengiz FB, et al. Comprehensive Analysis via Exome Sequencing Uncovers Genetic Etiology in Autosomal Recessive Non-Syndromic Deafness in a Large Multiethnic Cohort. Genet Med 2016;18:364-71. <br />7. Starr DA, Han M. ANChors away: an actin based mechanism of nuclear positioning. J Cell Sci 2003;116:211-6. <br />8. Dallos P. Cochlear amplification, outer hair cells and prestin. Curr Opin Neurobiol 2008;18:370-6. <br />9. Horn HF, Brownstein Z, Lenz DR, Shivatzki S, Dror AA, Dagan-Rosenfeld O, et al. The LINC complex is essential for hearing. J Clin Invest 2013;123:740-50.<br />10. Richards S, Aziz N, Bale S, Bick D, Das S, Gastier-Foster J, et al. Standards and guidelines for the interpretation of sequence variants: a joint consensus recommendation of the American College of Medical Genetics and Genomics and the Association for Molecular Pathology. Genet Med 2015;17:405-24.</p>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<!--\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t<a class=\"menu\" href=\"#summary_en\">Summary</a><a class=\"menu\" href=\"#introduction\">Introduction</a><a class=\"menu\" href=\"#methods\">Methods</a><a class=\"menu\" href=\"#results\">Results</a><a class=\"selected_menu\" href=\"#reference\">Reference</a>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\n\t\t\n\t\t\t\t\t\t\t\t\t\t\t        \t</div>";
