window._articleFullText = window._articleFullText || {};
window._articleFullText[2032] = "<div class=\"col-lg-12 col-md-12 col-sm-12 col-xs-12\">\n\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t<div  style=\"width:100%; float:left;  padding-bottom:0px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:100%; float:left; min-height:45px; text-align:justify\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<a href=\"\" name=\"summary_en\"></a>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<!--<h2>Summary</h2>-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<h4>Abstract</h4>\r\n<p><strong>Background</strong>: Multiple myeloma is a plasma cell dyscrasia characterized by transformation of B cells into malignant cells. Although there are data regarding the molecular pathology of multiple myeloma, the molecular mechanisms of the disease have not been fully elucidated.<br /><strong>Aims</strong>: To investigate the gene expression profiles in bone marrow myeloma cells via RNA-sequencing technology.<br /><strong>Study Design</strong>: Cell study.<br /><strong>Methods</strong>: Myeloma cells from four patients with untreated multiple myeloma and B cells from the bone marrow of four healthy donors were sorted using a FACSAria II flow cytometer. The patient pool of myeloma cells and the control pool of B cells were the two comparative groups. A transcriptome analysis was performed and the results were analyzed using bioinformatics tools.<br /><strong>Results</strong>: In total, 18.806 transcripts (94.4%) were detected in the pooled multiple myeloma patient cells. A total of 992 regions were detected as new exon candidates or alternative splicing regions. In addition, 490 mutations (deletions or insertions), 1.397 single nucleotide variations, 415 fusion transcripts, 132 frameshift mutations, and 983 fusions, which were reported before in the National Center for Biotechnology Information, were detected with unknown functions in patients. A total of 35.268 transcripts were obtained (71%) (25.355 transcripts were defined previously) in the control pool. In this preliminary study, the first 50 genes were analyzed with the MSigDB, Enrichr, and Panther gene set enrichment analysis programs. The molecular functions, cellular components, pathways, and biological processes of the genes were obtained and statistical values were determined using bioinformatics tools and are presented as a supplemental file.<br /><strong>Conclusion</strong>: <em>EEF1G</em>, <em>ITM2C</em>, <em>FTL</em>, <em>CLPTM1L</em>, and C<em>YBA</em> are identified as possible candidate genes associated with myelomagenesis.</p>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<!--\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t<a class=\"selected_menu\" href=\"#summary_en\">Summary</a><a class=\"menu\" href=\"#introduction\">Introduction</a><a class=\"menu\" href=\"#reference\">Reference</a>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t<div  style=\"width:100%; float:left;  padding-bottom:0px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:100%; float:left; min-height:45px; text-align:justify\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<a href=\"\" name=\"introduction\"></a>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<!--<h2>Introduction</h2>-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<hr size=\"1\" color=\"#f2f2f2\">\r\n<p>Multiple myeloma (MM) is a cancer of plasma cells in which abnormal types of immunoglobulins are produced that can be measured in blood and urine (1). MM is a clonal B cell malignancy characterized by a clinical pentad in the bone marrow (2). According to the National Cancer Institute Surveillance, Epidemiology, and Results Program in 2017, 30,280 new cases of MM were diagnosed, and 12,590 deaths occurred in the United States (3).</p>\r\n<p>The etiology of MM depends on many factors, such as the environment, chemical agents, viruses, and genetic factors (4). MM is caused by several molecular mechanisms, such as activation of oncogenes, genomic instability, and chromosome abnormalities (5). Accumulated plasma cells in the bone marrow of patients with MM are identified by surface membrane antigens, such as CD38, CD138, CD56, CD117, and CD33. Although most myeloma cells typically express CD38(+), CD138(+), CD56(+), and CD19(-), these cells may indicate genetic heterogeneity (6). Next generation sequencing is evolving rapidly, and RNA-seq has become a widely used tool for understanding the molecular mechanisms of human cancers. The transcriptome is the complete set of transcripts in a cell that allows for the identification of exons and alternative splicing/isoforms and novel RNAs. Massive parallel sequencing platforms are used to measure the differential expression of transcripts (7).</p>\r\n<p>In the present study, we provide comprehensive insight into the transcriptomes of patients with MM and healthy donors. Many of the genes were determined in transcriptome profiles of pooled myeloma cells and were determined to play a role in MM pathogenesis via intracellular pathways, gene expression patterns, biological functions, and protein-protein interactions using <em>in silico</em> tools. Our transcriptomic profile obtained data to evaluate differential expression of all transcripts, alternative new splicing variants, mutations, and fusion genes. These results will contribute to the understanding of myeloma pathogenesis and provide valuable information for prognostication and new therapies.</p>\r\n<h3>MATERIALS AND METHODS</h3>\r\n<h4>Sample collection</h4>\r\n<p>This study was approved (2010/108-28) by the Ethics Committee of İstanbul University Faculty of Medicine. Written informed consent Helsinki Declaration and ethics committee documents was obtained from all patients and healthy donors. We performed RNA-seq using the Ion Torrent Personal Genome Machine (PGM) platform to compare the transcriptome profiles of four newly diagnosed patients with untreated MM and four healthy donors. Bone marrow was aspirated from the hip bones of all patients and donors. The bone marrow samples were subjected to Ficoll gradient centrifugation (1.077 g/mL Ficoll), and the mononuclear cells were collected. The viability and absolute cell counts were determined by the Vi-CELL&trade; XR Cell Counter (Beckman Coulter, Brea, CA, USA).</p>\r\n<h4>Fluorescence-activated cell sorting</h4>\r\n<p>Myeloma cells (CD38+, CD138+, CD19-, and CD56+) and healthy B cells (CD38+, CD138+, CD19+, and CD56-) were selected from bone marrow mononuclear cells using a gating strategy by simultaneously specifying cell surface markers, and by determining forward and side light scattering characteristics on the FACSAria II Cell Sorter (Becton Dickinson, San Jose, CA, USA) (<a href=\"javascript:openWin('uploads/grafik/figure_BMJ_2032_0.jpg', 640, 480)\"><b>Figures 1</b></a>, <a href=\"javascript:openWin('uploads/grafik/figure_BMJ_2032_1.jpg', 640, 480)\"><b>2</b></a>). The antibodies used for activating fluorescence and cell sorting were CD138/SYNDECAN-1 (cat: 347216) allophycocyanin, CD38 (cat: 340909) fluorescein isothiocyanate, CD19 (cat: 345777) phycoerythrin, CD56 (cat: 557747), and phycoerythrin cyanin (Becton Dickinson). The cells were passed through a 100 &micro;m nozzle tip at a speed of approximately 50,000 events per sec. The images were taken and the analysis was performed using FACS Diva Software 6.1.2. The sorted cells were frozen for RNA isolation.</p>\r\n<h4>RNA isolation</h4>\r\n<p>RNA was extracted from the sorted cells using the PureLink RNA Microkit (cat: 12183_016; Invitrogen, Carlsbad, CA, USA). Before proceeding to rRNA depletion, the quantity and quality of total RNA was evaluated using the RNA 6000 Pico kit on the Agilent 2100 Bioanalyzer (Agilent Technologies, Anaheim, CA, USA). After checking the quantity and quality of the RNA, we pooled the RNA samples from the four untreated MM patients and four healthy donors. The workflow of the study is summarized in <a href=\"javascript:openWin('uploads/grafik/table_BMJ_2032_0.jpg', 640, 480)\"><b>Table 1</b></a>.</p>\r\n<h4>rRNA depletion</h4>\r\n<p>rRNA depletion was performed using the Eukaryotic Ribominus kit (cat no: A10837_2/A10837_08; Invitrogen). The quantity and quality of the mRNA was evaluated using the RNA 6000 Pico kit on Agilent 2100 Bioanalyzer.</p>\r\n<h4>Library preparation and RNA sequencing</h4>\r\n<p>The RNA-sequencing libraries were prepared from the pooled RNA of the patients and healthy donors. The libraries were constructed using the Ion Total RNA-seq V2 kit protocol for transcriptome profiling of low-input RNA samples (April 2011). The quality of the libraries was validated using the Agilent High Sensitivity DNA kit on the Agilent 2100 Bioanalyzer (<a href=\"javascript:openWin('uploads/grafik/figure_BMJ_2032_2.jpg', 640, 480)\"><b>Figure 3</b></a>). The final library concentrations were calculated after validating the libraries, using the Ion PGM&trade; 200 Xpress Template kit (cat no: 4474280). Each library was amplified by Emulsion polymerase chain reaction of ion sphere particles (ISPs). These ISPs were recovered from the emulsion. The templated ISPs and unbound ISPs were determined by fluorometric assay on Qubit&reg; 2.0.</p>\r\n<p>The Ion PGM&trade; system was cleaned and initialized before sequencing. Sequencing was accomplished using the Ion PGM&trade; 200 Sequencing v2 kit (cat: 4474004; Life Technologies, Carlsbad, CA, USA) according to the manufacturer&rsquo;s instructions. Pooled samples were sequenced with a semiconductor-based sequencing system using the Ion 318&trade; Chip (lots: P30518.1, AA0150428, and AA0150429; Life Technologies).</p>\r\n<h4>RNA-sequencing analysis</h4>\r\n<p>The pooled RNA from the patients with MM and that from the controls was subjected to massively parallel cDNA sequencing at İstanbul University Whole Genome Sequencing Laboratory Aziz Sancar Institute of Experimental Medicine. The sequencing data were analyzed with the PSSC Labs Big Data Server (Lake Forest, CA, USA). The quality of the raw sequencing data was checked and the data were preprocessed according to two criteria: adaptors and low quality sequences (readings with ambiguous N&rsquo;s) were removed, and data with a quality score (Qscore) &lt;30 was trimmed.</p>\r\n<h4>Bioinformatics analysis</h4>\r\n<p>The trimmed sequencing reads were aligned to the UCSC human reference genome (build GRCh37/hg19) using TopHat v2.0.6, which incorporates Bowtie v0.12.8 software to build the alignment. The expression level for each transcript was normalized to the reads per kilobase of the exon model per million mapped reads (RPKM) (8). Cufflinks v2.0.2 was used to operate the original alignment file generated by tophat and the gene transfer format file for genome annotation to determine the difference between the expressed genes. The first 50 genes from the pooled MM RNA that were highly differentially expressed were analyzed using the Gene Set Enrichment Analysis (GSEA) program (<a href=\"javascript:openWin('uploads/grafik/figure_BMJ_2032_3.jpg', 640, 480)\"><b>Figure 4</b></a>) and also demonstrated protein interactions.</p>\r\n<h3>RESULTS</h3>\r\n<p>The cDNA libraries from the patients with MM and the control group were subjected to massively parallel transcriptome sequencing. Of the 18,806 total transcripts obtained by the transcriptome analysis, 17,760 were reported previously in pooled patients with MM. These transcripts were used in the downstream analysis.</p>\r\n<p>In total, 992 regions were detected and were candidates for new exons or alternative splicing regions. In addition, 490 deletions or insertions and 1.397 single nucleotide variations were detected; 415 fusion transcripts were defined. A total of 983 fusions, which were reported before in National Center for Biotechnology Information, were detected with unknown functions. In total, 132 frame shift mutations were identified in pooled patients with MM; 35,268 transcripts were obtained (71%) (25,355 transcripts were defined previously) in the control pool. Qscore values of about 35% on average for each chromosome were calculated. We measured the transcript values and identified the differentially expressed genes between the two groups using Cuffdiff/Cuflink. In total, we detected 12,453 expressed genes by calculating reads per kilobase million (RPKM) values and analyzed the data from the first 50 highly expressed selected genes in the pooled MM cells (<a href=\"javascript:openWin('uploads/grafik/table_BMJ_2032_1.jpg', 640, 480)\"><b>Table 2</b></a>) and compared the expression levels with the controls (<a href=\"javascript:openWin('uploads/grafik/table_BMJ_2032_2.jpg', 640, 480)\"><b>Table 3</b></a>). The eukaryotic elongation factor 2 (<em>EEF2</em>) gene was the most significantly expressed gene among the MM and normal cells according to our RPKM results. Our analysis included the majority of annotated human genes. The analysis of the whole transcriptome data revealed different expression levels of several genes, such as <em>JAK1</em>, <em>JAK2</em>, <em>JAK3</em>, <em>RAF</em>, <em>IL6R</em>, <em>NCAM (CD56)</em>, <em>WHSC1</em>, <em>MCL1</em>, <em>BCL2,</em> and <em>IGF1</em>, which showed myeloma pathogenesis as reported previously. These 50 genes were subjected to the GSEA using MSigDB. As a result, 11 of these genes had increased expression in plasma cells from patients with MM that significantly overlapped between <em>EEF1A1</em>, <em>UBC</em>, <em>UBB</em>, <em>CALR</em>, <em>CXCR4</em>, <em>JUND</em>, <em>FOS</em>, <em>PIM2</em>, <em>JUN</em>, <em>GAPDH</em>, and <em>HSP90B1</em> and were previously reported as upregulated in the Munshi_multiple _myeloma data set (<a href=\"javascript:openWin('uploads/grafik/figure_BMJ_2032_3.jpg', 640, 480)\"><b>Figure 4</b></a>ffigure4>). The biological functions, molecular processes, and pathways of these genes were determined using the online tool Panther-GO (Figures 5-7). These genes were also analyzed by the String v9.0 program to demonstrate protein-protein interactions (<a href=\"javascript:openWin('uploads/grafik/figure_BMJ_2032_7.jpg', 640, 480)\"><b>Figure 8</b></a>). We performed the bioinformatics calculations using the Enrichr GSEA program. Computational bioinformatics was used to explore the deep relationships between the genes in the annotated gene sets and between other data sets. We investigated and visualized the overlap between our data sets within the Enrichr program to compare with other web-server tools and resources that serve gene set libraries. The molecular functions, cellular components, pathways, and biological processes are presented as a supplemental file.</p>\r\n<h3>DISCUSSION</h3>\r\n<p>Decades of scientific research data backed up with array technologies and next generation sequencing technologies have shown us two important contributors to MM pathogenesis. One is the interactions between myeloma cells and the microenvironment and the other is malignant clone genetic transformation (9,10). In this study, we purified and directly sorted myeloma cells. Multicolor flow cytometry is a sensitive method to analyze plasma cell immunophenotypes and identify normal and neoplastic plasma cell populations. In our study, the surface markers of malignant B cells obtained from the literature were compared with those of the MM group with CD38(+), 138(+), CD56(+), CD19(-) B cell separation of the control group using CD38(+), 138(+), CD56(-), CD19(+) surface markers by multicolor flow cytometry (11). The transcriptome patterns of the first 50 highly expressed genes in <a href=\"javascript:openWin('uploads/grafik/figure_BMJ_2032_3.jpg', 640, 480)\"><b>Figure 4</b></a> and in those in <a href=\"javascript:openWin('uploads/grafik/figure_BMJ_2032_4.jpg', 640, 480)\"><b>Figures 5</b></a>,<a href=\"javascript:openWin('uploads/grafik/figure_BMJ_2032_5.jpg', 640, 480)\"><b>6</b></a>,<a href=\"javascript:openWin('uploads/grafik/figure_BMJ_2032_6.jpg', 640, 480)\"><b>7</b></a> were analyzed, and a pathway network was constructed to better understand the relationships among them. Technologies must be developed to identify related genes that show potential to play a role in transforming normal cells into myeloma cells. GSEA MSigD identified genes were included in previously known and myelomagenesis signaling related pathways. In addition, the families of these genes were analyzed with the same program. According to the results, <em>JUN</em> was in the oncogene family; <em>FOSB</em>, <em>JUN</em>, <em>JUNB</em>, <em>JUND</em>, and <em>KLF </em>were in the transcription factor family; <em>PIM2</em> and <em>SIK1</em> were in the protein kinase family; and <em>CXCR4</em>, <em>CD74,</em> <em>ICAM3</em>, and <em>SDC1</em> were cell differentiation markers. In particular, the first 50 genes supported that the ubuquitin genes, such as <em>UBB</em>, <em>UBC</em>, <em>EEF2</em>, were related to the pathogenesis of MM. The ubiquitin cascade system is a central contributor to cellular processes that regulate protein stability, trafficking, and activation (12,13). Proteosome inhibitors have been used for many years as a basic therapeutic strategy for treating MM and have been developed as antimyeloma therapy by focusing on this ubiquitin proteosome cascade (14). Losada et al. (15) reported that plitidepsin has successfully concluded a phase-III clinical trial for MM. Antitumor activity was achieved by targeting plitidepsin to <em>EEF1A2</em>. <em>EEF1A2</em> has proto-oncogenic activity, and it has been reported to be abnormally expressed in many human tumors including MM. In our study, the <em>EEF2</em>, <em>EEF1G</em>, and <em>EEF1A1</em> genes were overexpressed and may be responsible for inhibiting apoptosis and controlling unfolded protein degradation by proteasomes similar to the EEF1G gene. Prosaposin is a lysosomal protein that has pleiotropic growth factor activity. It is known to be related to the growth of breast cancer and to increase ER levels through the mitogen activated protein kinase (MAPK) pathway. In addition to gallbladder cancer, it operates as a biomarker and promotes increased degradation of ceramides, ensuring a survival advantage to cancer cells (16). Starlets et al. (17) reported that a cell surface molecule expressed on B cells binds CD74 to the macrophage migration-inhibition factor, activating CD74; thus, initiating a survival pathway. The humanized anti-CD74 monoclonal antibody acts as a potential therapeutic agent by exhibiting cell proliferation activity in MM (18). Our findings show that CD74 is strongly expressed in myeloma cells compared to healthy cells and has a role in the oncogenic process of cell proliferation and survival. Prosaposin is a protein encoded by the <em>PSAP</em> gene that interacts with CD74 and may play a role in MM carcinogenesis (Figure 8). Despite advances in the understanding of MM pathogenesis, the molecular pathways underlying the development of MM are still unknown. Our functional and pathway enrichment analysis proposed that major histocompatibility complex class I molecules called human leukocyte antigens (HLA-A, HLA-B, HLA-C, and HLA-E) are involved. Known as antigen processing and presenting machinery (AMP), these molecules are important for cell survival, cell cycle progression, and inhibition of apoptosis. Defects in the AMP lead to immune escape and continuity by cancer. As a result, it enables malignant transformation of cells.</p>\r\n<p>The enriched pathways for these upregulated genes are involved in peptide transport from the cytosol into the endoplasmic reticulum, antigen processing, peptide trimming, and assembly of the major histocompatibility complex class I loading complex (19). Leone et al. (20) compared the expression levels of calnexin, calreticulin, tapasin, and <em>ERp57</em> genes in premalignant plasma cells obtained from patients with monoclonal gammopathy of undetermined significance, those with MM, and normal plasma cells from healthy donors and showed that these levels are higher in patients with monoclonal gammopathy of undetermined significance and MM. It has been documented in cell lines from primary cells and various tumors, particularly MM, that <em>TAP1</em> and/or <em>TAP2</em> mRNA and protein levels are not detectable in small quantities. Defects in <em>TAP</em> genes play a role in the development of hematological malignancies (21). Our results included a mutated <em>TAP</em> gene (data not shown). Another related gene is calreticulin. This gene produces a calcium-binding protein that is a major component of the endoplasmic reticulum and has been shown in various cell types to be involved in regulating calcium homeostasis, as a ligand for integrins, and as a component of phagocytic synapses (22). Upregulation of the calreticulin gene is an adverse prognostic factor as the dominant pro-phagocytic signal in diverse tumors and is correlated with increased CD47 expression in cancer cells (23). The cisplatin resistance-related protein CRR9p (<em>CLPTM1L</em>) gene is overexpressed in lung cancer and knockdown of this gene prevents 95-D lung cancer cells from migrating and invading (24). Although the function of this gene is largely unknown, high expression levels of <em>CLPTM1L</em> have been observed in many cancers, and it is linked with cisplatin-induced apoptosis (25). Another study demonstrated that blocking <em>CLPTM1L</em> with interfering RNA inhibits lung tumorigenesis induced by K-RAS. That study suggested that <em>CLPTM1L</em> interacts with PI3 kinase and has a major role in <em>RAS</em>-induced <em>AKT</em> phosphorylation (26). Our results suggest that the <em>CLPTM1L</em> gene has a higher RPKM value in myeloma cells than in healthy donor cells. Cytokines and growth factors activate the phosphoinositide 3-kinase/AKT signaling cascade, creating life signals for myeloma cells by inhibiting apoptosis in MM. The RAS-MAPK pathway provides for proliferation of myeloma cells (27). The results of other studies on this gene are particularly related to PI3-K/AKT cascades, indicating that it should be considered a plausible candidate gene. Dytfell et al. reported that increased <em>TXNDC5</em> expression in plasma cells and serum is related to a poor response to bortezomid-based therapy in patients with newly diagnosed MM and in those with relapsed MM. Understanding MM biology and identifying drug-resistance biomarkers are vital to enable the development of individualized treatments. Proteomic signature results indicate that <em>TXNDC5</em>, which is a member of the protein disulfide isomerase family, shows increased expression and is involved in protection against oxidative stress and plays a major role in bortezomib treatment (28). Using in silico tools, we identified significant overlaps between <em>EEF1A1</em>, <em>UBC,</em> <em>UBB</em>, <em>CALR</em>, <em>CXCR4</em>, <em>JUND</em>, <em>FOS</em>, <em>PIM2</em>, <em>JUN</em>, <em>GAPDH</em>, and <em>HSP90B1</em>, which were previously reported to be upregulated in MM according to the Munıshı_ MM data set (<a href=\"javascript:openWin('uploads/grafik/figure_BMJ_2032_3.jpg', 640, 480)\"><b>Figure 4</b></a>). The Panther program revealed the results of molecular features, biological functions, and related pathways of the first 50 genes with the highest RPKM values. As we compared our findings with the results from other studies, we saw that biologic adhesion, biological processes, and metabolic processes were the same as these previous studies (Figures 5-7). Analyzing these genes using the String program may lead to further functional studies on protein-protein interactions (Figure 8). This transcriptome analysis of MM was performed for the first time in a Turkish population. As results, we determined some variations and different mRNA expression patterns in some of the previously reported genes particularly those in the ubiqutin-proteosomal pathway.</p>\r\n<p>In conclusion, <em>EEF1G</em>, <em>ITM2C</em>, <em>FTL</em>, <em>CLPTM1L</em>, and <em>CYBA</em> are possible candidate genes associated with myelomagenesis.</p>\r\n<p><strong>Conflict of Interest: </strong>No conflict of interest was declared by the authors.</p>\r\n<p><strong>Financial Disclosure: </strong>This study was supported by the Research Fund of İstanbul University (Grant number: 7348) İstanbul University, İstanbul, Turkey.</p>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<!--\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t<a class=\"menu\" href=\"#summary_en\">Summary</a><a class=\"selected_menu\" href=\"#introduction\">Introduction</a><a class=\"menu\" href=\"#reference\">Reference</a>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t<div  style=\"width:100%; float:left;  padding-bottom:0px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:100%; float:left; min-height:45px; text-align:justify\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<a href=\"\" name=\"reference\"></a>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<!--<h2>Reference</h2>-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<h4>REFERENCES</h4>\r\n<ol>\r\n<li>Hameed A, Brady JJ, Dowling P, Clynes M, O'Gorman P. Bone disease in multiple myeloma: pathophysiology and management. Cancer Growth Metastasis 2014;7:33-42.</li>\r\n<li>Dispenzieri A, Lacy QM, Greipp RP. Wintrobe's Clinical Hematology-Volume 2 Multiple Myeloma. In: Greer JP, Foerster J, Rodgers GM, Paraskevas F, Glader B, Arber DA, et al, editors. 12th ed. USA, Philadelphia; Lippincott W &amp; Wilkins; 2009:2372-438.</li>\r\n<li>Kumar SK, Callander NS, Alsina M, Atanackovic D, Biermann JS, Castillo J, et al. NCCN Guidelines Insights: Multiple Myeloma, Version 3. 2018. J Natl Compr Canc Netw 2018;16:11-20.</li>\r\n<li>Sergentanis TN, Zagouri F, Tsilimidos G, Tsagianni A, Tseliou M, Dimopoulos MA, et al. Risk Factors for Multiple Myeloma: A Systematic Review of Meta-Analyses. Clin Lymphoma Myeloma Leuk 2015;15:563-77.</li>\r\n<li>Sawyer JR. The prognostic significance of cytogenetics and molecular profiling in multiple myeloma. Cancer Genet 2011;204:3-12.</li>\r\n<li>Cao FF, Chen F, Hu YP, Zhang JH. Immunophenotype in multiple myeloma cells detected by multiparameter flow cytometry. Zhongguo Shi Yan Xue Ye Xue Za Zhi 2012;20:620-3.</li>\r\n<li>Han Y, Gao S, Muegge K, Zhang W, Zhou B. Advanced Applications of RNA Sequencing and Challenges. Bioinform Biol Insights 2015;9(Suppl 1):29-46.</li>\r\n<li>Conesa A, Madrigal P, Tarazona S, Gomez-Cabrero D, Cervera A, McPherson A, et al. A survey of best practices for RNA-seq data analysis. Genome Biol 2016;17:13.</li>\r\n<li>Hideshima T, Mitsiades C, Tonon G, Richardson PG, Anderson KC. Understanding multiple myeloma pathogenesis in the bone marrow to identify new therapeutic targets. Nat Rev Cancer 2007;7:585-98.</li>\r\n<li>Furukawa Y, Kikuchi J. Molecular pathogenesis of multiple myeloma. Int J Clin Oncol 2015;20:413-22.</li>\r\n<li>Flores-Montero J, de Tute R, Paiva B, Perez JJ, B&ouml;ttcher S, Wind H, et al. Immunophenotype of normal vs. myeloma plasma cells: Toward antibody panel specifications for MRD detection in multiple myeloma. Cytometry B Clin Cytom 2016;90:61-72.</li>\r\n<li>Lub S, Maes K, Menu E, De Bruyne E, Vanderkerken K, Van Valckenborgh E. Novel strategies to target the ubiquitin proteasome system in multiple myeloma. Oncotarget 2016;7:6521-37.</li>\r\n<li>Myung J, Kim KB, Crews CM. The ubiquitin-proteasome pathway and proteasome inhibitors. Med Res Rev 2001;21:245-73.</li>\r\n<li>Kubiczkova L, Pour L, Sedlarikova L, Hajek R, Sevcikova S. Proteasome inhibitors - molecular basis and current perspectives in multiple myeloma. J Cell Mol Med 2014;18:947-61.</li>\r\n<li>Losada A, Mu&ntilde;oz-Alonso MJ, Garc&iacute;a C, S&aacute;nchez-Murcia PA, Mart&iacute;nez-Leal JF, Dom&iacute;nguez JM, et al. Translation Elongation Factor eEF1A2 is a Novel Anticancer Target for the Marine Natural Product Plitidepsin. Sci Rep 2016;6:35100.</li>\r\n<li>Sahasrabuddhe NA, Barbhuiya MA, Bhunia S, Subbannayya T, Gowda H, Advani J, et al. Identification of prosaposin and transgelin as potential biomarkers for gallbladder cancer using quantitative proteomics. Biochem Biophys Res Commun 2014;446:863-9.</li>\r\n<li>Starlets D, Gore Y, Binsky I, Haran M, Harpaz N, Shvidel L, et al. Cell-surface CD74 initiates a signaling cascade leading to cell proliferation and survival. Blood 2006;107:4807-16.</li>\r\n<li>Stein R, Qu Z, Cardillo TM, Chen S, Rosario A, Horak ID, et al. Antiproliferative activity of a humanized anti-CD74 monoclonal antibody, hLL1, on B-cell malignancies. Blood 2004;104:3705-11.</li>\r\n<li>Lecker SH, Goldberg AL, Mitch WE. Protein degradation by the ubiquitin-proteasome pathway in normal and disease states. J Am Soc Nephrol 2006;17:1807-19.</li>\r\n<li>Leone P, Shin EC, Perosa F, Vacca A, Dammacco F, Racanelli V. MHC class I antigen processing and presenting machinery: organization, function, and defects in tumor cells. J Natl Cancer Inst 2013;105:1172-87.</li>\r\n<li>Ozbas-Gerceker F, Bozman N, Gezici S, Pehlivan M, Yilmaz M, Pehlivan S, et al. Association of TAP1 and TAP2 gene polymorphisms with hematological malignancies. Asian Pac J Cancer Prev 2013;14:5213-7.</li>\r\n<li>Zhou P, Teruya-Feldstein J, Lu P, Fleisher M, Olshen A, Comenzo RL. Calreticulin expression in the clonal plasma cells of patients with systemic light-chain (AL-) amyloidosis is associated with response to high-dose melphalan. Blood 2008;111:549-57.</li>\r\n<li>Chao MP, Jaiswal S, Weissman-Tsukamoto R, Alizadeh AA, Gentles AJ, Volkmer J, et al. Calreticulin is the dominant pro-phagocytic signal on multiple human cancers and is counterbalanced by CD47. Sci Transl Med 2010;2:63ra94.</li>\r\n<li>Ni Z, Chen Q, Lai Y, Wang Z, Sun L, Luo X, et al. Prognostic significance of CLPTM1L expression and its effects on migration and invasion of human lung cancer cells. Cancer Biomark 2016;16:445-52.</li>\r\n<li>Yin J, Li Y, Yin M, Sun J, Liu L, Qin Q, et al. TERT-CLPTM1L polymorphism rs401681 contributes to cancers risk: evidence from a meta-analysis based on 29 publications. PLoS One 2012;7:50650.</li>\r\n<li>James MA, Vikis HG, Tate E, Rymaszewski AL, You M. CRR9/CLPTM1L regulates cell survival signaling and is required for Ras transformation and lung tumorigenesis. Cancer Res 2014;74:1116-27.</li>\r\n<li>Steinbrunn T, St&uuml;hmer T, Sayehli C, Chatterjee M, Einsele H, Bargou RC. Combined targeting of MEK/MAPK and PI3K/Akt signalling in multiple myeloma. Br J Haematol 2012;159:430-40.</li>\r\n<li>Dytfeld D, Luczak M, Wrobel T, Usnarska-Zubkiewicz L, Brzezniakiewicz K, Jamroziak K, et al. Comparative proteomic profiling of refractory/relapsed multiple myeloma reveals biomarkers involved in resistance to bortezomib-based therapy. Oncotarget 2016;7:56726-36.</li>\r\n</ol>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<!--\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t<a class=\"menu\" href=\"#summary_en\">Summary</a><a class=\"menu\" href=\"#introduction\">Introduction</a><a class=\"selected_menu\" href=\"#reference\">Reference</a>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\n\t\t\n\t\t\t\t\t\t\t\t\t\t\t        \t</div>\n\t\t\t</div";
