<div class="col-lg-12 col-md-12 col-sm-12 col-xs-12">
																		        				
												        				<div  style="width:100%; float:left;  padding-bottom:0px;">
												        					<div	style="width:100%; float:left; min-height:45px; text-align:justify">
												        						<a href="" name="summary_en"></a>
												        						<!--<h2>Summary</h2>-->
												        						</hr><h4>Abstract</h4>
<p>The introduction and advances on next-generation sequencing have led to novel ways to integrate simultaneous assessment of multiple target genes in routine laboratory analysis. Assessment of myeloid neoplasms with targeted next-generation sequencing panels shows evidence to improve diagnosis, assist therapeutic decisions, provide better information about prognosis, and better detection of minimal residual disease. Herein, we provide information for application and utilization of next-generation sequencing studies with a focus on the most important mutations in acute myeloid leukemia, myelodysplastic syndrome, myeloproliferative neoplasms, and other myelodysplastic/myeloproliferative neoplasms in order to integrate them into the daily clinical practice.</p>												        					</div>
												        					<!--
												        					<div	style="width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;">
												        									<a class="selected_menu" href="#summary_en">Summary</a><a class="menu" href="#introduction">Introduction</a><a class="menu" href="#reference">Reference</a>												        					</div>
												        					-->
												        				</div>
												        				
												        				
												        																        				
												        				<div  style="width:100%; float:left;  padding-bottom:0px;">
												        					<div	style="width:100%; float:left; min-height:45px; text-align:justify">
												        						<a href="" name="introduction"></a>
												        						<!--<h2>Introduction</h2>-->
												        						<hr size="1" color="#f2f2f2">
<p>Next-generation sequencing (NGS) technology can be of great utility in the clinical assessment of various neoplasms. NGS can be rapidly integrated into the diagnostic laboratories and merged into current clinical practice. However, bioinformatics and genetic code technology and language are difficult to understand for practitioners who are not familiar with molecular biology. We will review the recent developments in NGS, technical progress, and focus on the approach of diseases affecting the myeloid lineage, including acute myeloid leukemia (AML), myelodysplastic syndrome (MDS), chronic myeloproliferative diseases (CMPDs) such as polycythemia vera (PV), essential&nbsp;thrombocythemia&nbsp;(ET), primary myelofibrosis (PMF), atypical CMPD, chronic&nbsp;myelomonocytic&nbsp;leukemia (CMML) and chronic eosinophilic leukemia (CEL) and mast cell disease. </p>
<p>Due to the increasing number of mutations that are necessary for diagnosis, it is becoming impractical to do a single gene testing analysis. Furthermore, multigene assessment is providing guidance for determining prognosis in many diseases. Since many of the diseases encountered in clinical practice show more than one molecular abnormality, we could adapt NGS based assays to have simultaneous detection of multiple somatic mutations in tens or hundreds of target genes that are associated with specific diseases. NGS assays are becoming cheaper, faster, and rapidly adopted by many laboratories and since there is an increasing number of actionable targets  that could benefit from these assays. However, NGS technology requires highly complex bioinformatics and laboratory technology to develop a diagnostic pipeline. Currently, there are&nbsp;two commonly used&nbsp;NGS.&nbsp;Sequencing differs in these technologies but they&nbsp;use&nbsp;similar&nbsp;bioinformatics&nbsp;principles for determining&nbsp;abnormal sequences that are not matching reference sequences. </p>
<p>There are also&nbsp;three&nbsp;different enrichment methods used for NGS&nbsp;sequencing:&nbsp;Hybrid capture, Amplicons, and&nbsp;Anchored Multiplex polymerase chain reaction (1). However, until now there is no single database available for clinical annotations to determine the significance of variants. Moreover, these assays are expensive and reimbursement remains low.&nbsp;</p>
<p>The introduction of NGS into clinical laboratories has significantly changed the use of molecular techniques. Testing for multiple genes in a panel using NGS platforms is more practical and economical (2,3).&nbsp;Furthermore, the cost of adding a new gene to the panel is relatively inexpensive, in comparison to performing multiple single-gene tests. The presence of specific genetic mutations could aid clinicians in 4 ways: 1) diagnostic algorithms, 2) prognosis assessment, 3) use of targeted therapy, and 4) monitoring treatment response and residual disease.&nbsp;Despite increased utilization, NGS assays remain challenging particularly for the identifications of variants without a known clinical significance. To address these concerns, new terms in NGS reports have been adopted: &ldquo;actionable&rdquo;, &ldquo;potentially actionable&rdquo;, &ldquo;Variant of Undetermined Significance&rdquo;, &ldquo;likely benign&rdquo; or &ldquo;benign&rdquo;.</p>
<h4>Mutation profiles in acute myeloid leukemia</h4>
<p>The Cancer Genome Atlas Network has many studies that report recurrent mutations and show a complex network of genetic mutations (4,5). Despite these exhaustive studies, there is no gene-specific diagnostic entity. However,&nbsp;there are some mutations leading to an aberrant activation of proteins that have a crucial effect on hematopoietic progenitor cell proliferation and differentiation. The mutations in AML involve epigenetic modifiers (<em>TET2, IDH1/IDH2,&nbsp;DNMT3A, ASXL1, KMT2A, EZH2</em>), activated signaling pathway (<em>FLT3, KRAS, NRAS, KIT</em>), tumor&nbsp;suppressor&nbsp;genes (<em>TP53, WT1</em>),&nbsp;RNA splicing (<em>SF3B1</em>),&nbsp;nucleophosmin&nbsp;mutation (NPM1), and genes coding for transcription-differentiation (<em>CEBPA, RUNX1</em>) (<a href="javascript:openWin('uploads/grafik/figure_BMJ_2043_0.jpg', 640, 480)"><b>Figure 1</b></a>) (6-8).&nbsp;Recurrently mutated genes in AML and&nbsp;MDS&nbsp;are listed in <a href="javascript:openWin('uploads/grafik/table_BMJ_2043_0.jpg', 640, 480)"><b>Table 1.</b></a></p>
<p>Recent studies have demonstrated that genes mutated in AML with a normal karyotype AML (NK-AML) share significant overlap with genes mutated in MDS&nbsp;(9,10). The World Health Organization (WHO) 2016 classification recognizes AML with mutated&nbsp;<em>NPM1</em>&nbsp;and AML with&nbsp;bi-allelic&nbsp;mutation of&nbsp;<em>CEBPA</em>&nbsp;as specific to AML classification categories and AML with mutated&nbsp;<em>RUNX1</em>&nbsp;as a provisional entity&nbsp;(8-10).&nbsp;The most frequently mutated genes in NK-AML occur in the following genes: nucleophosmin (<em>NPM1</em>),&nbsp;Fms-related tyrosine kinase 3 (<em>FLT3</em>)&nbsp;and&nbsp;DNA methyltransferase 3A (<em>DNMT3A</em>), Ten-eleven translocation-2 (<em>TET2</em>), isocitrate dehydrogenase 1 and 2 (<em>IDH1</em>&nbsp;and&nbsp;<em>IDH2</em>),&nbsp;CCAAT/enhancer binding protein alpha (<em>CEBPA</em>), NRAS, Additional sex comb-like 1 (<em>ASXL1</em>),&nbsp;WT1&nbsp;and&nbsp;runt-related transcription factor-1 (<em>RUNX1</em>). Mutations of&nbsp;<em>FLT3,&nbsp;DNMT3A</em>, and&nbsp;<em>NPM1</em>&nbsp;are often present concurrently, while other mutations of&nbsp;<em>NPM1,&nbsp;RUNX1,&nbsp;CEBPA</em>, and&nbsp;<em>TP53</em>&nbsp;are almost always mutually exclusive both at diagnosis and during disease transformation&nbsp;(11-19).</p>
<p>The WHO classification of AML includes many specific chromosomal abnormalities determined by karyotyping (11).&nbsp;The gene mutations recognized as specific entities by the 2016 WHO classification include <em>NPM1,&nbsp;CEBPA</em>, and&nbsp;<em>RUNX1</em>&nbsp;mutations.&nbsp;However, since the publication of this classification, it became very important to clinically recognize&nbsp;other pathogenic variants such as those involving <em>FLT3</em>&nbsp;(<em>ITD</em>&nbsp;and&nbsp;<em>TKD</em>) and&nbsp;<em>IDH1/2</em>. This can drastically change the clinical management of AML patients since&nbsp;the US Food and Drug Administration (FDA)&nbsp;has approved specific inhibitors for&nbsp;<em>FLT3</em>&nbsp;and&nbsp;<em>IDH2</em>&nbsp;positive AML (20,21).</p>
<p>The gene&nbsp;<em>NPM1</em>&nbsp;is the most frequently mutated gene in the adult type AML. It occurs approximately in 50 to 60% of cytogenetically normal AMLs and in 2 to 5% of MDS cases. It is often characterized by CD34 antigen negativity with a bright expression of CD33 and monocytic differentiation.&nbsp;<em>NPM1</em>&nbsp;mutations&nbsp;include small insertions (4-11bp in size), resulting in an aberrant cytoplasmic localization of the mutant protein&nbsp;nucleophosmin&nbsp;which can be detected immunohistochemically (21,22). <em>NPM1</em>&nbsp;mutations increase disease-free and overall survival in patients with AML. Secondary mutations [<em>FLT3-ITD,&nbsp;DNMT3A</em> (21),&nbsp;<em>IDH,&nbsp;TET2</em>&nbsp;mutations] are frequently accompanying these patients.&nbsp;<em>FLT3-ITD</em>&nbsp;mutations are observed in 40% of&nbsp;<em>NPM1</em>&nbsp;mutated AML patients. In&nbsp;general,&nbsp;<em>FLT3 ITD</em>&nbsp;positive patients without other molecular abnormalities show poor clinical outcome while&nbsp;<em>NPM1</em>&nbsp;positive patients have a better clinical response. Coexistence of&nbsp;<em>FLT3-ITD</em>&nbsp;mutation in <em>NPM1</em>&nbsp;positive patients is associated with a poor prognosis in young AML patients compared with patients with&nbsp;<em>NPM1<sup>mut</sup>/FLT3<sup>wild</sup></em>&nbsp;type (4,11,17,23).</p>
<p>Among&nbsp;transcription factor mutations&nbsp;in myeloid neoplasia,&nbsp;<em>CEBPA</em>&nbsp;and&nbsp;<em>RUX1</em>&nbsp;mutations are deemed important by the WHO due to their prognostic implications.&nbsp;CCAAT&nbsp;Enhancer&nbsp;Binding&nbsp;Protein-alpha&nbsp;(<em>CEBPA</em>) is a transcription factor responsible for promoting granulocytic maturation in regulating cellular function.&nbsp;<em>CEBPA</em>&nbsp;mutations are observed in approximately 10% of <em>de novo</em> AML patients and are commonly&nbsp;bi-allelic. The&nbsp;bi-allelic&nbsp;mutation is a favorable prognostic marker. Germline mutations of CEBPA predispose to the development of AML in young patients. The concurrence of&nbsp;<em>FLT3-ITD</em>&nbsp;mutation is an adverse prognostic finding in&nbsp;<em>CEBPA</em>&nbsp;mutated-AML patients&nbsp;(23).&nbsp;<em>RUNX1</em>&nbsp;encodes the DNA binding alpha subunit of the core binding factor (<em>CBFalpha</em>) that is involved in the normal differentiation of hematopoietic cells. <em>RUNX1</em>&nbsp;has also tumor suppressor properties and loss of function of the gene that contributes to tumorigenesis in myeloid cancers. <em>De novo</em> AML with mutated&nbsp;<em>RUNX1</em>&nbsp;in the absence of MDS related cytogenetic abnormalities appears to be a distinct entity with adverse outcome. <em>RUNX1</em>&nbsp;mutations are detected in approximately&nbsp;4% to 16% of AML and 9% of MDS&nbsp;(11). They are commonly seen in older patients with NK-AML and MDS-associated AML. In several multivariate analyses, they have been associated with worse overall survival&nbsp;(23-25).</p>
<p><em>FLT3</em>&nbsp;is a class III receptor tyrosine kinase coding for&nbsp;signal transduction gene. This receptor is important for proliferation and differentiation of hematopoietic stem cells. Its expression is downregulated during differentiation. <em>FLT3</em> mutations occur in 30% to 40%&nbsp;of&nbsp;AML patients with normal cytogenetics and 2% to 6% of MDS cases. Two types of mutations are observed in&nbsp;<em>FLT3</em>&nbsp;gene: <em>FLT3-ITD&nbsp;(internal tandem duplications)</em>&nbsp;mutations (with a frequency of 75% to 80%) and&nbsp;<em>FLT3-TKD&nbsp;(tyrosine kinase domain)</em>&nbsp;(with a frequency of 20% to 25%). FLT3 mutated patients show higher white blood cell count, loss of HLA-DR and CD34 antigen negativity, and impaired survival&nbsp;(26). AML patients with&nbsp;<em>FLT3-ITD</em>&nbsp;mutation appear to benefit from allogeneic stem cell transplantation. However, the prognostic effect of&nbsp;<em>FLT3-TKD</em>&nbsp;remains controversial&nbsp;(27). In 2017, the FDA approved treatment with midostaurin for newly diagnosed patients with FLT3-mutated AML due to the observation of a significant beneficial outcome (28,29). In MDS,&nbsp;<em>FLT3</em>&nbsp;mutations are observed in high-risk subgroups and are associated with complex karyotype&nbsp;(30,31).&nbsp;<em>NPM1</em>&nbsp;and&nbsp;<em>FLT3</em>&nbsp;mutations are found primarily in AML; however, the presence of these mutations in an MDS patient should alert the clinician about the risk of rapid progression to AML (22). Alterations on <em>FLT3</em>&nbsp;signaling pathway seems to be the most important prognostic factor for overall survival in AML patients younger than 60 years.</p>
<p>The other signal transduction genes are&nbsp;<em>KIT</em>&nbsp;mutations that are frequent in gastrointestinal stromal&nbsp;tumors, germ cell&nbsp;tumors, melanomas, and systemic mast cell diseases. They can be detected in approximately 6% of patients with newly diagnosed AML and 20% of AML patients with t(8;21)&nbsp;<em>RUNX1-RUNXT1</em>&nbsp;and&nbsp;30% of AML patients with&nbsp;inv(16) or t(16;16)&nbsp;<em>CBFB-MYH11</em>&nbsp;(the core binding&nbsp;factor AMLs-CBF-AML). <em>KIT</em> mutation in AML increases the risk of relapse and worsens the good prognosis of CBF-AML (32-36).</p>
<p>There are a number of other genes playing a role in the pathogenesis of AML and appear to be important for overall survival despite not being recognized as specific entities. DNA methylation mutations and chromatin/histone modifications belong to epigenetic modifiers.&nbsp;<em>DNMT3A</em>&nbsp;encodes an epigenetic regulator and mediates&nbsp;<em>de novo</em>&nbsp;methylation of CpG dinucleotides and are seen in approximately 20% of AML patients with normal karyotype.&nbsp;This mutation tends to shorten overall survival&nbsp;(14,37). <em>TET2</em>&nbsp;is also involved in the epigenetic regulation of DNA methylation. <em>TET2</em>&nbsp;mutations occur in approximately 10% to 15% of AMLs. As&nbsp;<em>TET2</em>&nbsp;is a key regulator in hematopoietic stem cell renewal and differentiation, their mutations result in increased stem cell renewal and myeloid hyperplasia with impaired differentiation. These tend to have a poorer prognosis in NK-AML. <em>TET2</em>&nbsp;mutations predict the response to hypomethylating agents&nbsp;(4,38).</p>
<p><em>IDH1/IDH2</em>&nbsp;mutations prevent the conversion of isocitrate to alpha-ketoglutarate&nbsp;in the Krebs cycle; create an oncometabolite that inhibits the function of&nbsp;<em>TET2. IDH1/IDH2</em>&nbsp;mutations often mutate with&nbsp;<em>TET2</em>&nbsp;and&nbsp;<em>WT1</em>&nbsp;mutations (39). Between 15% to 25%&nbsp;of&nbsp;AML and approximately 2% to 12% of MDS cases display <em>IDH</em> mutations&nbsp;(40). The prognostic significance of&nbsp;<em>IDH</em>&nbsp;mutations in AML is currently unclear.&nbsp;IDH2 inhibitor had been successful in the management of <em>IDH2</em> positive AML patients while IDH1 inhibitor is showing promising results in ongoing clinical trials&nbsp;(4,20,34,40).</p>
<p>Chromatin/Histone Modification mutations&nbsp;such as&nbsp;Additional Sex Coms Like Transcriptional Regulator 1 (<em>ASXL1</em>)&nbsp;mutations and&nbsp;Enhancer of Zeste 2 Polycomb Repressive Complex 2 Subunit (<em>EZH2</em>)&nbsp;mutations are a part of the epigenetic modifiers.&nbsp;<em>ASXL1</em>&nbsp;gene is involved in the epigenetic regulation of gene expression. They are uncommon in <em>de novo</em> AML (6.5%), but more frequent in AML arising from a prior myeloid neoplasm (up to 30%)&nbsp;(41-43). These mutations are associated with shorter overall survival and resistance to&nbsp;chemotherapy. <em>KMT2A: lysine(K)-specific methyltransferase 2A (KMT2A/MLL)</em>&nbsp;gene plays a role in hematopoiesis and cell differentiation. <em>KMT2A</em>&nbsp;mutations co-occur with an additional gene mutation (<em>IDH2/DNMT3A/U2AF1/TET2</em>).&nbsp;This mutation is frequently associated with trisomy 11 (90%) and a poorer prognosis (17,44,45).</p>
<p>TP53 protein is a&nbsp;transcription factor and tumor suppressor&nbsp;gene that determines whether the cell undergoes repair, senescence or apoptosis. Although somatic&nbsp;<em>TP53</em>&nbsp;mutations are frequently seen in 50% of solid&nbsp;tumors, they are uncommon in <em>de novo</em> AML (5% to 18%) and they are associated with secondary or therapy-related AMLs.&nbsp;<em>TP53</em>&nbsp;mutations often coexist with complex karyotypes, chemotherapy resistance, and reduced overall survival compared with <em>TP53<sup>wild</sup></em> AML patients. <em>TP53<sup>mut</sup></em> AML patients have also high relapse rates after stem cell transplantation (46-48). Despite these findings, a recent study indicated that AML patients with cytogenetic abnormalities associated with unfavorable risk,&nbsp;<em>TP53</em>&nbsp;mutations, or both had a favorable clinical response to decitabine (49,50).</p>
<p><a href="javascript:openWin('uploads/grafik/figure_BMJ_2043_1.jpg', 640, 480)"><b>Figure 2</b></a> represents these risk categories associated with distinct molecular mutations that should be screened by NGS in AML patients.</p>
<p>For the monitoring of minimal residual disease (MRD) in AML; NGS based technologies can be used besides multiparameter flow cytometry (MFC). MFC is a more reliable method for monitoring remission than conventional morphology-based assessment in AML patients; however, MFC lacks sensitivity for detecting residual disease. Therefore, a combined result of sequencing and flow cytometry detects residual disease more specifically. A recent study about molecular MRD in AML showed that the detection of molecular MRD is associated with a significantly higher relapse rate (51). During the complete remission, persistent mutations associated with clonal hematopoiesis (<em>DNMT3A, TET2</em>, and <em>ASXL1</em> referred as <em>DTA</em> mutations) and non-<em>DTA</em> mutations can be detected by NGS. While <em>DTA</em> mutations do not have prognostic value for MRD, non-<em>DTA</em> mutations (such as mutations in <em>TP53, IDH1, IDH2</em> and genes related to the Ras pathway) with a higher allele frequency can predict relapse rates (51,52).</p>
<p>Among&nbsp;AML with myelodysplasia-related changes&nbsp;(AML-MRC) and therapy-related AML (t-AML), mutations in splicing factors genes such as <em>SRSF2,&nbsp;SF3B1,&nbsp;U2AF1,&nbsp;ZRSR2</em>, mutations on chromatin/histone modifications like <em>ASXL1,&nbsp;EZH2,&nbsp;BCOR,&nbsp;STAG2</em>, and&nbsp;<em>TP53</em>&nbsp;mutations are common&nbsp;(53). <em>NPM1</em>&nbsp;mutations in the absence of&nbsp;<em>FLT3-ITD</em>&nbsp;and double&nbsp;<em>CEBPA</em>&nbsp;mutations are fairly uncommon in AML-MRC and are associated with favorable overall survival. Patients in this category with&nbsp;<em>TP53</em> mutations&nbsp;had a poor prognosis regardless of age (46,54-56).&nbsp;After treating patients with alkylating agents/ionizing radiation, patients often have increased blasts with associated multilineage dysplasia that cause similar mutation profiles like in AML-MRC. Mutations of <em>TP53</em> can be seen in as many as 50% of t-AML cases with worse survival (11,53,57,58). Following a topoisomerase II inhibitor therapy, 20 to 30% of patients develop overt acute leukemia without a preceding myelodysplastic phase. These patients often show balanced chromosomal translocations involving 11q23 (<em>KMT2A/MLL</em>) or 21q22 (<em>RUNX1</em>) and are associated with monoblastic or myelomonocytic morphology (11,53,59,60). Mutations in <em>TP53, TET2</em>, and <em>PTPN11</em> (protein tyrosine phosphatase, non-receptor type 11), <em>IDH1/2, NRAS</em> are frequent in t-AML, but <em>FLT3</em> and <em>NPM1</em> mutations are less frequent than de novo AML (58,61).</p>
<h4>Mutation profiles&nbsp;in Myelodysplastic Syndrome</h4>
<p>Approximately half of the myelodysplastic syndrome is associated with recurrent cytogenetic abnormalities such as -5/del(5q), -7/del(7q), +8del(20q) or complex karyotypes (62). Targeted panels and whole-genome NGS assay detect somatic mutations in up to 90% of MDS patients (63-65). Some of the mutations overlap with those observed in AML, none of these mutations are specific for MDS diagnosis. However, having multiple genetic abnormalities with high variant allele frequency is starting to be recognized for its supportive evidence of evolving MDS (66).</p>
<p>The most frequently mutated genes in MDS are SF3B1&nbsp;(~25%),&nbsp;<em>TET2</em>&nbsp;(~25%-20%),&nbsp;<em>ASXL1</em>&nbsp;(~15%),&nbsp;<em>SRSF2</em> (~15%),&nbsp;<em>RUNX1</em>&nbsp;(~15%-10%),&nbsp;<em>DNMT3A</em> (~15%-10%)&nbsp;(23). Mutations in&nbsp;<em>ASXL1,&nbsp;TP53,&nbsp;EZH2,&nbsp;ETV6</em>, and&nbsp;<em>RUNX1</em>&nbsp;are also found to be recurrent in MDS patients and predict a poor overall survival&nbsp;(41). <em>DNMT3A</em> mutations occur early in the course of MDS and suggest an early genetic event in leukemogenesis. Patients with <em>DNMT3A</em> mutations have a worse overall survival and more rapid progression to AML. They are associated with mutations in <em>FLT3</em> (<em>ITD</em> or <em>TKD</em>), <em>NPM1, IDH1</em> genes&nbsp;(67). <em>TP53</em> mutations in MDS are associated with the very aggressive disease. Even in low-risk MDS with del(5q), the presence of a <em>TP53</em> mutation at low frequency is correlated with a higher rate of leukemic transformation and poor response to lenalidomide (11,68).</p>
<p>Mutations in&nbsp;splicing factors&nbsp;are commonly detected in MDS patients with a frequency of 50%. The most common of these mutations are&nbsp;<em>SF3B1,&nbsp;SRSF2,&nbsp;U2AF1</em>, and&nbsp;<em>ZSR2</em>&nbsp;mutations. <em>SF3B1</em>&nbsp;mutations are the most common among splicing mutations in MDS. These mutations are highly correlated with the presence of ring&nbsp;sideroblast&nbsp;(RS). They are observed in 20% to 28% of MDS, more frequent in MDS with RSS and the frequency of&nbsp;<em>SF3B1</em>&nbsp;mutation in AML is &lt;5%&nbsp;(11,23,69-71).&nbsp;Mutations in&nbsp;Cohesin&nbsp;Complex&nbsp;genes&nbsp;(<em>SMC1, SMC3, SCC1/RAD, STAG2</em>) can be germline resulting in Congenital Malformation syndrome. These mutations can be detected in 10% to 25% of MDS and 10% to 15% of <em>de novo</em> AML (17,69).</p>
<p>Nearly all cases of MDS show at least one mutation and demonstrate that specific mutation with a targeted NGS panel&nbsp;is highly useful for the diagnosis of MDS. However, determination of mutational burden/variant allele frequency (VAF) (usually &gt;25% for MDS) is critical for the diagnostic assessment to distinguish MDS from clonal hematopoiesis of indeterminate potential (CHIP) or clonal cytopenia of undetermined significance (CCUS). CHIP is the proposed term for healthy individuals, lacking hematological malignancy or clonal disorder, but&nbsp;carrying a hematological&nbsp;somatic mutation. Key findings in CHIP are single gene mutations (mostly&nbsp;<em>DNMT3A, TET2</em> or <em>ASXL1</em>&nbsp;mutation), increased risk of developing hematological malignancy, and increased rate of cardiovascular mortality regardless of their cancer risk. The single mutation burden in CHIP has a VAF more than 2% but typically not exceeding 20%-30%, which allows the distinction from underlying dysplastic process (3). Patients who are not meeting the WHO defined criteria for a hematologic neoplasm, but show a clonal mutation with unexplained cytopenia, are referred to as CCUS. The mutation spectrum of CCUS patients is similar to mutations seen in MDS.&nbsp;Mutations in spliceosome genes (<em>SF3B1, SRSF2, U2AF1, ZRSR2</em>) have the highest predictive value for CCUS (72). Co-mutations of <em>DNMT3A,&nbsp;TET2</em>, and <em>ASXL1</em> genes remain also common in CCUS. There are no clear recommendations to follow up CCUS patients. However, cytopenia, mutation burden, number and type of the mutations or the detection of additional mutations in the follow-up of a patient with suspected MDS appear to have a course similar to MDS (72-76).</p>
<h4>Mutation profiles&nbsp;in myeloproliferative and myelodysplastic/myeloproliferative neoplasms</h4>
<p>Disorders altering the myeloid elements include the following disorders: Polycythemia&nbsp;vera&nbsp;(PV), essential thrombocythemia (ET), primary myelofibrosis (PMF), chronic myelogenous leukemia (CML), chronic neutrophilic leukemia&nbsp;(CNL),&nbsp;hypereosinophilic&nbsp;leukemia/CEL, CMML, and mast cell disease.&nbsp;Among CMPD (PV, ET, PMF, and CML), molecular markers have been essential for diagnosis.&nbsp;Recurrent somatic mutations in <em>BCR-ABL2</em> negative MPNs and MDS/MPNs&nbsp;are listed&nbsp;in <a href="javascript:openWin('uploads/grafik/table_BMJ_2043_1.jpg', 640, 480)"><b>Table 2</b></a>.</p>
<p>The most frequent mutation in <em>BCR-ABL2</em> negative MPN,&nbsp;<em>JAK2</em> exon 14 (V617F) mutations are observed in 95% of PV patients&nbsp;(77). The majority of remaining PV patients may harbor a mutation in <em>JAK2</em> exon 12 (missense mutation, deletion, insertion).&nbsp;<em>JAK2</em>&nbsp;(V617F)&nbsp;mutation&nbsp;occurs in&nbsp;approximately 50-60%&nbsp;of ET&nbsp;or&nbsp;PMF patients. Homozygous mutations of&nbsp;<em>JAK2</em>&nbsp;(V617F) (with a VAF value &gt;50%) are more common in PV than in ET (78,79). <em>JAK2</em> (V617F) mutation may not be present in half of the MPN patients. This type of mutation has rarely been reported in patients with MDS/MPN; however,&nbsp;they&nbsp;are found in patients with MDS, CMML, atypical&nbsp;CML,&nbsp;and <em>de novo</em> AML&nbsp;(approx.&nbsp;&lt;5%)&nbsp;and&nbsp;MDS/MPN-RS-T&nbsp;(approx. 50%). In&nbsp;PV and ET;&nbsp;<em>JAK2</em>&nbsp;(V617F) allele&nbsp;burden is also associated with a more aggressive behavior. Thrombotic&nbsp;events and fibrotic transformation&nbsp;are more common in patients with&nbsp;a&nbsp;high&nbsp;<em>JAK2</em>&nbsp;(V617F) allele burden (higher&nbsp;<em>JAK2</em>&nbsp;VAF).&nbsp;These findings suggest that monitoring&nbsp;<em>JAK2</em>&nbsp;(V617F)&nbsp;allele burden could be useful to identify patients at higher risk of myelofibrotic transformation. <em>JAK2</em> mutation in PMF&nbsp;is associated with intermediate prognosis when compared with <em>CALR</em> mutation (3,11,17,23,80).</p>
<p>Patients with ET&nbsp;have mutations&nbsp;in the <em>CALR</em> gene that encodes the endoplasmic reticulum and are associated with chaperone calreticulin&nbsp;at a frequency of 20% to 25%.&nbsp;CALR&nbsp;mutations are frequently seen in ET and PMF patients with&nbsp;<em>JAK2<sup>wild</sup></em>&nbsp;and&nbsp;<em>MPL<sup>wild</sup></em>&nbsp;type. They&nbsp;are rarely detected in PV, CMML, MDS/MPN&nbsp;patients.&nbsp;<em>CALR</em>&nbsp;mutations are not&nbsp;seen in AML,&nbsp;mastocytosis, lymphoid&nbsp;neoplasia&nbsp;and solid tumors&nbsp;(81,82). <em>CALR</em>&nbsp;mutated MPNs demonstrate distinct features; such as younger age group, lower hemoglobin levels, higher platelet counts, decreased risk of&nbsp;thrombosis, and improved overall survival compared to&nbsp;patients with&nbsp;<em>JAK2</em>&nbsp;or&nbsp;<em>MPL</em>&nbsp;mutations (83,84). In ET and PMF patients,&nbsp;<em>CALR</em>&nbsp;show 2 different mutations: type-I that typically has 52-bp deletion (p.L367fs*46), and  type-II characterized by 5-bp TTGTC insertion (p.K385fs*47). Type I&nbsp;CALR&nbsp;mutation&nbsp;is detected more frequently in PMF than ET.&nbsp;Type II&nbsp;<em>CALR</em>&nbsp;mutations tend to show leukemic transformation (namely increased leucocyte count and higher circulating blasts)&nbsp;than type I <em>CALR</em> mutation (81,85,86). Platelet count is significantly higher in type II vs type I&nbsp;<em>CALR<sup>mut</sup></em> ET patients,&nbsp;but no difference was noted in thrombosis-free survival in a study with a large cohort of ET patients.&nbsp;However, there is a significant survival difference in the setting of PMF, as CARL type-I positive PMF patients do much better than&nbsp;<em>CALR</em>&nbsp;type-II or&nbsp;<em>JAK2</em>&nbsp;(V617F)&nbsp;positive PMF patients&nbsp;(84).</p>
<p>The&nbsp;activating&nbsp;mutation&nbsp;on&nbsp;codon W515&nbsp;of&nbsp;<em>MPL</em>&nbsp;gene&nbsp;is seen in 5-10% of&nbsp;ET and PMF patients who are negative for&nbsp;<em>JAK2&nbsp;</em>(V617F) (87).&nbsp;Patients that are&nbsp;<em>JAK2</em><sup>wild</sup>&nbsp;and&nbsp;<em>CALR</em><sup>wild</sup>,&nbsp;but harboring ET or PMF clinical features&nbsp;should be screened for&nbsp;<em>MPL</em>&nbsp;mutation. <em>MPL</em>&nbsp;mutated patients have increased the risk for thrombotic episodes and transfusion requirement compared with&nbsp;<em>JAK2</em><sup>mut</sup>&nbsp;and&nbsp;<em>CALR</em><sup>mut</sup>&nbsp;patients&nbsp;(80,88,89).&nbsp;Patients with PMF have mutations for&nbsp;<em>JAK2</em>&nbsp;in 60%,&nbsp;<em>CALR</em>&nbsp;in&nbsp;20-30% and&nbsp;<em>MPL&nbsp;</em>in<em>&nbsp;</em>5-10%&nbsp;of the cases (<a href="javascript:openWin('uploads/grafik/figure_BMJ_2043_2.jpg', 640, 480)"><b>Figure 3</b></a>). The remaining 10% to 15% of patients with ET or PMF patients&nbsp;have none of the above-mutated genes and these are named &ldquo;triple negative&rdquo;&nbsp;(85,90,91).&nbsp;This is a heterogeneous category and some of the ET patients may have rare non-canonical mutations in <em>JAK2</em>, <em>MPL, </em>and<em> SH2B3</em>; however,&nbsp;the large majority remain without a specific mutation. For&nbsp;&ldquo;triple negative&rdquo; PMF patients,&nbsp;a&nbsp;detailed&nbsp;search for the&nbsp;mutation&nbsp;profiles&nbsp;regarded&nbsp;as a marker of&nbsp;clonality (such as&nbsp;<em>ASXL1</em>,&nbsp;<em>EZH2</em>,&nbsp;<em>TET2</em>,&nbsp;<em>IDH1/IDH2</em>,&nbsp;<em>SRSF2</em>,&nbsp;<em>SF3B1</em>)&nbsp;is currently recommended (77).&nbsp;&ldquo;Triple negative&rdquo;&nbsp;patients tend to show inferior outcome compared to&nbsp;<em>JAK2</em><sup>mut</sup><em>/CALR</em><sup> must</sup>&nbsp;and&nbsp;<em>MPL</em><sup>mut</sup>&nbsp;CMPD&nbsp;patients. The presence of at least one of the 3 variants/mutations (<em>ASXL1, SRSF2</em>, and&nbsp;<em>IDH2)&nbsp;</em>is&nbsp;associated with inferior overall&nbsp;and myelofibrosis-free survival in patients with PV.&nbsp;<em>SH2B3, IDH2, U2AF1, SF3B1, EZH2, </em>and<em> TP53&nbsp;</em>mutations are identified as significant risk factors for inferior overall, myelofibrosis-free survival, and leukemia-free survival in patients with ET (85).&nbsp;</p>
<p>Recent studies about leukemic transformation in MPNs show <em>ASXL1</em>, <em>SRSF2</em>, <em>IDH1</em>, <em>IDH2</em>, <em>RUNX1</em> mutations have been associated with leukemic transformation in PMF, whereas <em>SRSF2</em>, <em>IDH2</em> mutations in PV and <em>TP53</em>, <em>EZH2</em> mutations in ET predicts more leukemic transformation (92). These findings show that the mutation spectrum in primary NK-AML is different from secondary AML. Recurrent somatic gene mutations are observed in up to 90% of CMML patients. <em>TET2 </em>(~60%), <em>SRSF2 </em>(~50%), <em>ASXL1 </em>(~50%), <em>RUX1</em>, <em>NRAS,</em> and <em>TP53</em> are the most frequent mutations.&nbsp;However,&nbsp;these mutations&nbsp;can also be encountered in&nbsp;healthy&nbsp;aging&nbsp;people such as CHIP with a single mutation of a VAF &lt;20-30%. Therefore, a low VAF of these mutations&nbsp;should not be used as a definitive diagnosis&nbsp;for CMML (23). <em>ASXL1</em> mutations are associated with a higher white blood cell count, lower hemoglobin, extramedullary disease, and an abnormal karyotype. <em>ASXL1</em> mutations lead to the aggressive clinical course in CMML patients (93,94). The <em>JAK2</em> mutation also occurs in CMML patients sharing some mutual features with <em>JAK2</em><sup>mut</sup> MPNs. These cases predict more reticulin fibrosis, occasional megakaryocytic clustering with atypia, and erythroid and megakaryocytic hyperplasia (95). In case of presence, <em>NPM1</em><sup>mut</sup> AML with monocytic differentiation should be suspected. Variable results are also expected on <em>JAK</em><sup>mut</sup> CMML patients treated with JAK2 inhibitors (11,96,97). Juvenile&nbsp;myelomonocytic&nbsp;leukemia&nbsp;(JMML) patients harbor also clonal cytogenetic abnormalities. The commonly mutated genes are&nbsp;<em>KRAS/NRAS</em>,&nbsp;<em>NF1,</em>&nbsp;<em>PTPN11</em>,&nbsp;<em>CBL</em>, which encode proteins of the Ras oncogene pathway.&nbsp;<em>PTPN11</em> mutations are the most frequent alterations (~35%). Somatic <em>NRAS </em>and <em>KRAS</em> mutations occur in 20-25% of JMML cases. Germline mutations in <em>CBL</em> and <em>NF1</em> genes are also frequent and are associated with Noonan Syndrome-like disorder and neurofibromatosis type-I on JMML patients, respectively (98,99).</p>
<p><em>SETBP1</em> and ethanolamine kinase 1 (<em>ETNK1</em>) mutations are relatively common in atypical CML (aCML). <em>CSF3R</em> mutation is frequently associated with CNL, this type of mutation is uncommon (&lt;10%) in aCML and rarely encountered in CMML and AML cases (100,101). Correlation of <em>CSF3R</em> mutation with CNL has been included in the diagnostic criteria in the revised 2016 WHO classification (10,21). In myelodysplastic/myeloproliferative neoplasm with ring&nbsp;sideroblasts&nbsp;and thrombocytosis (MDS/MPN-RS-T),&nbsp;<em>SF3B1</em>&nbsp;mutations&nbsp;are pesent in 70-90% of cases. They&nbsp;are frequently commutated with <em>JAK2&nbsp;</em>(V617F) (50-65%)&nbsp;and less frequently&nbsp;with&nbsp;<em>CALR</em>/<em>MPL </em>(11,23,70,71).</p>
<p>Among myeloid neoplasms with eosinophilia; CEL-NOS, idiopathic hypereosinophilic syndrome, myeloid/lymphoid neoplasms with eosinophilia, and gene rearrangement are considered in the revised 2016 WHO classification. However, hypereosinophilia can be encountered in systemic mastocytosis, CML, other types of MPNs, MDS/MPNs, AML, MDS and T cell lymphoproliferative disorders. Once, cytogenetically <em>PDGFRA</em>, <em>PDGFRB</em>, <em>FGFR1</em> rearrangements, and <em>PCM1-JAK2</em> fusion and other causes of hypereosinophilia are excluded, NGS should be performed for the patients with eosinophilia.</p>
<p>Lastly, mast cell diseases show point mutations of the <em>KIT</em> gene (D816V) in 95% of cases. Although the presence of D816V mutations is important at diagnosis for systemic mastocytosis, a minority of patients harbor mutations on exon 17 or other exons (102) Additionally; <em>TET2</em>, <em>ASXL1</em>, <em>RUX1</em>, <em>SRSF2,</em> and <em>JAK2</em> mutations can be seen on the patients with mastocytosis (103,104).</p>
<p>We outlined some of the common genes that are important to assess in various hematologic neoplasms with emphasis on&nbsp;myeloid neoplasia. Despite the significant progress in lymphoid neoplasms, particularly for understanding its pathogenesis, NGS is no ready to be integrated into routine clinical testing.&nbsp;Most of the&nbsp;NGS&nbsp;panels&nbsp;offered for assessment of&nbsp;myeloid&nbsp;neoplasms&nbsp;contain between&nbsp;30-60 targeted genes.&nbsp;The&nbsp;market for NGS-based diagnostic procedures is rapidly growing,&nbsp;therefore&nbsp;it is impossible to list all the available tests. However, based on current literature,&nbsp;a panel with the following genes will most likely be informative in clinical assessment for most of the myeloid neoplasms: <em>FLT3</em>,&nbsp;<em>NPM1</em>, <em>CEBPA</em>,&nbsp;<em>TP53</em>, <em>IDH1/2</em>, <em>DNMT3A</em>, <em>TET2</em>, <em>CSF3R</em>, <em>SRSF2</em>, <em>KIT</em>, <em>NRAS</em>, <em>RUNX1</em>, <em>WT1</em>, <em>ASXL1</em>, <em>SF3B1</em>&nbsp;for AML; <em>NRAS</em>, <em>KRAS</em>, <em>IDH1/2</em>, <em>TET2</em>, <em>EZH2</em>, <em>ASXL1</em>, <em>RUNX1</em>, <em>TP53</em>, <em>DNMT3A</em>,&nbsp;<em>SF3B1</em>,&nbsp;<em>U2AF1</em>,&nbsp;<em>ETV6</em>&nbsp;for MDS;&nbsp;<em>JAK2</em>,&nbsp;<em>CALR,</em>&nbsp;<em>MPL</em>,&nbsp;<em>IDH1/2</em>,&nbsp;<em>ASXL1</em>,&nbsp;<em>TET2</em>,&nbsp;<em>EZH2</em>, <em>SRSF2</em>, <em>SF3B1</em>, <em>TP53</em>&nbsp;for MPN;&nbsp;<em>TET2</em>,&nbsp;<em>SRSF2</em>, <em>ASXL1</em>, <em>RUNX1</em>, <em>NRAS</em>, <em>TP53&nbsp;</em>for CMML;&nbsp;<em>SF3B1</em>&nbsp;for MD/MPN RS-T;&nbsp;<em>CSF3R</em>&nbsp;for&nbsp;CNL,&nbsp;<em>SETBP1</em>&nbsp;for aCML,&nbsp;<em>KIT</em>&nbsp;(D816V) for mast cell disease.</p>
<p>In conclusion; the progress of NGS and widen utilization on hematologic malignancies will&nbsp;facilitate a more precise diagnosis and treatment for patients with hematologic neoplasms. AML, MDS, and MPNs are characterized by morphologic or phenotypic similarities but these disorders harbor different mutations types with different prognostic implications. Myeloid neoplasms that lack a cytogenetic alteration but show a somatic mutation can be diagnosed by the above-mentioned mutation profiles. These mutations can also facilitate new novel therapeutic agents. In the coming years, it appears that major efforts with more comprehensive high throughput NGS sequencing and inclusion of genomic data in clinical management will enable the delivery of precision medicine.</p>
<p><strong>Conflict of Interest:</strong> No conflict of interest was declared by the authors.</p>
<p><strong>Financial Disclosure:</strong> No financial disclosure was declared by the authors.</p>												        					</div>
												        					<!--
												        					<div	style="width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;">
												        									<a class="menu" href="#summary_en">Summary</a><a class="selected_menu" href="#introduction">Introduction</a><a class="menu" href="#reference">Reference</a>												        					</div>
												        					-->
												        				</div>
												        				
												        				
												        																        				
												        				<div  style="width:100%; float:left;  padding-bottom:0px;">
												        					<div	style="width:100%; float:left; min-height:45px; text-align:justify">
												        						<a href="" name="reference"></a>
												        						<!--<h2>Reference</h2>-->
												        						<h3>REFERENCES</h3>
<p>1. Kuo FC. Next generation sequencing in hematolymphoid neoplasia. Seminars in Hematology 2018. <br />2. Yohe S, Thyagarajan B. Review of Clinical Next-Generation Sequencing. Arch Pathol Lab Med 2017;141:1544-57.<br />3. Sallman DA, Padron E. Integrating mutation variant allele frequency into clinical practice in myeloid malignancies. Hematol Oncol Stem Cell Ther 2016;9:89-95.<br />4. Jaffe E AD, Campo E, Harris NL, Quintanilla-Martinez L. Hematopathology Molecular Diagnosis in Hematopathology. Molecular Diagnosis in Hematopathology. Philadelphia PA: Elsevier; 2017:1-1216.<br />5. Cancer Genome Atlas Research Network, Ley TJ, Miller C, Ding L, Raphael BJ, Mungall AJ, et al. Genomic and epigenomic landscapes of adult de novo acute myeloid leukemia. N Engl J Med 2013;368:2059-74.<br />6. Mrozek K, Marcucci G, Paschka P, Whitman SP, Bloomfield CD. Clinical relevance of mutations and gene-expression changes in adult acute myeloid leukemia with normal cytogenetics: are we ready for a prognostically prioritized molecular classification? Blood 2007;109:431-48. <br />7. Dohner H, Weisdorf DJ, Bloomfield CD. Acute Myeloid Leukemia. N Engl J Med 2015;373:1136-52. <br />8. Arber DA, Orazi A, Hasserjian R, Thiele J, Borowitz MJ, Le Beau MM, et al. The 2016 revision to the World Health Organization classification of myeloid neoplasms and acute leukemia. Blood 2016;127:2391-405.<br />9. Duncavage EJ, Tandon B. The utility of next-generation sequencing in diagnosis and monitoring of acute myeloid leukemia and myelodysplastic syndromes. Int J Lab Hematol 2015;37(Suppl 1):115-21. <br />10. Sperling AS, Gibson CJ, Ebert BL. The genetics of myelodysplastic syndrome: from clonal haematopoiesis to secondary leukaemia. Nat Rev Cancer 2017;17:5-19. <br />11. Swerdlow SH CE, Harris NL, Jaffe ES, Pileri SA, Stein H, Thiele J. WHO Classification of Tumours of the Haematopoietic and Lymphoid Tissues 2017.<br />12. Dang L, Jin S, Su SM. IDH mutations in glioma and acute myeloid leukemia. Trends Mol Med 2010;16:387-97.<br />13. Falini B, Mecucci C, Tiacci E, Alcalay M, Rosati R, Pasqualucci L, et al. Cytoplasmic nucleophosmin in acute myelogenous leukemia with a normal karyotype. N Engl J Med 2005;352:254-66. <br />14. Ley TJ, Ding L, Walter MJ, McLellan MD, Lamprecht T, Larson DE, et al. DNMT3A mutations in acute myeloid leukemia. N Engl J Med 2010;363:2424-33.<br />15. Marcucci G, Haferlach T, D&ouml;hner H. Molecular genetics of adult acute myeloid leukemia: Prognostic and therapeutic implications. J Clin Oncol 2011;29:475-86.<br />16. Metzeler KH, Maharry K, Radmacher MD, Mrozek K, Margeson D, Becker H, et al. TET2 mutations improve the new European LeukemiaNet risk classification of acute myeloid leukemia: A cancer and leukemia group B study. J Clin Oncol 2011;29:1373-81.<br />17. Patel U, Luthra R, Medeiros LJ, Patel KP. Diagnostic, Prognostic, and Predictive Utility of Recurrent Somatic Mutations in Myeloid Neoplasms. Clin Lymphoma Myeloma Leuk 2017;17:62-74. <br />18. Paschka P, Schlenk RF, Gaidzik VI, Habdank M, Kronke J, Bullinger L, et al. IDH1 and IDH2 mutations are frequent genetic alterations in acute myeloid leukemia and confer adverse prognosis in cytogenetically normal acute myeloid leukemia with NPM1 mutation without FLT3 internal tandem duplication. J Clin Oncol 2010;28:3636-43. <br />19. Chang C-C OR. Precision Molecular Pathology of Myeloid Neoplasms. In: PT C, editor. Precision Molecular Pathology of Myeloid Neoplasms. Acute Myeloid Leukemia with Recurrent Genetic Abnormalities, Part II: Mutations Involving CEBPA, NPM1, and RUNX1: Springer International Publishing AG; 2018:427.<br />20. Perl AE. The role of targeted therapy in the management of patients with AML. Hematology Am Soc Hematol Educ Program 2017;2017:54-65. <br />21. DiNardo CD, Stein EM, de Botton S, Roboz GJ, Altman JK, Mims AS, et al. Durable Remissions with Ivosidenib in IDH1-Mutated Relapsed or Refractory AML. N Engl J Med 2018;378:2386-98. <br />22. Bains A, Luthra R, Medeiros LJ, Zuo Z. FLT3 and NPM1 mutations in myelodysplastic syndromes: Frequency and potential value for predicting progression to acute myeloid leukemia. Am J Clin Pathol 2011;135:62-9. <br />23. E. H. Hematopathology: A Volume in the Series: Foundations in Diagnostic Pathology. In: JR G, editor. Hematopathology. Molecular Hematopathology. 3rd ed. Philadelphia, PA: Elsevier; 2018:800.<br />24. Mendler JH, Maharry K, Radmacher MD, Mrozek K, Becker H, Metzeler KH, et al. RUNX1 mutations are associated with poor outcome in younger and older patients with cytogenetically normal acute myeloid leukemia and with distinct gene and MicroRNA expression signatures. J Clin Oncol 2012;30:3109-18. <br />25. Gaidzik VI, Bullinger L, Schlenk RF, Zimmermann AS, R&ouml;ck J, Paschka P, et al. RUNX1 mutations in acute myeloid leukemia: results from a comprehensive genetic and clinical analysis from the AML study group. J Clin Oncol 2011;29:1364-72. <br />26. Kussick SJ, Stirewalt DL, Yi HS, Sheets KM, Pogosova-Agadjanyan E, Braswell S, et al. A distinctive nuclear morphology in acute myeloid leukemia is strongly associated with loss of HLA-DR expression and FLT3 internal tandem duplication. Leukemia 2004;18:1591-8. <br />27. O&rsquo;Donnell MR, Tallman MS, Abboud CN, Altman JK, Appelbaum FR, Arber DA, et al. Acute Myeloid Leukemia, Version 3.2017, NCCN Clinical Practice Guidelines in Oncology. J Natl Compr Canc Netw 2017;15:926-57. <br />28. Stone RM, Mandrekar SJ, Sanford BL, Laumann K, Geyer S, Bloomfield CD, et al. Midostaurin plus Chemotherapy for Acute Myeloid Leukemia with a FLT3 Mutation. N Engl J Med 2017;377:454-64. <br />29. Assi R, Ravandi F. FLT3 inhibitors in acute myeloid leukemia: Choosing the best when the optimal does not exist. Am J Hematol 2018;93:553-63. <br />30. Kayser S, Levis MJ. FLT3 tyrosine kinase inhibitors in acute myeloid leukemia: clinical implications and limitations. Leuk Lymphoma 2014;55:243-55.<br />31. Nardi V, Hasserjian RP. Genetic Testing in Acute Myeloid Leukemia and Myelodysplastic Syndromes. Surg Pathol Clin 2016;9:143-63. <br />32. Kiyoi H, Naoe T. FLT3 mutations in acute myeloid leukemia. Methods Mol Med 2006;125:189-97. <br />33. Papaemmanuil E, Gerstung M, Bullinger L, Gaidzik VI, Paschka P, Roberts ND, et al. Genomic classification and prognosis in acute myeloid leukemia. N Engl J Med 2016;374:2209-21.<br />34. Patel JP, Gonen M, Figueroa ME, Fernandez H, Sun Z, Racevskis J, et al. Prognostic relevance of integrated genetic profiling in acute myeloid leukemia. N Engl J Med 2012;366:1079-89. <br />35. Allen C, Hills RK, Lamb K, Evans C, Tinsley S, Sellar R, et al. The importance of relative mutant level for evaluating impact on outcome of KIT, FLT3 and CBL mutations in core-binding factor acute myeloid leukemia. Leukemia 2013;27:1891-901. <br />36. Boissel N, Leroy H, Brethon B, Philippe N, de Botton S, Auvrignon A, et al. Incidence and prognostic impact of c-Kit, FLT3, and Ras gene mutations in core binding factor acute myeloid leukemia (CBF-AML). Leukemia 2006;20:965-70. <br />37. Balsat M, Renneville A, Thomas X, de Botton S, Caillot D, Marceau A, et al. Postinduction Minimal Residual Disease Predicts Outcome and Benefit From Allogeneic Stem Cell Transplantation in Acute Myeloid Leukemia With NPM1 Mutation: A Study by the Acute Leukemia French Association Group. J Clin Oncol 2017;35:185-93. <br />38. Bejar R, Lord A, Stevenson K, Bar-Natan M, P&eacute;rez-Ladaga A, Zaneveld J. TET2 mutations predict response to hypomethylating agents in myelodysplastic syndrome patients. Blood 2014;124:2705-12.<br />39. Medeiros BC, Fathi AT, DiNardo CD, Pollyea DA, Chan SM, Swords R. Isocitrate dehydrogenase mutations in myeloid malignancies. Leukemia 2017;31:272-81.<br />40. Patnaik MM, Hanson CA, Hodnefield JM, Lasho TL, Finke CM, Knudson RA, et al. Differential prognostic effect of IDH1 versus IDH2 mutations in myelodysplastic syndromes: A Mayo Clinic Study of 277 patients. Leukemia 2012;26:101-5.<br />41. Bejar R, Stevenson K, Abdel-Wahab O, Galili N, Nilsson B, Garcia-Manero G, et al. Clinical effect of point mutations in myelodysplastic syndromes. N Engl J Med 2011;364:2496-506.<br />42. Gelsi-Boyer V, Brecqueville M, Devillier R, Murati A, Mozziconacci MJ, Birnbaum D. Mutations in ASXL1 are associated with poor prognosis across the spectrum of malignant myeloid diseases. J Hematol Oncol 2012;5:12. <br />43. Gelsi-Boyer V, Trouplin V, Adelaide J, Bonansea J, Cervera N, Carbuccia N, et al. Mutations of polycomb-associated gene ASXL1 in myelodysplastic syndromes and chronic myelomonocytic leukaemia. Br J Haematol 2009;145:788-800. <br />44. Bacher U, Haferlach T, Kern W, Haferlach C, Schnittger S. A comparative study of molecular mutations in 381 patients with myelodysplastic syndrome and in 4130 patients with acute myeloid leukemia. Haematologica 2007;92:744-52.<br />45. Sun QY, Ding LW, Tan KT, Chien W, Mayakonda A, Lin DC, et al. Ordering of mutations in acute myeloid leukemia with partial tandem duplication of MLL (MLL-PTD). Leukemia 2017;31:1-10.<br />46. Devillier R, Mansat-De Mas V, Gelsi-Boyer V, Demur C, Murati A, Corre J, et al. Role of ASXL1 and TP53 mutations in the molecular classification and prognosis of acute myeloid leukemias with myelodysplasia-related changes. Oncotarget 2015;6:8388-96. <br />47. Lindsley RC, Mar BG, Mazzola E, Grauman PV, Shareef S, Allen SL, et al. Acute myeloid leukemia ontogeny is defined by distinct somatic mutations. Blood 2015;125:1367-76. <br />48. Rucker FG, Schlenk RF, Bullinger L, Kayser S, Teleanu V, Kett H, et al. TP53 alterations in acute myeloid leukemia with complex karyotype correlate with specific copy number alterations, monosomal karyotype, and dismal outcome. Blood 2012;119:2114-21. <br />49. Welch JS, Petti AA, Miller CA, Fronick CC, O&rsquo;Laughlin M, Fulton RS, et al. TP53 and Decitabine in Acute Myeloid Leukemia and Myelodysplastic Syndromes. N Engl J Med 2016;375:2023-36. <br />50. Welch JS, Petti AA, Ley TJ. Decitabine in TP53-Mutated AML. N Engl J Med 2017;376:797-8. <br />51. Jongen-Lavrencic M, Grob T, Hanekamp D, Kavelaars FG, Al Hinai A, Zeilemaker A, et al. Molecular Minimal Residual Disease in Acute Myeloid Leukemia. N Engl J Med 2018;378:1189-99. <br />52. Dohner H, Estey E, Grimwade D, Amadori S, Appelbaum FR, Buchner T, et al. Diagnosis and management of AML in adults: 2017 ELN recommendations from an international expert panel. Blood 2017;129:424-47. <br />53. Li P. ORS. Acute Myeloid Leukemia with Myelodysplasia-Related Changes, Therapy-Related Myeloid Neoplasms, and Acute Myeloid Leukemia, Not Otherwise Specified. In: Chang CC, Ohgami R, editors. Precision Molecular Pathology of Myeloid Neoplasms. Springer, Cham; 2018.<br />54. Metzeler KH, Herold T, Rothenberg-Thurley M, Amler S, Sauerland MC, G&ouml;rlich D, et al. Spectrum and prognostic relevance of driver gene mutations in acute myeloid leukemia. Blood 2016;128:686-98.<br />55. Devillier R, Gelsi-Boyer V, Brecqueville M, Carbuccia N, Murati A, Vey N, et al. Acute myeloid leukemia with myelodysplasia-related changes are characterized by a specific molecular pattern with high frequency of ASXL1 mutations. Am J Hematol 2012;87:659-62.<br />56. Ohgami RS, Ma L, Merker JD, Gotlib JR, Schrijver I, Zehnder JL, et al. Next-generation sequencing of acute myeloid leukemia identifies the significance of TP53, U2AF1, ASXL1, and TET2 mutations. Mod Pathol 2015;28:706-14.<br />57. Bhatia S. Therapy-related myelodysplasia and acute myeloid leukemia. Semin Oncol 2013;40:666-75. <br />58. Ok CY, Patel KP, Garcia-Manero G, Routbort MJ, Fu B, Tang G, et al. Mutational profiling of therapy-related myelodysplastic syndromes and acute myeloid leukemia by next generation sequencing, a comparison with de novo diseases. Leuk Res 2015;39:348-54. <br />59. Arber DA, Slovak ML, Popplewell L, Bedell V, Ikle D, Rowley JD, et al. Therapy-related acute myeloid leukemia/myelodysplasia with balanced 21q22 translocations. Am J Clin Pathol 2002;117:306-13. <br />60. Czader M, Orazi A. Acute Myeloid Leukemia and Other Types of Disease Progression in Myeloproliferative Neoplasms. Am J Clin Pathol 2015;144:188-206. <br />61. Shih AH, Chung SS, Dolezal EK, Zhang SJ, Abdel-Wahab OI, Park CY, et al. Mutational analysis of therapy-related myelodysplastic syndromes and acute myelogenous leukemia. Haematologica 2013;98:908-12. <br />62. Tiu RV, Visconte V, Traina F, Schwandt A, Maciejewski JP. Updates in cytogenetics and molecular markers in MDS. Curr Hematol Malig Rep 2011;6:126-35. <br />63. Haase D, Germing U, Schanz J, Pfeilst&ouml;cker M, N&ouml;sslinger T, Hildebrandt B, et al. New insights into the prognostic impact of the karyotype in MDS and correlation with subtypes: evidence from a core dataset of 2124 patients. Blood 2007;110:4385-95. <br />64. Haferlach T, Nagata Y, Grossmann V, Okuno Y, Bacher U, Nagae G, et al. Landscape of genetic lesions in 944 patients with myelodysplastic syndromes. Leukemia 2014;28:241-7.<br />65. Papaemmanuil E, Gerstung M, Malcovati L, Tauro S, Gundem G, Van Loo P, et al. Clinical and biological implications of driver mutations in myelodysplastic syndromes. Blood 2013;122:3616-27.<br />66. Bartels S, Schipper E, Hasemeier B, Kreipe H, Lehmann U. Routine clinical mutation profiling using next generation sequencing and a customized gene panel improves diagnostic precision in myeloid neoplasms. Oncotarget 2016;7:30084-93.<br />67. Walter MJ, Ding L, Shen D, Shao J, Grillot M, McLellan M, et al. Recurrent DNMT3A mutations in patients with myelodysplastic syndromes. Leukemia 2011;25:1153-8.<br />68. Jadersten M, Saft L, Smith A, Kulasekararaj A, Pomplun S, G&ouml;hring G, et al. TP53 mutations in low-risk myelodysplastic syndromes with del (5q) predict disease progression. J Clin Oncol 2011;29:1971-9.<br />69. Kuo FC, Dong F. Next-generation sequencing-based panel testing for myeloid neoplasms. Curr Hematol Malig Rep 2015;10:104-11.<br />70. Papaemmanuil E, Cazzola M, Boultwood J, Malcovati L, Vyas P, Bowen D, et al. Somatic SF3B1 mutation in myelodysplasia with ring sideroblasts. N Engl J Med 2011;365:1384-95.<br />71. Brecqueville M, Rey J, Bertucci F, Coppin E, Finetti P, Carbuccia N, et al. Mutation analysis of ASXL1, CBL, DNMT3A, IDH1, IDH2, JAK2, MPL, NF1, SF3B1, SUZ12, and TET2 in myeloproliferative neoplasms. Genes Chromosomes Cancer 2012;51:743-55. <br />72. Malcovati L, Galli A, Travaglino E, Ambaglio I, Rizzo E, Molteni E, et al. Clinical significance of somatic mutation in unexplained blood cytopenia. Blood 2017;129:3371-8. <br />73. Jain M, Tripathi A. ICUS/CCUS/CHIP: basics &amp; beyond. Expert Rev Hematol 2017;10:915-20. <br />74. Jaiswal S, Fontanillas P, Flannick J, Manning A, Grauman PV, Mar BG, et al. Age-related clonal hematopoiesis associated with adverse outcomes. N Engl J Med 2014;371:2488-98.<br />75. Zink F, Stacey SN, Norddahl GL, Frigge ML, Magnusson OT, Jonsdottir I, et al. Clonal hematopoiesis, with and without candidate driver mutations, is common in the elderly. Blood 2017;130:742-52.<br />76. Brauninger A, Blau W, Kunze K, Desch AK, Brobeil A, Tur MK, et al. Targeted Next-Generation Sequencing Is a Sensitive Tool for Differential Diagnosis of Myelodysplastic Syndromes in Bone Marrow Trephines. J Mol Diagn 2018;20:344-54. <br />77. Barbui T, Thiele J, Gisslinger H, Kvasnicka HM, Vannucchi AM, Guglielmelli P, et al. The 2016 WHO classification and diagnostic criteria for myeloproliferative neoplasms: document summary and in-depth discussion. Blood Cancer J 2018;8:15.<br />78. Tefferi A, Lasho TL, Schwager SM, Steensma DP, Mesa RA, Li CY, et al. The JAK2(V617F) tyrosine kinase mutation in myelofibrosis with myeloid metaplasia: lineage specificity and clinical correlates. Br J Haematol 2005;131:320-8. <br />79. Baxter EJ, Scott LM, Campbell PJ, East C, Fourouclas N, Swanton S, et al. Acquired mutation of the tyrosine kinase JAK2 in human myeloproliferative disorders. Lancet 2005;365:1054-61. <br />80. Vannucchi AM, Antonioli E, Guglielmelli P, Rambaldi A, Barosi G, Marchioli R, et al. Clinical profile of homozygous JAK2 617V&gt;F mutation in patients with polycythemia vera or essential thrombocythemia. Blood 2007;110:840-6.<br />81. Klampfl T, Gisslinger H, Harutyunyan AS, Nivarthi H, Rumi E, Milosevic JD, et al. Somatic mutations of calreticulin in myeloproliferative neoplasms. N Engl J Med 2013;369:2379-90.<br />82. Nangalia J, Massie CE, Baxter EJ, Nice FL, Gundem G, Wedge DC, et al. Somatic CALR mutations in myeloproliferative neoplasms with nonmutated JAK2. N Engl J Med 2013;369:2391-405.<br />83. Rotunno G, Mannarelli C, Guglielmelli P, Pacilli A, Pancrazzi A, Pieri L, et al. Impact of calreticulin mutations on clinical and hematological phenotype and outcome in essential thrombocythemia. Blood 2014;123:1552-5.<br />84. Tefferi A, Lasho TL, Finke C, Belachew AA, Wassie EA, Ketterling RP, et al. Type 1 vs type 2 calreticulin mutations in primary myelofibrosis: differences in phenotype and prognostic impact. Leukemia 2014;28:1568-70. <br />85. Mesa RA, Jamieson C, Bhatia R, Deininger MW, Fletcher CD, Gerds AT, et al. NCCN Guidelines Insights: Myeloproliferative Neoplasms, Version 2.2018. J Natl Compr Canc Netw 2017;15:1193-207.<br />86. Tefferi A, Guglielmelli P, Larson DR, Finke C, Wassie EA, Pieri L, et al. Long-term survival and blast transformation in molecularly annotated essential thrombocythemia, polycythemia vera, and myelofibrosis. Blood 2014;124:2507-13. <br />87. Pikman Y, Lee BH, Mercher T, McDowell E, Ebert BL, Gozo M, et al. MPLW515L is a novel somatic activating mutation in myelofibrosis with myeloid metaplasia. PLoS Med 2006;3:270.<br />88. Langabeer SE, Andrikovics H, Asp J, Bellosillo B, Carillo S, Haslam K, et al. Molecular diagnostics of myeloproliferative neoplasms. Eur J Haematol 2015;95:270-9.<br />89. Rumi E, Pietra D, Pascutto C, Guglielmelli P, Mart&iacute;nez-Trillos A, Casetti I, et al. Clinical effect of driver mutations of JAK2, CALR, or MPL in primary myelofibrosis. Blood 2014;124:1062-9.<br />90. Tefferi A. Novel mutations and their functional and clinical relevance in myeloproliferative neoplasms: JAK2, MPL, TET2, ASXL1, CBL, IDH and IKZF1. Leukemia 2010;24:1128-38.<br />91. Tefferi A, Vainchenker W. Myeloproliferative neoplasms: molecular pathophysiology, essential clinical understanding, and treatment strategies. J Clin Oncol 2011;29:573-82. <br />92. Yogarajah M, Tefferi A. Leukemic Transformation in Myeloproliferative Neoplasms: A Literature Review on Risk, Characteristics, and Outcome. Mayo Clin Proc 2017;92:1118-28. <br />93. Itzykson R, Kosmider O, Renneville A, Gelsi-Boyer V, Meggendorfer M, Morabito M, et al. Prognostic score including gene mutations in chronic myelomonocytic leukemia. J Clin Oncol 2013;31:2428-36. <br />94. Wassie EA, Itzykson R, Lasho TL, Kosmider O, Finke CM, Hanson CA, et al. Molecular and prognostic correlates of cytogenetic abnormalities in chronic myelomonocytic leukemia: a Mayo Clinic-French Consortium Study. Am J Hematol 2014;89:1111-5. <br />95. Pich A, Riera L, Sismondi F, Godio L, Davico Bonino L, Marmont F, et al. JAK2V617F activating mutation is associated with the myeloproliferative type of chronic myelomonocytic leukaemia. J Clin Pathol 2009;62:798-801. <br />96. Padron E, Dezern A, Andrade-Campos M, Vaddi K, Scherle P, Zhang Q, et al. A Multi-Institution Phase I Trial of Ruxolitinib in Patients with Chronic Myelomonocytic Leukemia (CMML). Clin Cancer Res 2016;22:3746-54. <br />97. E.D. GMH. Chronic Myelomonocytic Leukemia: Clinical and Pathologic Features. In: Chung-Che (Jeff) Chang RSO, editor. Precision Molecular Pathology of Myeloid Neoplasms. VOL 12: Springer International Publishing AG 2018:2018.<br />98. Caye A, Strullu M, Guidez F, Cassinat B, Gazal S, Fenneteau O, et al. Juvenile myelomonocytic leukemia displays mutations in components of the RAS pathway and the PRC2 network. Nat Genet 2015;47:1334-40. <br />99. Niemeyer CM, Kang MW, Shin DH, Furlan I, Erlacher M, Bunin NJ, et al. Germline CBL mutations cause developmental abnormalities and predispose to juvenile myelomonocytic leukemia. Nat Genet 2010;42:794-800. <br />100. Gambacorti-Passerini CB, Donadoni C, Parmiani A, Pirola A, Redaelli S, Signore G, et al. Recurrent ETNK1 mutations in atypical chronic myeloid leukemia. Blood 2015;125:499-503.<br />101. Meggendorfer M, Bacher U, Alpermann T, Haferlach C, Kern W, Gambacorti-Passerini C, et al. SETBP1 mutations occur in 9% of MDS/MPN and in 4% of MPN cases and are strongly associated with atypical CML, monosomy 7, isochromosome i(17)(q10), ASXL1 and CBL mutations. Leukemia 2013;27:1852-60. <br />102. Wang SA, Tam W, Tsai AG, Arber DA, Hasserjian RP, Geyer JT, et al. Targeted next-generation sequencing identifies a subset of idiopathic hypereosinophilic syndrome with features similar to chronic eosinophilic leukemia, not otherwise specified. Mod Pathol 2016;29:854-64.<br />103. Arock M, Sotlar K, Akin C, Broesby-Olsen S, Hoermann G, Escribano L, et al. KIT mutation analysis in mast cell neoplasms: recommendations of the European Competence Network on Mastocytosis. Leukemia 2015;29:1223-32. <br />104. Schwaab J, Schnittger S, Sotlar K, Walz C, Fabarius A, Pfirrmann M, et al. Comprehensive mutational profiling in advanced systemic mastocytosis. Blood 2013;122:2460-6.</p>												        					</div>
												        					<!--
												        					<div	style="width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;">
												        									<a class="menu" href="#summary_en">Summary</a><a class="menu" href="#introduction">Introduction</a><a class="selected_menu" href="#reference">Reference</a>												        					</div>
												        					-->
												        				</div>
												        				
												        				
												        															        			
		
											        	</div>