<div class="col-lg-12 col-md-12 col-sm-12 col-xs-12">
																		        				
												        				<div  style="width:100%; float:left;  padding-bottom:0px;">
												        					<div	style="width:100%; float:left; min-height:45px; text-align:justify">
												        						<a href="" name="summary_en"></a>
												        						<!--<h2>Summary</h2>-->
												        						<h4>Abstract</h4>
<p><strong>Background</strong>: Coagulase-negative staphylococci, which belong to the normal microbiota of the skin and mucous membranes, are opportunistic pathogens. <em>sasX</em>, a newly described protein, is thought to play an important role in nasal colonization and methicillin-resistant <em>Staphylococcus aureus </em>virulence, and it may be acquired from coagulase-negative staphylococci by horizontal gene transfer. It has been considered that understanding the function of <em>sasX</em> gene may help clarify the relevance of the different adhesion mechanisms in the pathogenesis of infections associated with biofilm.<br /><strong>Aims</strong>: To investigate the <em>sasX</em> gene presence, staphylococcal cassette chromosome <em>mec</em> types, and antimicrobial resistance patterns of invasive and noninvasive coagulase-negative staphylococci isolates.<br /><strong>Study Design</strong>: Cross-sectional study.<br /><strong>Methods</strong>: The study included a total of 180 coagulase-negative staphylococci strains. Non-invasive isolates (n=91) were obtained from the hands of healthy volunteers who do not work at the hospital (n=30), the nasal vestibule of healthy volunteer hospital workers (n=26), and central venous catheter (n=35). Invasive isolates (n=89) were isolated from peripheral blood cultures of inpatients who do not have catheters. All isolates were identified by conventional microbiological methods, automated systems, and, if needed, with matrix-assisted laser desorption/ionization-time of flight. Staphylococcal cassette chromosome <em>mec</em> typing, <em>sasX</em> and <em>mec</em> gene detection, antibiotic susceptibility, and <em>sasX</em> gene sequence analysis were performed.<br /><strong>Results</strong>: Peripheral blood, central venous catheter colonization, and nasal vestibule isolates were positive for the <em>sasX</em> gene, whereas hand isolates were negative. <em>sasX </em>gene was present in 17 isolates, and no statistical significance was found between invasive and noninvasive isolates (p=0.173). Sequence analysis of the <em>sasX </em>genes showed high homology to related proteins of <em>Staphylococcus</em> phage SPbeta-like and <em>Staphylococcus epidermidis</em> RP62A. staphylococcal cassette chromosome <em>mec</em> type V was the most prevalent regardless of species. staphylococcal cassette chromosome <em>mec</em> type II was more frequent in invasive isolates and found to be statistically important for invasive and noninvasive <em>S. epidermidis</em> isolates (p=0.029). <em>Staphylococcus haemolyticus</em> isolates had the overall highest resistance rates. Resistance to ciprofloxacin, trimethoprim-sulfamethoxazole, and erythromycin was found to be higher in isolates from catheter and blood culture.<em> Staphylococcus hominis</em> isolates had the highest rate for inducible clindamycin resistance. None of the isolates were resistant to vancomycin, teicoplanin, and linezolid.<br /><strong>Conclusion</strong>: The <em>sasX</em> gene is detected in 9.44%&nbsp; of the isolates. There is no statistical difference between the <em>sasX</em>-positive and -negative isolates in terms of antibacterial resistance and the presence of <em>sasX </em>and SCC<em>mec</em> types. Further studies about the role of <em>sasX</em> at virulence in coagulase-negative staphylococci, especially from clinical samples such as tracheal aspirate and abscess isolates, and distribution of staphylococcal cassette chromosome <em>mec</em> types are needed.</p>												        					</div>
												        					<!--
												        					<div	style="width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;">
												        									<a class="selected_menu" href="#summary_en">Summary</a><a class="menu" href="#introduction">Introduction</a><a class="menu" href="#methods">Methods</a><a class="menu" href="#results">Results</a><a class="menu" href="#discussion">Discussion</a><a class="menu" href="#reference">Reference</a>												        					</div>
												        					-->
												        				</div>
												        				
												        				
												        																        				
												        				<div  style="width:100%; float:left;  padding-bottom:0px;">
												        					<div	style="width:100%; float:left; min-height:45px; text-align:justify">
												        						<a href="" name="introduction"></a>
												        						<!--<h2>Introduction</h2>-->
												        						<hr size="1" color="#f2f2f2">
<p>Coagulase-negative staphylococci (CoNS), which belong to the normal microbiota of the skin and mucous membranes, are opportunistic pathogens, particularly in patients who are immunocompromised or have indwelling devices (1). Especially, <em>Staphylococcus epidermidis</em> and <em>Staphylococcus haemolyticus</em> are considered to be the major nosocomial pathogens, as well as others like <em>Staphylococcus saprophyticus</em> (2,3). These bacteria are also the cause of bacteremia in nearly 30% of bloodstream-related infections and responsible for sepsis episodes in immunocompromised patients (4).</p>
<p>Resistance to methicillin and almost all beta-lactam antibiotics in staphylococci is mostly caused by the acquisition of the <em>mecA </em>gene, encoding an additional altered low-affinity penicillin-binding protein, which is located on a mobile genetic island, termed the staphylococcal cassette chromosome <em>mec</em> (SCC<em>mec</em>). SCC<em>mec</em> contains <em>mec</em> gene complex and <em>ccr</em> gene complex of varying size and genetic structure (5). There is strong evidence suggesting the diversity of SCC<em>mec</em> in CoNS. New combinations of a <em>mec</em> complex and <em>ccr</em> complex, differences from the amplification of the <em>ccr</em> gene, and multiple presence of <em>ccr</em> gene complex make the typing difficult in CoNS (6-9).</p>
<p><em>sasX</em>, a homolog of <em>sesI</em>, is a newly described protein that is thought to play an important role in nasal colonization and methicillin-resistant <em>Staphylococcus aureus</em> (MRSA) virulence, and <em>sasX</em> gene is located on a ϕSP&beta;-like prophage (10). <em>sesI</em> is one of the <em>S. epidermidis</em> surface (ses) proteins that can be expressed during biofilm expression (11) and is encoded by ϕSP&beta; prophage (12). The sequence similarity of <em>sesI</em> and <em>sasX</em> genes indicates that the latter may be acquired from CoNS by horizontal gene transfer, and it has been considered that understanding the function of <em>sasX</em> gene may help clarify the relevance of the different adhesion mechanisms in the pathogenesis of infections associated with biofilm (13).</p>
<p>This study aimed to investigate the <em>sasX</em> gene carriage, which was thought to play a role at attachment, pathogenicity, and distribution of the SCC<em>mec</em> types of invasive and noninvasive CoNS.</p>												        					</div>
												        					<!--
												        					<div	style="width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;">
												        									<a class="menu" href="#summary_en">Summary</a><a class="selected_menu" href="#introduction">Introduction</a><a class="menu" href="#methods">Methods</a><a class="menu" href="#results">Results</a><a class="menu" href="#discussion">Discussion</a><a class="menu" href="#reference">Reference</a>												        					</div>
												        					-->
												        				</div>
												        				
												        				
												        																        				
												        				<div  style="width:100%; float:left;  padding-bottom:0px;">
												        					<div	style="width:100%; float:left; min-height:45px; text-align:justify">
												        						<a href="" name="methods"></a>
												        						<!--<h2>Methods</h2>-->
												        						<h3>MATERIALS AND METHODS</h3>
<h4>Bacterial strains and identification</h4>
<p>A total of 180 CoNS strains from İbni Sina Hospital Central Microbiology Laboratory Biobank, which were previously approved by the ethical committee for clinical investigations of Ankara University School of Medicine - 21-680-12), were included in the study. Noninvasive isolates (n=91) were obtained between 2013 and 2014 from the hands of healthy volunteers who do not work at the hospital without chronic diseases (n=30), noses of healthy volunteers (nurses<strong>, </strong>doctors<strong>,</strong> and hospital staff) who worked at the university hospital (902 beads) without chronic diseases (n<em>=</em>26), and central venous catheter colonization (n<em>=</em>35). Catheter colonization is defined as the growth of an organism, using the semiquantitative roll-plate culture technique greater than 15 colony-forming units from the tip or the subcutaneous segment of the removed catheter in patients who have no growth at their blood cultures, which were taken simultaneously (14). On the other hand, invasive isolates (n=89) were isolated from peripheral blood cultures of inpatients who do not have catheters. In our study, we defined isolates from sterile body sites, including blood (which were causative agents of bloodstream infections according to guidelines) as invasive and isolates from hands, nasal vestibules, and central venous catheter colonization as noninvasive. Isolates were identified by conventional microbiological methods (Gram staining, catalase, and coagulase tests), BD Phoenix (Becton Dickinson, USA) automated systems, and, if needed, Bruker Microflex MS (Bruker Daltonics, Bremen, Germany).</p>
<h4>Susceptibility testing</h4>
<p>A disk diffusion test determined the susceptibilities of isolate to cefazolin 30 &mu;g, amoxicillin-clavulanate 20/10 &mu;g, rifampicin 5 &mu;g, cefoxitin 30 &mu;g, tetracycline 30 &mu;g, clindamycin 2 &mu;g, erythromycin 15 &mu;g, trimethoprim-sulfamethoxazole 1.25/23.75 &mu;g, gentamicin 10 &mu;g, linezolid 30 &mu;g, and teicoplanin 30 &mu;g, and broth microdilution method for vancomycin minimum inhibitor concentrations (MIC). The results were evaluated according to the Clinical and Laboratory Standards Institute standards (15). Isolates that were resistant to cefoxitin were regarded as methicillin resistant.</p>
<h4><em>sasX, mecA </em>detection, and SCC<em>mec</em> Typing</h4>
<p>Standard polymerase chain reaction (PCR) detected <em>sasX</em> and <em>mecA </em>genes (10,16) and multiplex PCR strategy for SCC<em>mec</em> types as previously described Kondo et al. (17).</p>
<h4>Sequence analysis of <em>sasX</em> gene</h4>
<p>Sequencing of <em>sasX</em> genes was performed using ABI Prism 310 genetic analyzer (Applied Biosystems, Foster City, CA, USA) with BigDye fluorescent terminator chemistry. Sequence data obtained were analyzed with the basic local alignment search tool of the National Center for Biotechnology Information (18).</p>
<h4>Statistical analysis</h4>
<p>The Kruskal-Wallis variance analysis compared the vancomycin MIC values, and chi-square or Fisher&rsquo;s exact tests for all other comparisons, as appropriate. All analyses were performed using SPSS Statistics for Windows, Version 15.0. A p-value &lt;0.05 was considered statistically significant.</p>												        					</div>
												        					<!--
												        					<div	style="width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;">
												        									<a class="menu" href="#summary_en">Summary</a><a class="menu" href="#introduction">Introduction</a><a class="selected_menu" href="#methods">Methods</a><a class="menu" href="#results">Results</a><a class="menu" href="#discussion">Discussion</a><a class="menu" href="#reference">Reference</a>												        					</div>
												        					-->
												        				</div>
												        				
												        				
												        																        				
												        				<div  style="width:100%; float:left;  padding-bottom:0px;">
												        					<div	style="width:100%; float:left; min-height:45px; text-align:justify">
												        						<a href="" name="results"></a>
												        						<!--<h2>Results</h2>-->
												        						<h3>RESULTS</h3>
<h4>Bacterial strains</h4>
<p>Of the 180 isolates, <em>S. epidermis</em> (n=60), <em>S. haemolyticus</em> (n=48), <em>S. hominis </em>(n=46), <em>Staphylococcus capitis</em> (n=13), <em>Staphylococcus warneri</em> (n=8), <em>Staphylococcus lugdunensis</em> (n=2), <em>S. saprophyticus</em> (n=2), and <em>Staphylococcus scuiri</em> (n=1) were isolated. Distribution of the strains according to the sample types are given in <a href="javascript:openWin('uploads/grafik/table_BMJ_2203_0.jpg', 640, 480)"><b>Table 1</b></a>.</p>
<h4>Antimicrobial susceptibility patterns</h4>
<p>Methicillin resistance was found in 135 of the isolates using the cefoxitin disk diffusion test. When overall antibiotic resistance was considered, <em>S. haemolyticus</em> isolates had the highest resistance rates except for clindamycin and tetracycline (<a href="javascript:openWin('uploads/grafik/table_BMJ_2203_1.jpg', 640, 480)"><b>Table 2</b></a>). Resistance to ciprofloxacin, trimethoprim-sulfamethoxazole, and erythromycin was found to be higher in isolates from catheter colonization (71.4%, 71.4%, and 80%, respectively) and blood culture (70.8%, 68.5%, and 85.5%, respectively). Considerable proportions of isolates from the nose were found to be susceptible to ciprofloxacin (96.2%), trimethoprim-sulfamethoxazole (96.2%), and rifampicin (96.2 %). <em>S. hominis</em> isolates had the highest inducible clindamycin resistance rates (36.9%), followed by <em>S. capitis</em> (23.07%), <em>S. epidermidis</em> (11.7%), and <em>S. haemolyticus</em> (8.3%). None of the isolates were found to be resistant to vancomycin, teicoplanin, and linezolid. The resistance patterns of the isolates are given in <a href="javascript:openWin('uploads/grafik/table_BMJ_2203_1.jpg', 640, 480)"><b>Table 2</b></a>. Vancomycin MIC50, MIC90, and MIC ranges were 1, 4, and 0.125-4 &mu;g/mL, respectively, by broth microdilution method, and the minimum, median, and maximum values of vancomycin for the CoNS spp. are given in <a href="javascript:openWin('uploads/grafik/table_BMJ_2203_2.jpg', 640, 480)"><b>Table 3</b></a>. MIC ranges of the samples are given in <a href="javascript:openWin('uploads/grafik/table_BMJ_2203_3.jpg', 640, 480)"><b>Table 4</b></a>. No difference was found between vancomycin MIC values and the isolates (p=0.477) and SCC<em>mec</em> types (p=0.277). No difference was found between <em>sasX</em>-positive and negative isolates in terms of antibiotic resistance patterns.</p>
<h4>Presence of <em>sasX </em>and <em>mecA </em>genes</h4>
<p><em>sasX</em> gene was detected in 9.44% (17/180) of the isolates. Of the strains isolated, 12.36% (n=89) from peripheral blood, 11.42% (n=35) from catheter colonization, and 7.69% (n=26) from nasal vestibules were positive for the <em>sasX</em> gene, whereas strains from healthy hands (n=30) were negative. <em>sasX</em> carriage rate distribution and p-values according to the sample types are presented in <a href="javascript:openWin('uploads/grafik/table_BMJ_2203_4.jpg', 640, 480)"><b>Table 5</b></a>.</p>
<p>Of the 17 <em>sasX</em>-positive strains (17/180), nine were <em>S. haemolyticus </em>(9/48, 18.42%), four <em>S. epidermidis </em>(4/60, 6.66%), three <em>S. hominis </em>(3/46, 6.52%), and one <em>S. capitis </em>(1/13, 7.69%). <em>S. warneri</em>,<em> S. lugdunensis</em>,<em> S. saprophyticus</em>, and <em>S. sciuri</em> isolates were found to be negative for the <em>sasX </em>gene, and no statistical significance was found between invasive and noninvasive isolates (p=0.173).</p>
<p>Of the 180 isolates,<em> mecA </em>carriage was found as 75% (n=135). <em>mecA ratios </em>of the strains were as follows: <em>S. haemolyticus</em>, 91.66% (44/48); <em>S. hominis</em>, 78.26% (36/46); <em>S. epidermidis</em>, 66.66% (40/60); <em>S. capitis</em>, 61.53% (8/13), <em>S. warneri</em>, 37.5% (3/8), <em>S. saprohyticus</em>, 50% (1/2), <em>S. lugdunensis</em>, 100% (2/2), and <em>S. sciuri</em>, 100% (1/1). The <em>mecA</em> gene was detected in 16 of 17 <em>sasX</em> gene-positive isolates (94.11%; p=0.056). The only <em>mecA</em>-negative and <em>sasX</em>-positive strain was a nasal colonizer<em> S. hominis</em> isolate.</p>
<h4>SCC<em>mec</em> types</h4>
<p>With the protocol of Kondo et al. (17), 130 of the 135 <em>mecA</em>-positive isolates were typed. For all isolates, regardless of species, type V (34%) was the most prevalent, followed by type I (23.7%), type II (17%), type VI (10.4%), type III (4.4%), type I-II (1.5%), type I-III (1.5%), and type I-V (3.7%). Five of the isolates (3.7%) could not be typed. SCC<em>mec</em> distribution of the isolates is given in <a href="javascript:openWin('uploads/grafik/table_BMJ_2203_5.jpg', 640, 480)"><b>Table 6</b></a>.</p>
<p>When all invasive and noninvasive isolates were taken into consideration, type II was more frequent than the other types in invasive isolates and found to be statistically significant (p=0.014). Also, the type II ratio was statistically significant (p=0.029) between invasive and noninvasive <em>S. epidermidis</em> isolates.</p>
<h4>Sequence analysis of <em>sasX</em> gene</h4>
<p>Sequence analysis with <em>sasX</em> primers, <em>S. haemolyticus</em> isolates (n=9), and <em>S. capitis </em>isolate (n=1) gave high homology to proteins of<em> Staphylococcus</em> phage SPbeta-like, <em>S. epidermidis</em> RP62A, <em>S. aureus</em> strain XN108, <em>S. aureus subsp aureus</em> Z172, and <em>S. aureus subsp aureus</em> TW20 with the identities of 99%, 99%, 94%, 94%, and 94%, respectively. Sequence analysis of <em>S. hominis</em> isolates (n=3) was similar to that of <em>S. haemolyticus</em> and <em>S. capitis</em>, but the identities of the sequences changed to 94%, 94%, 99%, 99%, and 99%, respectively. The alignment descriptions of <em>S. epidermidis</em> isolates (n=4) were 94%, 99%, 99%, 99%, and 94%. Results of sequence analysis and related protein ID definitions with <em>sasX</em> primers are given in <a href="javascript:openWin('uploads/grafik/table_BMJ_2203_6.jpg', 640, 480)"><b>Table 7</b></a>.</p>												        					</div>
												        					<!--
												        					<div	style="width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;">
												        									<a class="menu" href="#summary_en">Summary</a><a class="menu" href="#introduction">Introduction</a><a class="menu" href="#methods">Methods</a><a class="selected_menu" href="#results">Results</a><a class="menu" href="#discussion">Discussion</a><a class="menu" href="#reference">Reference</a>												        					</div>
												        					-->
												        				</div>
												        				
												        				
												        																        				
												        				<div  style="width:100%; float:left;  padding-bottom:0px;">
												        					<div	style="width:100%; float:left; min-height:45px; text-align:justify">
												        						<a href="" name="discussion"></a>
												        						<!--<h2>Discussion</h2>-->
												        						<h3>DISCUSSION</h3>
<p><em>SasX</em>, a newly described protein, which is secreted by a gene containing a signal peptide and an LPXTG motif, was found at 127.2 kb ϕSP&beta;-like prophage of MRSA strain TW20. It is thought to play an important role in nasal colonization and MRSA virulence and is very similar to the <em>sesI</em> protein encoded by ϕSP&beta; prophage of <em>S. epidermidis</em> RP62A (19). De Backer et al. (20) stated geographical variations between Indian and European MRSA isolates harboring <em>sasX</em> and a newly described variant, <em>sasX-1</em>, from a European MRSA strain.</p>
<p>The potential role of <em>sasX</em> at colonization and virulence, distribution, and approaches about immunization is still being investigated by the researchers (10,21,22). Li et al. (10) found an increase of <em>sasX</em> among invasive MRSA isolates from 2003 to 2011, with low prevalence at community isolates, indicating that clonal spread is predominantly within the hospital setting.</p>
<p>A few data are available about the carriage of <em>sasX</em> in CoNS. Soumya et al. (23) investigated the <em>sasX</em> gene in 40 CoNS clinical isolates and found one positive <em>S. epidermidis</em> isolate obtained from blood. In their study, none of the <em>S. haemolyticus</em>, <em>S. hominis</em>, and <em>S. saprophyticus</em> isolates harbored the <em>sasX</em> gene. In our study, we found <em>sasX</em> carriage in <em>S. haemolyticus</em> (18.75%), <em>S. epidermidis</em> (6.66%), <em>S. hominis</em> (6.52%), and <em>S. capitis</em> (7.69%) isolates. De Backer et al. (20) investigated a total of 32 CoNS; <em>S. epidermidis</em> (n=22), <em>S. capitis</em> (n=4), and <em>S. hominis</em> (n=6) were isolated from endotracheal tubes, but <em>sasX</em> could not be found (20). The different rates of <em>sasX</em> carriage in different species can be explained by the distribution of the isolates in different geographies and a variety of clinical samples.</p>
<p>The <em>sasX</em> gene was thought to facilitate the nasal colonization of MRSA (10). In our study, we found <em>sasX</em> carriage of CoNS at a rate of 7.69% of the isolates (<em>n</em>=26) from nasal vestibules of healthy volunteer hospital workers. It is well known that healthcare professionals have more frequent nasal colonization of <em>S. aureus</em> and other resistant CoNS, and because the <em>sasX</em> gene may facilitate the nasal colonization, the possible role of this gene in healthcare-related settings for nasal colonization should be investigated.</p>
<p>When our findings are considered in respect of invasive and noninvasive isolates, the carriage of <em>sasX</em> is found in 12.36% of isolates from peripheral blood cultures, and 11.43%, 7.69%, and 0% of isolates from central venous catheter (catheter colonization), nasal vestibule, and hands, respectively. Our peripheral blood and nasal isolate results were in concordance with the results of Li et al. (10). The high <em>sasX</em> ratio in catheter colonizers seems to support the role of <em>sasX</em> in adherence to surfaces, as in nasal isolates. The small number of CoNS rather than <em>S. epidermidis</em>, <em>S. haemolyticus</em>, and <em>S. hominis</em> isolates and the absence of tracheal aspirate and abscess samples were the limitations of our study.</p>
<p>Also, Li et al. (21) stated that invasive infections caused by <em>sasX</em>-positive isolates were higher than <em>sasX</em>-negative isolates in accordance with our isolates from blood cultures. There was no <em>sasX</em> carriage in isolates from hands, and this finding also supports the results of Li et al. (10,20), which underlined the potential role of <em>sasX</em> in invasive rather than noninvasive isolates.</p>
<p>Sequence analysis of the <em>sasX</em> gene of the isolates showed 99% homology with related proteins of <em>Staphylococcus</em> phage SPbeta-like and <em>S. epidermidis</em> RP62A, especially for <em>S. haemolyticus</em> and <em>S. capitis</em> isolates. <em>S. hominis</em> and <em>S. epidermidis</em> isolates had 94% and 99% homology to related proteins of <em>Staphylococcus</em> phage SPbeta-like and <em>S. epidermidis</em> RP62A, respectively, indicating that the phages of CoNS rather than <em>S. epidermidis</em> isolates were also highly conserved.</p>
<p>In a study about MRSA ST239 isolates, Holden et al. (19) stated that the <em>sasX</em> gene is located at the 3&prime; end region of a ϕSP&beta;-like prophage, which had coding sequences for aminoglycoside resistance. In our study, no difference was found for gentamycin and other antimicrobial resistance between <em>sasX-</em>positive and <em>-</em>negative isolates in accordance with Li et al. (21).</p>
<p>In our study, type V SCC<em>mec</em> (34%) was the most prevalent type in all isolates regardless of species, except <em>S. hominis</em> (<a href="javascript:openWin('uploads/grafik/table_BMJ_2203_5.jpg', 640, 480)"><b>Table 6</b></a>). The SCC<em>mec</em> types differ among CoNS species and between studies. McManus et al. (24) reported that type IVc is the most common type in methicillin-resistant <em>S. epidermidis</em> isolated from cerebrospinal fluid and external ventricular drains in device-associated meningitis patients. Garza-Gonz&aacute;lez et al. (25) found that types III and IVa were the predominant types in <em>S. epidermidis</em> isolated from blood samples, whereas Saffari et al. (26) and Chen et al. (27) found type IV and type III/SCCHg, respectively. Pinheiro et al. (28) characterized <em>S. epidermidis</em> (n<em>=</em>79) blood culture isolates and found SCC<em>mec</em> types III (53.2%) and II (29.1%) as the most prevalent types. The diversity of the types found in these studies may depend on the geographical differences, and the clinical samples from which the isolates were obtained, the epidemiologic similarity of the isolates, and the methodological limitations were used to detect SCC<em>mec </em>types in CoNS. In our study, no statistical difference was found between the presence of the <em>sasX</em> gene and the SCC<em>mec </em>type.</p>
<p>In our study, with respect to invasive isolates, SCC<em>mec</em> type II was more frequent than the other types when compared with noninvasive isolates, and it was found to be statistically important (p=0.014). Also, the type II ratio was found to be statistically important (p=0.029) between invasive and noninvasive <em>S. epidermidis</em> isolates. In some studies, it is seen that there is no significant difference in invasive and noninvasive isolates in terms of SCC<em>mec</em> type (9,29). It is thought that this difference may be due to the number of isolates and diversity of sample origin (cat, pig, human isolates, etc.) used in the studies.</p>
<p>Some CoNS isolates could not be typed in our study with the multiplex PCR method of Kondo et al. (17). As Garza-Gonz&aacute;lez et al. (30) also stated in their study, new methodologies are needed to be designed because of the inconsistent results for SCC<em>mec</em> typing of CoNS strains. The usage of two or more different multiplex PCR protocols/conditions (new primer designs for conserved gene sequences) directed to different targets of the SCC<em>mec</em> complex may be helpful to solve this issue.</p>
<p>In this study, we characterized our invasive and noninvasive CoNS isolates from different samples of human origin and for the presence of <em>sasX</em> gene and SCC<em>mec</em> types. For <em>sasX</em>, we found 9.44% positivity. Remarkably, the <em>sasX</em> gene was found to be negative from strains isolated from healthy volunteers&rsquo; hands, which indicate the possible role of this gene in invasive isolates. No statistical difference was found between <em>sasX</em>-positive and -negative isolates in respect to antibacterial resistance and between<em> sasX</em> presence and SCC<em>mec </em>types. SCC<em>mec</em> type V is the most prevalent type of <em>S. haemolyticus (</em>47.7%) and <em>S. epidermidis </em>(35%) isolates. As far as we know, this is the first data about the distribution SCC<em>mec</em> types and <em>sasX</em> carriage of different CoNS from our country.</p>
<p>As a result, more large scale and comprehensive studies are needed to define the SCC<em>mec</em> type distribution and to find out the molecular epidemiological characteristics of <em>sasX</em> and its possible role at colonization and infection in particular species. Also, more clinical samples of CoNS are needed.</p>
<p><strong>Conflict of Interest:</strong> No conflict of interest was declared by the authors.</p>
<p><strong>Financial Disclosure:</strong> The study was carried out using consumables in our laboratory.</p>												        					</div>
												        					<!--
												        					<div	style="width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;">
												        									<a class="menu" href="#summary_en">Summary</a><a class="menu" href="#introduction">Introduction</a><a class="menu" href="#methods">Methods</a><a class="menu" href="#results">Results</a><a class="selected_menu" href="#discussion">Discussion</a><a class="menu" href="#reference">Reference</a>												        					</div>
												        					-->
												        				</div>
												        				
												        				
												        																        				
												        				<div  style="width:100%; float:left;  padding-bottom:0px;">
												        					<div	style="width:100%; float:left; min-height:45px; text-align:justify">
												        						<a href="" name="reference"></a>
												        						<!--<h2>Reference</h2>-->
												        						<h3>REFERENCES</h3>
<ol>
<li>Becker K, Heilmann C, Peters G. Coagulase-negative staphylococci. Clin Microbiol Rev 2014;27:870-926.</li>
<li>Schoenfelder SM, Lange C, Eckart M, Hennig S, Kozytska S, Ziebuhra W. Success through diversity-how Staphylococcus epidermidis establishes as a nosocomial pathogen. Int J Med Microbiol 2010;300:380-6.</li>
<li>Widerstr&ouml;m M, Wistr&ouml;m J, Sj&ouml;stedt A, Monsen T. Coagulase- negative staphylococci: update on the molecular epidemiology and clinical presentation, with a focus on Staphylococcus epidermidis and Staphylococcus saphrophyticus. Eur J Clin Microbiol Infect Dis 2012;31:7-20.</li>
<li>Widerstr&ouml;m M, Monsen T, Karlsson C, Edebro H, Johansson A. Wistr&ouml;m J. Clonality among multidrug-resistant hospital associated Staphylococcus epidermidis in northern Europe. Scan J Infect Dis 2009;41:642-9.</li>
<li>Baig, S, Johannesen TB, Overballe-Petersen S, Larsen J, Larsen AR, Stegger M. Novel SCCmec type XIII (9A) identified in an ST152 methicillin-resistant Staphylococcus aureus. Infect Genet Evol 2018;61:74-6.</li>
<li>Barbier F, Rupp&eacute; E, Hernandez D, Lebeaux D, Francois P, Felix B, et al. Methicillin-resistant coagulase-negative staphylococci in the community: High homology of SCCmec IVa between Staphylococcus epidermidis and major clones of methicillin-resistant Staphylococcus aureus. J Infect Dis 2010;15:270-81.</li>
<li>Hanssen AM, Sollid JU. Multiple staphylococcal cassette chromosomes and allelic variants of cassette chromosome recombinases in Staphylococcus aureus and coagulase-negative staphylococci from Norway. Antimicrob Agents Chemother 2007;51:1671-7.</li>
<li>Ibrahem S, Salmenlinna S, Virolainen A, Kerttula AM, Lyytik&auml;inen O, J&auml;gerroos H, et al. Carriage of methicillin-resistant Staphylococci and their SCCmec types in a long-term-care facility. J Clin Microbiol 2009;47:32-7.</li>
<li>Rupp&eacute; E, Barbier F, Mesli Y, Maiga A, Cojocaru R, Benkhalfat M, et al. Diversity of staphylococcal cassette chromosome mec structures in methicillin-resistant Staphylococcus epidermidis and Staphylococcus haemolyticus strains among outpatients from four countries. Antimicrob Agents Chemother 2009;53:442-9.</li>
<li>Li M, Du X, Villaruz AE, Diep BA, Wang D, Song Y, et al. MRSA epidemic linked to a quickly spreading colonization and virulence determinant. Nat Med 2012;18:816-9.</li>
<li>S&ouml;derquist B, Anderson M, Nilsson M, Nilsdotter-Augutinsson A, Persson l, Friberg &Ouml;, et al. Staphylococcus epidermidis surface protein I (SesI): a marker of the invasive capacity of S.epidermidis? J Med Microbiol 2009;58:1395-7.</li>
<li>Bowden MG, Chen W, Singvall J, Xu Y, Peacock SJ, Valtulina V, et al. Identification and preliminary characterization of cell-wall-anchored proteins of Staphylococcus epidermidis. Microbiology 2005;151:1453-64.</li>
<li>Otto M. Coagulase-negative staphylococci as reservoirs of genes facilitating MRSA infection: Staphylococcal commensal species such as Staphylococcus epidermidis are being recognized as important sources of genes promoting MRSA colonization and virulence. Bioassays 2013;35:4-11.</li>
<li>Maki DG, Weise CE, Sarafin HW. A semiquantitative culture method for identifying intravenous-catheter-related infection. N Eng J Med 1977;296:1305-9.</li>
<li>Clinical and Laboratory Standards Institute. Performance standards for antimicrobial suscepti&not;bility testing, Twenty-fourth Informational Supplement, CLSI Document M100-S24, CLSI, Wayne PA, 2014.</li>
<li>Geha DJ, Uhl JR, Gustaferro CA, Persing DH. Multiplex PCR for identification of methicillin-resistant staphylococci in the clinical laboratory. J Clin Microbiol 1994;32:1768-72.</li>
<li>Kondo Y, Ito T, Ma XX, Watanabe S, Kreiswirth BN, Etienne J, et al. Combination of multiplex PCRs for staphylococcal cassette chromosome mec type assignment: Rapid identification system for mec, ccr, and major differences in junkyard regions. Antimicrob Agents Chemother 2007;51:264-74.</li>
<li>Altschul SF, Gish W, Miller W, Myers EW, Lipman DJ. Basic local alignment search tool. J Mol Microbiol 1990;215:403-10.</li>
<li>Holden MT, Lindsay JA, Corton C, Quail MA, Cockfield JD, Pathak S, et al. Genome Sequence of a Recently Emerged, Highly Transmissible, Multi-Antibiotic- and Antiseptic-Resistant Variant of Methicillin-Resistant Staphylococcus aureus, Sequence Type 239 (TW). J Bacteriol 2010;192:888-92.</li>
<li>De Backer S, Xavier BB, Vanjari L, Coppens J, Lammens C, Vemu L, et al. Remarkable geographical variations between India and Europe in carriage of the staphylococcal surface protein-encoding sasX/sesI and in the population structure of methicillin-resistant Staphylococcus aureus belonging to clonal complex 8. Clin Microbiol Infect 2019;25:628.</li>
<li>Li S, Sun J, Zhang J, Li X, Tao X, Wang L, et al. Comparative analysis of the virulence characteristics of epidemic methicillin-resistant Staphylococcus aureus (MRSA) strains isolated from Chinese children: ST59 MRSA highly expresses core gene-encoded toxin. APMIS 2014;122:101-4.</li>
<li>Monecke S, M&uuml;ller E, Dorneanu OS, Vremeră T, Ehricht R. Molecular Typing of MRSA and of Clinical Staphylococcus aureus Isolates from Iaşi, Romania. PLOS One 2014;9:97833.</li>
<li>Soumya KR, Mathew S, Sugathan S, Mathew J, Radhakrıshnan EK. Studies on prevalence of biofilm associated genes and primary observation on sasX gene in clinical isolates of coagulase negative staphylococci (CoNS). APMIS 2016;124:319-26.</li>
<li>McManus BA, Coleman DC, Deasy EC, Brennan GI, O'Connell B, Monecke S, et al. Comparative Genotypes, Staphylococcal Cassette Chromosome mec (SCCmec) Genes and Antimicrobial Resistance amongst Staphylococcus epidermidis and Staphylococcus haemolyticus Isolates from Infections in Humans and Companion Animals. PLoS One 2015;10:e0138079.</li>
<li>Garza-Gonz&aacute;lez E, Morf&iacute;n-Otero R, Llaca-D&iacute;az JM, Rodriguez-Noriega E. Staphylococcal cassette chromosome mec (SCC mec) in methicillin-resistant coagulase-negative staphylococci. A review and the experience in a tertiary-care setting. Epidemiol Infect 2010;138:645-54.</li>
<li>Saffari F, Widerstr&ouml;m M, Gurram BK, Edebro H, Hojabri Z, Monsen T. Molecular and Phenotypic Characterization of Multidrug-Resistant Clones of Staphylococcus epidermidis in Iranian Hospitals: Clonal Relatedness to Healthcare-Associated Methicillin-Resistant Isolates in Northern Europe. Microb Drug Resist 2016;22:570-7.</li>
<li>Chen HJ, Lin YT, Hung WC, Tsai JC, Hsueh PR, Teng LJ. Distribution of Staphylococcal Cassette Chromosome (SCC) mec Element Types in Fusidic Acid-Resistant Staphylococcus epidermidis and Identification of a Novel SCC7684 Element. Antimicrob Agents Chemother 2016;60:5006-9.</li>
<li>Pinheiro L, Brito CI, Pereira VC, Oliveira A, Bartolomeu AR, Camargo CH, et al. Susceptibility Profile of Staphylococcus epidermidis and Staphylococcus haemolyticus Isolated from Blood Cultures to Vancomycin and Novel Antimicrobial Drugs over a Period of 12 Years. Microb Drug Resist 2016;22:283-93.</li>
<li>Iravani Mohammad Abadi M, Moniri R, Khorshidi A, Piroozmand A, Mousavi SG, Dastehgoli K, et al. Molecular Characteristics of Nasal Carriage Methicillin-Resistant Coagulase Negative Staphylococci in School Students. Jundishapur J Microbiol 2015;8:e18591.</li>
<li>Garza-Gonz&aacute;lez E, Morfin-Otero R, Mart&iacute;nez-V&aacute;zquez MA, Gonzalez-Diaz E, Gonz&aacute;lez-Santiago O, Rodr&iacute;guez-Noriega E. Microbiological and molecular characterization of human clinical isolates of Staphylococcus cohnii, Staphylococcus hominis, and Staphylococcus sciuri. Scand J Infect Dis 2011;43:930-6.</li>
</ol>												        					</div>
												        					<!--
												        					<div	style="width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;">
												        									<a class="menu" href="#summary_en">Summary</a><a class="menu" href="#introduction">Introduction</a><a class="menu" href="#methods">Methods</a><a class="menu" href="#results">Results</a><a class="menu" href="#discussion">Discussion</a><a class="selected_menu" href="#reference">Reference</a>												        					</div>
												        					-->
												        				</div>
												        				
												        				
												        															        			
		
											        	</div>
			<