<div class="col-lg-12 col-md-12 col-sm-12 col-xs-12">
																		        				
												        				<div  style="width:100%; float:left;  padding-bottom:0px;">
												        					<div	style="width:100%; float:left; min-height:45px; text-align:justify">
												        						<a href="" name="summary_en"></a>
												        						<!--<h2>Summary</h2>-->
												        						<h4>Abstract</h4>
<p><strong>Background:</strong> Widespread and effective use of molecular diagnostic tests is indispensable for protecting public health and containing the severe respiratory syndrome coronavirus 2 (SARS-CoV-2) pandemic. More than 1 year into the pandemic, as resources have reached a point of depletion, grouping samples in pools of certain sizes appears to be a reasonable method to reduce both the costs and the processing time without necessitating additional training, equipment, or materials.<br /><strong>Aims:</strong> To assess whether the pooling strategy that was used in past outbreaks and is used in blood tests prior to transfusion for screening large populations can also be used in SARS CoV-2 tests.<br /><strong>Study Design:</strong> Diagnostic accuracy study.<br /><strong>Methods:</strong> This prospective study was conducted with 2815 samples, sent to the coronavirus disease 2019 (COVID-19) Laboratory of our hospital between February 12 and 21, 2021, to be tested for the presence of SARS-CoV-2. The samples were examined individually and in pools of five 100 &micro;l taken from each sequential sample, using 3 different SARS-CoV-2 reverse transcription-polymerase chain reaction (RT-PCR) kits, the Allplex&trade; 2019-nCoV Assay kit (Seegene, Republic of Korea), the GeneMAP&trade; 2019-nCoV detection V.3 kit (GenMark, T&uuml;rkiye), and the Bio-Speedy&trade; SARS-CoV-2 Double Gene&trade; RT-qPCR kit (Bioeksen, T&uuml;rkiye) on the BioRAD CFX96&trade; Touch (Bio-Rad Laboratories Inc., Hercules, CA, USA) platform available in our laboratory.<br /><strong>Results:</strong> Following the extraction of serial dilutions prepared from the SARS-CoV-2 RNA positive (cycle of threshold: 20) sample, the standard curves of RT-PCR were analyzed. By evaluating the efficiency (E) values, all 3 kits showed high sensitivity and similar results; while the highest level was detected with the Allplex&trade; 2019-nCoV Assay kit in the nucleocapsid (N) gene (E: 124%), the lowest was detected with the Double Gene&trade; RT-qPCR kit in the N and ORF 1ab genes (E: 90%). Of the samples included in the study, only 1 positive sample with low viral load was found to be negative when studied by pooling. The total number of kits to be used in pooled tests and then to individually retest the 5 samples in positive pools was calculated as 827 and the savings rate as 69.91% (1968/2815).<br /><strong>Conclusion:</strong> The pooling strategy is an effective approach to extend the impact of limited testing resources and reagents available in certain periods of the COVID-19 pandemic. Testing by pooling samples requires improvement of RNA extraction methods and careful monitoring of RT-PCR test sensitivity to avoid missing low-positive entities. Therefore, based on the prevalence of COVID-19 in their regions, laboratories should conduct their own validation of pooling studies for RNA extraction and amplification methods they use.</p>												        					</div>
												        					<!--
												        					<div	style="width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;">
												        									<a class="selected_menu" href="#summary_en">Summary</a><a class="menu" href="#introduction">Introduction</a><a class="menu" href="#methods">Methods</a><a class="menu" href="#results">Results</a><a class="menu" href="#discussion">Discussion</a><a class="menu" href="#reference">Reference</a>												        					</div>
												        					-->
												        				</div>
												        				
												        				
												        																        				
												        				<div  style="width:100%; float:left;  padding-bottom:0px;">
												        					<div	style="width:100%; float:left; min-height:45px; text-align:justify">
												        						<a href="" name="introduction"></a>
												        						<!--<h2>Introduction</h2>-->
												        						<hr size="1" color="#f2f2f2">
<h3>INTRODUCTION</h3>
<p>Since its first detection in Wuhan, China in December 2019 the severe respiratory syndrome coronavirus 2 (SARS-CoV-2), which is the causative agent of the coronavirus disease 2019 (COVID-19), spread across the continents by human-to-human transmission and was defined as a pandemic.<sup>1</sup> Given that COVID-19 cases can be contagious 3 days before the onset of symptoms and a considerable portion of patients remain asymptomatic or minimally symptomatic throughout the course of the disease is responsible for the spread of the virus, the World Health Organization recommends increasing the number of tests as much as possible as the most effective way to limit the epidemic.<sup>2</sup> Although, intensive efforts aimed at developing nucleic acid-based or viral antigen detection-based formats to enable faster results and ease of application in the field have begun to yield positive results, these new methods do not allow for testing large numbers of samples in a short period of time.<sup>3,4</sup> Another approach is the pooling technique, where the samples are grouped to reduce the number of tests required. As the pooling strategy reduces both the costs and the processing time without necessitating additional training, equipment, or material, this method is more attractive. In this way, both laboratory facilities and human resources can be used more efficiently.<sup>5,6</sup></p>
<p>The real-time reverse transcription polymerase chain reaction (RT-PCR) tests developed for detecting SARS-CoV-2 are widely used during the pandemic by laboratories authorized by the Ministry of Health. In these tests, the samples collected by nasopharyngeal and oropharyngeal swabs undergo reverse transcription and amplification with real-time RT-PCR after sample inactivation and lysis. The sample pooling technique, on the other hand, has already been used in past outbreaks and is being used prior to blood product transfusion for detecting human immunodeficiency virus, hepatitis B and C viruses.<sup>7,8</sup> This technique, which is implemented according to the pooling guidelines of the The European Center for Disease Prevention and Control, was first suggested by Dorfman in 1943 and improved over the years.<sup>9</sup> Here, test samples are combined in equal volumes in a single pool, and if the pooled test result is negative, then all the samples can be presumed negative. If the pooled test result is positive, then each of the samples in the pool are tested individually. Successful implementation of the pooling technique essentially requires knowing the detection limit of the test, its sensitivity and specificity, and the prevalence of the disease in the population. Many studies that have focused on determining pool sizes concluded that an effective pooling study could be carried out without affecting test specificity and sensitivity with a pool size of up to 8 samples.<sup>10,11</sup> Recent guidelines recommend leaving the decision of pool size to the laboratories given the sensitivity and specificity of different extraction and PCR methods, as well as the disease prevalence rate that can vary among populations.<sup>12</sup> Furthermore, since SARS-CoV-2 is a newly identified pathogen, it is still unclear how much dilution would affect the detection of the RNA-containing sample, hence leading to false negative results. Therefore, modifications and extensive validation studies are needed before the worldwide adaptation of the test.<sup>13,14</sup></p>
<p>While the pooling technique has generally been used to screen large asymptomatic populations, the data obtained with this technique provides a reliable resource that can then be used to estimate the actual prevalence of the virus in the population. Determining the prevalence of SARS-CoV-2 is crucial for an accurate estimation of mortality. Considering that 85% of cases are asymptomatic, the mortality rate calculated only from the symptomatic cases will be higher than the actual rate. This can lead to an unnecessary panic in societies. In fact, it was recently found that previous serological studies estimating the prevalence of the SARS-CoV-2 virus have calculated the mortality rate 5 times higher than the actual.<sup>15</sup> In this respect, the pooling technique will help to obtain more accurate prevalence data.</p>
<p>In this study, we aimed to assess whether a strategy, which was used in past outbreaks and is used in blood tests prior to transfusion for screening large populations, can also be used in SARS CoV-2 tests with a pool size that enables the best use of resources while maintaining the reliability of the tests.</p>												        					</div>
												        					<!--
												        					<div	style="width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;">
												        									<a class="menu" href="#summary_en">Summary</a><a class="selected_menu" href="#introduction">Introduction</a><a class="menu" href="#methods">Methods</a><a class="menu" href="#results">Results</a><a class="menu" href="#discussion">Discussion</a><a class="menu" href="#reference">Reference</a>												        					</div>
												        					-->
												        				</div>
												        				
												        				
												        																        				
												        				<div  style="width:100%; float:left;  padding-bottom:0px;">
												        					<div	style="width:100%; float:left; min-height:45px; text-align:justify">
												        						<a href="" name="methods"></a>
												        						<!--<h2>Methods</h2>-->
												        						<h3>MATERIAL and METHODS</h3>
<p>This study was performed in line with the principles of the Declaration of Helsinki. Approval was granted by the Ethics in Non-&shy;Pharmaceutical Clinical Research Ethics Committee of İstanbul University-Cerrahpaşa (date: August 4, 2021 and decision no: A-23).</p>
<h4>Sample Collection</h4>
<p>This prospective study was conducted with 2815 samples, sent to the COVID-19 Laboratory of the İstanbul University-Cerrahpaşa, Cerrahpaşa School of Medicine, between February 12 and February 21, 2021, to be tested for the presence of SARS-CoV-2. Naso-oropharyngeal swab samples taken and placed in 2 ml vNAT (viral nucleic acid buffer) transfer tubes by our hospital&rsquo;s healthcare staff were delivered under suitable conditions to the laboratory where routine SARS-CoV-2 RT-PCR tests were done, and the remaining parts were portioned into 1.5 ml RNAse-free microcentrifuge tubes. The samples were renumbered and examined anonymously on the same day.</p>
<h4>Nucleic Acid Isolation and Amplification Validation Tests</h4>
<p>To quantitatively determine the limit of detection (LoD) of the RT-PCR kits, the lowest detectable copy number was determined using 10-fold serial dilutions prepared from the extraction of virus culture supernatant obtained from an external cell culture laboratory with the virus concentration predetermined sample (107 copy/ml) by nano-drop spectrophotometer. In addition, to determine the lowest concentration level at which the 3 different RT-PCR kits to be used would be able to detect SARS-CoV-2, serial dilutions prepared from a SARS-CoV-2 RNA positive (cycle of threshold (Ct): 20) sample were extracted and the RT-PCR test was done.</p>
<p>The slope (the standard curve-Ct against log/DNA copy), is a measure of efficiency (E) and depends on the dilution factors. Using the same standards, if operated many times, the slope should stay the same. The intercept (y-int) is the value of Y when X equal zero and is a measure of the sensitivity of the assay. As long as threshold is used with the same value, during the whole assay, a longer PCR product gives more fluorescence per copy and we get higher Ct values and therefore higher intercepts. Therefore, keeping the number of cycles longer than the intercept value on the y-axis when the x-axis is zero will not yield more PCR products, and the fluorescence obtained above this value will be false positive. Kits were compared for the efficiency and sensitivity using standard curves drawn individually for each of the E, nucleocapsid (N), RdRP, and ORF 1ab gene regions targeted by all 3 kits.</p>
<h4>Determining the Most Effective Pool Size for Detecting SARS-CoV-2 RNA</h4>
<p>The optimal pool size was computed using the https​://ww​w.chr​isbil​der.c​om/sh​iny platform which uses mathematical modeling based on the parametric values of the prevalence of the disease in the region, the sensitivity and specificity of the test used, and the number of pooling stages.12 While we were using Chris Bilder&rsquo;s calculator, we compared the prevalence of SARS-CoV-2 in Turkish population and SARS-CoV-2 positivite detection rate in our own hospital in the period of study, and used whichever was higher in the formula, in order to keep the pooling size at a safe value, since the prevalence rate in the population declared by the government is not always up-to-date.</p>
<h4>Nucleic Acid Isolation and Amplification Tests</h4>
<p>The study was planned to be carried out over 10 days. The nucleic acid isolations of the samples alone and as sequential pools of 5, created by taking 100 &micro;l from each sample in vNAT transfer tubes were done with the SEEPREP32&trade; automated extraction system (Seegene, Republic of Korea), according to the manufacturer&rsquo;s instructions. While the volume put into the system was 200 &micro;l, the final elution volume was 60 &micro;l. All samples were run on the BioRAD CFX96&trade; Touch (Bio-Rad Laboratories Inc., USA) platform available in our laboratory, separately with all 3 SARS-CoV-2 RT-PCR kits, the Allplex&trade; 2019-nCoV Assay kit (LOD: 4.167 copy/ml, sensitivity: 99.97%) (Seegene, Republic of Korea), the GeneMAP&trade; 2019-nCoV detection V.3 kit (LOD: 100 copy/ml, sensitivity: 99.81%) (GenMark, T&uuml;rkiye), and the Bio-Speedy&trade; SARS-CoV-2 Double Gene&trade; RT-qPCR kit (LOD: 1000 copy/ml, sensitivity: 99.53%) (Bioeksen, T&uuml;rkiye) alone and as sequential pools of 5. The Allplex&trade; 2019-nCoV kit is multiplex real-time PCR assay for simultaneous detection of 3 target genes of SARS-CoV-2. The assay is designed to detect RdRP (in the Cal Red 610 channel) and N (in the Quasar 670 channel) genes specific for SARS-CoV-2, and E gene (in the FAM channel) for all of sarbecovirus including SARS-CoV-2. The GeneMAP&trade; 2019-nCoV detection V.3 kit target gene regions are RdRp and N in 2 dye channels of FAM and VIC, respectively. The Bio-Speedy&trade; SARS-CoV-2 Double Gene&trade; RT-qPCR kit is an 1-step real-time RT-PCR assay for the qualitative detection of SARS-CoV-2 specific ORF1ab gene and N gene in a single FAM channel. The comparison of sensitivity, specificity, positive predictive value, negative predictive value, and accuracy values of the 3 kits used in the study is summarized in <a href="javascript:openWin('uploads/grafik/table_BMJ_2346_0.jpg', 640, 480)"><b>Table 1</b></a>.</p>
<p>All detected Ct values were recorded. Correlations were evaluated by comparing the Ct values obtained separately for each gene region. While in groups containing positive samples with high-Ct (low viral load) levels, the lowest viral copy detection limit of the pooling technique for the diagnosis of COVID-19 and the possible risks for false negative results were evaluated. The number of test kits used for pooled tests was compared with the number of those that would have been used if tests were done individually to find the reduction in test kit consumption.</p>
<h4>Statistical Analysis</h4>
<p>Statistical analysis was done using the Statistical Package for Social Sciences (SPSS) v22.0. Results of the parametric data were expressed as mean and standard deviation, and results of non-parametric data were expressed as median and minimum-maximum (median, min-max). The confidence interval was 95% and <em>P</em> &lt; 0.05 was considered as significant.</p>												        					</div>
												        					<!--
												        					<div	style="width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;">
												        									<a class="menu" href="#summary_en">Summary</a><a class="menu" href="#introduction">Introduction</a><a class="selected_menu" href="#methods">Methods</a><a class="menu" href="#results">Results</a><a class="menu" href="#discussion">Discussion</a><a class="menu" href="#reference">Reference</a>												        					</div>
												        					-->
												        				</div>
												        				
												        				
												        																        				
												        				<div  style="width:100%; float:left;  padding-bottom:0px;">
												        					<div	style="width:100%; float:left; min-height:45px; text-align:justify">
												        						<a href="" name="results"></a>
												        						<!--<h2>Results</h2>-->
												        						<h3>RESULTS</h3>
<p>LoDs calculated with the SARS-CoV-2 RT-PCR kits using the lowest detectable 10-fold serial dilutions prepared from extract of virus culture supernatant were &lt;10 copies/ml for Allplex&trade; and GeneMAP&trade; and &lt;100 copies/ml for Double Gene&trade;. The standard curves of RT-PCR were analyzed after the extraction of serial dilutions prepared from the SARS-CoV-2 RNA positive (Ct: 20) sample. Based on the intercept values, we interpreted that false positive could be detected with the Allplex&trade; 2019-nCoV Assay kit in the FAM channel (E gene) after 41.70 Ct, in the Quasar 670 channel (N gene ) after 39.64 Ct, in the Cal Red 610 channel (RdRp gene) after 44.34 Ct; with the GeneMAP&trade; 2019-nCoV detection V.3 kit in the FAM channel (RdRp gene) after 43.32 Ct, in the Cy5 channel (N gene) after 43.32 Ct; and with the SARS-CoV-2 Double Gene&trade; RT-qPCR kit in the FAM channel (N and ORF 1ab genes) after 45.21 Ct; and therefore, the cycles should not exceed the indicated values. By evaluating the E values, all 3 kits showed high sensitivity (over 90%) and similar results. While the highest level was detected with the Allplex&trade; 2019-nCoV Assay kit in the N gene (E: 124%), the lowest was detected with the Double Gene&trade; RT-qPCR kit in the N and ORF 1ab genes (E: 90%) (<a href="javascript:openWin('uploads/grafik/figure_BMJ_2346_0.jpg', 640, 480)"><b>Figure 1</b></a>).</p>
<p>However, an analytical sensitivity difference of 3-6 Ct was noticed between the RT-PCR kits used, which increased especially as the Ct value was rising (<a href="javascript:openWin('uploads/grafik/figure_BMJ_2346_1.jpg', 640, 480)"><b>Figure 2</b></a>).</p>
<p>Failure in RNA extraction can create a significant limitation as positive samples will not be detected. To prevent this, internal control (human transcript) was additionally performed on the samples before extraction. Human gene amplification was present in all of the samples included in the study, and no false negatives were detected that could potentially be associated with extraction errors.</p>
<p>SARS-CoV-2 RNA positivity rate was 2.3% (66/2815) in the samples included in the study. Of these, 22 were weak positive (Ct&ge;30), 26 were moderate positive (30&gt;Ct&ge;25), and 18 were strong positive (25&gt;Ct). Of the positive pools, 46 had 1 positive sample and 10 had 2 positive samples (66 positives, totally). However, out of 567 pools of 5 samples, 512 were negative and 55 were positive, false negatives occurred in 1 pool, and 1 positive sample could not be detected (The potential risk of missing positive samples was calculated as 0.33%). The mean Ct values of the positive samples and pools are summarized in <a href="javascript:openWin('uploads/grafik/table_BMJ_2346_1.jpg', 640, 480)"><b>Table 2</b></a>.</p>
<p>When the difference between the Ct value of the positive samples and the Ct values of the pools containing these samples was calculated, a median decrease of 4.31 (3-5) Ct was found. Of the samples included in the study, only 1 positive sample with low-viral load (Allplex&trade; E gene: 36.15 Ct, N gene: 36.16 Ct, RdRp gene: 39.61 Ct, GeneMAP&trade; RdRp gene: 38.25 Ct, N gene: 36.62 Ct, and SARS-CoV-2 Double Gene&trade; RT-qPCR kit N and ORF 1ab genes: 38.55 Ct) was found to be negative. The number of kits to be used when samples are tested in pools of 5 and then positive pools are retested was calculated as 827 and the savings rate as 69.91% (1968/2815) (<a href="javascript:openWin('uploads/grafik/figure_BMJ_2346_2.jpg', 640, 480)"><b>Figure 3</b></a>).</p>												        					</div>
												        					<!--
												        					<div	style="width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;">
												        									<a class="menu" href="#summary_en">Summary</a><a class="menu" href="#introduction">Introduction</a><a class="menu" href="#methods">Methods</a><a class="selected_menu" href="#results">Results</a><a class="menu" href="#discussion">Discussion</a><a class="menu" href="#reference">Reference</a>												        					</div>
												        					-->
												        				</div>
												        				
												        				
												        																        				
												        				<div  style="width:100%; float:left;  padding-bottom:0px;">
												        					<div	style="width:100%; float:left; min-height:45px; text-align:justify">
												        						<a href="" name="discussion"></a>
												        						<!--<h2>Discussion</h2>-->
												        						<h3>DISCUSSION</h3>
<p>Grouping samples in pools of a certain size has been successfully applied for many years in blood banking and screening of infectious diseases. Today, this strategy is becoming more widespread with the development of highly sensitive molecular-based tests; and mathematical models have been designed to determine the appropriate pool sizes for use in cases of new diseases and outbreaks. In this study, we examined the possibility of a pooling strategy that would allow for more efficient use of the testing resources, which have reached a point of depletion more than 1 year into the COVID-19 pandemic, by comparing 3 RT-PCR tests that target the E, N, RdRp, and ORF 1ab gene regions in a large sample size.</p>
<p>According to a study, higher sample pool sizes provide more gain in conditions when the prevalence rate is 1% or lower, and pool sizes of 5 or more samples would be inefficient if the prevalence rate is 5% or higher as this would increase the risk of dilution and thereby the risk of false negative.<sup>16</sup> In the referred study, the authors conclude that a pool size of 5 samples would be suitable when prevalence rate is lower than 5%, but the sample pooling method should not be used when the rate exceeds 20%. While there are no epidemiological study data available on the prevalence of COVID-19 in Turkey, the positivity rate in the population tested in our hospital in the study period remained below 5%. Our study showed that the pool size of 5 samples that we calculated using the mathematical modeling tool at https​://ww​w.chr​isbil​der.c​om/sh​iny maintained both testing sensitivity and specificity and helped save from reagent consumption as well as staff time, and thereby provided high efficiency.</p>
<p>The calculated increase in efficiency is only for RT-PCR and does not include the viral isolation and reporting processes. Since, the routine workflow for molecular diagnosis of SARS-CoV-2 in Turkey includes only the viral inactivation and lysis steps without a complete extraction, the process will inevitably take longer to complete when RNA extraction and pool creation are added to this typical algorithm. Pooling-dependent groups such as individuals from the same family or increasing the efficiency of the RNA isolation process with automated extraction devices or using primary kits that screen more gene regions, as we did in our study, can help to enhance testing sensitivity and thereby to increase efficiency and tackle the issue of inefficiency due to loss of time.<sup>5,17</sup> Because test results are automatically sent to the data network, the pooling scheme should be adapted to this system to obviate the need for additional time and work. Since this approach will allow to screen more people in shorter period of time by providing increased efficiency, it will enable isolation and prevent the spread of the disease within the group as soon as positivity is detected in closed communities such as nurseries or factories.</p>
<p>The major issue which is likely to be encountered in the pooling strategy is false positive and false negative. We had no false positives in the pools of our 2815 samples. False negatives, however, are more likely. Because amplification could not be obtained for 1 weak positive sample in the pool of 5 samples, the potential risk of missing positive samples in our pooled-testing protocol was calculated as 0.33%. This was deemed tolerable considering the potentially high rates of false negative results due to especially preanalytical errors (e.g., insufficient sampling, transport times, and impact of temperature). Large-scale implementation of the pooling strategy must be done carefully so that preanalytical processes do not lead to major signal losses.</p>
<p>Since, the prevalence of COVID-19 in the tested population is not always known, it may affect the decision of an optimum pool size. In such an event, data such as patient histories, serological data, and the number of symptomatic cases can be used to determine the pool size.<sup>18,19</sup> Once the pooling strategy is routinely implemented, it will also be possible to adapt the pool size according to the dynamic prevalence data.</p>
<p>Ultimately, in a rapidly changing pandemic, it is essential that testing strategies are adapted to the conditions brought about by the potential increases in positive test rates. Testing by pooling samples requires improvement of RNA extraction methods and careful monitoring of RT-PCR test sensitivity to avoid missing low-positive entities. Therefore, based on the prevalence of COVID-19 in their regions, laboratories should conduct their own validation pool studies for the RNA extraction and amplification kits they use. Our study demonstrated that the pooling strategy is an effective approach to extend the impact of limited testing resources and reagents available in certain periods of the COVID-19 pandemic.</p>
<p><strong>Ethics Committee Approval:</strong> Ethics Committee approval was received from the Non-Pharmaceutical Clinical Research Ethics Committee of İstanbul University-Cerrahpaşa, Cerrahpaşa School of Medicine (date: 04.08.2021 and decision no: A-23). This study was performed in line with the principles of the Declaration of Helsinki.</p>
<p><strong>Patient Consent for Publication:</strong> Informed consent was obtained from the patients.</p>
<p><strong>Data Sharing Statement:</strong> Data available on request from the authors. The data that &shy;support the findings of this study are available from the corresponding author.</p>
<p><strong>Author Contributions:</strong> Concept &ndash; K.M.; Design &ndash; K.M., M.A.K.; Supervision &ndash; K.M., M.A.K.; Resources &ndash; K.M., M.A.K.; Data Collection and/or Processing &ndash; Y.T.T., D.S., H.E., S.S.Y, O.K.N.; Analysis and/or Interpretation &ndash; Y.T.T., M.A.K.; Literature Review &ndash; R.K., İ.İ.B., B.M. G.A.; Writing &ndash; Y.T.T., K.M.; Critical Review &ndash; R.K., İ.İ.B., B.M, &Ouml;.F.T., G.A., K.M.</p>
<p><strong>Conflict of Interest:</strong> The authors have no conflict of interest to declare.</p>
<p><strong>Funding:</strong> This study was carried out with the contribution of the İstanbul University-Cerrahpaşa Scientific Research and Project Fund.</p>												        					</div>
												        					<!--
												        					<div	style="width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;">
												        									<a class="menu" href="#summary_en">Summary</a><a class="menu" href="#introduction">Introduction</a><a class="menu" href="#methods">Methods</a><a class="menu" href="#results">Results</a><a class="selected_menu" href="#discussion">Discussion</a><a class="menu" href="#reference">Reference</a>												        					</div>
												        					-->
												        				</div>
												        				
												        				
												        																        				
												        				<div  style="width:100%; float:left;  padding-bottom:0px;">
												        					<div	style="width:100%; float:left; min-height:45px; text-align:justify">
												        						<a href="" name="reference"></a>
												        						<!--<h2>Reference</h2>-->
												        						<h3>REFERENCES</h3>
<ol>
<li>Del Rio&nbsp;C, Malani&nbsp;PN. COVID-19: new insights on a rapidly changing epidemic. <em>JAMA.</em> 2020;323:1339-1340.</li>
<li>World Health Organization (WHO). Country &amp; Technical Guidance - Coronavirus Disease (COVID-19). 2020. Available at: https​://ww​w.who​.int/​emerg​encie​s/dis​eases​/nove​l-cor​onavi​rus-2​019/t​echni​cal-g​uidan​ce.</li>
<li>Fomsgaard&nbsp;AS, Rosenstierne&nbsp;MW. An alternative workflow for molecular detection of SARS-CoV-2 - escape from the NA extraction kit-shortage, Copenhagen, Denmark, March 2020. <em>Euro Surveill</em>. 2020;25(14).</li>
<li>KalikiriMKR, Hasan&nbsp;MR, Mirza&nbsp;F, Xaba&nbsp;T, Tang&nbsp;P, Lorenz&nbsp;S. High-throughput extraction of SARS-CoV-2 RNA from nasopharyngeal swabs using solid-phase reverse immobilization beads. <em>medRxiv</em>. 2020.</li>
<li>Aldridge&nbsp;M, Johnson&nbsp;O, Scarlett&nbsp;J. Group testing: an information theory perspective. <em>Found Trends Commun Inf Theory.</em> 2019;15:196-392.</li>
<li>Shani-Narkiss H, Gilday&nbsp;OD, Yayon&nbsp;N, Landau&nbsp;ID. Efficient and practical sample pooling high-throughput PCR diagnosis of COVID-19. medRxiv. 2020.</li>
<li>Van&nbsp;TT, Miller&nbsp;J, Warshauer&nbsp;DM,&nbsp;et&nbsp;al. Pooling nasopharyngeal/throat swab specimens to increase testing capacity for influenza viruses by PCR. <em>J Clin Microbiol.</em> 2012;50:891-896.</li>
<li>Dwyre&nbsp;DM, Fernando&nbsp;LP, Holland&nbsp;PV. Hepatitis B, hepatitis C and HIV transfusion-transmitted infections in the 21st century<em>. Vox Sang.</em> 2011;100:92-98.</li>
<li>Dorfman&nbsp;R. The detection of defective members of large populations. <em>Ann Math Statist.</em> 1943;14:436-440.</li>
<li>Gupta&nbsp;E, Padhi&nbsp;A, Khodare&nbsp;A,&nbsp;et&nbsp;al. Pooled RNA sample reverse transcriptase real time PCR assay for SARS CoV-2 infection: a reliable, faster and economical method. <em>PLoS One. </em>2020;15:e0236859.</li>
<li>Ministry of Health and Family Welfare, Government of India. Advisory on Feasibility of Using Pooled Samples for Molecular Testing of COVID-19. Available at: https://www.mohfw.gov.in/.</li>
<li>Abdalhamid&nbsp;B, Bilder&nbsp;CR, McCutchen&nbsp;EL, Hinrichs&nbsp;SH, Koepsell&nbsp;SA, Iwen&nbsp;PC. Assessment of specimen pooling to conserve SARS CoV-2 testing resources. <em>Am J Clin Pathol. </em>2020;153:715-718.</li>
<li>Centers for Disease Control and Prevention (U.S.). Research Use Only Real-Time RT-PCR Protocol for Identification of 2019-nCoV Available at: https​://ww​w.cdc​.gov/​coron​aviru​s/201​9-nco​v/abo​ut/te​sting​.html​.</li>
<li>Li&nbsp;C, Debruyne&nbsp;D, Spencer&nbsp;J,&nbsp;et&nbsp;al. High sensitivity detection of SARS-CoV-2 using multiplex PCR and a multiplex-PCR-based metagenomic method. <em>bioRxiv</em>. 2020.</li>
<li>Guha&nbsp;P, Guha&nbsp;A, Bandyopadhyay&nbsp;T. Application of pooled testing in screening and estimating the prevalence of Covid-19. <em>Health Serv Outcomes Res Method.</em> 2021.</li>
<li>Goyal&nbsp;S, Bist&nbsp;P, Sharma&nbsp;R. Optimal sample pooling: an efficient tool against SARS-CoV-2. medRxiv. 2020.</li>
<li>Ben-Ami&nbsp;R, Klochendler&nbsp;A, Seidel&nbsp;M,&nbsp;et&nbsp;al. Large-scale implementation of pooled RNA extraction and RT-PCR for SARS-CoV-2 detection. <em>Clin Microbiol Infect. </em>2020;26:1248-1253.</li>
<li>Randazzo&nbsp;W, Truchado&nbsp;P, Cuevas-Ferrando&nbsp;E, Sim&oacute;n&nbsp;P, Allende&nbsp;A, S&aacute;nchez&nbsp;G. SARS-CoV-2 RNA in wastewater anticipated COVID-19 occurrence in a low prevalence area. <em>Water Res.</em> 2020;181:115942.</li>
<li>Orive&nbsp;G, Lertxundi&nbsp;U, Barcelo&nbsp;D. Early SARS-CoV-2 outbreak detection by sewage-based epidemiology. <em>Sci Total Environ.</em> 2020;732:139298.</li>
</ol>												        					</div>
												        					<!--
												        					<div	style="width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;">
												        									<a class="menu" href="#summary_en">Summary</a><a class="menu" href="#introduction">Introduction</a><a class="menu" href="#methods">Methods</a><a class="menu" href="#results">Results</a><a class="menu" href="#discussion">Discussion</a><a class="selected_menu" href="#reference">Reference</a>												        					</div>
												        					-->
												        				</div>
												        				
												        				
												        															        			
		
											        	</div>
			</div>