window._articleFullText = window._articleFullText || {};
window._articleFullText[2387] = "<div class=\"col-lg-12 col-md-12 col-sm-12 col-xs-12\">\n\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t<div  style=\"width:100%; float:left;  padding-bottom:0px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:100%; float:left; min-height:45px; text-align:justify\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<a href=\"\" name=\"summary_en\"></a>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<!--<h2>Summary</h2>-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<h4>Abstract</h4>\r\n<p><strong>Background:</strong> Mitochondrial diseases are a clinically heterogeneous group of rare hereditary disorders that are defined by a genetic defect predominantly affecting mitochondrial oxidative phosphorylation. Mitochondrial diseases are caused by mutations of genes encoded by either nuclear DNA or mitochondrial DNA. Hundreds of different mitochondrial DNA point mutations and large-scale mitochondrial DNA rearrangements have been shown to cause mitochondrial diseases including Kearns&ndash;Sayre syndrome, Leber&rsquo;s hereditary optic neuropathy, Leigh syndrome, myoclonic epilepsy with ragged-red fibers, mitochondrial encephalopathy lactic acidosis stroke.<br /><strong>Aims:</strong> To investigate new variants that could be associated with mitochondrial diseases and to determine the effect of mitochondrial DNA mutations on the clinical spectrum.<br /><strong>Study Design:</strong> Cross-sectional study.<br /><strong>Methods:</strong> We screened whole mitochondrial DNA genome using next-generation sequencing in 16 patients who are considered to have mitochondrial disease. CentoGene and Mikrogen Genetic Diseases Diagnostic Center&rsquo;s database were used to investigate sequence variants. Detected variants were evaluated in bioinformatic databases to determine pathogenicity and were classified as class 1 (pathogenic), class 2 (likely pathogenic), and class 3 (variant of uncertain significance) according to CentoGene-ACMG database.<br /><strong>Results:</strong> As a result of the study, 2 patients were diagnosed with Leigh syndrome as previously reported class 1 mutations in <em>MT-ATP6</em> and <em>MT-ND5</em> genes. Four variants were identified for the first time in literature and 2 variants, previously reported but with uncertain pathogenic effect, are thought to be associated with mitochondrial disease.<br /><strong>Conclusion:</strong> Mitochondrial DNA screening should be among the primary clinical tests in patients with suspected mitochondrial disease to rule out DNA-associated mutations.</p>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<!--\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t<a class=\"selected_menu\" href=\"#summary_en\">Summary</a><a class=\"menu\" href=\"#introduction\">Introduction</a><a class=\"menu\" href=\"#methods\">Methods</a><a class=\"menu\" href=\"#results\">Results</a><a class=\"menu\" href=\"#discussion\">Discussion</a><a class=\"menu\" href=\"#reference\">Reference</a>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t<div  style=\"width:100%; float:left;  padding-bottom:0px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:100%; float:left; min-height:45px; text-align:justify\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<a href=\"\" name=\"introduction\"></a>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<!--<h2>Introduction</h2>-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<hr size=\"1\" color=\"#f2f2f2\">\r\n<h3>INTRODUCTION</h3>\r\n<p>Mitochondrial DNA (mtDNA) contains 37 genes which encode 13 proteins that form the oxidative phosphorylation (OXPHOS) complexes (I, III, IV, V), 22 transfer RNAs (tRNAs), and 2 ribosomal RNAs (rRNAs). Protein-coding genes in mitochondrial genome are <em>MT-ND</em>1, 2, 3, 4, 4L, 5, 6 in complex I, <em>MT-CYB</em> in complex III, <em>MT-CO</em>1, 2, 3 in complex IV, and <em>MT-ATP6</em> and <em>MT-ATP8</em> in complex V.<sup>1</sup> Apart from protein-coding genes, tRNA mutations are also responsible for mitochondrial disease (MD). Mitochondrial DNA mutations cause MD which are clinically heterogeneous group of rare hereditary disorders characterized by abnormal OXPHOS and result in mitochondrial dysfunction involving mitochondrial respiration chain.<sup>2</sup></p>\r\n<p>Mitochondrial diseases are caused by mutations of genes encoded by either nuclear DNA (nDNA) or mtDNA.<sup>3</sup> Mitochondrial diseases involve one or more organs usually with high energy demand such as the brain, heart, liver, and skeletal system and are often progressive with high morbidity and mortality.<sup>4</sup> Approximately 1 in 5000 people have a genetic MD. Mitochondrial DNA-related diseases can occur as sporadic or maternal transient. Hundreds of different mtDNA point mutations and large-scale mtDNA rearrangements have been shown to cause disease.<sup>5</sup> Kearns&ndash;Sayre syndrome, Leber&rsquo;s hereditary optic neuropathy (LHON), Leigh syndrome (LS), Myoclonic epilepsy with ragged-red fibers (MERRF), itochondrial encephalopathy lactic acidosis stroke (MELAS) are the early identified syndromes associated with mtDNA mutations.<sup>3</sup> In MD, clinical manifastation is so wide, even with same mutation differents type of MD can occur. Apart from protein-coding gene mutations, mutations in tRNA also take important place in MD such as MELAS and MERRF. In approximately 80% of MELAS patients, the causative mutation is the A3243G pathogenic variant in <em>MT-TL1</em> 1. Additionally, it is known that T8993G mutation in <em>MT-ATP6</em> gene was found be the most common mutation in terms of mtDNA linked MD.</p>\r\n<p>The diagnosis and management of these disorders is very difficult by an often tenuous relationship between the genotype and the observed clinical phenotype. Diagnosis often relies on genetic testing, in addition to histochemical and biochemical analysis of tissue biopsies. Advances in next-generation sequencing (NGS) techniques have substantially improved diagnosis by performing targeted exome and whole genome sequencing and have changed traditional approaches to diagnose MD. Next-generation sequencing allows to identify the mutant mtDNA rates in tissues and whole mtDNA mutations with cost-effective analyses.<sup>6,7</sup></p>\r\n<p>In this study, we analyzed whole mtDNA in patients with MD by using NGS to investigate the effect of mtDNA. We also identified all mtDNA variants to find potential mutations related to MD.</p>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<!--\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t<a class=\"menu\" href=\"#summary_en\">Summary</a><a class=\"selected_menu\" href=\"#introduction\">Introduction</a><a class=\"menu\" href=\"#methods\">Methods</a><a class=\"menu\" href=\"#results\">Results</a><a class=\"menu\" href=\"#discussion\">Discussion</a><a class=\"menu\" href=\"#reference\">Reference</a>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t<div  style=\"width:100%; float:left;  padding-bottom:0px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:100%; float:left; min-height:45px; text-align:justify\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<a href=\"\" name=\"methods\"></a>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<!--<h2>Methods</h2>-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<h3>MATERIAL and METHODS</h3>\r\n<p><strong>Patients</strong></p>\r\n<p>Sixteen patients whose ages ranged from 1 month to 6 years and who were pre-diagnosed with MD and their family members consented to inclusion were included in this study. Patients were chosen according to clinical and radiological findings and have MD. Median age of patients was 18.5 months (1 month to 6 years) (<a href=\"javascript:openWin('uploads/grafik/table_BMJ_2387_0.jpg', 640, 480)\"><b>Table 1</b></a>). Patients in the study did not have any other genetic diseases. Patients were categorized and chosen according to the algorithm for mitochondrial disorders (<a href=\"javascript:openWin('uploads/grafik/figure_BMJ_2387_0.jpg', 640, 480)\"><b>Figure 1</b></a>). We did not exclude any patient related by paternal kinship. However, families incompatible with maternal inheritance were excluded from the study.</p>\r\n<p><strong>DNA Extraction</strong></p>\r\n<p>Peripheral blood samples were collected from all 16 patients and their family members (total of 35 specimens). DNA extraction was performed according to the standard phenol-chloroform method.<sup>8</sup></p>\r\n<p><strong>Next-Generation Sequencing and Sanger Sequencing Analyses</strong></p>\r\n<p>The whole mtDNA was sequenced using Illumina-MiSeq Next Generation Sequencer in Mikrogen Genetic Diseases Diagnosis Center. QIAseq Targeted DNA Panel (96) Human Mitochondrial Panel (Qiagen, Hilden, Germany, Catalog No: DHS-105Z-96) was used, and all procedures were completed according to manufacturers&rsquo; recommendations. Prior to the NGS step, samples were diluted to a final volume of 10-40 ng/30 &mu;L. Results were analyzed using NextGENe Analyze Software (SoftGenetics, State College, USA). Detected variants with NGS were confirmed by Sanger sequencing (Beckman Coulter, California, USA) in patients and their family members (total 35 reactions).</p>\r\n<p>Whole exome sequencing was performed in 2 patients (P10, P14) and whole genome sequencing (WGS) was performed in 1 patient (P16) by an external center.</p>\r\n<p><strong>Analysis of Mitochondrial DNA Variants</strong></p>\r\n<p>Sequences were aligned to the revised Cambridge Reference Sequence (NM_012920.1). CentoGene and Mikrogen Genetic Diseases Diagnostic Center&rsquo;s database (with approximately 5000 individual mitochondrial genome database of Turkish population) were used to investigate sequence variants. Variants which have been reported before in the literature were taken into consideration, and only variants that were considered as very rare were selected for further analysis.</p>\r\n<p><strong>Bioinformatic Analysis</strong></p>\r\n<p>Detected variants were evaluated in bioinformatic databases to determine pathogenicity (Polyphen-2, SIFT, Varsome, MITOMAP, MITOTIP) and were classified as class 1 (pathogenic), class 2 (likely pathogenic), and class 3 (variant of uncertain significance) according to CentoGene-ACMG database. Transfer RNA variants were evaluated using mamit-tRNA,<sup>9</sup> Yarham&nbsp;et&nbsp;al.&rsquo;s<sup>10</sup> heat map, Yarham&nbsp;et&nbsp;al.&rsquo;s<sup>11</sup> pathogenicity table, and RNA secondary structure prediction programme in RNA Fold web server (http:​//rna​.tbi.​univi​e.ac.​at/cg​i-bin​/RNAW​ebSui​te/RN​Afold​.cgi).</p>\r\n<p><strong>Computational Protein and RNA Characterization</strong></p>\r\n<p>The pathogenicity prediction was performed by Protter (http:​//wla​b.eth​z.ch/​prott​er/st​art/) examining the regions where the variants were found in the predicted transmembrane structures and the regions that were found to be in the vicinity of the predicted transmembrane structures.</p>\r\n<p>Using the Phyre2 program (http://www.sbg.bio.ic.ac.uk/~phyre2/html/page.cgi?id=index), the predicted two- and three-dimensional structures of the variants detected in the proteins in the region of interest were evaluated by sequence and mutation analysis in terms of mutation sensitivity of the changing amino acid.</p>\r\n<p>RNA fold web server program (http:​//rna​.tbi.​univi​e.ac.​at/cg​i-bin​/RNAW​ebSui​te/RN​Afold​.cgi) determined by using the wild type and variant of the tRNA and rRNA variants which are predicted to occur as a result of any changes in the secondary structure of the mutant was examined. ExPASy-Protscale program (https​://we​b.exp​asy.o​rg/pr​otsca​le/) was evaluated to determine whether the variants of the protein change the hydrophobicity.</p>\r\n<p>The results were evaluated in terms of phenotype&ndash;genotype correlation with the contribution of the clinicians included in the study.</p>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<!--\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t<a class=\"menu\" href=\"#summary_en\">Summary</a><a class=\"menu\" href=\"#introduction\">Introduction</a><a class=\"selected_menu\" href=\"#methods\">Methods</a><a class=\"menu\" href=\"#results\">Results</a><a class=\"menu\" href=\"#discussion\">Discussion</a><a class=\"menu\" href=\"#reference\">Reference</a>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t<div  style=\"width:100%; float:left;  padding-bottom:0px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:100%; float:left; min-height:45px; text-align:justify\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<a href=\"\" name=\"results\"></a>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<!--<h2>Results</h2>-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<h3>RESULTS</h3>\r\n<p>In this study, whole mitochondrial genome sequencing was performed using NGS in 16 patients with MD. In order to clarify all detected variants, bioinformatic analysis was performed to evaluate the pathogenicity of changes. Detected variants were confirmed with Sanger sequencing in patients and their family members.</p>\r\n<p>A total of 12 variants were detected in 9 patients (only class 1-3 variants were listed) (Table 1). Seven of these variants were found in protein-coding genes, 4 of them were in tRNA genes, and 1 of them was in an rRNA gene. Two of the protein-coding variants were identified as class 1 mutations according to CentoGene Database (ATP6; m.9185 T&gt;C p.L220P in patient (P)7 and ND5; m.13513 G&gt;A p.D393N in P16), and patients were diagnosed with LS. Three variants detected in 2 patients were classified as class 2; m.8296 A&gt;G and m.4025 C&gt;T p.T240M in P4; and m.4336 T&gt;C in P3. The remaining 7 variants were evaluated as class 3; m.3316 G&gt;A p.A4T in P2, m.1655 delA and m.9840 T&gt;A p.S212T in P7; m.8633 A&gt;G p.Y36C and m.3213 A&gt;G in P9; and m.15446 C&gt;T p.L234F P12, and m.1646 T&gt;C in P15 (<a href=\"javascript:openWin('uploads/grafik/table_BMJ_2387_1.jpg', 640, 480)\"><b>Table 2</b></a>). According to the read number of NGS data heteroplasmy rate, over 90% were grouped as homoplasmic and 6 of the variants were found to be homoplasmic. The remaining 6 variants were heteroplasmic with the heteroplasmy rate between 34% and 88% (<a href=\"javascript:openWin('uploads/grafik/table_BMJ_2387_1.jpg', 640, 480)\"><b>Table 2</b></a>). Detected variants were confirmed with Sanger sequencing in patients (<a href=\"https://balkanmedicaljournal.org/uploads/pdf/balkanmedj.2021.21141-Supplementary.pdf\">Supplementary Figures 1-6</a>) and family members (data not shown). All identified variants were also found in patients&rsquo; mothers and siblings. Sanger sequencing data for patients&rsquo; mother were also shown in Supplementary file, and heteroplasmy/homoplasmy evaluations were given in <a href=\"javascript:openWin('uploads/grafik/table_BMJ_2387_1.jpg', 640, 480)\"><b>Table 2</b></a>. Sanger sequencing for P11&rsquo;s mother could not be performed because the mother did not give concent to give blood. Detected variants were checked with suitable prediction programmes for their pathogenicity (<a href=\"javascript:openWin('uploads/grafik/table_BMJ_2387_2.jpg', 640, 480)\"><b>Table 3</b></a>). <a href=\"javascript:openWin('uploads/grafik/table_BMJ_2387_3.jpg', 640, 480)\"><b>Table 4</b></a> is a final summary of all variants evaluated in Bioinformatics programmes.</p>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<!--\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t<a class=\"menu\" href=\"#summary_en\">Summary</a><a class=\"menu\" href=\"#introduction\">Introduction</a><a class=\"menu\" href=\"#methods\">Methods</a><a class=\"selected_menu\" href=\"#results\">Results</a><a class=\"menu\" href=\"#discussion\">Discussion</a><a class=\"menu\" href=\"#reference\">Reference</a>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t<div  style=\"width:100%; float:left;  padding-bottom:0px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:100%; float:left; min-height:45px; text-align:justify\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<a href=\"\" name=\"discussion\"></a>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<!--<h2>Discussion</h2>-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<h3>DISCUSSION</h3>\r\n<p>Diagnoses of MD are often difficult due to the heterogeneous clinical manifestations.<sup>2,5 </sup>For a certain genetic diagnosis, clinicians must try to discern disease inheritance patterns, which is not always straightforward to do.<sup>12</sup> While evaluating mitochondrial inheritance, mtDNA allows the screening of 16 569 base pairs, and 37 genes encode 13 proteins namely complex I&nbsp; (ND1, ND2, ND3, ND4, ND4L, ND5, and ND6); complex III (Cytochrome b); complex IV (COXI, COXII, and COXIII) and complex V (<em>ATP6 and ATP8</em>), 22 tRNA, and 2 rRNA, and if mutations can be detected, hundreds of nuclear genes can be excluded in terms of MD.<sup>13,1</sup> Although today with NGS technologies, mtDNA can be entirely sequenced for a relatively low cost and it is possible to detect levels of heteroplasmy even less than 5% in tissues, it is still challenging to identify pathogenic variations due to unknown heteroplasmy in affected tissues. Also coincidentally identified pathogenic mtDNA mutations with low levels of heteroplasmy may be confusing.<sup>14,15</sup></p>\r\n<p>The <em>MT-ATP6</em> gene, which provides instructions for building a portion of complex V, often known as the ATP synthase protein complex, is the most prevalent mtDNA mutation in LS. The ATP synthase complex creates ATP, using the energy provided by the other protein complexes.<sup>16</sup> In this study, 2 changes have been identified in <em>MT-ATP6</em> gene in 2 patients. One of the patients who is carrying class 1 mutations was reevaluated and diagnosed as LS (m.9185 T&gt;C p.L220P change in <em>MT-ATP6</em> gene in P7). To date, 19 pathogenic mutations in <em>MT-ATP6</em> gene were reported in over 500 patients with <em>MT-ATP6</em>-associated disease such as LS, ataxia, and neuropathy, ataxia, and retinitis pigmentosa-like syndrome. Although LS is known as the most frequent phenotype of <em>MT-ATP</em> gene mutations, Ganetzky&nbsp;et&nbsp;al.<sup>16 </sup>demonstrate that <em>MT-ATP6</em> variants cause a continuous disease spectrum rather than a group of distinct clinical syndromes. Four point mutations including m.9185 T&gt;C comprise over 82% of reported disease. In P7, the laboratory investigations revealed lactic acidosis and central hypothyroidism. Neuroradiological findings of the proband indicated for encephalopathy. Proband&rsquo;s (P7) sister applied to the clinic with the same clinical findings and by confirming the mutation in the sibling (99% homoplasmic), it led to early intervention in the clinic and genetic counseling for the family.</p>\r\n<p>In our results, m.3213A&gt;G variant has found as a rare variant in the database; thus, it is thought that it could be related or contributed to the clinical findings. Another variant in the <em>MT-ATP6</em> gene, p.Y36C is located in the intracellular matrix of the transmembrane protein according to the Protter program (<a href=\"https://balkanmedicaljournal.org/uploads/pdf/balkanmedj.2021.21141-Supplementary.pdf\">Supplementary Figure 4C</a>). Our patient&rsquo;s clinical findings were speech delay, cognitive retardation, ataxia, and microcephaly, and a patient with similar clinical findings reported with insC frameshift mutation at position 8611 in a study.<sup>17</sup> m.8633 A&gt;G variation has been shown as pathogenic with high Genomic Evolutionary Rate Profiling (GERP) score (<a href=\"javascript:openWin('uploads/grafik/table_BMJ_2387_3.jpg', 640, 480)\"><b>Table 4</b></a>), and the patient has high degree of mutation heteroplasmy (82%). This variant was confirmed by Sanger sequencing in the patient&rsquo;s symptomatic mother and brother. All data suggest that m.8633 A&gt;G variant may be responsible for the patient&rsquo;s clinical features, so we classify it as possibly pathogenic, and A3213G variant could have been contributed to the phenotype.</p>\r\n<p>Due to heterogeneity in clinical findings of MD, first the disease in P16 was thought to be caused by defects in nuclear genes and the entire nuclear genome was scanned with WGS but no mutation was found to enable the patient to be diagnosed. As a result of our study, P16 and his maternal twin have been diagnosed with LS with low mutant load, G13513A p.D393N mutation which has been reported in the literature as class 1.<sup>18</sup></p>\r\n<p><em>MT-ND1</em> allelic variants have been reported in various diseases; LHON, Alzheimer&rsquo;s, and Parkinson&rsquo;s diseases are the most common among these studies.<sup>19</sup> Although detected variant in P1, in <em>MT-ND1</em> gene at position G3316A change was described as polymorphism in MITOMAP https​://ww​w.mit​omap.​org/f​oswik​i/bin​/view​////Main/SearchAllele, according to reported publications, it was found to be associated with diabetes, LHON, and Progressive External Ophthalmoplegia (PEO) diseases,<sup>20,21</sup> there are also evidence supporting that this change can be a polymorphism as well as studies showing the relationship of this variant with diseases that mentioned above.<sup>2</sup><sup>2</sup> Close positions to this region has been reported with LHON and MELAS.<sup>23,24</sup> The patients&rsquo; asymptomatic mother was also found to be homoplasmic for this variant by Sanger sequencing. <em>MT-ND1</em> gene in mtDNA encodes for a transmembrane protein and alanine amino acid in the fourth position. This position is located on the outer side of the inner membrane (<a href=\"https://balkanmedicaljournal.org/uploads/pdf/balkanmedj.2021.21141-Supplementary.pdf\">Supplementary Figure 1C</a>). Results of Phyre2 program according to the three-dimensional structure of wild type <em>MT-ND1</em> shows that mutation sensitivity of alanine amino acid is low, and replacement of alanine amino acid with threonine was considered to be moderately pathogenic. On the other hand, in ExPASy-ProtScale program, Kyte-Doolittle algorithm was evaluated for wild type and mutant ND1 protein, and p.A4T substitution change the hydrophobicity level which could affect protein flexibility (<a href=\"https://balkanmedicaljournal.org/uploads/pdf/balkanmedj.2021.21141-Supplementary.pdf\">Supplementary Figure 1E</a>). Although this variant is located in an evolutionarily unprotected region, it should not be ignored that this change may be related to MD.</p>\r\n<p>In P3, m.8296 A&gt;G change was previously reported with MELAS, hypertrophic cardiomyopathy (HCM) in the literature.<sup>25,26</sup> In addition to that, a patient who was carrying m.8296 A&gt;G mutation in the heteroplasmic form died due to cardiac failure, and the autopsy results indicated that in various tissues higher heteroplasmy rate of mutation was detected in muscles than blood.<sup>26</sup> When the m.8296 A&gt;G variation is examined in the MitoTIP database, it is evolutionarily preserved at a rate of 93.3% and appears to be pathogenic with a MitoTIP score of 72.30%. m.8296 A&gt;G also appears as pathogenic in mamit-tRNA (<a href=\"https://balkanmedicaljournal.org/uploads/pdf/balkanmedj.2021.21141-Supplementary.pdf\">Supplementary Figure 2C</a>). In addition, according to Sonney&nbsp;et&nbsp;al.&rsquo;s<sup>11</sup> scoring heat map, the variant is located as a structurally important region. However, according to the RNA fold web server program, this variant does not cause a change in the secondary structure of the protein (<a href=\"https://balkanmedicaljournal.org/uploads/pdf/balkanmedj.2021.21141-Supplementary.pdf\">Supplementary Figure 2D</a>). According to the Yarham&nbsp;et&nbsp;al.&rsquo;s<sup>10</sup> scoring table, this variant with a score of 10/20 seems to be pathogenic. In muscle biopsy performed in P3, complex II, III values were in the normal range but complex I and IV were low. Although the <em>MT-ND1</em> gene belongs to the complex I family, it is thought that the variant detected in <em>MT-TK</em> gene may be causative in terms of MD or these 2 variants may have shown pathogenic effects together. In order to investigate other variants detected in the <em>MT-ND1</em> gene m.4025 C&gt;T p.T240M in P3, it is reported in the literature that the mutation did not cause the disease alone, but it was also found with other mutations as in our patient.<sup>27,28</sup> With this knowledge, m.4025 C&gt;T p.T240M substitution does not seem to cause a disease itself; however, it may increase their effect when it is found with other mitochondrial or nuclear genome mutations. Considering the patient&rsquo;s clinical findings, this variant is thought to be related to MD.</p>\r\n<p>Seneca et al.<sup>29</sup> reported&nbsp; a mutation, p.L236P, which is very close to the homoplasmic m.1546 C&gt;T p.L234F variant found in P11, is associated with complex III deficiency. The mutation sensitivity of the Leucine amino acid in the predicted three-dimensional structure is slightly low in Protter program (<a href=\"https://balkanmedicaljournal.org/uploads/pdf/balkanmedj.2021.21141-Supplementary.pdf\">Supplementary Figure 5C</a>). In Phyre2 program, the replacement of phenylalanine aminoacid was considered to be moderately inappropriate (<a href=\"https://balkanmedicaljournal.org/uploads/pdf/balkanmedj.2021.21141-Supplementary.pdf\">Supplementary Figure 5D</a>). In the ExPASy-ProtScale program, p.L234F change caused a decrease in the level of hydrophobicity that could change the flexibility of the protein (<a href=\"https://balkanmedicaljournal.org/uploads/pdf/balkanmedj.2021.21141-Supplementary.pdf\">Supplementary Figure 5E</a>). According to Polyphen, this variation is probably damaging and GERP score is high so considering the patient&rsquo;s findings, this variant might cause MD.</p>\r\n<p>When it is evaluated detected variants in P6, related to p.S212T, m.9840 T&gt;A change, two variants close to p.S212T region in Protter program (G&gt;A change at position m.9804 p.A220T and at position m.9738G&gt;T p.A178S) were reported with LHON disease.<sup>30,31 </sup>(<a href=\"https://balkanmedicaljournal.org/uploads/pdf/balkanmedj.2021.21141-Supplementary.pdf\">Supplementary Figure 3E</a>). P6&rsquo;s muscle biopsy showed that complex I, II, III, V values were within normal range, but complex IV showed low activity. <em>MT-CO3</em> gene encodes a transmembrane protein which is a part of complex IV of Electron Transport Chain (ETC). Analysis of the bioinformatics databases Polyphen2 and SIFT shows that this mutation is pathological (<a href=\"javascript:openWin('uploads/grafik/table_BMJ_2387_3.jpg', 640, 480)\"><b>Table 4</b></a>). Although the mutation sensitivity of serine amino acid in the predicted three-dimensional structure is moderately low (<a href=\"https://balkanmedicaljournal.org/uploads/pdf/balkanmedj.2021.21141-Supplementary.pdf\">Supplementary Figure 3F</a>), in the ExPASy-ProtScale program, it is shown that p.S212T changes the hydrophobicity level and therefore the protein flexibility structure (<a href=\"https://balkanmedicaljournal.org/uploads/pdf/balkanmedj.2021.21141-Supplementary.pdf\">Supplementary Figure 3G</a>). Our findings strongly suggest that m.9840 T&gt; A variant in <em>MT-CO3</em> gene is related to MD for the first time in the literature. To confirm this position with MD, further studies need to be done and also it is always more reliable to perform NGS with muscle biopsy samples compared to blood sample. m.1646 T&gt;C variant <em>MT-TV</em> gene detected in patient P14 is reported as polymorphism in bioinformatics databases (<a href=\"javascript:openWin('uploads/grafik/table_BMJ_2387_3.jpg', 640, 480)\"><b>Table 4</b></a>). The variant has an evolutionary conservation rate of 86.67% at MitoTIP (https​://ww​w.mit​omap.​org/M​ITOMA​P/Mit​oTipI​nfo). According to Sonney&nbsp;et&nbsp;al.<sup>11</sup> scoring map, it appears moderately pathogenic in secondary structure, however pathogenicity score was reported as polymorphism. In the literature, m.1644 G&gt;A change was reported with LS and confirmed as a pathogenic class 1 mutation. This reported class 1 mutation is very close to the detected variant in P14 (<a href=\"https://balkanmedicaljournal.org/uploads/pdf/balkanmedj.2021.21141-Supplementary.pdf\">Supplementary Figure 6D</a>). In addition, in WES analyses, no pathogenic mutation was detected, it is thought that m.1646 T&gt;C variant might be pathogenic and responsible for MD. All detected variants were summarized in <a href=\"javascript:openWin('uploads/grafik/table_BMJ_2387_3.jpg', 640, 480)\"><b>Table 4</b></a>.</p>\r\n<p>In this study, we screened entire mtDNA by NGS in 16 MD suspected patients. Bioinformatic analysis and prediction programs were used to evaluate the entire dataset. Twelve distinct changes in 9 patients were studied one by one to see if they were linked to MD. Seven of the patients were found to be normal in terms of mtDNA changes. Four of detected variants were in tRNA, 1 was in rRNA, and 7 variants were in protein-encoding genes (Table 2), while 2 of these variants were identified as class 1 mutations causing LS in the literature. Four variants were thought to be associated with MD, for the first time in the literature. The remaining 7 variants and MD association was thought to be controversial. We eliminated P2 in terms of MD due to clinical findings which occurred later, thus we thought that m.4336 T&gt;C variant could not be disease-causing as it is recently reported as a polymorphism. In P10, we did not find any variation related to MD by mtDNA screening and WES analysis which was performed externally. The clinical table of the study indicates that approximately 1 out of 2 patients with mitochondrial prediagnosis may be due to a mitochondrial genome defect as a genetic background of the disease.</p>\r\n<p>In our study, all variants that are thought to be related to MD were also detected in patients&rsquo; asymptomatic mothers and siblings. This may be because the heteroplasmic mutant load in other family members is below the pathogenic threshold for that mutation. De novo mutations in mtDNA are also relatively common, so a mutation with high levels of heteroplasmy may not be detected in the mother and other maternal relatives.<sup>32,33</sup> Also functional tissue-specific analysis is limitation of this study. It would be better if real situation in tissues could be confirmed by muscle biopsy, especially for new variants.</p>\r\n<p>To our knowledge, this is the first study in Turkey to screen the whole mitochondrial genome of prediagnosed MD patients with a wide range of clinical symptoms. Mitochondrial disease collective case and related to mtDNA assessment studies in Turkey are rare. One of these studies similar to ours, Abaci&nbsp;et&nbsp;al.<sup>34</sup> screened mtDNA of the 22 patients&rsquo;cardiac tissue with congenital heart disease and reported 3 novel mutations associated with cardiomyopathy. Abaci&nbsp;et&nbsp;al.&rsquo;s<sup>34</sup> study and our results emphasize together the importance of the mtDNA effect especially in undiagnosed MD with wide clinical findings. We recommend first screening mitochondrial genes rather than nuclear genes in patients suspected of MD. Variants which were detected as class 3 to class 2 in our study, with further studies these variants may be included as class 1 status.</p>\r\n<p>In conclusion, it is thought that especially in patients with LS pre-diagnosis, mtDNA screening should be among the primary clinical tests in order to exclude mtDNA-related mutations for a genetic diagnosis. Further studies are needed in more patients to confirm these detected variants with MD associations.</p>\r\n<p><strong>Ethics Committee Approval:</strong> All procedures performed in studies involving human participants were in accordance with the ethical standards of the institutional and/or national research committee and with the 1964 Helsinki declaration and its later amendments or comparable ethical standards. Collection of all human materials was approved by Ankara University School of Medicine Ethics Committee (approval number: 15-967-17).</p>\r\n<p><strong>Patient Consent for Publication:</strong> Informed consent was obtained from all individual participants included in the study.</p>\r\n<p><strong>Author Contributions:</strong> Concept &ndash; T.B., N.A.B., P.B.; Design &ndash; T.B., N.A.B., P.B.; Supervision &ndash; N.B., P.B., P.F.Y.; Data Collection and/or Processing &ndash; T.B., K.E.A., F.E.&Ouml;., N.C.D., R.A., T.A.T., &Ouml;.E.; Analysis and/or Interpretation &ndash; M.A.&Ouml;.; Literature Review &ndash; T.B., F.D.; Writing &ndash; T.B.; Critical Review &ndash; P.F.Y.</p>\r\n<p><strong>Acknowledgments:</strong> The manuscript has been presented in International Inborn Errors of Metabolism and Nutrition Congress as an oral presentation in İstanbul between 10-14 April, 2019.</p>\r\n<p><strong>Conflict of Interest:</strong> The authors have declared that no conflicts of interest exist.</p>\r\n<p><strong>Funding:</strong> This research has been supported by Ankara University Scientific Research Projects Coordination Unit. Project Number: 18L0430004, 2019.</p>\r\n<p><strong>Supplementary: <a href=\"https://balkanmedicaljournal.org/uploads/pdf/balkanmedj.2021.21141-Supplementary.pdf\">https://balkanmedicaljournal.org/uploads/pdf/balkanmedj.2021.21141-Supplementary.pdf</a></strong></p>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<!--\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t<a class=\"menu\" href=\"#summary_en\">Summary</a><a class=\"menu\" href=\"#introduction\">Introduction</a><a class=\"menu\" href=\"#methods\">Methods</a><a class=\"menu\" href=\"#results\">Results</a><a class=\"selected_menu\" href=\"#discussion\">Discussion</a><a class=\"menu\" href=\"#reference\">Reference</a>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t<div  style=\"width:100%; float:left;  padding-bottom:0px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:100%; float:left; min-height:45px; text-align:justify\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<a href=\"\" name=\"reference\"></a>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<!--<h2>Reference</h2>-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<h3>REFERENCES</h3>\r\n<ol>\r\n<li>Anderson S, Bankier&nbsp; AT, Barrell BG, et al. Sequence and organization of the human mitochondrial genome. <em>Nature</em>.1981;290:457-465.</li>\r\n<li>Parikh&nbsp;S, Goldstein&nbsp;A, Koenig&nbsp;MK,&nbsp;et&nbsp;al. Diagnosis and management of mitochondrial disease: a consensus statement from the Mitochondrial Medicine Society. <em>Genet Med</em>. 2015;17:689-701.</li>\r\n<li>Gorman&nbsp;GS, Chinnery&nbsp;PF, DiMauro&nbsp;S,&nbsp;et&nbsp;al. Mitochondrial diseases. <em>Nat Rev Dis Primers</em>. 2016;2:16080.</li>\r\n<li>El-Hattab&nbsp;AW, Scaglia&nbsp;F. Mitochondrial DNA depletion syndromes: review and updates of genetic basis, manifestations, and therapeutic options. <em>Neurotherapeutics</em>. 2013;10(2):186-198.</li>\r\n<li>Alston&nbsp;CL, Rocha&nbsp;MC, Lax&nbsp;NZ, Turnbull&nbsp;DM, Taylor&nbsp;RW. The genetics and pathology of mitochondrial disease. <em>J Pathol</em>. 2017;241:236-250.</li>\r\n<li>Richards&nbsp;S, Aziz&nbsp;N, Bale&nbsp;S,&nbsp;et&nbsp;al. Standards and guidelines for the interpretation of sequence variants: a joint consensus recommendation of the American College of Medical Genetics and Genomics and the Association for Molecular Pathology. <em>Genet Med</em>. 2015;17:405-424.</li>\r\n<li>Coelho&nbsp;MP, Martins&nbsp;E, Vilarinho&nbsp;L. Diagnosis, management, and follow-up of mitochondrial disorders in childhood: a personalized medicine in the new era of genome sequence. <em>Eur J Pediatr</em>. 2019;178:21-32.</li>\r\n<li>Barker. Phenol-chloroform isoamyl alcohol (PCI) DNA extraction. Tampa, Florida: University of South Florida; 1998. Available at: http:​//cco​on.my​web.u​sf.ed​u/eco​immun​ology​.org/​PCI_e​xtrac​tion_​files​/PCI extraction.pdf.</li>\r\n<li>P&uuml;tz&nbsp;J, Dupuis&nbsp;B, Sissler&nbsp;M, Florentz&nbsp;C. Mamit-tRNA, a database of mammalian mitochondrial tRNA primary and secondary structures. <em>Rna</em>. 2007;13:1184-1190.</li>\r\n<li>Yarham&nbsp;JW, Al-Dosary&nbsp;M, Blakely&nbsp;EL,&nbsp;et&nbsp;al. A comparative analysis approach to determining the pathogenicity of mitochondrial tRNA mutations. <em>Hum Mutat</em>. 2011;32:1319-1325.</li>\r\n<li>Sonney&nbsp;S, Leipzig&nbsp;J, Lott&nbsp;MT,&nbsp;et&nbsp;al. Predicting the pathogenicity of novel variants in mitochondrial tRNA with MitoTIP. <em>PLoS Comput Biol</em>. 2017;13:e1005867.</li>\r\n<li>Parikh&nbsp;S, Karaa&nbsp;A, Goldstein&nbsp;A,&nbsp;et&nbsp;al. Diagnosis of &lsquo;possible&rsquo; mitochondrial disease: an existential crisis. <em>J Med Genet</em>. 2019;56:123-130.</li>\r\n<li>Bannwarth&nbsp;S, Procaccio&nbsp;V, Lebre&nbsp;AS,&nbsp;et&nbsp;al. Prevalence of rare mitochondrial DNA mutations in mitochondrial disorders. <em>J Med Genet</em>. 2013;50:704-714.</li>\r\n<li>Buermans&nbsp;HPJ, den Dunnen&nbsp;JT. Next generation sequencing technology: advances and applications. <em>Biochim Biophys Acta</em>. 2014;1842:1932-1941.</li>\r\n<li>Duan&nbsp;M, Tu&nbsp;J, Lu&nbsp;Z. Recent advances in detecting mitochondrial DNA heteroplasmic variations. <em>Molecules</em>. 2018;23:1-18.</li>\r\n<li>Ganetzky&nbsp;RD, Stendel&nbsp;C, Mccormick&nbsp;EM,&nbsp;et&nbsp;al. MT-ATP6 mitochondrial disease variants: phenotypic and biochemical features analysis in 218 published cases and cohort of 14 new cases. <em>Hum Mutat</em>. 2020;40:499-515.</li>\r\n<li>Jackson&nbsp;CB, Hahn&nbsp;D, Schr&ouml;ter&nbsp;B,&nbsp;et&nbsp;al. A novel mitochondrial ATP6 frameshift mutation causing isolated complex V deficiency, ataxia and encephalomyopathy. <em>Eur J Med Genet</em>. 2017;60:345-351.</li>\r\n<li>Brautbar&nbsp;A, Wang&nbsp;J, Abdenur&nbsp;JE,&nbsp;et&nbsp;al. The mitochondrial 13513G&gt;A mutation is associated with Leigh disease phenotypes independent of complex I deficiency in muscle. <em>Mol Genet Metab</em>. 2008;94:485-490.</li>\r\n<li>Danhelovska&nbsp;T, Kolarova&nbsp;H, Zeman&nbsp;J,&nbsp;et&nbsp;al. Multisystem mitochondrial diseases due to mutations in mtDNA-encoded subunits of complex I. <em>BMC Pediatr</em>. 2020;20:41.</li>\r\n<li>Dai&nbsp;Y, Wang&nbsp;C, Nie&nbsp;Z,&nbsp;et&nbsp;al. Mutation analysis of Leber&rsquo;s hereditary optic neuropathy using a multi-gene panel. <em>Biomed Rep</em>. 2018;8:51-58.</li>\r\n<li>Qu&nbsp;J, Zhou&nbsp;X, Zhao&nbsp;F,&nbsp;et&nbsp;al. Low penetrance of Leber&rsquo;s hereditary optic neuropathy in ten Han Chinese families carrying the ND6 T11484C mutation. <em>Biochim Biophys Acta</em>. 2010;1800:305-312.</li>\r\n<li>Lam&nbsp;CW, Yang&nbsp;T, Tsang&nbsp;MW, Pang&nbsp;CP. Homoplasmic 3316G--&gt;A in the ND1 gene of the mitochondrial genome: a pathogenic mutation or a neutral polymorphism? <em>J Med Genet.</em> 2001;38:E10.</li>\r\n<li>Ji&nbsp;Y, Liang&nbsp;M, Zhang&nbsp;J,&nbsp;et&nbsp;al. Mitochondrial ND1 variants in 1281 Chinese subjects with Leber&rsquo;s hereditary optic neuropathy. <em>Invest Ophthalmol Vis Sci</em>. 2016;57:2377-2389.</li>\r\n<li>Opdal&nbsp;SH, Vege&nbsp;A, Egeland&nbsp;T, Musse&nbsp;MA, Rognum&nbsp;TO. Possible role of mtDNA mutations in sudden infant death. <em>Pediatr Neurol</em>. 2002;27:23-29.</li>\r\n<li>Sakuta&nbsp;R, Honzawa&nbsp;S, Murakami&nbsp;N, Goto&nbsp;Y, Nagai&nbsp;T. Atypical MELAS associated with mitochondrial tRNA(Lys) gene A8296G mutation. <em>Pediatr Neurol. </em>2002;27:397-400.</li>\r\n<li>Akita&nbsp;Y, Koga&nbsp;Y, Iwanaga&nbsp;R,&nbsp;et&nbsp;al. Fatal hypertrophic cardiomyopathy associated with an A8296G mutation in the mitochondrial tRNA(Lys) gene. <em>Hum Mutat</em>. 2000;15:382.</li>\r\n<li>Arenas&nbsp;J, Campos&nbsp;Y, Bornstein&nbsp;B,&nbsp;et&nbsp;al. A double mutation (A8296G and G8363A) in the mitochondrial DNA tRNA(Lys) gene associated with myoclonus epilepsy with ragged-red fibers. <em>Neurology</em>. 1999;52:377-382.</li>\r\n<li>Bornstein&nbsp;B, Mas&nbsp;JA, Patrono&nbsp;C,&nbsp;et&nbsp;al. Comparative analysis of the pathogenic mechanisms associated with the G8363A and A8296G mutations in the mitochondrial tRNA(Lys) gene. <em>Biochem J</em>. 2005;387:773-778.</li>\r\n<li>Seneca&nbsp;S, Vancampenhout&nbsp;K, Van Coster&nbsp;R,&nbsp;et&nbsp;al. Analysis of the whole mitochondrial genome: translation of the ion torrent personal genome machine system to the diagnostic bench? Eur <em>J Hum Genet</em>. 2015;23:41-48.</li>\r\n<li>Dogulu&nbsp;CF, Kansu&nbsp;T, Seyrantepe&nbsp;V,&nbsp;et&nbsp;al. Mitochondrial DNA analysis in the Turkish Leber&rsquo;s hereditary optic neuropathy population. <em>Eye</em>. 2001;15:183-188.</li>\r\n<li>Johns&nbsp;DR, Neufeld&nbsp;MJ, Hedges&nbsp;TR. Mitochondrial dna mutations in Cuban optic and peripheral neuropathy. <em>J Neuroophthalmol</em>. 1994;14:135-140.</li>\r\n<li>Naue&nbsp;J, H&ouml;rer&nbsp;S, S&auml;nger&nbsp;T,&nbsp;et&nbsp;al. Evidence for frequent and tissue-specific sequence heteroplasmy in human mitochondrial DNA. <em>Mitochondrion</em>. 2015;20:82-94.</li>\r\n<li>Wallace&nbsp;DC, Chalkia&nbsp;D. Mitochondrial DNA genetics and the heteroplasmy conundrum in evolution and disease. <em>Cold</em> <em>Spring Harb Perspect Biol</em>. 2013;5:a021220.</li>\r\n<li>Abaci&nbsp;N, Arıkan&nbsp;M, Tansel&nbsp;T,&nbsp;et&nbsp;al. Mitochondrial mutations in patients with congenital heart defects by next generation sequencing technology. <em>Cardiol Young</em>. 2015;25:705-711.</li>\r\n</ol>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<!--\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t<a class=\"menu\" href=\"#summary_en\">Summary</a><a class=\"menu\" href=\"#introduction\">Introduction</a><a class=\"menu\" href=\"#methods\">Methods</a><a class=\"menu\" href=\"#results\">Results</a><a class=\"menu\" href=\"#discussion\">Discussion</a><a class=\"selected_menu\" href=\"#reference\">Reference</a>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\n\t\t\n\t\t\t\t\t\t\t\t\t\t\t        \t</div>\n\t\t\t<";
