<div class="col-lg-12 col-md-12 col-sm-12 col-xs-12">
																		        				
												        				<div  style="width:100%; float:left;  padding-bottom:0px;">
												        					<div	style="width:100%; float:left; min-height:45px; text-align:justify">
												        						<a href="" name="summary_en"></a>
												        						<!--<h2>Summary</h2>-->
												        						<h4>Abstract</h4>
<p><strong>Background:</strong> The miR-144/451a cluster acts as a tumor suppressor in various tumors by synergistically inhibiting the proliferation, migration, and invasion of oral squamous cell carcinoma (OSCC).<br /><strong>Aims:</strong> To achieve the synergistic delivery of the miR-144/451a cluster for OSCC treatment by constructing chitosan nanoparticles (CAs) camouflaged with macrophage membranes.<br /><strong>Study Design:</strong> A cell-culture study.<br /><strong>Methods:</strong> CAs were prepared using the ionic cross-linking method, and biomimetic nanoparticles coloaded with the miR-144/451a cluster (miR-144-source of macrophage-derived exosomes [MEXO]/CA-miR-451a) were prepared using the uptake&ndash;efflux method. The MEXO was detected by a bicinchoninic acid assay. The as-prepared biomimetic nanoparticles were then characterized to determine their protective effects on microRNAs (miRNAs). Moreover, the influence of the miR-144-MEXO/CA-miR-451a nanoparticles on the proliferation, migration, and invasion of OSCCs was evaluated. Finally, the effects of the biomimetic system on the expression of calcium-binding protein 39 (CAB39) and migration inhibitory factor (MIF) were detected using the real-time polymerase chain reaction and Western blot.<br /><strong>Results:</strong> After coating the CAs with MEXO, their particle size increased from 113.1 &plusmn; 3.4 nm to 143.2 &plusmn; 14 nm, and their surface potential decreased from 26.34 &plusmn; 0.4 mV to &minus;10.3 &plusmn; 1.6 mV. The expression of the MEXO marker protein was also observed on the biomimetic nanoparticles&rsquo; surface. The system can protect miRNAs from RNase A degradation. Compared with the CAs cotransfected with free miR-144/451a cluster, CAs that are coloaded with miR-144-MEXO/CA-miR-451a nanoparticles substantially reduced the viability (<em>p</em> &lt; 0.001), migration (<em>p</em> = 0.023), and invasion (<em>p</em> = 0.004) of OSCC. These findings revealed the successful construction of biomimetic nanoparticles coloaded with the miR-144/451a cluster. CAB39 and MIF expression in OSCC treated with miR-144-MEXO/CA-miR-451a nanoparticles have significantly decreased compared with the miR-144/451a group (<em>p</em> &lt; 0.05). Thus, the nanoparticles can effectively improve the inhibitory effects of the miR-144/451a cluster on OSCC.<br /><strong>Conclusion:</strong> This study provided a new idea for the application of gene cotransfection to tumor treatment.</p>												        					</div>
												        					<!--
												        					<div	style="width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;">
												        									<a class="selected_menu" href="#summary_en">Summary</a><a class="menu" href="#introduction">Introduction</a><a class="menu" href="#methods">Methods</a><a class="menu" href="#results">Results</a><a class="menu" href="#discussion">Discussion</a><a class="menu" href="#reference">Reference</a>												        					</div>
												        					-->
												        				</div>
												        				
												        				
												        																        				
												        				<div  style="width:100%; float:left;  padding-bottom:0px;">
												        					<div	style="width:100%; float:left; min-height:45px; text-align:justify">
												        						<a href="" name="introduction"></a>
												        						<!--<h2>Introduction</h2>-->
												        						<hr size="1" color="#f2f2f2">
<h3>INTRODUCTION</h3>
<p>Oral squamous cell carcinoma (OSCC) is one of the most common human malignant tumors worldwide, with an increasing incidence rate. Over 400,000 people die from OSCC, of whom 80% are from developing countries.<sup>1</sup> OSCC is currently viewed as a complex biological process caused by carcinogens and involves multiple factors and steps. Changes in cancer-related genes play an important role, which is generally believed to result from the loss, mutation, or overexpression of genetic information.<sup>2</sup> MicroRNAs (miRNAs) are non-coding RNAs that revealed their effects on target genes at the post-transcriptional level.<sup>3</sup></p>
<p>Alterations in miRNA content have been observed in many tumor tissues.<sup>4 </sup>The potential role of miRNAs as therapeutic molecules has transcended their utility from basic research to cancer treatment. However, reports on the relationship between OSCC and miRNA expression have primarily focused on the relationship between one miRNA and one target. However, the 3′--untranslated region has numerous miRNA sites, and most targets are regulated by multiple miRNAs.<sup>5,6</sup> miR-144/451 is a bicistronic miRNA gene locus that is located in chromosome 17 of humans and chromosome 11 of mice. Both are highly conserved in the evolution process<sup> 7,8 </sup>and play important roles in erythroid homeostasis.<sup>9,10</sup> A result comparison of small-RNA library construction has revealed the differential expression of the miR-144/451a cluster between the cancer tissues and normal bile ducts. Some studies revealed that miR-144 and miR-451a expression decreases in several types of malignant tumors, confirming a potential cancer suppressor.<sup>11,12</sup> Recent studies have confirmed the prominent role of the miR-144/451a cluster in the pathogenesis and progression of OSCC.<sup>13</sup></p>
<p>Inflammatory cells tend to adhere to the blood vessel wall and pass through it toward the inflammatory site. Due to this targeting characteristic, they can carry corresponding anti-inflammatory drugs or antibiotics to the focal point to treat infection and inflammation. Parodi et al.<sup>14</sup> were the first to apply nanoparticles encapsulated with inflammatory cell membranes. Nanoparticles coated with cell membranes can easily avoid the monitoring and clearance of the body&rsquo;s immune system due to the homologous recognition mechanism of antigen-presenting cells. Research on natural killer (NK) cell-membrane-encapsulated nanoparticles has revealed that this biomimetic strategy can utilize the receptor molecules on the NK cell membrane&rsquo;s surface, thereby prolonging the circulation time in vivo and substantially enhancing the tumor-homing potential. These features are expected to play a role in tumor treatment and immune monitoring.<sup>15</sup></p>
<p>Chitosan has many advantages as a drug delivery carrier. It forms a polyelectrolyte complex with anionic drugs as the only natural cationic polysaccharide to improve the delivery efficiency of anionic drugs.<sup>16</sup> Chitosan is an important nonviral gene vector as a cationic polymer. As early as the 1990s, researchers have used chitosan nanoparticles (CAs) to load plasmid DNA and obtain stable gene vectors.<sup>17,18</sup> Biomimetics is a new technology that extracts the cell membrane and then coats it onto nanoparticles to realize the functional modification of nanomaterials and obtain functions similar to those of the cells from which the cell membrane is derived.<sup>19</sup> The cell membrane has good biocompatibility and the capacity of innate immune recognition as a biological endogenous substance.<sup>20</sup> Extensive evidence has established that membrane-camouflaged nanosystems have a better therapeutic effect than noncamouflaged ones.<sup>21-23</sup> Meanwhile, macrophages are one of the most common cells in the tumor microenvironment and are chemotactic to the tumor area under some cytokine actions.<sup>24 </sup>The present study aimed to combine biomimetic technology and miRNA cotransfection to effectively deliver drugs to tumor sites. Macrophage-derived exosomes (MEXO) were used to coat CAs for the synergistic delivery of the miR-144/451a cluster for OSCC treatment.</p>												        					</div>
												        					<!--
												        					<div	style="width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;">
												        									<a class="menu" href="#summary_en">Summary</a><a class="selected_menu" href="#introduction">Introduction</a><a class="menu" href="#methods">Methods</a><a class="menu" href="#results">Results</a><a class="menu" href="#discussion">Discussion</a><a class="menu" href="#reference">Reference</a>												        					</div>
												        					-->
												        				</div>
												        				
												        				
												        																        				
												        				<div  style="width:100%; float:left;  padding-bottom:0px;">
												        					<div	style="width:100%; float:left; min-height:45px; text-align:justify">
												        						<a href="" name="methods"></a>
												        						<!--<h2>Methods</h2>-->
												        						<h3>MATERIALS AND METHODS</h3>
<p><strong>Main Reagents and Instruments</strong></p>
<p>UM-SCC083A and UPCI-SCC029B cell lines were provided by Shanghai Key Laboratory of Stomatology (China); human peripheral blood monocyte line THP-1 cells (Shanghai Cell Bank of the Chinese Academy of Sciences, China); miR-451a and miR-144-5p from Sigma (United States of America [USA]); MTT chromogenic agent, fetal bovine serum (FBS), RPMI-1640 medium, and Trypsin (0.25%) from Gibco (USA); Recombinant human interleukin (IL)-4 and IL-13 from Peprotech (USA); Chitosan hydrochloride solution from Aoxing Company (China); calcium-binding protein 39 (Cab39), migration inhibitory factor (MIF), CD63, CD81, TSG101, and &beta;-actin antibodies from Abcam Company (USA); and Transwell cell from Corning Company (USA). The polymerase chain reaction (PCR) primers were synthesized by the Shanghai Sangong Bioengineering Company (China). Trizol reagent, quantitative PCR kit, and reverse transcription kit were bought from Takara Company (Japan). The quantitative PCR instrument (7500) was obtained from ABI Company (USA); fluorescence microplate reader (iMark 680) from Biorad (USA); and WB exposure instrument (Tanon-5200 Multi) from Shanghai Tianneng Company (China).</p>
<p><strong>Macrophage Culture, Induction, and Identification</strong></p>
<p>THP-1 cells were cultured in RPMI-1640 medium + 10% FBS at 37 &deg;C and 5% CO<sub>2</sub>. M2 macrophages were induced by adding 100 ng/ml of phorbol 12-myristate 13-acetate to THP-1 cells and culturing them for 48 h to induce the THP-1 cells into M &phi; macrophages. Then, 40 ng/ml of IL-4 and 20 ng/ml of IL-13 were added and cultured for 48 h. The THP-1 cells were induced into M2 macrophages. The M2 macrophages were identified by detecting the M2 macrophage marker, CD206, and transforming growth factor (TGF)-&beta;, through Western blot. Both exhibited an upward trend, whereas inducible nitric oxide synthase (iNOS) showed a downward trend in M2 macrophage cells. The entire experimental process was supervised and guided by the ethics committee of the Fourth Affiliated Hospital.</p>
<p><strong>MEXO iIsolation</strong></p>
<p>The exosomes were extracted by differential centrifugation.<sup>25</sup> In a typical procedure, exosomes were initially centrifuged at 2000 &times; g and 4 &deg;C for 20 min, followed by 10,000 x g for 30 min. The precipitation was centrifuged twice at 100,000 x g for 70 min and resuspended in PBS. A 0.22 &mu;m filter membrane was used to remove the bacteria and purify exosomes. Finally, the exosome concentration was evaluated using the bicinchoninic acid assay.</p>
<p><strong>Preparation of Chitosan/miR-451a Nanoparticles (CA-miR-451a)</strong></p>
<p>CA-miR-451a nanoparticles were prepared using ion cross-linking.<sup>26</sup> First, 3 &mu;l (20 &mu;g/&mu;l) of miR-451a mimics was added to 1.2 ml (0.84 mg/ml) of tripolyphosphate solution (Sigma, USA), pipetted for &gt; 10 times, and mixed. Afterward, 3 ml of chitosan hydrochloride solution (2 mg/ml) was placed on a magnetic stirrer. Then, the mixture of tripolyphosphate solution and miR-451a mimics was added dropwise. Finally, CA-miR-451a was collected after centrifuging at 25,000 x g for 20 min.</p>
<p><strong>Preparation of miR-144-MEXO/CA-miR-451a Nanoparticles</strong></p>
<p>M2 macrophages were transfected by miR-144-5p. After adding CA-miR-451a solution (300 &mu;g/ml) into the culture medium of miR-144/M2 macrophages, the resulting mixture was incubated for 15&ndash;25 h. Cell supernatant was collected, centrifuged at 5,500 x g for 16 min to remove excess impurities, and centrifuged again at 20,000 x g for 30 min. The supernatant was removed and washed with PBS thrice. The supernatant was sonicated to form a high-concentration miR-144-MEXO/CA-miR-451a suspension of approximately 15 mg/ml.</p>
<p><strong>Characterization and Identification of Nanoparticles</strong></p>
<p>Nanoparticle morphology was examined using transmission electron microscopy (TEM). First, the obtained exosomes were suspended in PBS buffer containing 2% paraformaldehyde. The suspension was collected and placed on the copper mesh of an electron microscope. It was fixed with 1% glutaraldehyde after drying. TEM images of the nanoparticles were obtained for observation. An appropriate amount of the nanosystem solution was collected, into which deionized water was added a 2-ml dilution. The intensity average diameter (Z-average), intensity particle size distribution, and polydispersity index were measured using a laser particle size analyzer. The exosome markers CD63, CD81, and TSG101 were detected using the Western blot.<sup>27 </sup>A loading buffer was added into the exosome suspension for denaturation, electrophoresis, membrane transfer, antibody incubation, ECL development, and imaging.</p>
<h3>CYTOLOGY EXPERIMENTS</h3>
<p><strong>Cell Viability</strong></p>
<p>Free miRNA and nanoparticles were dissolved in a complete medium, and UM-SCC083A and UPCI-SCC029B cells were seeded onto a 96-well plate at a density of 2 x 10<sup>4</sup>/well. Then, 100 &mu;l of the complete medium was added per well. The experiment involved a control group, a free miRNA group, and a nanodrug-loading group, with four multiple pores in each group. Their absorbance value was detected using an MTT kit on the third day.</p>
<p><strong>Agarose-gel Electrophoresis Experiment</strong></p>
<p>DNase digestion experiment was divided into two groups, namely, a free miRNA + RNase A group and a miR-144-MEXO/CA-MIR-451a + RNase A group. The two groups were coincubated with 10 &micro;g of RNase A mixture (Sigma&ndash;Aldrich, USA) at 37 &deg;C. The incubated solution was mixed with 1wt% sodium dodecyl sulfate (SDS) for 3 min and then added to a 1.2% agarose-gel sample hole. In Tris-acetate-EDTA TAE buffer, the two groups were subjected to electrophoresis for 30 min at 100 V. A gel-imaging analysis system was used to observe and photograph the agarose-gel samples.</p>
<p><strong>Detecting Cell Invasion and Migration Ability</strong></p>
<p>Cell migration was assessed by a cell-scratch test.<sup>28</sup> A horizontal line (positioning) was evenly drawn through the midpoint of each hole on the back of a 12-well plate using a pen. Then, 3 x 10<sup>5</sup> cells were added onto each well. Afterward, a 200 &mu;l pipette gun head was used to mark 80%&ndash;90% cell confluence. Exfoliated cells were removed by PBS cleaning. The miR-144/451a cluster and miR-144-MEXO/CA-miR-451a nanoparticles were added, cultured in serum-free medium, sampled, and photographed at a preset time point. The mimic-negative control (NC) group (cells transfected with mimic NC) served as a control group, and the scratch length was measured with Image J software.</p>
<p>Cell invasion was evaluated by Transwell assay.<sup>29</sup> UM-SCC083A and UPCI-SCC029B cells were cultured in RPMI-1640 serum-free medium for 12 h. OSCC cells were resuspended in the medium containing the miR-144/451a cluster and miR-144-MEXO/CA-miR-451a nanoparticles. Cell density was adjusted to 5 x 10<sup>5</sup> cells/mL. A Transwell chamber (coated with matrix glue in advance) and 500 &mu;l of RPMI-1640 medium that contains 20% FBS were added into the lower chamber culture for 12 h. The cells on the bottom of the well were fixed with 4% paraformaldehyde and 200 &mu;l of 33% acetic acid-eluted crystal violet was used for staining. The eluent was moved onto a 96-well plate and placed on a multifunctional microplate reader (570 nm). The optical density of each group was measured to reflect the number of cells passing through the chamber.</p>
<p><strong>Quantitative Real-time PCR (qRT-PCR)</strong></p>
<p>The miR-144/451a cluster and miR-144-MEXO/CA-miR-451a nanoparticles were added into UM-SCC083A and UPCI-SCC029B cells, respectively, and the RNA was extracted after 72 h. The total RNA was initially extracted using the Trizol method. Then, the total RNA was reverse transcribed into cDNA following the reverse transcription kit instructions. The corresponding PCR primers and the reference gene (GAPDH) were analyzed using qRT-PCR. To calculate the expression of miRNAs, the 2-<sup>&Delta;&Delta;</sup>Ct method was employed. Gene primer sequences are shown in <a href="javascript:openWin('uploads/grafik/table_BMJ_2390_0.jpg', 640, 480)"><b>Table 1</b></a>.</p>
<p><strong>Western Blot</strong></p>
<p>UM-SCC083A and UPCI-SCC029B cells were cocultured with the miR-144/451a cluster and miR-144-MEXO/CA-miR-451a nanoparticles for 72 h. The total protein was extracted with radioimmunoprecipitation assay buffer lysate according to the instructions. The protein concentration was determined using a BCA kit. A corresponding volume of TBST (5x) was added to each protein sample, and the protein was kept in a water bath at 95 &deg;C&ndash;100 &deg;C for 15 min. Subsequently, 40 &mu;g of the protein was added to a 12% SDS PAGE gel, and electrophoretic separation was performed at 100 V. Afterward, the molecular weight of actin protein was cut into a glue according to the internal reference protein &beta;-actin. The protein in the glue was subsequently transferred onto polyvinylidene difluoride (PVDF) membranes at a constant current of 200 mA for 2 h. Then, the impurity protein in the PVDF membranes was blocked using an antibody blocking solution and washed with TBST solution five times (6 min each time). The corresponding primary antibody was further incubated overnight at room temperature. After washing with TBST five times, the corresponding secondary antibody was incubated for 2 h. Finally, it was washed three times with TBST. Protein bands were observed and analyzed using a Western blot exposure instrument.</p>
<p><strong>Statistical Analysis</strong></p>
<p>GraphPad Prism 9.0 was used for statistical analysis. The measurement data were tested by variance homogeneity test and normal distribution test. The conformers were expressed as mean &plusmn; standard deviation. Two-factor repeated measurement analysis of variance (ANOVA) and one-way ANOVA were used for statistical analysis. Tukey&rsquo;s honest significant difference test (HSD) was used to perform the post-hoc analysis. Sample sizes were ascertained to provide at least 80% power to identify a broad range of miRNA inhibitory effects. According to the abundance of cells, sufficient statistical power was achieved to differentially detect cell invasion and migration ability. The power calculation was performed based on Tukey&rsquo;s HSD test (global error rate &alpha; = 0.05, &beta; = 0.1, two-sided). According to our primer experimental setup (one 12-well plate containing four well of each UM-SCC08A, UPCI-SCC029B cell, and mimic NC). The miR-144/451a cluster and the miR-144-MEXO/CA-miR-451a nanoparticles were added as treatment (each tested in four replicates), and statistical power of 0.818, an effect size of 0.46, and a sample size of 48 well cells (3 x 10<sup>5</sup> cells in each well, two treatments with four duplicates, and four controls) were estimated, with &alpha; = 0.05. *<em>p</em> &lt; 0.05 represents significant difference.</p>												        					</div>
												        					<!--
												        					<div	style="width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;">
												        									<a class="menu" href="#summary_en">Summary</a><a class="menu" href="#introduction">Introduction</a><a class="selected_menu" href="#methods">Methods</a><a class="menu" href="#results">Results</a><a class="menu" href="#discussion">Discussion</a><a class="menu" href="#reference">Reference</a>												        					</div>
												        					-->
												        				</div>
												        				
												        				
												        																        				
												        				<div  style="width:100%; float:left;  padding-bottom:0px;">
												        					<div	style="width:100%; float:left; min-height:45px; text-align:justify">
												        						<a href="" name="results"></a>
												        						<!--<h2>Results</h2>-->
												        						<h3>RESULTS</h3>
<p><strong>Polarization Induction of THP-1 Cells</strong></p>
<p>The expression of M2 macrophage markers, CD206 and TGF-&beta;, significantly increased in the M &phi;+IL-4 + IL-13 group (<em>P</em> &lt; 0.05, <em>P </em>&lt; 0.01) compared with that in the M &phi; Group. The expression of iNOS, a marker of M1 macrophages, significantly decreased (<em>p</em> &lt; .01) (<a href="javascript:openWin('uploads/grafik/table_BMJ_2390_1.jpg', 640, 480)"><b>Table 2</b></a> and <a href="javascript:openWin('uploads/grafik/figure_BMJ_2390_0.jpg', 640, 480)"><b>Figure 1d</b></a>), indicating the successful induction of M2 cells.</p>
<p><strong>MEXO iIsolation and Characterization</strong></p>
<p>The miR-144-MEXO/CA-miR-451a nanoparticles were characterized. The average diameter of the miR-451a-loaded CAs (CA-miR-451a) was approximately 127 &plusmn; 11 nm. After coating the CA-miR-451a nanoparticles with MEXO, their particle size increased to 143 &plusmn; 14 nm (<a href="javascript:openWin('uploads/grafik/figure_BMJ_2390_0.jpg', 640, 480)"><b>Figures 1a</b></a> and <a href="javascript:openWin('uploads/grafik/figure_BMJ_2390_0.jpg', 640, 480)"><b>1b</b></a> and <a href="javascript:openWin('uploads/grafik/table_BMJ_2390_2.jpg', 640, 480)"><b>Table 3</b></a>). The surface potential of the CA-miR-451a nanoparticles was approximately 22.15 &plusmn; 0.6 mV, whereas that of the miR-144-MEXO/CA-miR-451a nanoparticles was &minus;10.3 &plusmn; 1.6 mV (<a href="javascript:openWin('uploads/grafik/table_BMJ_2390_2.jpg', 640, 480)"><b>Table 3</b></a>). CD63, CD81, and TSG101 are typical MEXO markers (<a href="javascript:openWin('uploads/grafik/figure_BMJ_2390_0.jpg', 640, 480)"><b>Figure 1c</b></a>). Unlike the macrophages, the expression of CD63, CD81, and TSG101 increased in the MEXO and MEXO/CA groups (<a href="javascript:openWin('uploads/grafik/figure_BMJ_2390_0.jpg', 640, 480)"><b>Figure 1c</b></a>).</p>
<p><strong>Protective Effects of the Biomimetic System on miRNAs</strong></p>
<p>The protective effects of the miRNA vector on miRNAs were observed using agarose-gel electrophoresis. Naked miRNA and miR-144-MEXO/CA-miR-451a nanoparticles were cultured in a medium that contains RNase A. Undegenerated miRNAs were analyzed according to the strip brightness of miRNAs that are separated by agarose-gel electrophoresis. The fluorescence intensity of naked miRNA rapidly decreased after RNase A incubation for 1 h, indicating that RNase A remarkably degraded the miRNAs (<a href="javascript:openWin('uploads/grafik/figure_BMJ_2390_1.jpg', 640, 480)"><b>Figures 2a</b></a> and <a href="javascript:openWin('uploads/grafik/figure_BMJ_2390_1.jpg', 640, 480)"><b>2c</b></a>). Contrarily, under the biomimetic nanosystem protection, the miRNAs did not considerably degrade, even when cultured in the RNase A-containing solution for 5 h. Therefore, MEXO/CA can effectively protect miRNAs from RNase A degradation.</p>
<p><strong>miRNA Expression and Cytotoxicity</strong></p>
<p>The target gene expression was detected using the qRT-PCR (<a href="javascript:openWin('uploads/grafik/figure_BMJ_2390_2.jpg', 640, 480)"><b>Figures 3a</b></a> and <a href="javascript:openWin('uploads/grafik/figure_BMJ_2390_2.jpg', 640, 480)"><b>3b</b></a>). The expression of miRNAs was significantly upregulated in each group after the transfection for 3 days (<em>p</em> &lt; 0.05). The target gene upregulation in the CA-miR-451a, miR-144-MEXO, and miR-144-MEXO/CA-miR-451a groups was more significant than that in the other groups (<em>p</em> &lt; 0.05; <a href="javascript:openWin('uploads/grafik/figure_BMJ_2390_2.jpg', 640, 480)"><b>Figures 3a</b></a> and <a href="javascript:openWin('uploads/grafik/figure_BMJ_2390_2.jpg', 640, 480)"><b>3b</b></a>).</p>
<p>The effects of each group on UM-SCC083A and UPCI-SCC029B cell line viability were detected using an MTT kit. Each group (miRNA concentration of 30 pmol) was cocultured with OSCC cells for 3 days. As shown in <a href="javascript:openWin('uploads/grafik/figure_BMJ_2390_3.jpg', 640, 480)"><b>Figure 4a</b></a>, the miR-144 and miR-451a groups cannot inhibit cell viability, whereas the miR-144/451a cotransfection group considerably inhibited the UM-SCC083A cell viability. Moreover, the miR-144-MEXO and miR-144-MEXO/CA-miR-451a groups substantially inhibited the UM-SCC083A cell proliferation, and the inhibitory effect of the latter was stronger than that of the other groups. miR-451a, miR-144/451a, CA-miR-451a, miR-144-MEXO, and miR-144-MEXO/CA-miR-451a nanoparticles significantly inhibited the UPCI-SCC029B cell viability (<a href="javascript:openWin('uploads/grafik/figure_BMJ_2390_3.jpg', 640, 480)"><b>Figure 4b</b></a>; <em>p</em> &lt; 0.05). The inhibitory effect of the cotransfection group was significantly better than that of the single-miRNA transfection group (<em>p</em> &lt; 0.05). The biomimetic nanoparticles exerted the optimum inhibitory effects on UPCI-SCC029B cells.</p>
<p><strong>Inhibitory Effects of the miR-144/451a Biomimetic System on OSCC Cell Migration and Invasion</strong></p>
<p>The effects of each group on cell migration were evaluated using the wound-healing experiment. The migration potential of OSCC cells transfected with miR-144/451a mimics was remarkably lower than that of OSCC cells in the control group. The miR-144-MEXO/CA-miR-451a nanoparticles notably decreased the migration of OSCC cells compared with free miR-144/451a (<a href="javascript:openWin('uploads/grafik/figure_BMJ_2390_4.jpg', 640, 480)"><b>Figure 5</b></a>). Moreover, the miR-144-MEXO/CA-miR-451a biomimetic system more effectively hindered the OSCC cell invasiveness than free miR-144/451a (<a href="javascript:openWin('uploads/grafik/figure_BMJ_2390_5.jpg', 640, 480)"><b>Figure 6</b></a>). These results suggested that the miR-144-MEXO/CA-miR-451a nanoparticles can effectively reduce the migration (<em>p</em> = 0.018) and invasion (<em>p</em> = 0.006) of OSCC cells through the miR-144/451a cluster.</p>
<p><strong>Inhibitory Effects of the miR-144/451a Cotransfection Biomimetic System on CAB39 and MIF Expression in OSCC Cells</strong></p>
<p>CAB39 and MIF gene expression in miRNA-transfected OSCC cells were determined using the qRT-PCR. The miR-144/451a cluster and miR-144-MEXO/CA-miR-451a nanoparticles significantly inhibited CAB39 and MIF gene expression compared with the control group (<em>p</em> &lt; 0.01). CAB39 expression in the miR-144-MEXO/CA-miR-451a group was significantly lower than that in the miR-144/451a group (<em>p</em> &lt; 0.01). The miR-144-MEXO/CA-miR-451a nanoparticle group reduced MIT expression compared with the miR-144/451a group, but significant differences were observed only in UPCI-SCC029B (<em>p</em> &lt; 0.05; <a href="javascript:openWin('uploads/grafik/figure_BMJ_2390_6.jpg', 640, 480)"><b>Figure 7</b></a>).</p>
<p>CAB39 and MIF protein expression in the miRNA-transfected OSCC cells was detected using the Western blot. The miR-144/451a cluster and miR-144-MEXO/CA-miR-451a nanoparticles significantly inhibited CAB39 and MIF protein expression (<em>p</em> &lt; 0.01). CAB39 and MIF expression in the OSCC cells treated with miR-144-MEXO/CA-miR-451a nanoparticles significantly decreased compared with the miR-144/451a group (<em>p</em> &lt; 0.05, <a href="javascript:openWin('uploads/grafik/figure_BMJ_2390_6.jpg', 640, 480)"><b>Figure 7</b></a>).</p>												        					</div>
												        					<!--
												        					<div	style="width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;">
												        									<a class="menu" href="#summary_en">Summary</a><a class="menu" href="#introduction">Introduction</a><a class="menu" href="#methods">Methods</a><a class="selected_menu" href="#results">Results</a><a class="menu" href="#discussion">Discussion</a><a class="menu" href="#reference">Reference</a>												        					</div>
												        					-->
												        				</div>
												        				
												        				
												        																        				
												        				<div  style="width:100%; float:left;  padding-bottom:0px;">
												        					<div	style="width:100%; float:left; min-height:45px; text-align:justify">
												        						<a href="" name="discussion"></a>
												        						<!--<h2>Discussion</h2>-->
												        						<h3>DISCUSSION</h3>
<p>The potential role of miRNAs as therapeutic molecules has transcended their utility from basic research to cancer treatment. Previous studies on tumor suppressor gene regulation using miRNAs have focused mostly on the relationship between a single-miRNA and a single target.<sup>30,31</sup> In recent years, the focus of basic research on miRNAs and their application in cancer treatment has shifted from a single-miRNA to the combined application of multiple miRNAs.<sup>32,33</sup> Kan et al.<sup>34 </sup>revealed that miRNA-21 and miRNA-221 can jointly act on the apoptosis of laryngeal squamous cell carcinoma cells. Mokhtari<sup>35 </sup>revealed that co-transfecting natural epigallocatechin gallate with miR-34a mimics and miR-93 inhibitors can more effectively inhibit the expression of androgen receptors and prostate-specific antigens in PC cells; thus, it may be a promising treatment to inhibit malignant cell growth. Xia et al.<sup>36</sup> revealed that increasing the level of miR-203-3p and reducing that of miR-21-5p can synergistically inhibit the pathogenesis and progression of esophageal cancer. The combined application of miRNA has become a promising new direction for tumor treatment.</p>
<p>Biomimetic technology is a new technology that involves cell membrane isolation and coating it with nanoparticles to realize the functional modification of nanomaterials and obtain similar functions to those of special cell membranes.<sup>37</sup> The application of this technology in coated drug-loaded nanoparticles can enhance their circulation and endow them with biological activities, such as the targeting ability, because of the presence of some functional proteins on the membrane.<sup>38,39</sup> Meanwhile, macrophages are some of the most common cells in the tumor microenvironment and perform chemotactic activities to the tumor area with some cytokines.<sup>40</sup> Tumor-associated macrophages (TAMs) can be polarized into M1 or M2 macrophages in different microenvironments. In degenerative tumors, under the stimulation of interferon-&gamma; and various Th1 cytokines, such as bacterial lipopolysaccharides, TAMs tend to polarize into M1 types. The surface marker CD86 is highly expressed and secrete iNOS and tumor necrosis factor &alpha;. The secreted products can inhibit the proliferation of tumor cells and destroy their vascular endothelium.<sup>41 </sup>In malignant tumors, TAMs tend to polarize into type M2 under the Th2 cytokine regulation (such as IL-4 and IL-13) with high expression of the surface marker CD206. They can secrete IL-10, TGF-&beta;, and other anti-inflammatory factors; degrade the extracellular matrix; and promote angiogenesis. Finally, they are associated with tumor growth and metastasis.<sup>42,43</sup> In the present study, CD206 and TGF-&beta; expression substantially increased after inducing the human peripheral blood monocyte line THP-1 cells using IL-4 and IL-13, but iNOS expression considerably decreased. This finding indicated that the THP-1 cells were successfully induced into M2 macrophages.</p>
<p>The membrane-extrusion method or the freeze-thaw cycle affects protein integrity in cell membranes, which may have a corresponding impact on circulation time and tumor-targeting ability.<sup>44</sup> Therefore, through the natural chemotaxis of macrophages, macrophage membranes are successfully coated onto the CA surface, endowing the CAs with long circulation and active targeting abilities. The TEM images showed that the CA surface was successfully coated with MEXO. The biomimetic nanoparticle surface potential decreased from 22.15 &plusmn; 0.6 mV for CAs to &minus;10.3 &plusmn; 1.6 mV for the miR-144-MEXO/CA-miR-451a nanoparticles. This result also confirmed that CAs were successfully coated with MEXO, consistent with the TEM results. Compared with that of M2 cells, the expression of CD63, CD81, and TSG101 proteins of MEXO and miR-144-MEXO/CA-miR-451a nanoparticles increased, thereby further validating the source of exosomes and the success of chitosan coating.</p>
<p>Like other biological macromolecules, miRNAs are difficult to apply in in vivo drug delivery. The nucleic acid drug administration faces the following problems: they cannot easily penetrate the cellular phospholipid bilayer, they are easily degraded by ribonuclease in the blood and tissues, and they are cleared by the reticuloendothelial system and transported to lysosomes for degradation.<sup>45 </sup>In the current study, the miR-144/451a cluster was co-delivered by MEXO/CA to protect the miRNAs from RNase A degradation. The molecular expression levels of miR-144 and miR-451a were upregulated in both UM-SCC083A and UPCI-SCC029B cell lines 72 h after administering the biomimetic nanoparticles. The inhibitory effects of the cotransfected group were stronger than those of the single-miRNA transfection group, consistent with previous study observations.<sup>13</sup> The CAs and MEXO increased the miRNA inhibition, and some results were significant. Moreover, the cotransfected biomimetic nanoparticle system exerted the best inhibition effect. Further, the effects of the miR-144/451a biomimetic system on OSCC cell migration and invasion were investigated. The inhibitory effects of the miR-144/451a biomimetic system remarkably reduced OSCC cell invasion and migration compared with the free miR-144/451a cotransfected group, which indicated the potential applications of the co-delivery system in preventing oral cancer progression.</p>
<p>CAB39 is a key regulator of aseptic 20 kinase<sup>46</sup> and is a recognized miR-451a target. MIF is extensively involved in tumorigenesis and development, thus it can promote tumor cell proliferation and differentiation and inhibit apoptosis. miR-451a may inhibit the expression of MIF, CAB39, and their downstream proteins by targeting and inhibiting cell proliferation and promoting cell apoptosis, consistent with the conclusions by Goto et al.<sup>47 </sup>Recent studies have revealed that CAB39 and MIF are the targets of miR-451a in oral cancer.<sup>13 </sup>The miR-144-MEXO/CA-miR-451a nanoparticles reduced the <em>CAB39</em> and <em>MIF</em> gene and protein expression in OSCC cells compared with the free cotransfection group, thereby suggesting that the biomimetic system can enhance the miR-144/451a cluster regulation on CAB39 and MIF signals and enhance the inhibitory effects of miRNAs on OSCC cells.</p>
<p>In conclusion, a novel biomimetic nanosystem was successfully constructed for the synergistic delivery of the miR-144/451a cluster. The developed biomimetic system can effectively deliver the miR-144/451a cluster and protect miRNAs from RNase A degradation. The biomimetic system more efficiently inhibited the proliferation, migration, and invasion of OSCC cells compared with the free miR-144/451a group. Overall, this biomimetic system provides an effective carrier platform for gene cotransfection, thereby offering a new idea for OSCC treatment.</p>
<p><strong>Ethics Committee Approval:</strong> The Ethics Committee of the Fourth Affiliated Hospital.(approval date: June 12<sup>th</sup> 2020/number: 2020ky269).</p>
<p><strong>Data Sharing Statement:</strong> The data that support the findings of this study are available from the corresponding author upon reasonable request.</p>
<p><strong>Author Contributions:</strong> Concept- K.L., F.H.; Design- X.L.; Data Collection or Processing- Y.Q.; Analysis or Interpretation- Y.Q.; Writing- U.K., X.L.</p>
<p><strong>Conflict of Interest:</strong> No conflict of interest was declared by the authors.</p>
<p><strong>Funding:</strong> This work was supported by Hebei key project plan of medical science research (no: 20211538).</p>												        					</div>
												        					<!--
												        					<div	style="width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;">
												        									<a class="menu" href="#summary_en">Summary</a><a class="menu" href="#introduction">Introduction</a><a class="menu" href="#methods">Methods</a><a class="menu" href="#results">Results</a><a class="selected_menu" href="#discussion">Discussion</a><a class="menu" href="#reference">Reference</a>												        					</div>
												        					-->
												        				</div>
												        				
												        				
												        																        				
												        				<div  style="width:100%; float:left;  padding-bottom:0px;">
												        					<div	style="width:100%; float:left; min-height:45px; text-align:justify">
												        						<a href="" name="reference"></a>
												        						<!--<h2>Reference</h2>-->
												        						<h3>REFERENCES</h3>
<ol>
<li>Ge L, Liu S, Xie L, Sang L, Ma C, Li H. Differential mRNA expression profiling of oral squamous cell carcinoma by high-throughput RNA sequencing. <em>J Biomed Res</em>. 2015;29:397-404.</li>
<li>Longacre M, Snyder NA, Housman G, et al. A comparative analysis of genetic and epigenetic events of breast and ovarian cancer related to tumorigenesis. <em>Int J Mol Sci.</em> 2016;17:759-777.</li>
<li>Reddy K B. MicroRNA (miRNA) in cancer. <em>Cancer Cell Int</em>. 2015;15:1-6.</li>
<li>Ding HX, Lv Z, Yuan Y, Xu Q. MiRNA polymorphisms and cancer prognosis: a systematic review and meta-analysis. <em>Front Oncol</em>. 2018;8:596.</li>
<li>Peter ME. Targeting of mRNAs by multiple miRNAs: the next step. <em>Oncogene</em>. 2010;29:2161-2164.</li>
<li>Gam JJ, Babb J, Weiss R. A mixed antagonistic/synergistic miRNA repression model enables accurate predictions of multi-input miRNA sensor activity. <em>Nat Commun.</em> 2018;9:2430.</li>
<li>Yang JS, Maurin T, Robine N, et al. Conserved vertebrate mir-451 provides a platform for Dicer-independent, Ago2-mediated microRNA biogenesis. P Natl Acad Sci USA. 2010;107:15163-15168.</li>
<li>Wang T, Wu F, Yu D. miR-144/451 in hematopoiesis and beyond. <em>ExRNA</em>. 2019;1:16.</li>
<li>Fang X, Shen F, Lechauve C, et al. miR-144/451 represses the LKB1/AMPK/mTOR pathway to promote red cell precursor survival during recovery from acute anemia.<em> Haematologica</em>. 2018;103:406-416.</li>
<li>Rasmussen KD, O&rsquo;Carroll D. The miR-144/451eGFP allele, a novel tool for resolving the erythroid potential of hematopoietic precursors. <em>Blood. </em>2011;118:2988-2992.</li>
<li>Kooshkaki O, Rezaei Z, Rahmati M, et al. MiR-144: A new possible therapeutic target and diagnostic/prognostic tool in cancers.<em> Int J Mol Sci</em>. 2020;21:2578.</li>
<li>Khordadmehr M, Jigari-Asl F, Ezzati H, et al. A comprehensive review on miR-451: a promising cancer biomarker with therapeutic potential. <em>J Cell Physiol</em>. 2019;234:21716-21731.</li>
<li>Manasa VG, Thomas S, Kannan S. MiR-144/451a cluster synergistically modulates growth and metastasis of Oral Carcinoma. <em>Oral Dis</em>. 2021.</li>
<li>Parodi A, Quattrocchi N, van de Ven AL, et al. Synthetic nanoparticles functionalized with biomimetic leukocyte membranes possess cell-like functions. <em>Nat Nanotechnol</em>. 2013;8:61-68.</li>
<li>Pitchaimani A, Nguyen TDT, Marasini R, et al. Biomimetic natural killer membrane camouflaged polymeric nanoparticle for targeted bioimaging. <em>Adv Funct Mater</em>. 2019;29: 1806817.</li>
<li>Wu D, Zhu L, Li Y, et al. Chitosan-based colloidal polyelectrolyte complexes for drug delivery: a review. <em>Carbohyd Polym</em>. 2020;238:116126.</li>
<li>Roy K, Mao HQ, Huang SK, Leong KW. Oral gene delivery with chitosan--DNA nanoparticles generates immunologic protection in a murine model of peanut allergy. <em>Nat Med</em>. 1999;5:387-391.</li>
<li>Lee KY, Kwon IC, Kim YH, Jo WH, Jeong SY. Preparation of chitosan self-aggregates as a gene delivery system.<em> J Control Release</em>. 1998;51:213-220.</li>
<li>Liu W L, Zou M Z, Qin S Y, et al. Recent Advances of Cell Membrane-Coated Nanomaterials for Biomedical Applications. Adv Funct Mater. 2020;30:2003559.</li>
<li>Fang R H, Kroll A V, Gao W, et al. Cell membrane coating nanotechnology. <em>Adv Mater</em>. 2018;30:1706759.</li>
<li>Liu Y, Luo J, Chen X, Liu W, Chen T. Cell membrane coating technology: a promising strategy for biomedical applications. <em>Nanommicro Lett</em>. 2019;11:100.</li>
<li>Narain A, Asawa S, Chhabria V, Patil-Sen Y. Cell membrane coated nanoparticles: next-generation therapeutics. <em>Nanomedicine</em>. 2017;12:2677-2692.</li>
<li>Oroojalian F, Beygi M, Baradaran B, Mokhtarzadeh A, Shahbazi MA. Immune Cell Membrane-Coated Biomimetic Nanoparticles for Targeted Cancer Therapy. <em>Small</em>. 2021;17:e2006484.</li>
<li>Chanmee T, Ontong P, Konno K, Itano N. Tumor-associated macrophages as major players in the tumor microenvironment. <em>Cancers (Basel)</em>. 2014;6:1670-1690.</li>
<li>&nbsp;Livshits MA, Khomyakova E, Evtushenko EG, et al. Isolation of exosomes by differential centrifugation: Theoretical analysis of a commonly used protocol. <em>Sci Rep</em>. 2015;5:17319.</li>
<li>&nbsp;Wu G, Feng C, Hui G, et al. Improving the osteogenesis of rat mesenchymal stem cells by chitosan-based-microRNA nanoparticles. <em>Carbohydr Polym</em>. 2016;138:49-58.</li>
<li>&nbsp;Rayamajhi S, Nguyen TDT, Marasini R, Aryal S. Macrophage-derived exosome-mimetic hybrid vesicles for tumor targeted drug delivery. <em>Acta Biomater</em>. 2019;94:482-494.</li>
<li>&nbsp;Bobadilla AVP, Ar&eacute;valo J, Sarr&oacute; E, et al. In vitro cell migration quantification method for scratch assays. <em>J R Soc Interface</em>. 2019;16:20180709.</li>
<li>&nbsp;Ding D, Zhang Y, Yang R, et al. miR-940 suppresses tumor cell invasion and migration via regulation of CXCR2 in hepatocellular carcinoma. <em>Biomed Res Int</em>. 2016;2016:7618342.</li>
<li>Chiyomaru T, Yamamura S, Fukuhara S, et al. Genistein up-regulates tumor suppressor microRNA-574-3p in prostate cancer. <em>PloS one</em>. 2013;8:e58929.</li>
<li>Zhang JG, Wang JJ, Zhao F, Liu Q, Jiang K, Yang GH. MicroRNA-21 (miR-21) represses tumor suppressor PTEN and promotes growth and invasion in non-small cell lung cancer (NSCLC). <em>Clin Chim Acta</em>. 2010;411:846-852.</li>
<li>Zeng D, Wang Z, Meng Z, et al. DNA tetrahedral nanostructure-based electrochemical miRNA biosensor for simultaneous detection of multiple miRNAs in pancreatic carcinoma. <em>ACS Appl Mater Interfaces</em>. 2017;9:24118-24125.</li>
<li>Miroshnichenko S, Patutina O. Enhanced inhibition of tumorigenesis using combinations of miRNA-targeted therapeutics. <em>Front Pharmacol</em>. 2019;10:488.</li>
<li>Kan X, Sun Y, Lu J, et al. Co‑inhibition of miRNA‑21 and miRNA‑221 induces apoptosis by enhancing the p53‑mediated expression of pro‑apoptotic miRNAs in laryngeal squamous cell carcinoma. <em>Mol Med Rep</em>. 2016;13:4315-4320.</li>
<li>Mokhtari H, Yaghmaei B, Sirati-Sabet M, et al. Epigallocatechin-3-gallate Enhances the Efficacy of MicroRNA-34a Mimic and MiR-93 Inhibitor Co-transfection in Prostate Cancer Cell Line. <em>Iran J Allergy Asthm Immunol</em>. 2020;19:612-623.</li>
<li>Xia Y, Wang Y, Wang Q, et al. Increased miR-203-3p and reduced miR-21-5p synergistically inhibit proliferation, migration, and invasion in esophageal cancer cells. <em>Anticancer Drugs</em>. 2019;30: 38-45.</li>
<li>Fang RH, Kroll AV, Gao W, Zhang L. Cell membrane coating nanotechnology. <em>Adv Mater</em>. 2018; 30: e1706759.</li>
<li>Zhen X, Cheng P, Pu K. Recent advances in cell membrane-camouflaged nanoparticles for cancer phototherapy. <em>Small</em>. 2019;15:e1804105.</li>
<li>Liu C, Su C. Design strategies and application progress of therapeutic exosomes. <em>Theranostics</em>. 2019;9:1015-1028.</li>
<li>Chanmee T, Ontong P, Konno K, Itano N. Tumor-associated macrophages as major players in the tumor microenvironment. <em>Cancers</em>. 2014;6:1670-1690.</li>
<li>Wang Y, Lin YX, Qiao SL, Wang J, Wang H. Progress in Tumor-Associated Macrophages: From Bench to Bedside. <em>Adv Biosyst</em>. 2019;3:e1800232.</li>
<li>Zheng P, Luo Q, Wang W, et al. Tumor-associated macrophages-derived exosomes promote the migration of gastric cancer cells by transfer of functional Apolipoprotein E. <em>Cell Death Dis</em>. 2018;9:434.</li>
<li>Zhu X, Shen H, Yin X, et al. Macrophages derived exosomes deliver miR-223 to epithelial ovarian cancer cells to elicit a chemoresistant phenotype. <em>Exp Clin Cancer Res</em>. 2019;38:81.</li>
<li>Rayamajhi S, Aryal S. Surface functionalization strategies of extracellular vesicles. <em>J Mater Chem B</em>. 2020;8:4552-4569.</li>
<li>Zheng M, Tao W, Zou Y, Farokhzad OC, Shi B. Nanotechnology-based strategies for siRNA brain delivery for disease therapy. <em>Trends Biotechnol</em>. 2018;36:562-575.</li>
<li>Wang HM, Lu YJ, He L, et al. HPV16 E6/E7 promote the translocation and glucose uptake of GLUT1 by PI3K/AKT pathway via relieving miR-451 inhibitory effect on CAB39 in lung cancer cells.<em> Ther Adv Chronic Dis</em>. 2020;11:2040622320957143.</li>
<li>Goto T, Fujiya M, Konishi H, et al. An elevated expression of serum exosomal microRNA-191, -21, -451a of pancreatic neoplasm is considered to be efficient diagnostic marker. <em>BMC Cancer</em>. 2018;18:116.</li>
</ol>												        					</div>
												        					<!--
												        					<div	style="width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;">
												        									<a class="menu" href="#summary_en">Summary</a><a class="menu" href="#introduction">Introduction</a><a class="menu" href="#methods">Methods</a><a class="menu" href="#results">Results</a><a class="menu" href="#discussion">Discussion</a><a class="selected_menu" href="#reference">Reference</a>												        					</div>
												        					-->
												        				</div>
												        				
												        				
												        															        			
		
											        	</div>