<div class="col-lg-12 col-md-12 col-sm-12 col-xs-12">
																		        				
												        				<div  style="width:100%; float:left;  padding-bottom:0px;">
												        					<div	style="width:100%; float:left; min-height:45px; text-align:justify">
												        						<a href="" name="summary_en"></a>
												        						<!--<h2>Summary</h2>-->
												        						<h4>Abstract</h4>
<p><strong>Background: </strong><em>RASD1 </em>encodes Dexamethasone-induced Ras-related protein 1 (Dexras1), a protein with a critical role in signal transduction in neurons. There is a strong suspicion that dysfunction of Dexras1 might contribute to the pathogenesis of neuropsychiatric diseases. Related to its functions in intracellular signaling pathways, Dexras1 has a potential role in the etiology of schizophrenia because of its close interaction with NOS1, NOS1AP, and NMDAR, which have previously been associated with schizophrenia.<br /><strong>Aims: </strong>To investigate the association of <em>RASD1 </em>variants with schizophrenia in a selected cohort from Turkey.<br /><strong>Study Design: </strong>A case-control study.<br /><strong>Methods: </strong>We performed targeted sequencing for the two exons, single intron, and untranslated regions of <em>RASD1 </em>gene in 200 individuals with schizophrenia and 100 healthy controls of Turkish origin.<br /><strong>Results: </strong>Two rare variants, <em>RASD1 </em>(NM_016084.5): c.722A&gt;T and c*31G&gt;A were identified in individuals with schizophrenia but not in any controls. The c.722A&gt;T was found in a single individual with schizophrenia and is a missense heterozygous variant in the second exon of <em>RASD1</em>, which is extremely rare in GnomAD. The other variant, c*31G&gt;A, which was found in another individual from this schizophrenia cohort, has not been reported previously. Seven previously identified common single nucleotide polymorphisms were also detected; however, they were not significantly associated with schizophrenia in this study cohort.<br /><strong>Conclusion: </strong>Our findings suggest that rare variants of <em>RASD1 </em>might be contributing to the etiopathogenesis of schizophrenia. Further studies are needed to elucidate the underlying mechanism of this association.</p>												        					</div>
												        					<!--
												        					<div	style="width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;">
												        									<a class="selected_menu" href="#summary_en">Summary</a><a class="menu" href="#introduction">Introduction</a><a class="menu" href="#methods">Methods</a><a class="menu" href="#results">Results</a><a class="menu" href="#discussion">Discussion</a><a class="menu" href="#reference">Reference</a>												        					</div>
												        					-->
												        				</div>
												        				
												        				
												        																        				
												        				<div  style="width:100%; float:left;  padding-bottom:0px;">
												        					<div	style="width:100%; float:left; min-height:45px; text-align:justify">
												        						<a href="" name="introduction"></a>
												        						<!--<h2>Introduction</h2>-->
												        						<hr size="1" color="#f2f2f2">
<h3>INTRODUCTION</h3>
<p>Schizophrenia (SCZ) is a psychiatric disorder characterized by chronic and recurrent psychosis. The lifetime prevalence of SCZ is about 1%, but there may be regional differences.<sup>1</sup> SCZ is a devastating disease not only for the individual but also for their families and society, and according to the World Health Organization, it is classified as one of the highest ranking disorders in the global burden of disease.<sup>2</sup> The disorder is complex and clinically heterogeneous. Nevertheless, in spite of the numerous investigations of post-mortem brain tissues and functional neuroimaging studies that have been conducted, the pathogenesis of SCZ is not clearly understood. Since no biomarker is yet available, the diagnosis of SCZ continues to be based on clinical criteria.<sup>3 </sup>The multifactorial nature of SCZ is acknowledged by many, and substantial heritability is indicated by studies that have shown 40%-50% concordance among monozygotic twins.<sup>4</sup> Numerous linkage and genome-wide association studies (GWAS) have identified genes that are involved in the etiopathogenesis of the disorder and revealed a number of loci and candidate genes.<sup>5,6</sup> In addition to the GWAS studies, a large number of studies are now focusing on identifying rare variants with potentially broader effects.<sup>7-9 </sup>N-methyl-D-aspartate receptor (NMDAR) dysfunction is one of the leading hypotheses for SCZ pathophysiology, and the genes encoding NMDARs and many NMDAR-related proteins have also been associated with SCZ in GWAS.<sup>10,11</sup> NMDARs are located in the postsynaptic membranes and are directly or indirectly associated with postsynaptic density, of which dexamethasone-induced Ras-related protein 1 (Dexras1) is a component. Dexras1, which belongs to the Ras superfamily of small GTPases, is encoded by <em>RASD1</em>, which is an evolutionarily well-conserved gene mapped to chromosome 17p11.2, consisting of two exons. Dexras1 is a 281-amino-acid residue protein, showing 35% homology with other Ras proteins.<sup>12 </sup>Dexras1 is expressed in heart, kidney, liver, and, predominantly in the brain.<sup>12</sup> In the glutamatergic postsynaptic terminal, Dexras1 is a component of the Dexras1-NOS1AP-NOS1 ternary complex.<sup>13</sup> NMDAR-mediated calcium influx activates NOS1, resulting in the production of nitric oxide, which leads to S-nitrosylation of Dexrasl on cysteine 11 and subsequent Dexras1 activation. Recent studies suggest that Dexras1 is involved in other key neuronal processes, such as regulation of circadian rhythms and neuronal iron homeostasis.<sup>12,14,15</sup>An association between SCZ and <em>NOS1</em>, <em>NOS1AP</em>, and <em>NMDAR</em> has previously been shown, and this suggests a possible role for <em>RASD1</em> in SCZ pathogenesis since <em>RASD1</em> has a close functional relationship with all three.<sup>16-18</sup> The specific role of <em>RASD1</em> in SCZ has been investigated in only one study that examined four single nucleotide polymorphisms (SNPs), but no association was found.<sup>16 </sup>Therefore, to the best of our knowledge, the association between rare variants in <em>RASD1</em> and SCZ has not yet been explored, despite the strong suggestion based on the functional relationship. In this study, we investigate the role of common and rare <em>RASD1</em> variants in the etiology of SCZ in a Turkish cohort.</p>												        					</div>
												        					<!--
												        					<div	style="width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;">
												        									<a class="menu" href="#summary_en">Summary</a><a class="selected_menu" href="#introduction">Introduction</a><a class="menu" href="#methods">Methods</a><a class="menu" href="#results">Results</a><a class="menu" href="#discussion">Discussion</a><a class="menu" href="#reference">Reference</a>												        					</div>
												        					-->
												        				</div>
												        				
												        				
												        																        				
												        				<div  style="width:100%; float:left;  padding-bottom:0px;">
												        					<div	style="width:100%; float:left; min-height:45px; text-align:justify">
												        						<a href="" name="methods"></a>
												        						<!--<h2>Methods</h2>-->
												        						<h3>MATERIALS AND METHODS</h3>
<p><strong>Participants</strong></p>
<p>The study utilized the biobank facility of Ankara University Brain Research Center (AUBAUM) to obtain DNA samples. The study group included 200 unrelated individuals with SCZ and 100 unrelated healthy controls who had undergone psychiatric evaluation. The individuals with SCZ were among the participants of a large gene-environment interaction study: the European Network of National SCZ Networks Studying Gene-Environment Interactions (EU-GEI). Details of the EU-GEI project are provided elsewhere.<sup>19 </sup>The inclusion criteria for the SCZ group were as follows: participants had been clinically diagnosed with SCZ according to the operational criteria checklist for diagnosis of psychosis and affective disorders in the International Statistical Classification of Diseases and Related Health Problems, Tenth Revision (ICD-10), had no history of traumatic brain injury or alcohol and substance abuse, had no diagnosis of any serious comorbid medical conditions, including type-2 diabetes, cardiovascular disease, or neurodegenerative disorders, and had no intellectual disability or developmental delay based on clinical evaluation or estimated IQ scores &lt;70 as measured by the brief Wechsler Adult Intelligence Scale, Third Edition (WAIS III). The healthy controls had no first-degree family history of major mental problems. The exclusion criteria for the controls were the same as those for the SCZ group, with the addition of a previous diagnosis of any psychotic disorder or previous use of antipsychotic medication for any reason. The study was conducted in accordance with the Declaration of Helsinki. The study was conducted in accordance with the Declaration of Helsinki. The study protocols for both the case and control groups were reviewed and approved by the Clinical Research Ethics Committee of Ankara University (No: 02-69-17, Date: 31 Jan 2017; No: I11-706-21, Date: 3 January 2022), as well as the Executive Board of the AUBAUM (No: 55, Date: 01 Feb 2017; No:75, Date:15.12.2021).</p>
<p><strong>Genotyping</strong></p>
<p>For all DNA samples, <em>RASD1</em> (NM_016084.5) promoter region,<sup>20</sup> 5&rsquo;UTR, two coding exons, intronic and 3&rsquo;UTR regions were amplified by polymerase chain reaction, analyzed by Sanger sequencing, and variants were annotated according to GRCh38. The primers were designed by Primer3 software<sup>21</sup> and are available upon request. The BigDye Terminator v3.1 Cycle Sequencing Kit on an Applied Biosystems (ABI) PRISM 3130 Genetic Analyzer (Thermo Fisher Scientific, Waltham, MA, USA) was used for Sanger sequencing and subsequently analyzed by SeqScape Software version 2.7 and Sequencing Analysis Software version 5.1. The minor allele frequencies (MAFs) for detected variants were obtained from the Genome Aggregation Database (gnomAD).<sup>22</sup> The potential impact of a rare exonic variant on protein function was predicted by Mutation Taster,<sup>23</sup> Sorting Intolerant from Tolerant (SIFT),<sup>24</sup> and combined annotation-dependent depletion (CADD) scores.<sup>25</sup></p>
<p><strong>Statistical Analysis</strong></p>
<p>A power analysis was performed using the Power Analysis and Sample Size software (NCSS, Kaysville, Utah, USA). Assuming the alpha error as 0.05, beta-error as 0.10, and Sample Allocation Ratio = 2, the power of this study was calculated to be 90%. To assess the differences between these groups, chi square test and t test for independent samples were used, and results were evaluated with 95.0% confidence intervals (CIs) and <em>p</em> &lt; 0.05 was considered statistically significant.To test the deviations from Hardy-Weinberg Equilibrium (HWE), the observed and expected frequencies were compared by a Fisher&rsquo;s exact test. The association between the SNPs and SCZ was investigated for co-dominant, dominant, recessive, overdominant, and log-additive inheritance models and was evaluated by odds ratios (ORs) with 95% CIs. For the assessment of linkage disequilibrium, D, D&rsquo;, and R statistics were calculated. Haplotype associations with SCZ were evaluated by OR with 95% CI. SNPStats were used to perform all statistical analysis and estimate haplotypes.<sup>26</sup></p>
<p>We applied Bonferroni corrections to reduce false-positive associations, and the resultant corrected <em>p</em> &lt; 0.0071 for SNPs (0.05/7) and <em>p </em>&lt; 0.0083 major haplotypes (0.05/6) were considered statistically significant.</p>												        					</div>
												        					<!--
												        					<div	style="width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;">
												        									<a class="menu" href="#summary_en">Summary</a><a class="menu" href="#introduction">Introduction</a><a class="selected_menu" href="#methods">Methods</a><a class="menu" href="#results">Results</a><a class="menu" href="#discussion">Discussion</a><a class="menu" href="#reference">Reference</a>												        					</div>
												        					-->
												        				</div>
												        				
												        				
												        																        				
												        				<div  style="width:100%; float:left;  padding-bottom:0px;">
												        					<div	style="width:100%; float:left; min-height:45px; text-align:justify">
												        						<a href="" name="results"></a>
												        						<!--<h2>Results</h2>-->
												        						<h3>RESULTS</h3>
<p>The mean ages were 33.24 &plusmn; 8.66 and 35.71 &plusmn; 12.3 years for the SCZ and control groups, respectively. There was no significant difference in age between the study groups (<em>p</em> &gt; 0.05). However, gender (139 men and 61 women in the SCZ group; 49 men and 51 women in the control group) was found to be statistically different between the groups (<em>p</em> &lt; 0.001). Male dominance is known in SCZ (male/female ratio: 1.4/1), and the gender difference detected in this study was thought to be related to this fact.<sup>27</sup></p>
<p>Nine variants were identified in different regions of <em>RASD1</em>, including one coding variant (rs550585475) and one intronic variant (rs139149360). Six other variants were located in the 3&rsquo; untranslated region (3&rsquo;UTR) (c*31G&gt;A, rs2232841, rs11545787, rs111300422, rs1671822, and rs711352) and one in 5&rsquo;UTR (rs2232839). While eight of these variants were previously described in gnomAD, the NM_016084.5 (RASD1):c*31G&gt;A, located in the 3&rsquo;UTR, has been identified for the first time in this study. Seven variants were identified in both the SCZ and control groups, and these were common in gnomAD with MAF &gt; 1%. Conversely, two variants (rs550585475 and c*31G&gt;A) were rare, and each of these variants was detected in two different individuals in the SCZ group but not in the control group. Two individuals with these variants demonstrated a typical SCZ phenotype and did not have any unexpected or unusual clinical findings. One of these rare variants is a unique variant in the 3&rsquo;UTR of <em>RASD1</em>, and the latter is an ultra-rare variant located in the second exon. Among them, the NM_016084.5(RASD1):c.722A&gt;T (p.Asp241Val) (rs550585475) variant is a nonsynonymous variant detected only in five individuals in gnomAD (MAF: 0.000036) and was located in the C-terminal extension of the Dexras1 protein (<a href="javascript:openWin('uploads/grafik/figure_BMJ_2445_0.jpg', 640, 480)"><b>Figure 1</b></a>). It was predicted to be disease-causing by the MutationTaster (<em>p</em> = 0.99) and damaging by SIFT (score: 0). Moreover, the variant achieved a CADD score (23.9) &gt;20, which places it among the top 1% most deleterious variants in the human genome. Detailed information of two rare variants detected in this study is summarized in <a href="javascript:openWin('uploads/grafik/table_BMJ_2445_0.jpg', 640, 480)"><b>Table 1</b></a>.</p>
<p>Of the nine variants, only rs11545787 and rs139149360 were not in HWE in the SCZ group. However, rs2232839, rs550585475, c*31G&gt;A, rs2232841, rs111300422, rs1671822, and rs711352 were all in HWE in both groups. Association was explored on all variants except for the two rare SNPs, which were observed only in the SCZ group. No significant differences were detected <br />between the SCZ and control groups for any variants when assessed on five different genetic models after Bonferroni correction. <a href="javascript:openWin('uploads/grafik/table_BMJ_2445_1.jpg', 640, 480)"><b>Table 2</b></a> details the association analysis data for all variants in the case and control groups. Haplotype estimation was performed using pairwise LD analysis, and six haplotypes with a frequency of &gt;0.01 were identified: G-C-A-C-G-G-G/G-C-A-C-G-G-C/A-C-A-T-G-G-G/G-C-A-T-G-G-G/G-C-G-C-C-G-G/A-T-A-T-G-G-G. Haplotype-based association analysis between cases and controls was also not statistically significant after Bonferroni correction (<a href="javascript:openWin('uploads/grafik/table_BMJ_2445_2.jpg', 640, 480)"><b>Table 3</b></a>).</p>												        					</div>
												        					<!--
												        					<div	style="width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;">
												        									<a class="menu" href="#summary_en">Summary</a><a class="menu" href="#introduction">Introduction</a><a class="menu" href="#methods">Methods</a><a class="selected_menu" href="#results">Results</a><a class="menu" href="#discussion">Discussion</a><a class="menu" href="#reference">Reference</a>												        					</div>
												        					-->
												        				</div>
												        				
												        				
												        																        				
												        				<div  style="width:100%; float:left;  padding-bottom:0px;">
												        					<div	style="width:100%; float:left; min-height:45px; text-align:justify">
												        						<a href="" name="discussion"></a>
												        						<!--<h2>Discussion</h2>-->
												        						<h3>DISCUSSION</h3>
<p>SCZ is a complex disease with a heterogeneous etiology consisting of considerable genetic contributions and environmental risk factors. Currently, one of the popular hypotheses employed in genetic studies of common complex disorders is the common variant-common disease hypothesis. Depending on this hypothesis, numerous multicenter and comprehensive GWAS have been conducted to identify genetic susceptibility loci. The heritability of SCZ is estimated to be nearly 80%.<sup>4,28</sup> Nonetheless, studies show that the association of common variants account for only approximately 25% of the heritability in SCZ.<sup>29,30</sup> Therefore, other hypotheses are in need of exploration to address the problem of missing heritability, which may close the gap between association studies and heritability estimates put forward by twin and family studies. The widely accepted common disease-rare variant hypothesis explores the role of individual rare variants with greater functional consequences in common diseases. Several studies have looked for contribution of rare variants (defined as those with an allele frequency of &lt; 1%) in defining susceptibility to common complex diseases and phenotypes since these variants do not appear in GWAS.<sup>7,31</sup> With the widespread use of whole exome/genome sequencing technologies, rare variant studies in SCZ are also increasing to close the gap between genotype and phenotype associations.<sup>32,33 </sup>In the present case-control study, targeted sequencing for all exons, intron and untranslated regions of <em>RASD1</em> together with the promoter region was performed, and two rare variants were identified. The c.722A&gt;T (p.Asp241Val) variant is located in the C-terminal extension of the Dexras1. Although Dexras1 contains all the conserved GTPase motifs found in Ras proteins, it differs from other Ras proteins by a 7 kDA elongation at its C terminus. The Dexras1 C-terminal domain (amino acids 223-276) is essential for activation and protein interactions.<sup>34,35</sup> Notably, Dexras1 binds with NOS1AP via this unique C-terminal domain during Dexras1-NOS1AP-NOS1 ternary complex formation. The consequence of the ternary complex formation is S-nitrosylation and activation of Dexras1.<sup>13</sup> S-nitrosylated Dexras1 mediates NMDAR-induced extracellular iron uptake and NMDAR-triggered release of the lysosomal iron store.<sup>15,36</sup> Since NMDAR-dependent calcium signaling is iron-dependent, these changes may have an impact on NMDAR signaling, leading to psychopathological processes.<sup>37</sup> Furthermore, p.Asp241 is located in an evolutionarily highly conserved region (Figure 1). Given these, it is probable that p.Asp241Val variant of Dexras1 may disrupt interaction between Dexras1 and its partners, leading to a disturbance of downstream NMDAR signaling cascade.The other rare variant discovered in this study, NM_016084.5:c*31 G&gt;A, is a rare non-protein-coding variant, which has not been reported in our control group or any healthy population database to date. It is well-conserved, particularly in mammals, suggesting that it may be under some selective pressure (Figure 1). The role of 3&rsquo;UTR in mRNA as a target of numerous regulatory elements, including microRNAs, RNA-binding proteins, and long non-coding RNAs is well-documented.<sup>38</sup> Several studies have demonstrated how some SNPs in the 3&rsquo;UTRs may contribute to the etiology of SCZ by disrupting miRNA binding stability.<sup>39, 40</sup>. Therefore, we used miRBase<sup>41</sup> to predict potential new miRNA binding sites that the variant (c*31 G&gt;A) can create, and miRWalk<sup>42</sup> and miRTarBase<sup>43</sup> databases to predict whether the variant affects currently identified miRNA binding sites. These in silico analysis showed that the variant did not affect either condition.</p>
<p>To date, only one study has explored role of <em>RASD1</em> variations in SCZ, in which rs4924755, rs711352, rs2232841, and rs2232838 variants in <em>RASD1</em> gene were examined in individuals with SCZ from different ethnic groups. As the quality scores of the data on the rs2232841 and rs2232838 variants did not meet the quality thresholds, no further analysis was conducted on these two variants. The two other variants (rs4924755 and rs711352) examined in this study were observed quite frequently in the population and were not found to be associated with SCZ.<sup>16 </sup>However, there is no study in the literature evaluating the rare variants of<em> RASD1</em> in individuals with SCZ. The data provided by this study may potentially indicate an association between these rare variants of <em>RASD1</em> and SCZ. These findings further cement the belief that genetic studies on multifactorial diseases should not be limited to the investigation of common variants, and rare variants should be investigated as well.Two rare variants found in this study raise the suspicion that rare variants of <em>RASD1</em> might contribute to the etiopathogenesis of SCZ. However, our study is limited by the small sample size and the lack of segregation analysis and functional characterization of these variants. Hence, independent replication studies with a larger sample size and functional validations are necessary to pinpoint the role of <em>RASD1</em> in the etiology of SCZ.</p>
<p><strong>Ethics Committee Approval:</strong> The study protocols for both the case and control groups were reviewed and approved by the Clinical Research Ethics Committee of Ankara University (no: 02-69-17, date: 31 Jan 2017; no: I11-706-21, date: 3 January 2022).</p>
<p><strong>Data Sharing Statement:</strong> The data that support the findings of this study are available from the corresponding author upon reasonable request.</p>
<p><strong>Authorship Contributions:</strong> Concept- C.D.D, H.G.K., H.I.R.; Design- C.D.D., H.G.K., M.C.S., C.S., G.G.A., C.A., H.I.R.; Data Collection or Processing- C.D.D., H.G.K., G.G.A.; Analysis or Interpretation- C.D.D., H.G.K., M.C.S., C.S., H.I.R.; <br />Literature Search- C.D.D., H.I.R.; Writing- C.D.D., H.G.K., C.S., G.G.A., H.I.R.</p>
<p><strong>Conflict of Interest:</strong> No conflict of interest was declared by the authors.</p>
<p><strong>Funding:</strong> This project was supported by Ankara University Scientific Research Projects Coordination Unit (Project No: 17L0230012 to H.I.R.). This work is based on the Medical Genetics Speciality Training Thesis by C.D.D. ; &ldquo;Durmaz, C.D. Investigation of <em>RASD1</em> Gene Mutations in Schizophrenia Patients. Ankara University, Faculty of Medicine, Ankara, 2018&rdquo; (Thesis advisor: H.I.R).</p>
<p><strong>Acknowledgment:</strong> "We thank Nurten Akarsu for facilitating RASD1 sequencing in control samples in Hacettepe University, Timur Tuncalı and Ziynet &Ccedil;ınar for their support in the statistical analysis. In addition, we thank Arda &Ccedil;etinkaya and Nurten Akarsu for critical reading and insightful comments. This study was presented as an oral presentation in the 13th National Medical Genetics Conference, 07-11 November, 2018 Antalya, TURKEY."</p>
<p>&nbsp;</p>												        					</div>
												        					<!--
												        					<div	style="width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;">
												        									<a class="menu" href="#summary_en">Summary</a><a class="menu" href="#introduction">Introduction</a><a class="menu" href="#methods">Methods</a><a class="menu" href="#results">Results</a><a class="selected_menu" href="#discussion">Discussion</a><a class="menu" href="#reference">Reference</a>												        					</div>
												        					-->
												        				</div>
												        				
												        				
												        																        				
												        				<div  style="width:100%; float:left;  padding-bottom:0px;">
												        					<div	style="width:100%; float:left; min-height:45px; text-align:justify">
												        						<a href="" name="reference"></a>
												        						<!--<h2>Reference</h2>-->
												        						<h3>REFERENCES</h3>
<ol>
<li>Kahn RS, Sommer IE, Murray RM, et al. Schizophrenia. <em>Nat Rev Dis Primers. </em>2015;1:15067.</li>
<li>Murray CJ, Lopez AD. The global burden of disease. 1996. <em>Gen&egrave;ve, OMS</em>. 1996:201-246.</li>
<li>Association AP. Diagnostic and statistical manual of mental disorders (DSM-5&reg;): <em>American Psychiatric Pub</em>; 2013.</li>
<li>Cardno AG, Marshall EJ, Coid B, et al. Heritability estimates for psychotic disorders: the Maudsley twin psychosis series. <em>Arch Gen Psychiatry</em>. 1999;56:162-168.</li>
<li>Butler MG, McGuire AB, Masoud H, Manzardo AM. Currently recognized genes for schizophrenia: High-resolution chromosome ideogram representation. <em>Am J Med Genet B Neuropsychiatr Genet</em>. 2016;171:181-202.</li>
<li>Stefansson H, Rujescu D, Cichon S, et al. Large recurrent microdeletions associated with schizophrenia. N<em>ature</em>. 2008;455:232-236.</li>
<li>Gratten J. Rare variants are common in schizophrenia. <em>Nat Neurosci</em>. 2016;19:1426-1428.</li>
<li>Genovese G, Fromer M, Stahl EA, et al. Increased burden of ultra-rare protein-altering variants among 4,877 individuals with schizophrenia. <em>Nat Neurosci</em>. 2016;19:1433-1441.</li>
<li>Zhou J, Ma C, Wang K, et al. Identification of rare and common variants in BNIP3L: a schizophrenia susceptibility gene. <em>Hum Genomics</em>. 2020;14:16.</li>
<li>Coyle JT. NMDA receptor and schizophrenia: a brief history. <em>Schizophr Bull</em>. 2012;38:920-926.</li>
<li>Balu DT. The NMDA receptor and schizophrenia: from pathophysiology to treatment. <em>Adv Pharmacol</em>. 2016;76:351-382.</li>
<li>Cheng HYM, Obrietan K. Dexras1: shaping the responsiveness of the circadian clock. Semin <em>Cell Dev Biol</em>. 2006;17:345-351.</li>
<li>Fang M, Jaffrey SR, Sawa A, Ye K, Luo X, Snyder SH. Dexras1: AG protein specifically coupled to neuronal nitric oxide synthase via CAPON. <em>Neuron</em>. 2000;28:183-193.</li>
<li>Chen Y, Khan RS, Cwanger A, et al. Dexras1, a small GTPase, is required for glutamate-NMDA neurotoxicity. <em>J Neurosci</em>. 2013;33:3582-3587.</li>
<li>White RS, Bhattacharya AK, Chen Y, et al. Lysosomal iron modulates NMDA receptor-mediated excitation via small GTPase, Dexras1. <em>Mol Brain</em>. 2016;9:38.</li>
<li>Weber H, Klamer D, Freudenberg F, et al. The genetic contribution of the NO system at the glutamatergic post-synapse to schizophrenia: further evidence and meta-analysis. <em>Eur Neuropsychopharmacol</em>. 2014;24:65-85.</li>
<li>Brzustowicz LM. NOS1AP in schizophrenia. <em>Curr Psychiatry Rep</em>. 2008;10:158-163.</li>
<li>Catts VS, Lai YL, Weickert CS, Weickert TW, Catts SV. A quantitative review of the postmortem evidence for decreased cortical N-methyl-d-aspartate receptor expression levels in schizophrenia: How can we link molecular abnormalities to mismatch negativity deficits? <em>Biol Psychol</em>. 2016;116:57-67.</li>
<li>European Network of National Networks studying Gene-Environment Interactions in Schizophrenia (EU-GEI); van Os J, Rutten BP, et al.&nbsp; Identifying gene-environment interactions in schizophrenia: contemporary challenges for integrated, large-scale investigations. <em>Schizophr Bull</em>. 2014;40:729-736.</li>
<li>Dreos R, Ambrosini G, P&eacute;rier RC, Bucher P. The Eukaryotic Promoter Database: expansion of EPDnew and new promoter analysis tools. <em>Nucleic Acids Res</em>. 2015;43:92-96.</li>
<li>Untergasser A, Cutcutache I, Koressaar T, et al. Primer3--new capabilities and interfaces. <em>Nucleic Acid Res</em>. 2012;40:115.</li>
<li>Karczewski KJ, Francioli LC, Tiao G, et al. The mutational constraint spectrum quantified from variation in 141,456 humans. <em>Nature</em>. 2020;581:434-443.</li>
<li>Schwarz JM, Cooper DN, Schuelke M, Seelow D. MutationTaster2: mutation prediction for the deep-sequencing age. <em>Nat Methods</em>. 2014;11:361-362.</li>
<li>Vaser R, Adusumalli S, Leng SN, Sikic M, Ng PC. SIFT missense predictions for genomes. <em>Nat Protoc</em>. 2016;11:1-9.</li>
<li>Kircher M, Witten DM, Jain P, O&rsquo;roak BJ, Cooper GM, Shendure J. A general framework for estimating the relative pathogenicity of human genetic variants. <em>Nat Genet</em>. 2014;46:310-315.</li>
<li>Sol&eacute; X, Guin&oacute; E, Valls J, Iniesta R, Moreno V. SNPStats: a web tool for the analysis of association studies. <em>Bioinformatics</em>. 2006;22:1928-1929.</li>
<li>Abel KM, Drake R, Goldstein JM. Sex differences in schizophrenia. <em>Int Rev Psychiatry</em>. 2010;22:417-428.</li>
<li>Hilker R, Helenius D, Fagerlund B, et al. Heritability of schizophrenia and schizophrenia spectrum based on the nationwide Danish twin register. <em>Biol Psychiatry</em>. 2018;83:492-498.</li>
<li>Schizophrenia Working Group of the Psychiatric Genomics Consortium. Biological insights from 108 schizophrenia-associated genetic loci. <em>Nature</em>. 2014;511:421-427.</li>
<li>Lee SH, DeCandia TR, Ripke S, et al. Estimating the proportion of variation in susceptibility to schizophrenia captured by common SNPs. <em>Nat</em> G<em>enet</em>. 2012;44:247-250.</li>
<li>Gibson G. Rare and common variants: twenty arguments. <em>Nat Rev Genet</em>. 2012;13:135-145.</li>
<li>Rhoades R, Jackson F, Teng S. Discovery of rare variants implicated in schizophrenia using next-generation sequencing. <em>J Transl Genet Genom</em>. 2019;3:1-20.</li>
<li>Singh T, Walters JT, Johnstone M, et al. The contribution of rare variants to risk of schizophrenia in individuals with and without intellectual disability. <em>Nat Genet.</em> 2017;49:1167-1173.</li>
<li>Kim HJ, Cha JY, Seok JW, et al. Dexras1 links glucocorticoids to insulin-like growth factor-1 signaling in adipogenesis. <em>Sci Rep.</em> 2016;6:28648.</li>
<li>Cha JY, Kim HJ, Yu JH, et al. Dexras1 mediates glucocorticoid-associated adipogenesis and diet-induced obesity. <em>Proc Natl Acad Sci U S A</em>. 2013;110:20575-20580.</li>
<li>Cheah JH, Kim SF, Hester LD, et al. NMDA receptor-nitric oxide transmission mediates neuronal iron homeostasis via the GTPase Dexras1. <em>Neuron</em>. 2006;51:431-440.</li>
<li>Mu&ntilde;oz P, Humeres A, Elgueta C, Kirkwood A, Hidalgo C, N&uacute;&ntilde;ez MT. Iron mediates N-methyl-D-aspartate receptor-dependent stimulation of calcium-induced pathways and hippocampal synaptic plasticity. <em>J Biol Chem</em>. 2011;286:13382-13392.</li>
<li>Mayr C. Regulation by 3&prime;-untranslated regions. <em>Annu Rev Genet</em>. 2017;51:171-194.</li>
<li>Yang J, Guo X, Zhu L, et al. Rs7219 regulates the expression of GRB2 by affecting miR-1288-mediated inhibition and contributes to the risk of schizophrenia in the Chinese Han population. <em>Cell Mol Neurobio</em>. 2019;39:137-147.</li>
<li>Wu S, Zhang R, Nie F, et al. MicroRNA-137 inhibits EFNB2 expression affected by a genetic variant and is expressed aberrantly in peripheral blood of schizophrenia patients. <em>EBioMedicine</em>. 2016;12:133-142.</li>
<li>Kozomara A, Birgaoanu M, Griffiths-Jones S. miRBase: from microRNA sequences to function. <em>Nucleic Acids Res.</em> 2019;47:155-162.</li>
<li>Sticht C, De La Torre C, Parveen A, Gretz N. miRWalk: An online resource for prediction of microRNA binding sites. <em>PloS One.</em> 2018;13:e0206239.</li>
<li>Huang HY, Lin YCD, Li J, et al. miRTarBase 2020: updates to the experimentally validated microRNA-target interaction database. <em>Nucleic Acids Res.</em> 2020;48:148-154.</li>
</ol>												        					</div>
												        					<!--
												        					<div	style="width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;">
												        									<a class="menu" href="#summary_en">Summary</a><a class="menu" href="#introduction">Introduction</a><a class="menu" href="#methods">Methods</a><a class="menu" href="#results">Results</a><a class="menu" href="#discussion">Discussion</a><a class="selected_menu" href="#reference">Reference</a>												        					</div>
												        					-->
												        				</div>
												        				
												        				
												        															        			
		
											        	</div>