window._articleFullText = window._articleFullText || {};
window._articleFullText[2451] = "<div class=\"col-lg-12 col-md-12 col-sm-12 col-xs-12\">\n\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t<div  style=\"width:100%; float:left;  padding-bottom:0px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:100%; float:left; min-height:45px; text-align:justify\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<a href=\"\" name=\"summary_en\"></a>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<!--<h2>Summary</h2>-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<h4>Abstract</h4>\r\n<p><strong>Background:</strong> Idiopathic generalized epilepsy is the most common group of epilepsy disorders in children and adolescents. Various types of genetic abnormality were identified among the hereditary factors that explain epilepsy.<br /><strong>Aims:</strong> To determine the variations in the etiopathogenesis, treatment protocol planning, and prognosis of idiopathic generalized epilepsy using the next-generation sequencing method.<br /><strong>Study Design:</strong> A cross-sectional study.<br /><strong>Methods:</strong> This study included 32 patients with idiopathic generalized epilepsy. Genomic DNA was obtained from peripheral venous blood samples taken from the patients. A total of 18 genes encoding ion channel subunits that are involved in monogenic disorders and are associated with idiopathic generalized epilepsy were included. The targeted custom next-generation sequencing panel was designed to cover all coding exons and all exon/intron splice site regions of 18 genes.<br /><strong>Results:</strong> We detected 9 (28%) variations, including 1 likely pathogenic (a variant in the <em>SCN1A</em> gene) and 8 of unknown clinical significance (2 in the <em>CLCN2</em> genes, <em>GABBR2</em>, <em>SCN1B</em>, <em>SLC2A1</em>, <em>SLC4A10</em> genes, and 2 in the <em>TBC1D24</em> gene).<br /><strong>Conclusion:</strong> Study results should be supported by functional advanced studies, with increased existing knowledge in the relevant variations.</p>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<!--\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t<a class=\"selected_menu\" href=\"#summary_en\">Summary</a><a class=\"menu\" href=\"#introduction\">Introduction</a><a class=\"menu\" href=\"#methods\">Methods</a><a class=\"menu\" href=\"#results\">Results</a><a class=\"menu\" href=\"#discussion\">Discussion</a><a class=\"menu\" href=\"#reference\">Reference</a>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t<div  style=\"width:100%; float:left;  padding-bottom:0px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:100%; float:left; min-height:45px; text-align:justify\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<a href=\"\" name=\"introduction\"></a>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<!--<h2>Introduction</h2>-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<hr size=\"1\" color=\"#f2f2f2\">\r\n<h3>INTRODUCTION</h3>\r\n<p>Idiopathic generalized epilepsy (IGE) is the most common group of epilepsy disorders in children and adolescents, constituting approximately one-third of all cases. IGE syndromes are characterized by generalized spike-wave discharges, differences in age of onset, different seizure types, characteristic electroencephalography (EEG) abnormalities, absence of structural brain lesions, and normal developmental abilities.<sup>1,2 </sup>Various types of genetic abnormality, such as monogenic, complex, mitochondrial, chromosomal, and imprinting, were identified among the hereditary factors that explain epilepsy. However, the underlying genetic abnormalities remain unclear in most clinical cases.<sup>3</sup> To date, studies have elucidated the genetic etiology of epilepsy syndromes whereby several genetic abnormalities are associated with or contribute to this condition.<sup>4</sup> In recent years, important discoveries have identified genes in monogenic forms of epilepsy.<sup>5</sup> Next-generation sequencing (NGS) has the potential to detect epilepsy-associated causative mutations and empower molecular diagnosis to explain variable phenotypic disorder status. NGS potentially detects epilepsy-associated causative mutations and empowers molecular diagnosis to explain variable phenotypic disorder status. The widespread use of NGS technologies in research and diagnostic laboratories has facilitated rapid epilepsy-related gene identification.<sup>4 </sup>Herein, we report a clinical genetic study of a Turkish population with IGE using targeted NGS. The targeted custom NGS panel contained 18 genes that are involved in monogenic disorders and are associated with IGE.</p>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<!--\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t<a class=\"menu\" href=\"#summary_en\">Summary</a><a class=\"selected_menu\" href=\"#introduction\">Introduction</a><a class=\"menu\" href=\"#methods\">Methods</a><a class=\"menu\" href=\"#results\">Results</a><a class=\"menu\" href=\"#discussion\">Discussion</a><a class=\"menu\" href=\"#reference\">Reference</a>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t<div  style=\"width:100%; float:left;  padding-bottom:0px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:100%; float:left; min-height:45px; text-align:justify\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<a href=\"\" name=\"methods\"></a>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<!--<h2>Methods</h2>-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<h3>MATERIALS AND METHODS</h3>\r\n<p><strong>Patients</strong></p>\r\n<p>The study included 32 patients (22 females and 10 males) diagnosed with IGE from December 2018 to October 2019. The patients&rsquo; ages ranged from 17 to 57 years, with a mean age of ~28.9 years.The patients and their parents were informed about the study before obtaining signed voluntary consent forms. This study was approved by the Scientific Research Ethics Committee of Trakya University Faculty of Medicine.Inclusion criteria were IGE diagnosis, ages of 16-60 years, and without additional neurological findings, consanguinity, traumatic findings in their epilepsy history, and clinically known syndrome or findings incompatible with the disease. Patients with a history of intellectual disability and major congenital anomalies were excluded from the study. Parents in cases with genetic variation were included in the segregation analysis.</p>\r\n<p>Genomic DNA (gDNA) was extracted from peripheral blood samples following the manufacturer&rsquo;s protocol using the EZ1 Advanced XL automated nucleic acid isolation device operating with the EZ1 DNA Blood isolation kit (Qiagen, Germany). The final volume of the obtained gDNA was adjusted to 100 &micro;l. All samples with DNA purity grades of 260/280 less than 2 and concentrations of &ge;20 &micro;l/ng were included in the study.</p>\r\n<p><strong>IGE NGS Panel</strong></p>\r\n<p>A total of 18 genes that are involved in monogenic disorders and are associated with IGE were included based on the Online Mendelian Inheritance in Man (OMIM) database and literature review. A Qiagen Targeted Custom NGS panel was designed to cover all coding exon regions and all exon/intron splice regions of the genes of interest. The <em>CHRNA2, CHRNA4, CHRNB2, CLCN2, GABRA1, GABR2, GABRD, KCNQ2, KCNQ3, KCNT1, SCN1A, SCN1B, SCN2A, </em>and<em> SCN9A</em> genes encoding the subunits of ion channels and IGE related/candidate genes <em>EFHC1, SLC2101, SLCD241, </em>and<em> TBC1D24 </em>genes were included in the study. Canonical transcripts, transcript ID, and exon numbers of the 18 studied genes are presented in <a href=\"javascript:openWin('uploads/grafik/table_BMJ_2451_0.jpg', 640, 480)\"><b>Table 1</b></a>.</p>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<!--\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t<a class=\"menu\" href=\"#summary_en\">Summary</a><a class=\"menu\" href=\"#introduction\">Introduction</a><a class=\"selected_menu\" href=\"#methods\">Methods</a><a class=\"menu\" href=\"#results\">Results</a><a class=\"menu\" href=\"#discussion\">Discussion</a><a class=\"menu\" href=\"#reference\">Reference</a>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t<div  style=\"width:100%; float:left;  padding-bottom:0px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:100%; float:left; min-height:45px; text-align:justify\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<a href=\"\" name=\"results\"></a>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<!--<h2>Results</h2>-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<h3>RESULTS</h3>\r\n<p><strong>Patient Characteristics</strong></p>\r\n<p>This study included 32 patients with clinical IGE pre-diagnoses. The patients consisted of 22 (68.75%) females and 10 (31.25%) males. The patients&rsquo; ages ranged from 17 to 57 years, with a median of 28.9 years.</p>\r\n<p>Clinical follow-up was indicated due to the suspicion of Marfan Syndrome based on the neurological examination of 32 patients. Therefore, the NGS results of the patients were excluded from the presented NGS data.</p>\r\n<p>The earliest age of onset was 8 years and the latest was 34 years. The age of onset for 3 patients could not be determined. <a href=\"javascript:openWin('uploads/grafik/table_BMJ_2451_1.jpg', 640, 480)\"><b>Table 2</b></a> shows the patients&rsquo; age, age of onset, gender distribution, seizure type, syndrome type, electroencephalogram (EEG)/video EEG findings, and notable features.</p>\r\n<p>The EEG results revealed anomalies in 19 (59.3%) patients. Venous malformation was detected in the right cerebellar hemisphere as a result of magnetic resonance in patient 7 only, and the other patients&rsquo; magnetic resonance imaging (MRI) results were evaluated as normal. Of the patients, 17 did not have any history of seizures in family or close relatives, while 15 had a family history of seizures (53%). A risk factor for IGE was detected in 4 (12.5%) patients.</p>\r\n<p>IGE syndrome type was found in 16 patients as juvenile myoclonic epilepsy (JME) (50%) and generalized tonic-clonic (TC) seizure (GTCS) in 6 (18.75%). <a href=\"javascript:openWin('uploads/grafik/table_BMJ_2451_2.jpg', 640, 480)\"><b>Table 3</b></a>.1 shows the syndrome type information of the other 11 patients. The seizure type was evaluated as TC and myoclonic (43.75%) in 14 patients. Seizure type information for 18 patients is given in <a href=\"javascript:openWin('uploads/grafik/table_BMJ_2451_2.jpg', 640, 480)\"><b>Table 3</b></a>.1. Additionally, 17 (53%) patients had no seizures in the past year, 10 (31%) had less than one seizure per month, and 5 (15.6%) had more than one seizure per month.</p>\r\n<p><strong>NGS Panel Screening Results</strong></p>\r\n<p>A coverage percentage of &ge;96 was found in 280 exons and exon/intron splicing regions in terms of the 18 targeted genes. The unique molecular index coverage of target genes was observed as &ge;99%, on average, at &ge;10 and &ge;30 times. The reading depth was considered at least 50 when evaluating the NGS data for detected nucleotide changes.</p>\r\n<p>We detected 9 (28%) variations, including one likely pathogenic (a variant in the <em>SCN1A</em> gene) and 8 of unknown clinical significance (2 in the <em>CLCN2</em> genes, <em>GABBR2</em>, <em>SCN1B</em>, <em>SLC2A1</em>, <em>SLC4A10</em> genes, and 2 in the <em>TBC1D24</em> gene).</p>\r\n<p><a href=\"javascript:openWin('uploads/grafik/table_BMJ_2451_2.jpg', 640, 480)\"><b>Table 3</b></a> provides patient genotypes, family segregation results, <em>in silico</em> prediction scores, Database of Single Nucleotide Polymorphisms (dbSNP) records, American College of Medical Genetics (ACMG)-2015 pathogenicity scores, and ClinVar data, as well as pathogenicity information of the variations detected in the patients. <a href=\"javascript:openWin('uploads/grafik/figure_BMJ_2451_0.jpg', 640, 480)\"><b>Figure 1</b></a> presents the integrative genomics viewer (IGV) image and sanger sequence analyzes of the segregation results of the variations.</p>\r\n<p>Patient 7 had a heterozygous c.560G&gt;A (p.Arg187Gln) variation in the <em>SCN1A</em> gene, which was defined with the number rs777631884 in the dbSNP database, and its global allele frequency was reported as 1/250522 in the genome aggregation database (GnomAD) exome. A missense variant was predicted as &ldquo;likely pathogenic&rdquo; and inherited from the patient&rsquo;s father.</p>\r\n<p>The c.2481T&gt;G, (p.Ile827Met) variation in the <em>CLCN2 </em>gene was detected in patient 11. A paternally inherited missense variant was revealed as damaging and rated as having &ldquo;unknown clinical significance&rdquo; according to the ACMG-2015 criteria. A maternally inherited c.1603A&gt;G (p.Met535Val) variation in the <em>CLCN2</em> gene was found in patient 29. It was defined in dbSNP as rs1313875037 and not found in the gnomAD database. The pathogenicity prediction was a &ldquo;variant of unknown clinical significance (VUS.&rdquo;).&rdquo;</p>\r\n<p>We found 2 heterozygous <em>TBC1D24</em> gene variations, c.*1C&gt;T and c.641G&gt;A (p.Arg214His), in patients 12 and 30, respectively. Segregation analysis was not performed in families because the inheritance pattern of the <em>TBC1D24</em> gene is autosomal recessive although the pathogenicity assessments of both variants were VUS.</p>\r\n<p>Patient 15 carried the c.2575G&gt;A (p.Asp859Asn) variation in the <em>GABBR2</em> gene as heterozygous. The related variation is defined in the dbSNP database with the code rs79773606.A heterozygous <em>SCN1B</em> gene variation c.632G&gt;A (p.Cys211Tyr) was found in patient 26. It was identified as rs150721582 in dbSNP, and its global allele frequency was reported as 104/251424 in the GnomAD_exome. The relevant variant was reported with a &ldquo;conflicting assessment of its pathogenicity&rdquo; according to the ClinVar assessment. <em>In silico </em>analysis of the variant was determined as damaging.</p>\r\n<p>Patient 2 had the heterozygous <em>SLC2A1</em> gene variant c.814A&gt;G (p. Ile272Val), which was not reported in the dbSNP and GnomAD_exome. This novel variation of the <em>SLC2A1</em> gene was predicted as &ldquo;VUS.&rdquo; Segregation analysis of the relevant variation could not be performed because the patient&rsquo;s parents were no longer alive.</p>\r\n<p>A heterozygous <em>SLC4A10</em> gene variation c.2852G&gt;A (p.Arg951Gln) was found in patient 16, which was reported with the number rs748518515 in dbSNP, and the global allele frequency was reported as 3/248636 in the GnomAD_exome. Pathogenicity assessment according to ACMG-2015 criteria in the in silico analysis was damaging.</p>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<!--\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t<a class=\"menu\" href=\"#summary_en\">Summary</a><a class=\"menu\" href=\"#introduction\">Introduction</a><a class=\"menu\" href=\"#methods\">Methods</a><a class=\"selected_menu\" href=\"#results\">Results</a><a class=\"menu\" href=\"#discussion\">Discussion</a><a class=\"menu\" href=\"#reference\">Reference</a>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t<div  style=\"width:100%; float:left;  padding-bottom:0px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:100%; float:left; min-height:45px; text-align:justify\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<a href=\"\" name=\"discussion\"></a>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<!--<h2>Discussion</h2>-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<h3>DISCUSSION</h3>\r\n<p>Advances in genetic technology have led to an increased number of discovered epilepsy-related genes since 2005. Recent studies identified more than 500 epilepsy-related genes that play critical roles in the steps of synaptic transmission, cortical development, and neuron excitability. In 2017, at least 66 of the ion channel genes were associated with epilepsy. The International League Against Epilepsy (ILAE) has reported 76 epilepsy-associated genes, mostly according to seizure types and other associated characteristics, most of which were defined as associated with various discrete age groups.<sup>6,7,26,27</sup>We identified the heterozygous NM_001165963.2 (<em>SCN1A</em>):c.560G&gt;A (p.Arg187Gln) variation in patient 7. The segregation analysis determined the variation as inherited from the father (paternally), but the patient&rsquo;s father did not have clinical complaints of IGE. The <em>SCN1A</em> (OMIM #182389) gene, which encodes the alpha-1-subunit of the sodium channel, was associated with some epilepsy syndromes and several other diseases <sup>8</sup>. It is the most clinically well-known epilepsy-associated gene with over 1,700 reported variants to date in various epilepsy phenotypes.<sup>9</sup> <em>SCN1A</em>, which is responsible for coding the pore-forming unit of the sodium ion channel, was associated with IGE.<sup>10</sup> <em>SCN1A</em>-related seizure disorders are inherited in an autosomal dominant manner. The missense variants near p.Arg187Gln (p.R187Q) (p.D188V, p.W190R, p.N191Y, and p.N191K) were reported in the Human Gene Mutation Database in association with an <em>SCN1A</em>-associated disorder, supporting the functional significance of this protein region. All the developed algorithms to predict the impact of missense variations on protein structure and function (SIFT, PolyPhen-2) suggest the destructiveness of this variant. However, these estimates have not been supported by published functional studies. Suls et al. revealed that <em>SCN1A</em> haploinsufficiency can cause significant intra-familial clinical variability, even in moderately affected patients with epilepsy. The inclusion of multiple genetic and environmental factors may underlie this difference in phenotype severity.<sup>11</sup> Our 21-year-old male patient, who had TC seizures, had a disease onset of 10 years old, and he also had a history of cerebral venous malformation suspicion on MRI. Among the presenting symptoms of cerebral venous malformation, epileptic seizures have been reported in previous studies.<sup>12,13</sup> Recurrence has rarely been reported although epileptic seizures are a common symptom of cerebral venous malformation and early seizure recurrence is common. This variant could be explained by the clinical findings and genetic variation of this case. Therefore, further functional studies are needed to predict the effect of the <em>SCN1A</em>:c.560G&gt;A (p.Arg187Gln) variant on the clinical findings in our patient.</p>\r\n<p>Single nucleotide polymorphisms in the chloride channel genes, <em>CLCN1</em> and <em>CLCN2,</em> were 3 times more common in many patients with epilepsy compared to controls.<sup>14</sup> Mutations in the <em>CLCN2</em> gene were screened in 52 unrelated patients from the IGE family and 23 patients with Doose syndrome to investigate the role of <em>CLCN2</em> in another independent sample and revealed that it may cause intracellular chloride accumulation and a loss of function that could contribute to neuronal overstimulation.<sup>15</sup> In 2003, Haug et al.<sup>16</sup> revealed that <em>CLCN2</em> mutations are responsible for common IGE subsyndromes, such as JME, childhood absence epilepsy (CAE), juvenile absence epilepsy, and GTCS.A paternally inherited NM_004366.6 (<em>CLCN2</em>):c.2481T&gt;G (p.Ile827Met) in patient 11 was indicated as damaging and rated as &ldquo;VUS&rdquo; according to the ACMG-2015 criteria. A maternally inherited c.1603A&gt;G (p.Met535Val) variation in the <em>CLCN2</em> gene was also detected in patient 29, which was a &ldquo;VUS&rdquo; variation pathogenicity prediction.&rdquo;. No effect was expected on the phenotype of patients because both variations were defined as inherited from the unaffected parent familial inheritance. Therefore, the pathogenicity evaluation of both variations supports a benign finding in terms of the literature and databases. The etiopathogenesis of epilepsy in these 2 cases could not be clarified with this study.</p>\r\n<p>Some studies revealed <em>SCN1B</em> gene variations as the cause of the epilepsy phenotype.<sup>17,18</sup> <em>SCN1B</em> mutations were initially identified in families with epilepsy and febrile seizures. The <em>SCN1B</em> phenotype displays similar clinical features as <em>SCN1A</em>, suggesting that the mechanism underlying the <em>SCN1B</em> mutation pathogenicity potentially involves the impaired function of the voltage-gated sodium channel (NaV1). Some of the <em>SCN1B</em> variants (p.C121W, p.I70 E74del, p.R85C, p.R85H, and p.R125L) were reported in patients with IGE with a history of febrile and absence seizure.<sup>19,20</sup>The heterozygous NM_001037.5 (<em>SCN1B</em>):c.632G&gt;A (p.Cys211Tyr) variant was detected in patient 26. A total of 7 records were found in the ClinVar database, but 2 of them were reported as having &ldquo;unknown clinical significance&rdquo; and 5 as &ldquo;possibly benign.&rdquo; The relevant variant was reported as a &ldquo;conflicting assessment of its pathogenicity&rdquo; in the ClinVar database. Pathogenicity scores according to the ACMG-2015 criteria in the <em>in silico </em>analysis of the <em>SCN1B</em>:c.632G&gt;A (p.Cys211Tyr) variant included &ldquo;PM2, PP3, and BP6&rdquo;; DANN score of 0.9974; Gerp score of 3.91; MutationTaster of Disease-Causing. It was determined as harmful in SIFT and evaluated as &ldquo;VUS.&rdquo; The <em>SCN1B</em> gene has also been associated with familial atrial fibrillation, Brugada 5 syndromes, nonspecific cardiac conduction defects, and epileptic encephalopathy according to ClinVar, apart from IGE disease. Our patient did not have any cardiac findings or a history of any other accompanying disease. The patient&rsquo;s parents were unavailable for segregation analysis; thus, it could not be performed. The relevant variant did not affect the phenotype when all the present findings of our patient were evaluated together. This variant is classified as benign according to ClinVar information although prediction tools were assessed as dangerous. The etiopathogenesis of epilepsy, in this case, could not be clarified in this study.</p>\r\n<p><em>SLC2A1</em> gene mutations were identified in various epilepsy phenotypes, from the complex phenotype of &ldquo;classic&rdquo; GLUT1-DS to IGE, which includes infantile seizures, developmental delay, microcephaly, hypotonia, spasticity, and complex movement disorder. The great clinical heterogeneity highlighted by various genetic errors in the <em>SLC2A1</em> gene has complicated the clinical and genetic GLUT1-DS diagnosis. Such <em>SLC2A1</em> mutations lead to premature protein degradation and reduce the GLUT1 concentration to 50%, similar to the pathological effects of hemizygous mutations.<sup>21</sup> Altıokka-Uzun et al.<sup>22</sup> examined patients with IGE associated with eyelid myoclonia for the possible presence of <em>SLC2A1</em> gene variants, considering GLUT1 deficiency as the cause of IGE in the study. <em>SLC2A1</em> was identified as one of the most important epilepsy genes in drug-resistant patients according to the ILAE guidelines, mainly due to its association with the ketogenic diet, which is a treatment option. However, Altıokka-Uzun et al.<sup>22</sup> reported that <em>SLC2A1</em> gene variants did not play an important role in eyelid myoclonia-associated IGE. A heterozygous NM_006516.3 (<em>SLC2A1</em>):c.814A&gt;G (p. Ile272Val) variant was identified in patient 2. The pathogenicity scores according to the ACMG-2015 criteria in our <em>in silico</em> analysis of the variant included the following: &ldquo;PM1, PM2, and PP2&rdquo;; DANN score of 0.9956; Gerp score of 5.13; and Mutation Taster: Disease-causing and evaluated as &ldquo;Unknown clinical significance.&rdquo; The neurological examination of a 57-year-old male patient revealed the age of onset of the first attack was 8 years, with normal EEG findings, without a family history of consanguinity, and without seizure history. We plan to conduct up-to-date functional research and applications regarding the variant we have detected in our patient due to the autosomal dominant inheritance in IGE disease associated with the <em>SLC2A1</em> gene. Segregation analysis could not be performed because the patient&rsquo;s parents were deceased. Additionally, we plan to study this variant in surviving siblings, children, and close relatives, if any, by contacting the patient.</p>\r\n<p>The<em> SLC4A10</em> gene is associated with complex epileptic conditions, ID, ASD, cognitive disability, and hearing impairment. <em>SLC4A10</em> gene variants were reported to decrease neuronal excitability in studies conducted in mice, resulting in increased seizure threshold, visual acuity and contrast sensitivity deterioration, and hearing loss. <em>SLC4A10</em> is highly expressed in the cerebral cortex and hippocampus, which are the 2 regions commonly associated with epilepsy. Gurnett et al.<sup>23</sup> reported the first patient with epilepsy and cognitive impairment with changes in the <em>SLC4A10</em> gene, a gene encoding the electroneutral sodium bicarbonate modifier.The heterozygous NM_001178015.2 (<em>SLC4A10</em>):c.2852G&gt;A (p.Arg951Gln) variant was identified with the number rs748518515 in dbSNP, and the global allele frequency was reported as 3/248636 in the GnomAD_exome. The constructor was determined to be SIFT: Tolerable, and this variant was evaluated as &ldquo;VUS.&rdquo;.&rdquo; The neurological examination of a 30-year-old female patient (patient 16) revealed that the age of onset of the first attack was 15 years, and EEG findings were accompanied by IGE anomalies. The patient has no family history of consanguinity and no relative had a history of seizures. Segregation analysis could not be performed because the patient&rsquo;s parents were unreachable. The variant we detected in our patient was not expected to affect the patient&rsquo;s phenotype considering the studies supporting the autosomal recessive inheritance pattern in <em>SLC4A10</em> gene-related IGE disease. Epilepsy-associated with the <em>SLC4A10</em> gene has not been well studied in humans. The etiopathogenesis of epilepsy, in this case, could not be clarified in this study.The c.2575G&gt;A variant detected in the <em>GABBR2</em> gene in patient 15 was benign in the ClinVar. Additionally, it does not explain the etiology of epilepsy in this case, since the case also had a clinically unaffected mother. The etiopathogenesis of epilepsy, in this case, could not be clarified with this study.</p>\r\n<p>Heterozygous variants were detected in the <em>TBC1D24</em> gene in patient 12 (c.*1C&gt;T) and in patient 30 c.641G&gt;A (p.Arg214His), and both variants were classified as VUS but the inheritance pattern of the <em>TBC1D24</em> gene is autosomal recessive. Pathogenic variations observed in the <em>TBC1D24</em> genes showed an autosomal recessive inheritance pattern. Therefore, the <em>TBC1D24</em> variation detected in patient 12 did not have a significant relationship with the clinical status of the patients.A heterozygous <em>SLC4A10</em> gene variation c.2852G&gt;A (p.Arg951Gln) was found in patient 16, which was reported with the number rs748518515 in dbSNP, and the global allele frequency was reported as 3/248636 in the GnomAD_exome. Pathogenicity assessment according to ACMG-2015 criteria in the <em>in silico</em> analysis was damaging.We anticipate that increasing the number of samples and investigating the phenotype&ndash;genotype relationship in patients with IGE would allow us to make significant contributions to the literature. The number of patients in the sample group is one of the limitations of the study. The effectiveness of our targeted NGS panel, which included 18 genes and was designed with unique primers in our study, can be increased with new candidate genes. The NGS panel gene content used in the study is another limitation. Additionally, the inability to perform segregation analyses in some of the reported genetic variations is an important limitation of our study. Large deletion or duplication-type mutations were detected by array-CGH analysis in the etiopathology of epilepsy cases, except for single gene mutations detected by NGS technology. Studies reported copy number changes in approximately 5%-12% of patients with different epilepsy types. The clinical contribution of aCGH findings in patients with IGE was not reported. NGS could be applied to patients due to the budget of the study. The aCGH study is also recommended to elucidate the etiopathogenesis of IGE. Our study findings should be supported by functional advanced studies, with an increased existing knowledge of the relevant variations.</p>\r\n<p>Inheritance patterns in idiopathic epilepsies are often complex. The effects of related genes and environmental factors on the disease are seen in this form of inheritance. Additionally, genetic heterogeneity is intensely observed in IGE syndromes, and the fact that more than one gene is associated with these syndromes or that a single gene is associated with more than one epileptic syndrome may create difficulties in determining the genetic etiology underlying the disease.<sup>24,25,28</sup></p>\r\n<p><strong>Ethics Committee Approval:</strong> The institutional review committee (Trakya University Faculty of Medicine, TUMF Scientific Research Ethics Committee Directive TUTF-BAEK 2018/25 Edirne, Turkey) approved the study.</p>\r\n<p><strong>Data Sharing Statement:</strong> The data that support the findings of this study are available from the corresponding author upon reasonable request.</p>\r\n<p><strong>Peer Review:</strong> Hakan G&uuml;rkan is the member of the Editorial Board of Balkan Medical Journal. However, He did not take place at any stage on the editorial decision of the manuscript.</p>\r\n<p><strong>Authorship Contributions:</strong> Concept- E.A., H.G., B.G.; Design- E.A., H.G.; Data Collection or Processing- D.E.; Analysis or Interpretation- E.A.; Literature Search- H.G., S.D., E.İ.A.; Writing- E.A.</p>\r\n<p><strong>Conflict of Interest:</strong> No conflict of interest was declared by the authors.</p>\r\n<p><strong>Funding:</strong> The authors declared that this study received no financial support.</p>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<!--\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t<a class=\"menu\" href=\"#summary_en\">Summary</a><a class=\"menu\" href=\"#introduction\">Introduction</a><a class=\"menu\" href=\"#methods\">Methods</a><a class=\"menu\" href=\"#results\">Results</a><a class=\"selected_menu\" href=\"#discussion\">Discussion</a><a class=\"menu\" href=\"#reference\">Reference</a>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t<div  style=\"width:100%; float:left;  padding-bottom:0px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:100%; float:left; min-height:45px; text-align:justify\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<a href=\"\" name=\"reference\"></a>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<!--<h2>Reference</h2>-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<h3>REFERENCES</h3>\r\n<ol>\r\n<li>Guerrini R, Marini C, Barba C. Generalized epilepsies. <em>Handb Clin Neurol.</em> 2019;161:3-15.</li>\r\n<li>Elmali AD, Auvin S, Bast T, Rubboli G, Koutroumanidis M. How to diagnose and classify idiopathic (genetic) generalized epilepsies. <em>Epileptic Disord.</em> 2020;22:399-420.</li>\r\n<li>Lemke JR, Riesch E, Scheurenbrand T, et al. Targeted next generation sequencing as a diagnostic tool in epileptic disorders. <em>Epilepsia.</em> 2012:53:1387-1398.</li>\r\n<li>Dunn P, Albury CL, Maksemous N, et al. Next Generation Sequencing Methods for Diagnosis of Epilepsy Syndromes. <em>Front Genet.</em> 2018;7:9:20.</li>\r\n<li>Meisler MH, Kearney JA. Sodium channel mutations in epilepsy and other neurological disorders. <em>J Clin Invest.</em> 2005;115:2010-2017.</li>\r\n<li>Symonds JD, McTague A. Epilepsy and developmental disorders: Next generation sequencing in the clinic. <em>Eur J Paediatr Neurol.</em> 2020;24:15-23.</li>\r\n<li>Wang J, Lin ZJ, Liu L, et al. Epilepsy-associated genes. <em>Seizure. </em>2017;44:11-20.</li>\r\n<li>Scheffer IE, Nabbout R. SCN1A-related phenotypes: epilepsy and beyond. <em>Epilepsia</em>. 2019;60:17-24.</li>\r\n<li>Parihar R, Ganesh S. The SCN1A gene variants and epileptic encephalopathies. <em>J Hum Genet.</em> 2013;58:573-580.</li>\r\n<li>Chan CK, Low JS, Lim KS, Low SK, Tan CT, Ng CC. Whole exome sequencing identifies a novel SCN1A mutation in genetic (idiopathic) generalized epilepsy and juvenile myoclonic epilepsy subtypes. <em>Neurol Sci.</em> 2019;41:591-598.</li>\r\n<li>Suls A, Velizarova R, Yordanova I, et al. Four generations of epilepsy caused by an inherited microdeletion of the SCN1A gene. <em>Neurology.</em> 2010;75:72-76.</li>\r\n<li>Ding H, Xie Y, Li L, et al. Clinical features of seizures after cerebral venous sinus thrombosis and its effect on outcome among Chinese Han population. <em>Stroke Vasc Neurol. </em>2017;2:184-188.</li>\r\n<li>S&aacute;nchez van Kammen M, Lindgren E, Silvis SM, et al. Late seizures in cerebral venous thrombosis. <em>Neurology.</em> 2020;95:1716-1723.</li>\r\n<li>Spillane J, Kullmann DM, Hanna MG. Genetic neurological channelopathies: molecular genetics and clinical phenotypes. <em>J Neurol Neurosurg Psychiatry.</em> 2016;87:37-48.</li>\r\n<li>Saint-Martin C, Gauvain G, Teodorescu G, et al. Two novel CLCN2 mutations accelerating chloride channel deactivation are associated with idiopathic generalized epilepsy. <em>Hum Mutat.</em> 2009;30:397-405.</li>\r\n<li>Haug K, Warnstedt M, Alekov AK, et al. Retraction: Mutations in CLCN2 encoding a voltage-gated chloride channel are associated with idiopathic generalized epilepsies. Nat Genet. 2009;41:1043.</li>\r\n<li>Kaplan DII, Isom LL, Petrou S. Role of sodium channels in epilepsy. <em>Cold Spring Harb Perspect Med.</em> 2016;6:a022814.</li>\r\n<li>Meisler MH, O&rsquo;Brien JE, Sharkey LM. Sodium channel gene family: Epilepsy mutations, gene interactions and modifier effects. <em>J Physiol.</em> 2010;588:1841-1848.</li>\r\n<li>Wei F, Yan LM, Su T, et al. Ion Channel Genes and Epilepsy: Functional Alteration, Pathogenic Potential, and Mechanism of Epilepsy. <em>Neurosci Bull.</em> 2017;33:455-477.</li>\r\n<li>Gong JE, Liao HM, Long HY, et al. SCN1B and SCN2B gene variants analysis in dravet syndrome patients: Analysis of 22 cases. <em>Medicine.</em> 2019;98:e14974.</li>\r\n<li>Ivanova N, Peycheva V, Kamenarova K, et al<em>. </em>Three novel SLC2A1 mutations in Bulgarian patients with different forms of genetic generalized epilepsy reflecting the clinical and genetic diversity of GLUT1-deficiency syndrome. <em>Seizure.</em> 2018;54:41-44.</li>\r\n<li>Altıokka-Uzun G, &Ouml;zdemir &Ouml;, Uğur-İşeri S, et al<em>.</em> Investigation of SLC2A1 gene variants in genetic generalized epilepsy patients with eyelid myoclonia. Epileptic Disord. 2018;20:396-400.</li>\r\n<li>Gurnett CA, Veile R, Zempel J, Blackburn L, Lovett M, Bowcock A. Disruption of sodium bicarbonate transporter SLC4A10 in a patient with complex partial epilepsy and mental retardation. <em>Arch Neurol.</em> 2008;65:550-553.</li>\r\n<li>Gardiner M. Genetics of Idıopathic Generalized Epilepsies. <em>Epilepsia.</em> 2005;46:15-20.</li>\r\n<li>Bebek N, Baykan B. Genetics aspects of epilepsies and recent advances in genetics of idiopathic epilepsies. <em>Journal of Neurological Sciences.</em> 2006;23:70-83.</li>\r\n<li>Weber YG, Biskup S, Helbig KL, Von Spiczak S, Lerche H. The role of genetic testing in epilepsy diagnosis and management. <em>Expert Rev Mol Diagn.</em> 2017;17:739-750.</li>\r\n<li>Fisher RS, Cross JH, French JA, et al<em>.</em> Operational classification of seizure types by the International League Against Epilepsy: Position Paper of the ILAE Commission for Classification and Terminology. The new classification of seizure types. <em>Epilepsia.</em> 2017;58:522-530.</li>\r\n<li>Berkovic SF, Mulley JC, Scheffer IE, Petrou S. Human epilepsies: interaction of genetic and acquired factors. <em>Trends Neurosci.</em> 2006;7:391-397.</li>\r\n</ol>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<!--\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t<a class=\"menu\" href=\"#summary_en\">Summary</a><a class=\"menu\" href=\"#introduction\">Introduction</a><a class=\"menu\" href=\"#methods\">Methods</a><a class=\"menu\" href=\"#results\">Results</a><a class=\"menu\" href=\"#discussion\">Discussion</a><a class=\"selected_menu\" href=\"#reference\">Reference</a>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\n\t\t\n\t\t\t\t\t\t\t\t\t\t\t        \t</div>\n\t\t\t</";
