window._articleFullText = window._articleFullText || {};
window._articleFullText[2492] = "<div class=\"col-lg-12 col-md-12 col-sm-12 col-xs-12\">\n\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t<div  style=\"width:100%; float:left;  padding-bottom:0px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:100%; float:left; min-height:45px; text-align:justify\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<a href=\"\" name=\"summary_en\"></a>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<!--<h2>Summary</h2>-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<h4>Abstract</h4>\r\n<p><strong>Background:</strong> Neurofibromatosis type 1 (NF1) is an autosomal dominant neurocutaneous disorder, characterized by multiple caf&eacute;-au-lait macules, axillary and inguinal freckling, tumors of the nervous system, and iris hamartomas. More than 3,100 different pathogenic variants have been reported in the <em>NF1</em> gene, including missense, nonsense, frameshift, in-frame, splicing, and large deletions.<br /><strong>Aims:</strong> To determine the <em>NF1</em> mutational spectrum in patients with NF1 from the Republic of North Macedonia.<br /><strong>Study Design:</strong> A cohort study.<br /><strong>Methods:</strong> Molecular analyses included reverse transcription and cDNA sequencing of the <em>NF1</em> gene and next-generation sequencing using the TruSight Cancer panel, along with the multiple ligation probe amplification method to detect single nucleotide variants and copy number variations. Direct DNA sequencing was also used for the family member analysis.<br /><strong>Results:</strong> Our 9-year study of patients suspected of having NF1 in the Republic of North Macedonia encompassed molecular characterization of 30 cases of the disease. We identified 28 unique pathogenic <em>NF1</em> variants (NM_001042492.3), of which ten were novel: c.208delA; c.341_364del; c.1480_1481delTT; c.2325+1G&gt;C; c.2495_2496dupAC; c.2533_2541del; c.4517delC; c.5844C&gt;G; c.6971delA; c.7605_7606delGAinsAT. In addition to the variant spectrum analysis, our research revealed two positive genotype-phenotype correlations. One between the clinical manifestation of cognitive impairment and gross deletions in the <em>NF1</em> gene, and the other between cognitive impairment and truncating variants located in the RAS-GAP functional domain.<br /><strong>Conclusion:</strong> This is the first study of NF1 patients in the Republic of North Macedonia, and it contributes ten novel variants to the global spectrum of pathogenic <em>NF1</em> variants. It also corroborates the crucial importance of <em>NF1</em> genetic testing for a prompt and precise diagnosis, particularly in younger patients.</p>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<!--\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t<a class=\"selected_menu\" href=\"#summary_en\">Summary</a><a class=\"menu\" href=\"#introduction\">Introduction</a><a class=\"menu\" href=\"#methods\">Methods</a><a class=\"menu\" href=\"#results\">Results</a><a class=\"menu\" href=\"#discussion\">Discussion</a><a class=\"menu\" href=\"#reference\">Reference</a>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t<div  style=\"width:100%; float:left;  padding-bottom:0px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:100%; float:left; min-height:45px; text-align:justify\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<a href=\"\" name=\"introduction\"></a>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<!--<h2>Introduction</h2>-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<hr size=\"1\" color=\"#f2f2f2\">\r\n<h3>INTRODUCTION</h3>\r\n<p>Neurofibromatosis type 1 (NF1), also known as Recklinghausen&rsquo;s disease, is an autosomal dominant neurocutaneous disorder with a prevalence of 1-5 in 10,000 people worldwide.<sup>1-3</sup> It is characterized by multiple caf&eacute;-au-lait macules (CALMs), axillary and inguinal freckling, tumors of the nervous system, and iris hamartomas (Lisch nodules). The most common NF1-associated tumors are neurofibromas, which are dermal or plexiform benign peripheral nerve sheath tumors. Although less common, optic glioma, macrocephaly, short stature, cognitive impairment, epilepsy, scoliosis, and certain malignancies can also be present in NF1 patients.<sup>4,5</sup></p>\r\n<p>The diagnosis of NF1 is based on the National Institute of Health (NIH) diagnostic criteria, defined at the Consensus Development Conference in 1988. An individual is considered to be NF1 positive&nbsp; if at least two of the following clinical features are met: six or more caf&eacute;-au-lait macules; axillary or inguinal region freckling; two or more neurofibromas of any type or one plexiform neurofibroma; optic glioma; two or more Lisch nodules; a distinctive osseous lesion, such as sphenoid dysplasia or thinning of a long bone cortex with or without pseudarthrosis, and a first-degree relative (parent, sibling, or offspring) with NF-1 diagnosed from these criteria.<sup>6</sup> In addition, the revised NF1 diagnostic criteria (reported in 2021) suggest that if only CALMs and freckling are present, at least one of the two pigmentary findings should be bilateral; sphenoid wing dysplasia is not a separate criterion in the case of an ipsilateral orbital plexiform neurofibroma; the presence of two or more choroidal abnormalities defined as bright, patchy nodules, and the presence of a heterozygous pathogenic <em>NF1</em> variant is also a diagnostic criterion.<sup>5</sup> Although NF1 is a monogenic disorder with complete penetrance after childhood, the phenotype of the disease is extremely variable and varies even within families carrying the same genetic variants.<sup>7,8</sup> The reasons for this phenotypic variability are poorly understood, but several contributing factors have been suggested, such as genetic modifiers, epigenetic alterations, or environmental causes.<sup>9</sup></p>\r\n<p>The <em>NF1</em> gene (17q11.2; NM_001042492.3; NP_001035957.1) is one of the longest protein-coding genes in the human genome. It encodes the 2,839 amino acid-long neurofibromin protein, which is produced in many cells, including fibroblasts, nerve cells, and cells surrounding nerve cells (oligodendrocytes and Schwann cells). This protein is a GTPase-activating protein (GAP) that converts the active RAS-GTP into its inactive RAS-GDP form. The RAS-GAP activity of the protein is managed by the GTPase-activating protein-related domain (GRD), which corresponds to amino acids 1,235 and 1,451 of the Nf1 protein. The active form of RAS binds and activates the kinases, which leads to the activation of mitogen-activated protein kinases (MAPK) and cell proliferation. Active neurofibromin downregulates the RAS/MAPK pathway and acts as a tumor suppressor in normal cells. Pathogenic variants of the <em>NF1</em> gene produce a non-functional protein that cannot regulate cell growth and division, and this, in turn, leads to the production of tumors along the nerves throughout the body (neurofibromas).<sup>4,5</sup> However, it remains unclear how variants in the <em>NF1</em> gene cause the other features of NF1, such as caf&eacute;-au-lait spots.<sup>10</sup></p>\r\n<p>Herein, we investigated a cohort of patients suspected of having NF1 from the Republic of North Macedonia. In addition to the variant spectrum analysis, we describe the patients&rsquo; phenotype and further investigate possible genotype-phenotype correlations.</p>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<!--\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t<a class=\"menu\" href=\"#summary_en\">Summary</a><a class=\"selected_menu\" href=\"#introduction\">Introduction</a><a class=\"menu\" href=\"#methods\">Methods</a><a class=\"menu\" href=\"#results\">Results</a><a class=\"menu\" href=\"#discussion\">Discussion</a><a class=\"menu\" href=\"#reference\">Reference</a>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t<div  style=\"width:100%; float:left;  padding-bottom:0px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:100%; float:left; min-height:45px; text-align:justify\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<a href=\"\" name=\"methods\"></a>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<!--<h2>Methods</h2>-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<h3>MATERIALS AND METHODS</h3>\r\n<p>In a period of 9 years (2013-2022), 48 patients suspected of having NF1 were referred to our laboratory for molecular analysis of the NF1 gene. Based on the referred clinical symptoms, 27 of the patients met the NIH diagnostic criteria for NF1 disease, while the remaining 21 had only isolated symptoms from the defined NIH criteria with some additional symptoms that were less specific for NF1 disease (<a href=\"javascript:openWin('uploads/grafik/table_BMJ_2492_0.jpg', 640, 480)\"><b>Table 1</b></a>). All patients underwent genetic testing for NF1. None of the patients&rsquo; parents had genetically confirmed NF1 or were previously diagnosed to have NF1 based on their clinical presentations. The patient&rsquo;s parents were also tested if a pathogenic variant was detected in a patient.</p>\r\n<p>DNA and RNA samples were extracted from peripheral blood from each patient and the available parent. DNA was isolated by Proteinase K/SDS digestion following standard phenol-chloroform extraction and ethanol precipitation. Total RNA was extracted from fresh peripheral blood collected in EDTA using TRIzol Reagent (ThermoFisher Scientific, Waltham, MA, USA) following the manufacturer&rsquo;s protocol. Analysis of the <em>NF1</em> gene was done using a couple of different methods depending on the period when the patient was referred to our laboratory.</p>\r\n<p>In the early stages of our study, we used reverse transcription and cDNA sequencing of the <em>NF1</em> gene to diagnose suspected NF1 cases, following a previously described protocol.<sup>11</sup> Therefore, the first patients and their family members were diagnosed using cDNA sequencing.</p>\r\n<p>As cDNA sequencing is a time and labor-consuming method, we implemented the next-generation sequencing (NGS) method in 2016 to analyze suspected NF1 cases. In addition, all previous unsolved NF1 cases were reanalyzed using NGS.</p>\r\n<p>The TruSight cancer sequencing panel was used for targeted NGS (Illumina, San Diego, CA, USA). This sequencing panel amplified 94 cancer-associated genes, including the <em>NF1</em> gene. Libraries were prepared according to the manufacturer&rsquo;s protocol and were sequenced on an Illumina MiSeq desktop sequencer using paired-end 150 bp sequencing reads. The raw sequence data were aligned against the reference genome as specified in the manifest file using MiSeq Reporter software (v2.6.3). The VCF output file was used for variant calling and filtering with Variant Studio software (v3.0 Illumina). The BAM output file was used as an input file in the Integrative Genome Viewer (IGV; https://www.broadinstitute.org/igv/ v2.4.5) to visualize the specific variant and coverage in that region. Coverage of the entire <em>NF1</em> gene in each patient was verified using the web app gene.iobio (https://gene.iobio.io/) and the BAM file as an input file in the application.</p>\r\n<p>Variants with a global frequency &lt; 1% according to the gnomAD database (https://gnomad.broadinstitute.org/) were thoroughly investigated, particularly truncating variants.<sup>12</sup> Their pathogenicity was evaluated based on the worldwide accepted guidelines of the American College of Medical Genetics and Genomics (ACMG), the Association for Molecular Pathology (AMP), and the Association for Clinical Genomic Science (ACGS).<sup>13-16</sup></p>\r\n<p>Despite the <em>NF1</em> gene analysis, patients who were analyzed using the TruSight Cancer panel were also screened for pathogenic, likely pathogenic, or &ldquo;hot&rdquo; VUS variants in the other cancer-associated genes covered by the panel.</p>\r\n<p>In addition to the sequencing methods, the multiplex ligation-dependent probe amplification (MLPA) method and the P081, P082, and P122 probe mixes were used to detect copy number variations, according to the manufacturer&rsquo;s protocol. Data analysis was performed using Coffalyser NET software (MRC-Amsterdam, The Netherlands).</p>\r\n<p>We used direct DNA sequencing or cDNA sequencing of a specific region in the <em>NF1</em> gene to analyze family members. The primers used for direct DNA sequencing were designed in-house using the Primer3Plus tool (www.bioinformatics.nl/cgi-bin/primer3plus/primer3plus.cgi) and their sequences are shown in <a href=\"https://balkanmedicaljournal.org/uploads/pdf/2022-12-28-supplement.pdf\">Supplementary Material Table S1</a>.</p>\r\n<p>We used a 2 x 2 contingency table, Fisher&rsquo;s exact one-tailed test, and an available online tool from GraphPad (www.graphpad.com/quickcalcs/contingency1/) to statistically analyze the small group of patients with a learning/intellectual disability.</p>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<!--\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t<a class=\"menu\" href=\"#summary_en\">Summary</a><a class=\"menu\" href=\"#introduction\">Introduction</a><a class=\"selected_menu\" href=\"#methods\">Methods</a><a class=\"menu\" href=\"#results\">Results</a><a class=\"menu\" href=\"#discussion\">Discussion</a><a class=\"menu\" href=\"#reference\">Reference</a>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t<div  style=\"width:100%; float:left;  padding-bottom:0px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:100%; float:left; min-height:45px; text-align:justify\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<a href=\"\" name=\"results\"></a>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<!--<h2>Results</h2>-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<h3>RESULTS</h3>\r\n<p>We detected 28 unique pathogenic variants among the 30 <em>NF1</em> pathogenic variant-positive patients. Six of the positive patients were some of the patients who did not meet the NIH diagnostic criteria based on their presenting symptoms. We diagnosed these patients as positive for NF1 disease based on detecting the pathogenic <em>NF1</em> variant. However, our analysis did not reveal a pathogenic <em>NF1</em> variant among these three patients that met the NIH diagnostic criteria for the disease (<a href=\"javascript:openWin('uploads/grafik/table_BMJ_2492_0.jpg', 640, 480)\"><b>Table 1</b></a>, <a href=\"javascript:openWin('uploads/grafik/table_BMJ_2492_1.jpg', 640, 480)\"><b>Table 2</b></a>). The analysis of the patients with available parents showed that 44% (11/25) of the detected variants were inherited from an affected parent, while 56% (14/25) of the variants had occurred de novo. No pathogenicity, likely pathogenic, or &ldquo;hot&rdquo; VUS variant was detected in the other cancer-associated genes covered by the TruSight Cancer panel.</p>\r\n<p>The unique <em>NF1</em> variants accounted for 11 non-sense variants, six frameshift variants, six splicing variants, one missense variant, two in-frame variants, and two gross deletions. The gross deletions were present as one multi-exon deletion of ~0.15 Mb, and one ~1.4 Mb deletion of the entire <em>NF1</em> gene and its surrounding genes. The splicing variants consisted of one missense and four intronic variants causing aberrations in the splicing and premature termination of the protein, and one intronic variant that caused an in-frame deletion of 58 amino acids in the protein. The 1.4 Mb deletion and the c.3916C&gt;T; p.(Arg1306Ter) variant were not unique; both were detected in two patients each. None of the detected <em>NF1</em> variants was present in the Macedonian population consisting of 500 clinical exomes, which served as the control group. The distribution of the variant types and the locations of the variants detected in our NF1 patients are given in <a href=\"javascript:openWin('uploads/grafik/figure_BMJ_2492_0.jpg', 640, 480)\"><b>Figures 1</b></a> and <a href=\"javascript:openWin('uploads/grafik/figure_BMJ_2492_1.jpg', 640, 480)\"><b>2</b></a>, while the effects of the variants on the protein are given in <a href=\"javascript:openWin('uploads/grafik/table_BMJ_2492_1.jpg', 640, 480)\"><b>Table 2</b></a>.</p>\r\n<p>Among all detected variants, 10 were novel: two nonsense variants (c.5844C&gt;G, c.7605_7606delGAinsAT), five frameshift variants (c.208delA, c.1480_1481delTT, c.2495_2496dupAC, c.4517delC, c.6971delA), and one splice variant (c.2325+1G&gt;C). All of these caused premature termination of protein synthesis; plus two in-frame deletions that caused deletions of eight and three amino acids, respectively (c.341_364del, c.2533_2541del). These variants were classified as pathogenic/likely pathogenic based on the ACMG/AMP guidelines for classifying genetic variants. The criteria used to classify the novel variants are given in <a href=\"https://balkanmedicaljournal.org/uploads/pdf/2022-12-28-supplement.pdf\">Supplementary Material Table S2</a>. Results from the conformational analysis (direct sequencing and MLPA) for the novel variants are given in <a href=\"https://balkanmedicaljournal.org/uploads/pdf/2022-12-28-supplement.pdf\">Supplementary Material Figure S1</a>.</p>\r\n<p>The most abundant symptoms among our patients were CALMs, axillary/inguinal freckling, and neurofibromas, whereas optic glioma, Lisch nodules, and bone lesions were less frequent. Different types of cognitive impairment, ranging from mild impairment characterized by memory loss, language problems, learning difficulties, or attention deficit, to severe intellectual disability, were also present in almost half of the patients.</p>\r\n<p>The genotype-phenotype correlations are schematically represented in <a href=\"javascript:openWin('uploads/grafik/figure_BMJ_2492_2.jpg', 640, 480)\"><b>Figure 3</b></a>. Learning difficulties and intellectual disabilities were more frequent in patients with gross deletions and patients with variants located in the RAS-GAP domain of the protein, compared with the remaining patients.</p>\r\n<p>Statistical analysis of the patients harboring a truncating variant in the RAS-GAP domain of the protein produced a significant one-tailed <em>P</em>-value of 0.0217 for the association with cognitive impairment, compared with patients harboring truncating variants outside the domain. Analysis of all of the patients with truncating variants located before and after the RAS-GAP domain did not indicate a correlation with cognitive impairment.</p>\r\n<p>The 21 patients that did not meet the NF1 diagnostic criteria, had fewer CALMs and neurofibromas, compared with the frequency of those symptoms in the group that met the NIH diagnostic criteria. Other symptoms in these patients included optic gliomas, scoliosis, hexadactyly, pes cavu, epilepsy, cognitive impairment, and other unspecific symptoms. The clinical findings of all patients referred to our laboratory are listed in <a href=\"javascript:openWin('uploads/grafik/table_BMJ_2492_0.jpg', 640, 480)\"><b>Table 1</b></a>.</p>\r\n<p>A reverse diagnostic analysis was performed after detecting a pathogenic variant in 11 of the available parents. All of the <em>NF1</em> pathogenic variant-positive parents had CALMs and most had neurofibromas. Headache, low vision, bone lesions, and epilepsy were also presenting symptoms among them. In addition, cases #13/M, #15/M, and #24/F also had learning difficulties (<a href=\"javascript:openWin('uploads/grafik/table_BMJ_2492_0.jpg', 640, 480)\"><b>Table 1</b></a>).</p>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<!--\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t<a class=\"menu\" href=\"#summary_en\">Summary</a><a class=\"menu\" href=\"#introduction\">Introduction</a><a class=\"menu\" href=\"#methods\">Methods</a><a class=\"selected_menu\" href=\"#results\">Results</a><a class=\"menu\" href=\"#discussion\">Discussion</a><a class=\"menu\" href=\"#reference\">Reference</a>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t<div  style=\"width:100%; float:left;  padding-bottom:0px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:100%; float:left; min-height:45px; text-align:justify\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<a href=\"\" name=\"discussion\"></a>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<!--<h2>Discussion</h2>-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<h3>DISCUSSION</h3>\r\n<p>The neurofibromin protein is a RAS GAP (RAS-GAP) that acts as a tumor suppressor by converting the active form of RAS (RAS-GTP) into the inactive form of RAS (RAS-GDP). The mutated neurofibromin protein results in an unchecked RAS signaling pathway, leading to uncontrolled cell growth and division.<sup>17</sup> More than 3,100 pathogenic variants in the <em>NF1</em> gene have been reported in the HGMD and ClinVar databases [https://www.ncbi.nlm.nih.gov/clinvar, http://www.hgmd.cf.ac.uk], and associated with NF1 disease. All types of pathogenic variants are prevalent in <em>NF1</em>, but the most abundantly reported are non-sense, frameshift, and splicing variants, all of which cause premature termination of the protein sequence and loss of protein function. In our cohort, we detected 23 truncating variants (12 non-sense, six frameshift, and five splicing variants) that cause premature termination of the protein, which represented 76.7% of all variants identified. We detected ten large deletions; variants that caused in-frame deletion of amino acids (two in-frame deletions and one splicing variant) had a prevalence of 10%, whereas classical missense variants were less frequent with a prevalence of 3.3% (<a href=\"javascript:openWin('uploads/grafik/figure_BMJ_2492_0.jpg', 640, 480)\"><b>Figure 1</b></a>, <a href=\"javascript:openWin('uploads/grafik/table_BMJ_2492_1.jpg', 640, 480)\"><b>Table 2</b></a>). One-half of the detected variants (56%) occurred as de novo events in the patients, which agreed with the usually reported frequencies.<sup>18</sup></p>\r\n<p>Ten of the detected pathogenic variants (33.3%) were novel and were not present in the <em>NF1</em> databases or the literature. Truncating variants were most common among them, with the presence of two nonsense variants, five frameshift variants, two in-frame deletions, and one splice variant (<a href=\"javascript:openWin('uploads/grafik/figure_BMJ_2492_1.jpg', 640, 480)\"><b>Figure 2</b></a>, <a href=\"javascript:openWin('uploads/grafik/table_BMJ_2492_1.jpg', 640, 480)\"><b>Table 2</b></a>). All of these novel variants were classified as pathogenic/likely pathogenic based on the ACMG, AMP, and ACGS guidelines (Table S2).<sup>13-16</sup></p>\r\n<p>Although the c.2325+1G&gt;C and c.5844C&gt;G; p.(Tyr1948Ter) variants were novel variants detected for the first time in our family cases (#10 and #19), alternative variants at the same coding positions have been described previously. The c.5844C&gt;A; p.(Tyr1948Ter) variant was described by Wu-Chou et al.<sup>19</sup> as a novel variant, but their study lacked clinical information on the patient. Patient #19 had CALMs, freckling, headache, and learning/intellectual disability, while her mother had CALMs, and neurofibromas. The c.2325+1G&gt;A variant was first described in a Slovak 18-year-old patient with CALMs, Lisch nodules, optic glioma, neurofibromas, premature skeletal development, hamartomas, and accelerated puberty.<sup>20</sup> Later, the variant was described in a 48-year-old woman with multiple neurofibromas, caf&eacute; au-lait spots, axillary and inguinal freckling, iris hamartomas, and an adrenal gland tumor (pheochromocytoma).<sup>21</sup> In addition, the same variant was described in the neurofibromas of a mosaic patient.<sup>22</sup> The c.2325+1G&gt;T variant was described in an Egyptian years old boy presenting with CALMs, freckling, and congenital cataracts, but no affected parents.<sup>23</sup> Compared to patients described in the literature, our patient #10, a carrier of the c.2325+1G&gt;C, had neurofibromas, caf&eacute; au-lait spots, bone lesions, intellectual disability, and occipital tumefaction, while his mother had neurofibromas, caf&eacute; au-lait spots, low vision, and frequent headaches.</p>\r\n<p>NF1 is a disorder with a diverse and variable phenotypic spectrum. It includes a mild phenotype in some patients and a severe phenotype in others. Symptom onset is usually in infancy or the neonatal period, with caf&eacute;-au-lait spots and neurofibromas present in more than 95% of the patients within the first year of life. The disease usually progresses with the onset of freckling, Lisch nodules, optic gliomas, or scoliosis present by the age of 8 years and full penetrance of the disease by the age of 20 years.<sup>18,24</sup> Although a diagnosis can be readily established by following the NIH diagnostic criteria, these criteria are less sensitive for younger patients. It is estimated that 30-50% of infants &lt; 1 year, without a family history, meet the NIH diagnostic criteria.<sup>25,26</sup> For example, six patients in our cohort (#28, #29, #30, #31, #32, and #33) did not meet the diagnostic criteria defined in 1988 by the NIH;<sup>6</sup> besides the presence of caf&eacute;-au-lait, they had learning difficulties/intellectual disability and other neurological symptoms (<a href=\"javascript:openWin('uploads/grafik/table_BMJ_2492_0.jpg', 640, 480)\"><b>Table 1</b></a>). These patients were diagnosed with NF1 based on a positive genetic test.</p>\r\n<p>Including genetic testing to establish an efficient diagnosis in patients, particularly when they are young, is important, as 28.6% (6/21) of patients who did not meet the NIH diagnostic criteria (based on the present symptoms) were positive for the pathogenic <em>NF1</em> variant.</p>\r\n<p>In contRASt, three of the patients from our cohort who met the NF1 diagnostic criteria (#25, #26, and #27) were negative for the pathogenic <em>NF1</em> gene variant (<a href=\"javascript:openWin('uploads/grafik/table_BMJ_2492_0.jpg', 640, 480)\"><b>Table 1</b></a>). They were negative for a pathogenic variant in other cancer-associated genes as well. We hypothesized that these patients may carry deep intronic variants or regulatory regional variants, which are out of the target range and were not detectable with the methods used. The low percentage of general mosaicism, or tissue-specific mosaicism (confined mosaicism), undetectable by our methods, was not excluded in these patients. In addition, they may have other underlying disorders that overlap with the NF1 disease spectrum. As the Legius syndrome phenotype overlaps with most of the clinical symptoms of NF1, these patients should be checked for pathogenic variants in the <em>SPRED1</em> gene. Furthermore, follow-up genetic tests for pathogenic variants that cause other RAS-opathies, such as MEN2B syndrome, Leopard, or Noonan syndrome, are recommended.<sup>18,26,27</sup></p>\r\n<p>The first of our patients were children, as our analysis was the first initiative for implementing and establishing NF1 testing in North Macedonia in collaboration with a children&rsquo;s hospital. Their parents were never registered as NF1 patients, or suspected of having the diagnosis. Parent #15/M had a medical history of a surgically removed meningioma and recurrent epileptic seizures; none of the other parents had any medical history with clinical issues associated with NF1. Our reverse approach (from the detected genetic defect toward the clinical diagnosis in the parents) showed that seven of them (#1/M, #6/M, #8/M, #10/M, #15/M, #30/F, and #19/M) were positive for the NIH diagnostic criteria, while the remaining four (#12/M, #13/M, #20/F, and #24/F) had isolated symptoms not specific to an NF1 diagnosis (<a href=\"javascript:openWin('uploads/grafik/table_BMJ_2492_0.jpg', 640, 480)\"><b>Table 1</b></a>). Given that the clinical data for the parents were limited, the correlation between members of the same family was not adequate. A detailed clinical examination and reevaluation of the phenotype are recommended for all parents positive for the <em>NF1</em> pathogenic variant.</p>\r\n<p>Many researchers have tried to establish a genotype-phenotype correlation in NF1 to predict the disease course. Unfortunately, as most of the mutations are unique to a single family, without any mutational hotspots, only a few genotypes have been related to specific phenotypes to date. For example, missense variants are negatively correlated with neurofibromas. The p.(Arg1241Ter) variant is correlated with the manifestation of structural brain lesions, and the p.(Tyr2285Ter) variant is correlated with Lisch nodules and endocrinological abnormalities.<sup>28</sup> Two of our patients carried the p.(Arg1241Ter) and p.(Tyr2285Ter) variants, respectively, but they did not present with any of the associated symptoms. Clinical reevaluation and investigation of endocrinological disorders and structural brain lesions are recommended.</p>\r\n<p>Frameshift variants and whole gene deletions have been associated with skeletal abnormalities, whereas intellectual disability is particularly correlated with large deletions in the <em>NF1</em> gene. A higher prevalence of severe phenotypes and earlier disease onset is generally associated with predominantly truncating variants, splicing variants, and large deletions.<sup>18,29-32</sup></p>\r\n<p>Three large deletions were present in our patients: 1.4 Mb (type 1) deletions in patient #31 and patient #32 and a deletion of 43 exons (including exons 9 and 51) of the <em>NF1</em> gene in patient #24. In addition to the CALMs and intellectual disability, patient #32 presented with ptosis, hexadactyly, and dysmorphism, while patient #24 presented with Lisch nodules, low vision, and epilepsy. Besides the very limited clinical presentation data of patient #31, intellectual disability was also reported. Thus, the association between cognitive impairment and large deletions was also present in our patients (<a href=\"javascript:openWin('uploads/grafik/figure_BMJ_2492_2.jpg', 640, 480)\"><b>Figure 3</b></a>). Moreover, cognitive impairment was frequent in the patients (#13, #13/M, #15, #15/M, and #29) with a pathogenic variant located between the RAS-GAP functional domain, which spans between amino acids 1,235 and 1,451 of the neurofibromin protein. Despite learning and/or intellectual disabilities, patient #29 and patient #13 presented with epilepsy, while patient #15 presented with dysmorphism. The mother of #13 (#13/M) had recurrent headache and learning difficulties, while the mother of #15 (#15/M) had a surgically removed meningioma at the age of 3 years, learning difficulties, and epilepsy. All of the <em>NF1</em> variants present in those patients were truncating and caused premature termination of the protein. These patients were analyzed using Fisher&rsquo;s exact test and showed a significant one-tailed <em>p</em>-value of 0.0217 for the association with cognitive impairment, compared with the patients harboring truncating variants outside the domain. The analysis of all patients with truncating variants before and after the RAS-GAP domain did not show any correlation with cognitive impairment.</p>\r\n<p>Klose et al.<sup>33</sup> reported in 1998 that even the missense variant p.(Arg1276Pro) in this domain can completely disable the GAP activity of the protein, which is critical for pathogenesis. Those patients presented with numerous symptoms, including multiple neurofibromas, schwannomas, and mild intellectual and motor retardation.<sup>33</sup> Genetically engineered mice lacking exon 31 (also known as exon 23a), which is part of the GAP domain, present with learning impairments without having a predisposition to tumors.<sup>34</sup> Experimental studies on mice and Drosophila have shown that increased RAS activity caused by inactivated GAP is closely related to impaired learning and cognition. In particular, the GRD of the <em>NF1</em> gene is necessary for long-term memory as shown by the expression of clinically relevant human <em>NF1</em> variants.<sup>35,36</sup> In contRASt, in a study with a large cohort of NF1 patients, RAS-GAP missense variants affecting the amino acids Arg1276 and Lys1444 were significantly associated with the Noonan-like phenotype (characterized by short stature, low set ears, hypertelorism, midface hypoplasia, webbed neck, pectus abnormality, and/or pulmonary stenosis), without any association with cognitive impairment.<sup>37</sup></p>\r\n<p>Although the RAS-GAP domain of the <em>NF1</em> gene is the most studied region of this gene, a clear correlation between this domain and a specific phenotype has not been established. The positive correlation between our patients with RAS-GAP domain variants and cognitive impairment may be a result of the location of the variant (RAS-GAP domain), the consequence of the variant (truncating), or a combination of both. This correlation should be further investigated with a larger number of samples.</p>\r\n<p>In conclusion, this is the first study of NF1 in the Republic of North Macedonia. We have identified and classified 10 novel pathogenic <em>NF1</em> variants and have shown an association between large deletions and truncating variants in the RAS-GAP domain with cognitive impairment. As the patients&rsquo; phenotype is often age-dependent, including <em>NF1</em> genetic testing in the NF1 diagnostics algorithm is of crucial significance for a prompt and precise diagnosis, particularly in young patients who usually do not meet the NIH clinical diagnostic criteria.</p>\r\n<p><strong>Ethics Committee Approval:</strong> The study was conducted according to the guidelines of the Declaration of Helsinki, and approved by the Ethics Committee of the Macedonian Academy of Science and Arts (03-2742/1, 29.11.2022, Skopje).</p>\r\n<p><strong>Informed Consent:</strong> Written informed consent was obtained from each patient.</p>\r\n<p><strong>Authorship Contributions:</strong> Concept- M.G.; Design- M.G.; Data Collection or Processing- M.G., G.B.; Analysis or Interpretation- M.G., G.B.; Literature Search- M.G.; Writing- D.P-K.; Critical Review- G.B., E.S-A., M.K., L.M.K., D.P-K.</p>\r\n<p><strong>Conflict of Interest:</strong> No conflict of interest was declared by the authors.</p>\r\n<p><strong>Funding:</strong> The authors declared that this study received no financial support.<strong>&nbsp;</strong></p>\r\n<p><strong>Supplementary:</strong> <a href=\"https://balkanmedicaljournal.org/uploads/pdf/2022-12-28-supplement.pdf\">http://balkanmedicaljournal.org/uploads/pdf/2022-12-28-supplement.pdf</a></p>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<!--\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t<a class=\"menu\" href=\"#summary_en\">Summary</a><a class=\"menu\" href=\"#introduction\">Introduction</a><a class=\"menu\" href=\"#methods\">Methods</a><a class=\"menu\" href=\"#results\">Results</a><a class=\"selected_menu\" href=\"#discussion\">Discussion</a><a class=\"menu\" href=\"#reference\">Reference</a>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t<div  style=\"width:100%; float:left;  padding-bottom:0px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:100%; float:left; min-height:45px; text-align:justify\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<a href=\"\" name=\"reference\"></a>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<!--<h2>Reference</h2>-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<h3>REFERENCES</h3>\r\n<ol>\r\n<li>Lammert M, Friedman JM, Kluwe L, Mautner VF. Prevalence of neurofibromatosis 1 in German children at elementary school enrollment. <em>Arch Dermatol.</em> 2005;141:71-74.</li>\r\n<li>Evans DG, Howard E, Giblin C, et al. Birth incidence and prevalence of tumor-prone syndromes: estimates from a UK family genetic register service. <em>Am J Med Genet A.</em> 2010;152A:327-332.</li>\r\n<li>Kallionp&auml;&auml; RA, Uusitalo E, Lepp&auml;virta J, P&ouml;yh&ouml;nen M, Peltonen S, Peltonen J. Prevalence of neurofibromatosis type 1 in the Finnish population. <em>Genet Med.</em> 2018;20:1082-1086.</li>\r\n<li>Pasmant E, Parfait B, Luscan A, et al. Neurofibromatosis type 1 molecular diagnosis: what can NGS do for you when you have a large gene with loss of function mutations? <em>Eur J Hum Genet.</em> 2015;23:596-601.</li>\r\n<li>Legius E, Messiaen L, Wolkenstein P, et al. Revised diagnostic criteria for neurofibromatosis type 1 and Legius syndrome: an international consensus recommendation. <em>Genet Med.</em> 2021;23:1506-1513.</li>\r\n<li>Neurofibromatosis. Conference statement. National Institutes of Health Consensus Development Conference. <em>Arch Neurol.</em> 1988;45:575-578.</li>\r\n<li>Ferner RE, Gutmann DH. Neurofibromatosis type 1 (NF1): diagnosis and management. <em>Handb Clin Neurol.</em> 2013;115:939-955.</li>\r\n<li>Dunning-Davies BM, Parker AP. Annual review of children with neurofibromatosis type 1. <em>Arch Dis Child Educ Pract Ed.</em> 2016;101:102-111.</li>\r\n<li>Szudek J, Joe H, Friedman JM. Analysis of intrafamilial phenotypic variation in neurofibromatosis 1 (NF1). <em>Genet Epidemiol.</em> 2002;23:150-164.</li>\r\n<li>Carroll SL, Stonecypher MS. Tumor suppressor mutations and growth factor signaling in the pathogenesis of NF1-associated peripheral nerve sheath tumors: II. The role of dysregulated growth factor signaling. <em>J Neuropathol Exp Neurol.</em> 2005;64:1-9.</li>\r\n<li>Valero MC, Martin Y, Hernandez-Imaz E, et al. A highly sensitive genetic protocol to detect NF1 mutations<em>. J Mol Diagn</em>. 2011;13:113-122.</li>\r\n<li>Han P, Wei G, Cai K, et al. Identification and functional characterization of mutations in LPL gene causing severe hypertriglyceridaemia and acute pancreatitis. <em>J Cell Mol Med.</em> 2020;24:1286-1299.</li>\r\n<li>Richards S, Aziz N, Bale S, et al. Standards and guidelines for the interpretation of sequence variants: a joint consensus recommendation of the American College of Medical Genetics and Genomics and the Association for Molecular Pathology. <em>Genet Med.</em> 2015;17:405-424.</li>\r\n<li>Abou Tayoun AN, Pesaran T, DiStefano MT, et al. Recommendations for interpreting the loss of function PVS1 ACMG/AMP variant criterion. <em>Hum Mutat</em>. 2018;39:1517-1524.</li>\r\n<li>Harrison SM, Biesecker LG, Rehm HL. Overview of Specifications to the ACMG/AMP Variant Interpretation Guidelines. <em>Curr Protoc Hum Genet.</em> 2019;103:e93.</li>\r\n<li>Sian Ellard ELB, Alison Callaway, Ian Berry, Natalie Forrester, Clare Turnbull, Martina Owens, Diana M Eccles, Stephen Abbs, Richard Scott, Zandra C Deans, Tracy Lester, Jo Campbell, William G Newman, Simon Ramsden and Dominic <em>J McMullan.</em> ACGS Best Practice Guidelines for Variant Classification in Rare Disease 2020 https://www.acgs.uk.com/media/11631/uk-practice-guidelines-for-variant-classification-v4-01-2020.pdf2020.</li>\r\n<li>Nix JS, Blakeley J, Rodriguez FJ. An update on the central nervous system manifestations of neurofibromatosis type 1. <em>Acta Neuropathol</em>. 2020;139:625-641.</li>\r\n<li>Friedman JM. Neurofibromatosis 1. In: Adam MP, Everman DB, Mirzaa GM, et al., eds. GeneReviews((R)). Seattle (WA)1993.</li>\r\n<li>Wu-Chou YH, Hung TC, Lin YT, et al. Genetic diagnosis of neurofibromatosis type 1: targeted next- generation sequencing with Multiple Ligation-Dependent Probe Amplification analysis<em>. J Biomed Sci.</em> 2018;25:72.</li>\r\n<li>Nemethova M, Bolcekova A, Ilencikova D, et al. Thirty-nine novel neurofibromatosis 1 (NF1) gene mutations identified in Slovak patients. <em>Ann Hum Genet.</em> 2013;77:364-379.</li>\r\n<li>Wan W, Nguyen B, Graybill S, Kim J. Clonidine suppression testing for pheochromocytoma in neurofibromatosis type 1. <em>BMJ Case Rep</em>. 2019;12:e228263.</li>\r\n<li>Maertens O, De Schepper S, Vandesompele J, et al. Molecular dissection of isolated disease features in mosaic neurofibromatosis type 1. <em>Am J Hum Genet.</em> 2007;81:243-251.</li>\r\n<li>Abdel-Aziz N, Y El-Kamah G, A Khairat R, et al. Mutational spectrum of NF1 gene in 24 unrelated Egyptian families with neurofibromatosis type 1. <em>Mol Genet Genomic Med.</em> 2021;9:e1631.</li>\r\n<li>Banerjee S, Lei D, Liang S, et al. Novel phenotypes of NF1 patients from unrelated Chinese families with tibial pseudarthrosis and anemia<em>. Oncotarget</em>. 2017;8:39695-39702.</li>\r\n<li>Kang E, Yoon HM, Lee BH. Neurofibromatosis type I: points to be considered by general pediatricians. <em>Clin Exp Pediatr.</em> 2021;64:149-156.</li>\r\n<li>Pinti E, Nemeth K, Staub K, Lengyel A, Fekete G, Haltrich I. Diagnostic difficulties and possibilities of NF1-like syndromes in childhood.<em> BMC Pediatr.</em> 2021;21:331.</li>\r\n<li>Kehrer-Sawatzki H, Cooper DN. Challenges in the diagnosis of neurofibromatosis type 1 (NF1) in young children facilitated by means of revised diagnostic criteria including genetic testing for pathogenic NF1 gene variants. <em>Hum Genet.</em> 2022;141:177-191.</li>\r\n<li>Scala M, Schiavetti I, Madia F, et al. Genotype-Phenotype Correlations in Neurofibromatosis Type 1: A Single-Center Cohort Study. <em>Cancers (Basel).</em> 2021;13:1879.</li>\r\n<li>Kang E, Kim YM, Seo GH, et al. Phenotype categorization of neurofibromatosis type I and correlation to NF1 mutation types. J Hum Genet. 2020;65:79-89.</li>\r\n<li>B&uuml;ki G, Zsigmond A, Czak&oacute; M, et al. Genotype-Phenotype Associations in Patients With Type-1, Type-2, and Atypical NF1 Microdeletions. <em>Front Genet.</em> 2021;12:673025.</li>\r\n<li>Geoffray MM, Robinson L, Ramamurthy K, et al. Predictors of cognitive, behavioural and academic difficulties in NF1. <em>J Psychiatr Res.</em> 2021;140:545-550.</li>\r\n<li>Kehrer-Sawatzki H, Cooper DN. Classification of NF1 microdeletions and its importance for establishing genotype/phenotype correlations in patients with NF1 microdeletions. <em>Hum Genet.</em> 2021;140:1635-1649.</li>\r\n<li>Klose A, Ahmadian MR, Schuelke M, et al. Selective disactivation of neurofibromin GAP activity in neurofibromatosis type 1. <em>Hum Mol Genet.</em> 1998;7:1261-1268.</li>\r\n<li>Costa RM, Yang T, Huynh DP, et al. Learning deficits, but normal development and tumor predisposition, in mice lacking exon 23a of Nf1. <em>Nat Genet.</em> 2001;27:399-405.</li>\r\n<li>Ho IS, Hannan F, Guo HF, Hakker I, Zhong Y. Distinct functional domains of neurofibromatosis type 1 regulate immediate versus long-term memory formation. <em>J Neurosci.</em> 2007;27:6852-6857.</li>\r\n<li>Anastasaki C, Orozco P, Gutmann DH. RAS and beyond: the many faces of the neurofibromatosis type 1 protein. <em>Dis Model Mech</em>. 2022;15:dmm049362.</li>\r\n<li>Koczkowska M, Callens T, Chen Y, et al. Clinical spectrum of individuals with pathogenic NF1 missense variants affecting p.Met1149, p.Arg1276, and p.Lys1423: genotype-phenotype study in neurofibromatosis type 1. <em>Hum Mutat</em>. 2020;41:299-315.</li>\r\n</ol>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<!--\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t<a class=\"menu\" href=\"#summary_en\">Summary</a><a class=\"menu\" href=\"#introduction\">Introduction</a><a class=\"menu\" href=\"#methods\">Methods</a><a class=\"menu\" href=\"#results\">Results</a><a class=\"menu\" href=\"#discussion\">Discussion</a><a class=\"selected_menu\" href=\"#reference\">Reference</a>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\n\t\t\n\t\t\t\t\t\t\t\t\t\t\t        \t</div>";
