<div class="col-lg-12 col-md-12 col-sm-12 col-xs-12">
																		        				
												        				<div  style="width:100%; float:left;  padding-bottom:0px;">
												        					<div	style="width:100%; float:left; min-height:45px; text-align:justify">
												        						<a href="" name="summary_en"></a>
												        						<!--<h2>Summary</h2>-->
												        						<h4>Abstract</h4>
<p><strong>Background:</strong> The fetal monogenic causes of early pregnancy losses (EPLs) are mainly unknown, with only a few articles on the subject published. In our previous study of EPLs using whole-exome sequencing analysis, we confirmed a genetic diagnosis of <em>CPLANE1</em>-related Joubert syndrome (JS) in three EPLs from two couples and identified a relatively common <em>CPLANE1 </em>allele among our population (NM_001384732.1:c.1819delT;c.7817T&gt;A, further after referred as &ldquo;complex allele&rdquo;). Pathogenic variants in the<em> CPLANE1 (C5orf42)</em> gene are reported to cause JS type 17, a primary ciliopathy with various system defects.<br /><strong>Aims:</strong> To examine the hypothesis that the <em>CPLANE1</em> &ldquo;complex allele,&rdquo; whether homozygous or compound heterozygous, is a common cause of EPLs in our population.<br /><strong>Study Design:</strong> Cohort study/case-control study.<br /><strong>Methods:</strong> In this study, we used polymerase chain reaction-based methods to screen for <em>CPLANE1 </em>&ldquo;complex allele&rdquo; presence among 246 euploid EPLs (&lt; 12 gestational weeks) from families in North Macedonia. We also investigated the impact of this allele in 650 women with EPLs versus 646 women with no history of pregnancy loss and at least one livebirth, matched by ethnic origin.<br /><strong>Results:</strong> We found a high incidence of JS in the total study group of EPLs (2.03%), with a considerably higher incidence among Albanian families (6.25%). Although not statistically significant, women with EPLs had a higher allele frequency of the <em>CPLANE1 </em>&ldquo;complex allele&rdquo; (AF&nbsp;=&nbsp;1.38%) than the controls (AF&nbsp;=&nbsp;0.85%; <em>p</em>&nbsp;=&nbsp;0.2). Albanian women had significantly higher frequency of the &ldquo;complex allele&rdquo; than the Macedonians (AF&nbsp;=&nbsp;1.65% and 0.39%, respectively; <em>p</em>&nbsp;=&nbsp;0.003).<br /><strong>Conclusion:</strong> To the best of our knowledge, this is the highest reported incidence of fetal monogenic disease that might cause EPLs. Targeted screening for the <em>CPLANE1 </em>&ldquo;complex allele&rdquo; would be warranted in Albanian ethnic couples because it would detect one JS in every 16 euploid EPLs. Our findings have a larger impact on the pathogenesis of pregnancy loss and contribute to a better understanding of the pathogenicity of the variants in the <em>CPLANE1</em> gene.</p>												        					</div>
												        					<!--
												        					<div	style="width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;">
												        									<a class="selected_menu" href="#summary_en">Summary</a><a class="menu" href="#introduction">Introduction</a><a class="menu" href="#methods">Methods</a><a class="menu" href="#results">Results</a><a class="menu" href="#discussion">Discussion</a><a class="menu" href="#reference">Reference</a>												        					</div>
												        					-->
												        				</div>
												        				
												        				
												        																        				
												        				<div  style="width:100%; float:left;  padding-bottom:0px;">
												        					<div	style="width:100%; float:left; min-height:45px; text-align:justify">
												        						<a href="" name="introduction"></a>
												        						<!--<h2>Introduction</h2>-->
												        						<hr size="1" color="#f2f2f2">
<h3>INTRODUCTION</h3>
<p>Early pregnancy loss (EPL) is a common medical condition during pregnancy caused by the spontaneous demise of conception, affecting up to 20% of women in the reproductive period, with recurring events occurring in only a small proportion of cases.<sup>1,2</sup> EPLs have a wide range of causes, both maternal and fetal. Up to 50% of the products of conception (POCs) of patients with EPLs have chromosomal imbalances. However, the remaining half of EPL cases are expected to remain primarily unknown.<sup>3,4</sup> Recent studies using next-generation sequencing technology provide some insights into the role of fetal monogenic causes in EPLs. These technologies allow for the identification of underlying genetic factors that may lead to pregnancy failure and provide a better understanding of the pathogenesis of this condition. To date, only a few studies on the fetal monogenic causes of pregnancy losses have been published, and only a few of these studies have focused on POCs from <br />EPLs.<sup>5-9</sup> Most of the recurrent genes associated with fetal death, such as <em>CHD7, FBN1, FGFR3, NIPBL</em>, and <em>SOS1</em>, have been associated with multisystem disorders. Other recurring genes have been associated with cardiac anomalies or arrhythmias (<em>CSRP3, GATA4, GPD1L, </em>and<em> SCN5A </em>genes), skeletal dysplasia (<em>COL1A1, FGFR2</em>, and <em>FGFR3</em>), kidney diseases (<em>GREB1L </em>and <em>NPHS1</em>), and central nervous system abnormality (<em>PIK3R2</em>). Multiple disease categories and recurring genes associated with fetal death have been identified, indicating that EPL might be caused by various etiologies.</p>
<p>Joubert syndrome [(JS); OMIM#614615] is caused by pathogenic variants in the cytogenesis and planar polarity effector 1, also known as chromosome 5 open reading frame 42 (<em>C5orf42</em>, OMIM#614571) gene. JS is an autosomal recessive primary ciliopathy with multiple system defects, the most notable of which are abnormally deep interpeduncular fossa, elongated, thick, and mal-oriented superior cerebellar peduncles, and absent or hypoplastic cerebellar vermis, which results in the molar tooth sign on magnetic resonance imaging. JS has been found in several fetuses with multiple anomalies, resulting in the termination of pregnancies in the second or third trimester.<sup>10-14</sup> However, there have been no reports of JS in EPLs.</p>
<p>Previously, in a pilot study conducted in our laboratory, we confirmed a genetic diagnosis of JS in three EPLs from two couples of Albanian ethnic origin (unpublished data). Two EPLs were found to be compound heterozygous for a <em>CPLANE1</em> (NM_001384732.1) allele c.1819delT;c.7817T&gt;A, further referred to as the &ldquo;complex allele,&rdquo; and a novel splice site variant c.5820+3_5820+6del, whereas the other EPL was homozygous for the &ldquo;complex allele.&rdquo; The &ldquo;complex allele&rdquo; consists of two known <em>CPLANE1 </em>pathogenic variants, c.1819delT and c.7817T&gt;A (ClinVar IDs 217590 and 217575, respectively), which are located on the same chromosome. Furthermore, given the high prevalence of carriers of this <em>CPLANE1 </em>&ldquo;complex allele&rdquo; in our internal whole-exome sequencing (WES) database and the absence of any known homozygous patients, we hypothesized that it might be a common cause of EPLs in our population, whether homozygous or compound heterozygous.</p>												        					</div>
												        					<!--
												        					<div	style="width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;">
												        									<a class="menu" href="#summary_en">Summary</a><a class="selected_menu" href="#introduction">Introduction</a><a class="menu" href="#methods">Methods</a><a class="menu" href="#results">Results</a><a class="menu" href="#discussion">Discussion</a><a class="menu" href="#reference">Reference</a>												        					</div>
												        					-->
												        				</div>
												        				
												        				
												        																        				
												        				<div  style="width:100%; float:left;  padding-bottom:0px;">
												        					<div	style="width:100%; float:left; min-height:45px; text-align:justify">
												        						<a href="" name="methods"></a>
												        						<!--<h2>Methods</h2>-->
												        						<h3>MATERIALS AND METHODS</h3>
<p><strong><em>Materials</em></strong></p>
<p><strong><em>Products of conception</em></strong></p>
<p>Screening for the <em>CPLANE1 </em>&ldquo;complex allele&rdquo; involved 246 euploid POCs from EPLs (&lt; 12 gestational age), including 121 (49.2%) males and 125 (50.8%) females. Individuals&rsquo; demographic characteristics included families of Macedonian (n&nbsp;=&nbsp;155), Albanian (n&nbsp;=&nbsp;80), and other ethnic origin (n&nbsp;=&nbsp;11). The POC samples, previously selected by a gynecologist and/or pathologist and accompanied by parental whole blood samples, were referred to our laboratory for primary chromosomal abnormality analysis. Consequently, euploid POC materials were selected for further research. Positive samples for the &lsquo;complex allele&rsquo; were further analyzed by WES.</p>
<p><strong><em>Women with EPLs and control women</em></strong></p>
<p>A group of 650 women with EPLs, with Macedonian (n&nbsp;=&nbsp;260), Albanian (n&nbsp;=&nbsp;364), and other ethnic origin (n&nbsp;=&nbsp;26), as well as 646 controls (women without EPL and with livebirth) of Macedonian (n&nbsp;=&nbsp;255), Albanian (n&nbsp;=&nbsp;364), and other ethnic origin (n&nbsp;=&nbsp;27), were screened for the <em>CPLANE1 </em>&ldquo;complex allele&rdquo; c.1819delT;c.7817T&gt;A. The mothers of the 246 screened POCs were among the 650 women with EPLs. The study also included 11 control fetuses and five partners of carrier women.</p>
<p>All participants in the study provided informed consent. The Macedonian Academy of Sciences and Arts Ethical Committee approved this study (approval number: 09-1047/6, date: 04.05.2016).</p>
<h3>METHODS</h3>
<p><strong><em>Histopathological analyses</em></strong></p>
<p>Initially, all submitted POCs were examined macroscopically. Fresh placental tissue from each POC was collected for molecular analysis and stored in liquid nitrogen. In addition, representative placental and decidua tissue samples were selected and preserved for 24&nbsp;h in 10% neutral buffered formalin. The samples were then processed in a tissue processor according to the standard tissue processing procedure. The paraffin tissue blocks were cut into 4 &mu; thin sections, which were then deparaffinized, dehydrated, and stained with hematoxylin and eosin. Microscopic analysis of the placental tissue was performed using the criteria outlined below. Microscopic analysis was performed using a standard light microscope (Leica DM2500) with 4, 10, 20, 40, and 100 times objectives. The following placental tissue characteristics were examined: villous contours, villous stroma appearance (mucoid or hydropic change, cavitation), fetal erythrocyte presence, trophoblastic hyperplasia, and trophoblastic stromal inclusions.<sup>15</sup></p>
<p><strong><em>Nucleic acid extraction and chromosomal abnormality exclusion</em></strong></p>
<p>DNA/RNA extraction from fetal/chorionic villus tissues and peripheral blood from women with EPLs, their partners, control women, and control fetuses from carrier women was performed using the magnetic bead automated nucleic acid extraction instrument Mag Core Super (RBC Bioscience).</p>
<p>To rule out chromosomal abnormalities and maternal DNA contamination, a quantitative fluorescence-polymerase chain reaction (QF-PCR) method using Short Tandem Repeats markers on chromosomes 13, 18, and 21 and sex chromosomes and subtelomere Multiplex Ligation Probe dependent Amplification (MLPA) (MRC-Holland) was used.<sup>16</sup></p>
<p><strong><em>Allele-specific PCR (AS-PCR) and allele discrimination methods</em></strong></p>
<p>Because the variants c.1819delT and c.7817T&gt;A in the <em>CPLANE1</em> gene were always coinherited on the same chromosome and were found at a high frequency among our population, AS-PCR and allele discrimination assays were performed for screening only c.7817T&gt;A (<a href="javascript:openWin('uploads/grafik/figure_BMJ_2581_0.jpg', 640, 480)"><b>Figure 1 a-c</b></a>), as these methods did not work well for the c.1819delT variant. However, Sanger sequencing revealed the presence of the c.1819delT variant in all positive samples for c.7817T&gt;A. Direct DNA sequencing of exons 12 and 40 of the <em>CPLANE1</em> gene, which detected c.1819delT and c.7817T&gt;A, is shown in <a href="javascript:openWin('uploads/grafik/figure_BMJ_2581_0.jpg', 640, 480)"><b>Figure 1d, e</b></a>.</p>
<p>The primers for screening the <em>CPLANE1</em> c.7817T&gt;A variant were designed using Primer 3 software.<sup>17</sup> The following primer sequences were used for AS-PCR: <em>CPLANE1</em>_ex40_F: TGGGTTTGTAGGAGGAGAGGT; <em>CPLANE1</em>_ex40_R: CATACTTCCTGCTCCTTTTCCT; and modified allele-specific primer for the <em>CPLANE1</em> c.7817T&gt;A variant: GAAGGCTCTTCTCTCACAGGTT. Briefly, PCR reactions were performed in a total volume of 25 &mu;l containing 1&nbsp;&times;&nbsp;Rb Buffer II, 1.3 mM MgCl<sub>2</sub>, 2 mM dNTPs, 0.3 &micro;M of the forward primer, 0.6 &micro;M of the reverse primer, 0.6 &micro;M of the AS primer, and 0.75 unit of AmpliTaq Gold DNA polymerase (Thermo Fisher Scientific). Each reaction began with 10&nbsp;min of denaturation at 95 &deg;C, followed by 35 cycles of 30&nbsp;s at 95 &deg;C, 30&nbsp;s at 60 &deg;C, 45&nbsp;s at 72 &deg;C, and 10&nbsp;min at 72 &deg;C. Amplified PCR products were separated and observed using 1.5% agarose electrophoresis. The presence (homozygous/heterozygous) of the <em>CPLANE1</em> c.7817T&gt;A variant resulted in two bands with sizes of 471 and 251&nbsp;bp, whereas the absence of the variant resulted in only a 471-bp fragment. Following the manufacturer&rsquo;s protocol, an allele discrimination assay identifying the <em>CPLANE1</em> c.7817T&gt;A variant using a custom-made design from Thermo Fisher Scientific for real-time PCR. The mutant allele was FAM-labeled, and the normal allele was VIC (HEX)-labeled. Each reaction consisted of a 15 min denaturation at 95 &deg;C, followed by 40 cycles of 20 s denaturation at 95 &deg;C and 1&nbsp;min amplification at 60 &deg;C.</p>
<p><strong><em>Sanger sequencing</em></strong></p>
<p>To confirm and establish the zygosity of the detected variants and to determine whether the c.1819delT variant was present on the same haplotype with c.7817T&gt;A, PCR amplification of exons 12 and 40 of the <em>CPLANE1</em> gene was performed using the following designed primers for exon 12: <em>CPLANE1</em>_ex12_F: TGAAAAATGAGTTCCAAGACCA and <em>CPLANE1</em>_ex12_R: AGGCAGACACTTTCCAGACAA, and the primers mentioned above for exon 40. Sanger sequencing was performed using the same primers and a Big Dye 1.1 Cycle Sequencing kit (Life Technologies, Thermo Fisher Scientific). Fragments were isolated and detected using an AB 3500 Genetic Analyzer (Life Technologies, Thermo Fisher Scientific).</p>
<p><strong><em>Whole-exome sequencing</em></strong></p>
<p>WES was performed using the TWIST exome library preparation protocol (Twist Bioscience) and sequencing on the Illumina NovaSeq platform (Illumina). Variants in the <em>CPLANE1</em> gene were examined using an in-house bioinformatics pipeline.</p>
<p><strong><em>RNA analyses</em></strong></p>
<p>To investigate the effect of the novel splicing mutation on mRNA, expression RNA analysis was performed using a Luna Universal One-Step RT-qPCR Kit (New England Biolabs) and custom-designed primers: Forward_RNA: TTGCAGTAGCAACTCCAGGT and Reverse_RNA: GGGTCTTCTGAGGTGTTGGA. The PCR products were then visualized by 1.5% agarose gel electrophoresis and Sanger sequencing using the same primers used for amplification was performed.</p>												        					</div>
												        					<!--
												        					<div	style="width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;">
												        									<a class="menu" href="#summary_en">Summary</a><a class="menu" href="#introduction">Introduction</a><a class="selected_menu" href="#methods">Methods</a><a class="menu" href="#results">Results</a><a class="menu" href="#discussion">Discussion</a><a class="menu" href="#reference">Reference</a>												        					</div>
												        					-->
												        				</div>
												        				
												        				
												        																        				
												        				<div  style="width:100%; float:left;  padding-bottom:0px;">
												        					<div	style="width:100%; float:left; min-height:45px; text-align:justify">
												        						<a href="" name="results"></a>
												        						<!--<h2>Results</h2>-->
												        						<h3>RESULTS</h3>
<p><strong><em>JS in the POC material</em></strong></p>
<p>Maternal DNA contamination and chromosomal aneuploidies were excluded from all studied POCs using QF-PCR and MLPA analyses. Therefore, our results revealed a high incidence of JS in the total study group of POCs (2.03%; 5/246), and the incidence was considerably higher among Albanian families (6.25%; 5/80) because the genetic diagnosis of JS was only confirmed in POCs from Albanian ethnic origin families (<a href="javascript:openWin('uploads/grafik/table_BMJ_2581_0.jpg', 640, 480)"><b>Table 1a</b></a>).</p>
<p>Among the 246 studied EPLs, we identified three POCs (from three different families) that were homozygous for the pathogenic <em>CPLANE1</em> &ldquo;complex allele.&rdquo; The results were confirmed by WES analysis, and Sanger sequencing analysis confirmed that the parents were heterozygous carriers of this &ldquo;complex allele.&rdquo; Additionally, another two POCs (from the same family) were found to be compound heterozygous for the &ldquo;complex allele&rdquo; and a second novel splice site variant c.5820+3_5820+6del in <em>CPLANE1</em>.</p>
<p>The splice site variant is a novel, non-classical splicing variant found in intron 29 of <em>CPLANE1</em>. In-silico analysis using the VarSeak software predicted that this variant affects splicing and results in exon skipping at the RNA level. To confirm this finding, a cDNA sequencing analysis was performed, which revealed a shortened mRNA caused by <em>CPLANE1</em> exon 29 skipping (<a href="javascript:openWin('uploads/grafik/figure_BMJ_2581_1.jpg', 640, 480)"><b>Figure 2</b></a>). The splicing variant causes an amino acid change from glutamic acid to glycine at position 1913, followed by a frameshift that creates a termination codon after 23 amino acids (p. Glu1913GlyfsTer23), resulting in a shortened protein. Furthermore, this variant was missing from the gnomAD database, and the nucleotides were highly conserved across species. The ACMG variant classification system identified this variant as pathogenic based on the following criteria: PS3, PM3, PM2, and PP3. Sanger sequencing of the identified variants on WES confirmed the POC findings and indicated that the mother had the &ldquo;complex allele,&rdquo; whereas the father carried the splicing variant in the <em>CPLANE1</em> gene. Another POC sample from the same pair showed compound heterozygosity for the same variants.</p>
<p>All families with a JS genetic diagnosis in their POCs had at least two EPLs and no previous live birth (<a href="javascript:openWin('uploads/grafik/figure_BMJ_2581_2.jpg', 640, 480)"><b>Figure 3a-d</b></a>). The histopathological analysis of the placental tissue revealed degenerated chorionic villi with localized hydropic swelling or hyalinization of the villous stroma. Fetal erythrocytes were found in villous blood vessels. The decidua tissue showed regions of necrosis, consistent with spontaneous miscarriage, but no distinct specific histopathologic characteristics.</p>
<p>Three additional POC samples were found to be heterozygous for the &ldquo;complex allele,&rdquo; and subsequent WES analysis revealed no second pathogenic variant in <em>CPLANE1</em>. These POCs originated from two families (<a href="javascript:openWin('uploads/grafik/figure_BMJ_2581_2.jpg', 640, 480)"><b>Figure 3e, f</b></a>). One was of Macedonian origin, and the other was of Albanian origin.</p>
<p><strong><em>Women with EPL</em></strong></p>
<p>We found that women with EPLs had a higher allele frequency of the &ldquo;complex allele&rdquo; (18/1300, AF&nbsp;=&nbsp;1.38%) than controls (11/1292, AF&nbsp;=&nbsp;0.85%), but this difference was not statistically significant (<em>p </em>=&nbsp;0.2), most likely due to the small sample sizes (<a href="javascript:openWin('uploads/grafik/table_BMJ_2581_0.jpg', 640, 480)"><b>Table 1b, c</b></a>). In general, the &ldquo;complex allele&rdquo; was significantly more common in the Albanian population (24/1456, AF&nbsp;=&nbsp;1.65%,<em> p</em>&nbsp;=&nbsp;0.003) than in the Macedonian population (4/1030, AF&nbsp;=&nbsp;0.39%) based on the total number of tested subjects of Macedonian and Albanian ethnic origins, including both women with EPLs and control women.</p>
<p>Additional analysis in one of the five partners of the female carriers revealed heterozygosity of the &ldquo;complex allele.&rdquo; The couple was of Albanian ancestry and had 10 EPLs, with the untested conceptions being eliminated between the sixth and seventh weeks of gestation (<a href="javascript:openWin('uploads/grafik/figure_BMJ_2581_2.jpg', 640, 480)"><b>Figure 3g</b></a>). The karyotypes of both partners were normal, and there were no physiological or anatomical anomalies in their reproductive systems. We assume that at least some of their POCs were affected by JS because both partners in this couple carry the &ldquo;complex allele.&rdquo;</p>
<p>Eleven fetuses from the control group of women confirmed to carry the &ldquo;complex allele&rdquo; were available for analysis. Seven of them carried the &ldquo;complex allele.&rdquo; We contacted all 11 families and found that 10 newborns were born healthy, and one baby had symptoms consistent with JS (9.1%; 1/11). A genetic study revealed compound heterozygosity of the &ldquo;complex allele&rdquo; and the c.8263dupA variant in <em>CPLANE1</em>. This family reported that the baby was born with clinical symptoms of JS and revealed that they previously had a child with JS symptoms and another phenotypically normal child. We did not have access to the detailed phenotypes of the JS-affected children of this family (<a href="javascript:openWin('uploads/grafik/figure_BMJ_2581_2.jpg', 640, 480)"><b>Figure 3h</b></a>). Previously, the <em>CPLANE1</em>c.1819delT;c.7817T&gt;A];[8263dupA] genotype was found in our laboratory among three patients with JS.</p>												        					</div>
												        					<!--
												        					<div	style="width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;">
												        									<a class="menu" href="#summary_en">Summary</a><a class="menu" href="#introduction">Introduction</a><a class="menu" href="#methods">Methods</a><a class="selected_menu" href="#results">Results</a><a class="menu" href="#discussion">Discussion</a><a class="menu" href="#reference">Reference</a>												        					</div>
												        					-->
												        				</div>
												        				
												        				
												        																        				
												        				<div  style="width:100%; float:left;  padding-bottom:0px;">
												        					<div	style="width:100%; float:left; min-height:45px; text-align:justify">
												        						<a href="" name="discussion"></a>
												        						<!--<h2>Discussion</h2>-->
												        						<h3>DISCUSSION</h3>
<p>The current study highlights pathogenic <em>CPLANE1</em> variants as a novel cause of EPL. <em>CPLANE1</em> is found in the 5p13.2 region. This gene encodes a transmembrane protein of 3197 amino acids and is highly conserved in vertebrates. The encoded protein is presumed to function as a transmembrane protein with a putative coiled-coil motif.<sup>18</sup> Although the precise function of the <em>CPLANE1</em> gene is not fully explored, several studies have shown that it plays a vital role in the process of ciliogenesis, which is required for cell proliferation, polarity, differentiation, and response maintenance to various stimuli.<sup>19,20</sup> According to the Human Protein Atlas Database, the <em>CPLANE1</em> gene is expressed in placental tissue during pregnancy, indicating that the function of the <em>CPLANE1</em> gene product may be crucial in pregnancy maintenance.<sup>21,22</sup> Defects in this gene cause JS type 17, which is caused by pathogenic homozygous or compound heterozygous loss of function variants, as well as orofaciodigital syndrome VI, which is caused by a combination of pathogenic loss of function and missense variants.<sup>18</sup> JS type 17 is an autosomal recessive condition found primarily in patients of French-Canadian origin. Clinically, it is characterized by the presence of the molar tooth sign, cerebellar ataxia, mental disability, motor delay, oculomotor apraxia, and, in some cases, renal disease and retinal dystrophy.<sup>23</sup> JS has not previously been reported in EPLs, but it has been confirmed in several fetuses with multiple anomalies in the second trimester or later in the pregnancy, resulting in termination.<sup>11-14</sup></p>
<p>Considering the causative variants found within our cohort in this study, c.1819delT (p. Tyr607Thrfs*6) is a known pathogenic variant in the <em>CPLANE1</em> gene that has been identified as pathogenic/likely pathogenic in the ClinVar database several times.<sup>24</sup> The protein impact of this variant is thought to be the formation of a premature translational stop signal (p. Tyr607Thrfs*6) in the <em>CPLANE1</em> gene, which is predicted to result in a missing or defective protein. This early translational stop signal variant has been found in patients with JS but, to the best of our knowledge, has not been reported in a homozygous condition in a living patient. In our cohort of positive samples, a second variant in exon 40 of the <em>CPLANE1</em> gene, c.7817T&gt;A (p. Leu2624Ter), was coinherited with the c.1819delT variant. This variant is also classified as pathogenic in the ClinVar database, and its protein product causes a premature translational stop signal (p. Leu2624Ter) in the <em>CPLANE1</em> gene.<sup>25</sup> In the <em>CPLANE1</em> &ldquo;complex allele&rdquo; c.1819delT;c.7817T&gt;A, the c.1819delT variant, located in exon 12 of the gene, has the LoF effect on the protein and causes JS. In <a href="javascript:openWin('uploads/grafik/table_BMJ_2581_1.jpg', 640, 480)"><b>Table 2</b></a>, we compare <em>CPLANE1</em> selected alleles from the population database gnomAD to our internal database of 545 individuals analyzed by WES. It is clear that all selected <em>CPLANE1</em> alleles are more common in our cohort and the European (non-Finnish) population with gnomAD.</p>
<p>The second most common <em>CPLANE1</em> pathogenic variant in our group was c.8263dupA (AF&nbsp;=&nbsp;0.28%, 3/1,090). It has already been identified in trans with the &ldquo;complex allele&rdquo; in three patients with JS tested in our laboratory. Furthermore, our present study revealed a novel non-classical splicing pathogenic <em>CPLANE1</em> gene variant, c.5820+3_5820+6del, which is discussed in the results section of this study.</p>
<p>Our present study found a high incidence of JS in EPLs, with 2.03% of POC samples having a genetic diagnosis of JS. The frequency is even higher (6.25% of POCs from EPLs) among couples of Albanian origin. To the best of our knowledge, this is the most common fetal monogenic cause of EPL to date. Targeted screening for the <em>CPLANE1</em> &ldquo;complex allele&rdquo; c.1819delT;c.7817T&gt;A, followed by NGS analysis in heterozygotes, would be warranted, particularly in couples of Albanian ethnic origin, because it would detect one JS in 16 euploid EPLs.</p>
<p>Further analysis of women with EPLs showed a higher <em>CPLANE1</em> &ldquo;complex allele&rdquo; carrier rate (<em>p </em>=&nbsp;0.2). It was found to be significantly more common among women of Albanian origin than among Macedonians (AF&nbsp;=&nbsp;1.65% vs. 0.39%, respectively), which is consistent with the findings among POC materials. <em>CPLANE1</em>c.1819delT;c.7817T&gt;A homozygosity is expected to affect approximately 1 of 25,600 JS children based on our findings of seven carriers in 545 individuals from the general population and a carrier frequency of about 1/80. However, no such patient with JS has been identified using clinical practice data in our country. Thus, the absence of patients with JS who are homozygous for the &ldquo;complex allele&rdquo; and the relatively high incidence of this genotype among EPLs in our study suggest that homozygosity of this allele is incompatible with life and results in pregnancy loss early in embryonic development. Nevertheless, our findings should be further confirmed by functional analyses.</p>
<p>Furthermore, we reviewed studies on JS individuals with the c.1819delT and c.7817T&gt;A variants and present these findings in <a href="javascript:openWin('uploads/grafik/table_BMJ_2581_2.jpg', 640, 480)"><b>Table 3</b></a>. These variants have been found in a compound heterozygous condition with other pathogenic variants; however, the c.1819delT variant has not been reported in a homozygous condition in a single patient.</p>
<p>Previous studies have revealed that homozygosity of particular variants in other genes can have lethal embryonic effect.<sup>30</sup> Arnadottir et al. observed a high deficit of homozygous individuals for a specific GLE1 variant and postulated that homozygosity for this variant (previously reported in compound heterozygous genotype with other variants in the same gene, causing a severe neonatal condition) would result in EPL. This assumption was confirmed by the fact that carrier couples for that particular variant had a high incidence of early miscarriage. In another similar study, Oddsson et al.<sup>31</sup> found that disease-causing LoF variants in the <em>DHCR7, GBE1, GLE1, PMM2, PNKP</em>, and <em>TSFM</em> genes have homozygous deficits and have only been reported in compound heterozygous cases, along with a hypomorphic allele that causes partial loss of function. They proposed that variants with a minimum activity level are necessary for successful embryonic development.</p>
<p>This is the first study to show such a high prevalence of fetal monogenic causes of pregnancy loss. This shows that, while rare, fetal monogenic diseases can be a common cause of EPLs. Our findings have a larger impact on the pathogenesis of pregnancy loss and contribute to a better understanding of the pathogenicity of the variants in the <em>CPLANE1</em> gene. Our results further suggest that focused screening for the <em>CPLANE1</em> &ldquo;complex allele&rdquo; might be warranted in couples of Albanian ethnicity, as it would detect one JS in every 16 euploid EPLs.</p>
<p><strong>Acknowledgments:</strong> We thank all gynecologists and couples that contributed to the study.</p>
<p><strong>Ethics Committee Approval:</strong> The Macedonian Academy of Sciences and Arts Ethical Committee approved this study (approval number: 09-1047/6, date: 04.05.2016).</p>
<p><strong>Informed Consent:</strong> All participants in the study provided informed consent.</p>
<p><strong>Data Sharing Statement:</strong> The data that support the findings of this study are available from the corresponding author upon reasonable request.</p>
<p><strong>Authorship Contributions:</strong> Concept- D.P.K.; Design- D.P.K.; Supervision- D.P.K.; Fundings- D.P.K.; Materials- K.K.S.; Data Collection or Processing- Gj.B., M.T., K.K.S.; Analysis or Interpretation- Gj.B., M.T., D.P.K.; Literature Search- Gj.B., M.T., D.P.K.; Writing- Gj.B., M.T., K.K.S., D.P.K.; Critical Review- Gj.B., M.T., K.K.S., D.P.K.</p>
<p><strong>Conflict of Interest:</strong> No conflict of interest was declared by the authors.</p>
<p><strong>Funding:</strong> Funded by Macedonian Academy of Sciences and Arts project: Molecular basis of spontaneous abortion, project number: 08-707.</p>												        					</div>
												        					<!--
												        					<div	style="width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;">
												        									<a class="menu" href="#summary_en">Summary</a><a class="menu" href="#introduction">Introduction</a><a class="menu" href="#methods">Methods</a><a class="menu" href="#results">Results</a><a class="selected_menu" href="#discussion">Discussion</a><a class="menu" href="#reference">Reference</a>												        					</div>
												        					-->
												        				</div>
												        				
												        				
												        																        				
												        				<div  style="width:100%; float:left;  padding-bottom:0px;">
												        					<div	style="width:100%; float:left; min-height:45px; text-align:justify">
												        						<a href="" name="reference"></a>
												        						<!--<h2>Reference</h2>-->
												        						<h3>REFERENCES</h3>
<ol>
<li>Dimarco CS. Speroff L, Glass RH, Kase NG, editors. Clinical Gynecologic Endocrinology and Infertility. Baltimore: Lippincott, Williams &amp; Wilkins, 1999: 1-1200. <em>Fertility and Sterility.</em> 2000;74:425-426.</li>
<li>Bricker L, Farquharson RG. Types of pregnancy loss in recurrent miscarriage: implications for research and clinical practice. <em>Hum Reprod.</em> 2002;17:1345-1350.</li>
<li>Bulletti C, Flamigni C, Giacomucci E. Reproductive failure due to spontaneous abortion and recurrent miscarriage. Hum Reprod Update. 1996;2:118-136.</li>
<li>Warren JE, Silver RM. Genetics of pregnancy loss. <em>Clin Obstet Gynecol.</em> 2008;51:84-95.</li>
<li>Rajcan-Separovic E. Next generation sequencing in recurrent pregnancy loss-approaches and outcomes. <em>Eur J Med Genet.</em> 2020;63:103644.</li>
<li>Zhao C, Chai H, Zhou Q, et al. Exome sequencing analysis on products of conception: a cohort study to evaluate clinical utility and genetic etiology for pregnancy loss. <em>Genet Med. </em>2021;23:435-442.</li>
<li>Al Qahtani NH, AbdulAzeez S, Almandil NB, et al. Whole-genome sequencing reveals exonic variation of ASIC5 gene results in recurrent pregnancy loss. <em>Front Med (Lausanne).</em> 2021;8:699672.</li>
<li>Buonaiuto S, Biase ID, Aleotti V, et al. Prioritization of putatively detrimental variants in euploid miscarriages. <em>Sci Rep.</em> 2022;12:1997.</li>
<li>Kline J, Vardarajan B, Abhyankar A, et al. Embryonic lethal genetic variants and chromosomally normal pregnancy loss. <em>Fertil Steril.</em> 2021;116:1351-1358.</li>
<li>Bruel AL, Franco B, Duffourd Y, et al. Fifteen years of research on oral-facial-digital syndromes: from 1 to 16 causal genes. <em>J Med Genet.</em> 2017;54:371-378.</li>
<li>Xiang J, Zhang L, Jiang W, et al. Prenatal diagnosis and genetic analysis of a fetus with Joubert syndrome. <em>Biomed Res Int.</em> 2018;2018:7202168.</li>
<li>Bonnard C, Shboul M, Tonekaboni SH, et al. Novel mutations in the ciliopathy-associated gene CPLANE1 (C5orf42) cause OFD syndrome type VI rather than Joubert syndrome. <em>Eur J Med Genet.</em> 2018;61:585-595.</li>
<li>Liu Y, Wang H, Jin X, Shao Q, Pan Q. Molecular Diagnosis and Prenatal Phenotype Analysis of Eight Fetuses With Ciliopathies. <em>Front Genet.</em> 2021;12:705808.</li>
<li>Zhu H, Chen W, Ren H, et al. Non-classic splicing mutation in the CPLANE1 (C5orf42) gene cause Joubert syndrome in a fetus with severe craniocerebral dysplasia. <em>Eur J Med Genet. </em>2021;64:104212.</li>
<li>Kubelka-Sabit K, Bozinovski G, Jasar D, et al. Detection of placental chromosomal aberrations in early spontaneous abortions in correlation with the histologic findings. <em>Mac Med Rev.</em> 2017;71:64-71.</li>
<li>Noveski P, Terzic M, Vujovic M, Kuzmanovska M, Sukarova Stefanovska E, Plaseska-Karanfilska D. Multilevel regression modeling for aneuploidy classification and physical separation of maternal cell contamination facilitates the QF-PCR based analysis of common fetal aneuploidies. <em>PLoS One.</em> 2019;14:e0221227.</li>
<li>Untergasser A, Cutcutache I, Koressaar T, et al. Primer3--new capabilities and interfaces. <em>Nucleic Acids Res. </em>2012;40:e115.</li>
<li>Romani M, Mancini F, Micalizzi A, et al. Oral-facial-digital syndrome type VI: is C5orf42 really the major gene? <em>Hum Genet.</em> 2015;134:123-126.</li>
<li>Toriyama M, Lee C, Taylor SP, et al. The ciliopathy-associated CPLANE proteins direct basal body recruitment of intraflagellar transport machinery. <em>Nat Gene.</em> 2016;48:648-656.</li>
<li>Hong H, Joo K, Park SM, et al. Extraciliary roles of the ciliopathy protein JBTS17 in mitosis and neurogenesis. <em>Ann Neurol.</em> 2019;86:99-115.</li>
<li>Uhl&eacute;n M, Fagerberg L, Hallstr&ouml;m BM, et al. Tissue-based map of the human proteome. <em>Science.</em> 2015;347:1260419.</li>
<li>Qian W, Liu X, Wang Z, et al. Whole‐exome sequencing identified novel variants in CPLANE1 that causes oral‐facial‐digital syndrome VI by inducing primary cilia abnormality. <em>J Cell Mol Med.</em> 2022;26:3213-3222.</li>
<li>Srour M, Schwartzentruber J, Hamdan FF, et al. Mutations in C5ORF42cause Joubert syndrome in the French Canadian population. <em>Am J Hum Genet.</em> 2012;90:693-700.</li>
<li>National Center for Biotechnology Information. ClinVar; [VCV000217590.43], (accessed June 4, 2023).</li>
<li>National Center for Biotechnology Information. ClinVar; [VCV000217575.31], (accessed June 4, 2023).</li>
<li>Bachmann-Gagescu R, Dempsey JC, Phelps IG, et al. Joubert syndrome: a model for untangling recessive disorders with extreme genetic heterogeneity. <em>J Med Genet.</em> 2015;52:514-522.</li>
<li>Summers AC, Snow J, Wiggs E, et al. Neuropsychological phenotypes of 76 individuals with Joubert syndrome evaluated at a single center. <em>Am J Med Genet A.</em> 2017;173:1796-1812.</li>
<li>Marinakis NM, Svingou M, Veltra D, et al. Phenotype-driven variant filtration strategy in exome sequencing toward a high diagnostic yield and identification of 85 novel variants in 400 patients with rare Mendelian disorders. <em>Am J Med Genet A</em>. 2021;185:2561-2571.</li>
<li>Shelby ES, Cocos R, Leanca MC, Mirea A, Barca D. Phenotypic manifestations of C5orf42 pathogenic variants. <em>Ro J Pediatr.</em> 2022;71:39-46.</li>
<li>Arnadottir GA, Oddsson A, Jensson BO, et al. Population-level deficit of homozygosity unveils CPSF3 as an intellectual disability syndrome gene. <em>Nat Commun.</em> 2022;13:705.</li>
<li>Oddsson A, Sulem P, Sveinbjornsson G, et al. Deficit of homozygosity among 1.52 million individuals and genetic causes of recessive lethality. <em>Nat Commun.</em> 2023;14:3453.</li>
</ol>												        					</div>
												        					<!--
												        					<div	style="width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;">
												        									<a class="menu" href="#summary_en">Summary</a><a class="menu" href="#introduction">Introduction</a><a class="menu" href="#methods">Methods</a><a class="menu" href="#results">Results</a><a class="menu" href="#discussion">Discussion</a><a class="selected_menu" href="#reference">Reference</a>												        					</div>
												        					-->
												        				</div>
												        				
												        				
												        															        			
		
											        	</div>