window._articleFullText = window._articleFullText || {};
window._articleFullText[2636] = "<div class=\"col-lg-12 col-md-12 col-sm-12 col-xs-12\">\n\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t<div  style=\"width:100%; float:left;  padding-bottom:0px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:100%; float:left; min-height:45px; text-align:justify\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<a href=\"\" name=\"summary_en\"></a>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<!--<h2>Summary</h2>-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<h4>Abstract</h4>\r\n<p><strong>Background:</strong> Emerging carbapenem-resistant <em>Klebsiella pneumoniae (K. pneumoniae)</em> (CRKP) bacteremias are presenting significant public health risks due to limited treatment options and increased mortality. <em>K. pneumoniae</em> isolates exhibit carbapenem resistance rates that vary from 25% to 50% throughout the European continent, including our country.<br /><strong>Aims:</strong> To assess the characteristics of CRKP bacteremia, a condition that has recently demonstrated an increasing prevalence in our center. We sought to ascertain the resistance rates of isolated strains to antibiotics other than carbapenems, identify the responsible carbapenemase genes, evaluate the efficacy of antibiotics, determine mortality rates, explore clonality among strains, and investigate the influence of the COVID-19 pandemic on all these factors.<br /><strong>Study Design:</strong> Retrospective observational study.<br /><strong>Methods:</strong> This study included patients aged 18 and older who had experienced meropenem-resistant <em>K. pneumoniae</em> bacteremia. Meropenem resistance was confirmed by employing the Kirby-Bauer disk diffusion method. Meropenem minimum inhibitory concentration (MIC) levels were determined using the gradient test, while colistin MIC levels were ascertained using the disk elution technique. Carbapenemase genes were evaluated via colony polymerase chain reaction (PCR), and clonality analysis was performed using the arbitrarily primed PCR technique.<br /><strong>Results:</strong> The study comprised 230 patients, with a mean age of 63.1 &plusmn; 15.9 years, of whom 58.7% were male. Oxacillinase-48 (OXA-48) was detected in 74.8% of the patients, New Delhi metallo-beta-lactamase (NDM) in 12.6%, OXA-48 + NDM in 7.8%, and KPC in 4.8%. The 14-day and 30-day mortality rates were 57% and 69.6%, respectively. Multivariate analysis of the 30-day mortality revealed several crucial factors, including bacteremia development in the intensive care unit, the occurrence of bacteremia during the COVID-19 pandemic, polymicrobial bacteremia, the use of indwelling intravenous catheters, a platelet count of &le; 140,000/&micro;l, procalcitonin levels of &ge; 6 &micro;g/l, and a Charlson comorbidity score &ge; 3. Notably, the <em>OXA-48</em> and <em>KPC</em> genes were upregulated significantly during the COVID-19 pandemic, while the <em>NDM</em> gene groups were downregulated. Additionally, both 14-day and 30-day mortality rates increased significantly.<br /><strong>Conclusion:</strong> In this study, the most prevalent carbapenemase gene was OXA-48; however, there has been a recent increase in <em>KPC</em> genes. No dominant epidemic strain was identified through clonality analysis. The clustering rate was 68% before the pandemic, increasing to 85.7% during the pandemic. The significance of infection control measures is underscored by the rise in both clustering and mortality rates during the COVID-19 pandemic.</p>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<!--\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t<a class=\"selected_menu\" href=\"#summary_en\">Summary</a><a class=\"menu\" href=\"#introduction\">Introduction</a><a class=\"menu\" href=\"#methods\">Methods</a><a class=\"menu\" href=\"#results\">Results</a><a class=\"menu\" href=\"#discussion\">Discussion</a><a class=\"menu\" href=\"#reference\">Reference</a>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t<div  style=\"width:100%; float:left;  padding-bottom:0px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:100%; float:left; min-height:45px; text-align:justify\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<a href=\"\" name=\"introduction\"></a>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<!--<h2>Introduction</h2>-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<hr size=\"1\" color=\"#f2f2f2\">\r\n<h3>INTRODUCTION</h3>\r\n<p><em>Klebsiella</em> species have developed resistance to carbapenem group antibiotics in recent years, which is primarily due to the irrational use of antibiotics.<sup>1</sup> The 2019 report from the United States Center for Disease Control and Prevention (CDC) identifies carbapenem-resistant <em>Enterobacteriaceae</em> as an urgent threat.<sup>2</sup> The carbapenem resistance rate in <em>Klebsiella pneumoniae (K. pneumoniae)</em> isolates has increased from 40.1% in the 2016 national antimicrobial resistance surveillance system report, which investigates healthcare-associated infections in our country, to 63.5% by 2021.<sup>3,4</sup></p>\r\n<p>The primary resistance mechanisms against carbapenems include beta-lactamase production, which catalyzes carbapenem hydrolysis, as well as efflux pumps, porins, and mutations that result in alterations in penicillin-binding proteins. Among these mechanisms, beta-lactamases, particularly carbapenemases that hydrolyze carbapenems, are of major significance.<sup>5</sup> The prevalence of carbapenemase resistance genes in carbapenem-resistant <em>K. pneumoniae </em>(CRKP) isolates demonstrates regional variation. <em>K. pneumoniae</em> carbapenemase (KPC) is the most prevalent carbapenemase in the United States, while in South and Southeast Asia, metallo-beta-lactamases (MBL) predominate.<sup>6,7</sup> Notably, the most frequently identified carbapenemase in CRKP isolates in our country belongs to the oxacillinase (OXA) group and is designated as OXA-48.<sup>8</sup></p>\r\n<p>Although the risk factors for CRKP bacteremia-associated mortality vary, the following are frequently reported: the presence of septic shock, admission to the intensive care unit (ICU), mechanical ventilation, corticosteroid use prior to bacteremia, high comorbidity-mortality/bacteremia scores, immunosuppression, neutropenia, and colistin resistance.<sup>9-11</sup></p>\r\n<p>Mortality rates associated with nosocomial CRKP bacteremias remain elevated, exhibiting variability across different periods and regions. This is primarily attributed to the challenges of treatment and the presence of concurrent comorbidities.<sup>12,13</sup> Given the rapid surge in resistant microorganisms and the constrained availability of novel antibiotic alternatives, antibiotic stewardship and strict adherence to infection control measures hold paramount significance in combating CRKP infections.</p>\r\n<p>In this study, we aimed to identify the characteristics of CRKP bacteremia, a condition that has been increasingly encountered at our center in recent years. We sought to determine the resistance rates of isolated strains to antibiotics other than carbapenems, identify responsible carbapenemase genes, assess the efficacy of antibiotic therapy, investigate mortality rates, and explore clonality among strains. Since we encountered treatment failures and elevated mortality due to CRKP bacteremias during the COVID-19 pandemic, we explored the pandemic&rsquo;s influence on all these factors and compared the periods before and after the pandemic.</p>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<!--\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t<a class=\"menu\" href=\"#summary_en\">Summary</a><a class=\"selected_menu\" href=\"#introduction\">Introduction</a><a class=\"menu\" href=\"#methods\">Methods</a><a class=\"menu\" href=\"#results\">Results</a><a class=\"menu\" href=\"#discussion\">Discussion</a><a class=\"menu\" href=\"#reference\">Reference</a>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t<div  style=\"width:100%; float:left;  padding-bottom:0px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:100%; float:left; min-height:45px; text-align:justify\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<a href=\"\" name=\"methods\"></a>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<!--<h2>Methods</h2>-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<h3>MATERIALS AND METHODS</h3>\r\n<p><strong><em>Study design</em></strong></p>\r\n<p>This retrospective observational single-center study was conducted at a tertiary university hospital with a 1000-bed capacity. The investigation was conducted over a seven-year period, from January 2015 to December 2021, and included patients aged 18 and older who had experienced meropenem-resistant <em>K. pneumoniae</em> bacteremia. Conventional methods were initially employed to identify the strains isolated from blood culture as <em>Klebsiella</em> spp. Subsequently, <em>K. pneumoniae </em>was confirmed using either the BD Phoenix 100 (Becton, Dickinson and Company, USA) or MALDI-TOF MS (bioM&eacute;rieux, France) systems. The Kirby-Bauer disk diffusion method was employed to ascertain meropenem resistance. Patients with inadequate clinical data and those with recurrent bacteremia were excluded from the study. To assess the influence of various antibiotic combinations on mortality, patients who passed away within 24 hours and/or experienced polymicrobial bacteremia were also excluded.</p>\r\n<p><strong><em>Ethics</em></strong></p>\r\n<p>During hospitalization, informed consent form was signed by the patients themselves or, if not possible, by their legal relatives. This study was conducted with the approval of the Clinical Research Ethics Committee of İstanbul University-Cerrahpaşa, Cerrahpaşa Faculty of Medicine (approval number: E-83045809-604.01.01-329361, date: 04.03.2022).</p>\r\n<p><strong><em>Data collection</em></strong></p>\r\n<p>Demographic information, risk factors for bacteremia, laboratory parameters during the bacteremia episode, sepsis severity, colonization, source of bacteremia (if applicable), and polymicrobial growth were meticulously analyzed from patients&rsquo; medical records. Additionally, antibiotic susceptibility data, types of antibiotic combinations administered, the Charlson comorbidity index score to predict mortality risk, the Sequential Organ Failure Assessment score (SOFA) and Pitt bacteremia scores were also investigated.</p>\r\n<p><strong><em>Definitions</em></strong></p>\r\n<p>Septic shock was defined as &ldquo;the detection of a serum lactate level exceeding 2 mmol/l, despite adequate fluid resuscitation, and the need for vasopressors to maintain a mean arterial blood pressure of at least 65 mmHg&rdquo;.<sup>14</sup> To differentiate between primary and secondary bloodstream infections, we adhered to CDC definitions.<sup>15</sup> A polymicrobial bloodstream infection was defined as the isolation of two or more distinct species in either the same or separate blood cultures obtained within a 48-hour window.<sup>16</sup></p>\r\n<p><strong><em>Antibiotic susceptibility tests, determination of resistance genes and clonality analysis</em></strong></p>\r\n<p>The Kirby-Bauer disk diffusion method was employed to evaluate susceptibility to meropenem, imipenem, amikacin, gentamicin, trimethoprim-sulfamethoxazole, and ceftazidime-avibactam, in accordance with the guidelines outlined by the European Committee on Antimicrobial Susceptibility Testing (EUCAST).<sup>17</sup> Furthermore, tigecycline susceptibility was ascertained using the Kirby-Bauer disk diffusion method, complying with the U.S. Food and Drug Administration recommendations.<sup>18,19</sup></p>\r\n<p>The minimum inhibitory concentration (MIC) levels of meropenem were determined using the gradient method, in accordance with the EUCAST recommendations.<sup>17</sup> To ascertain colistin MIC levels, the colistin disk elution method was employed, aligning with the recommendations of the Clinical and Laboratory Standards Institute.<sup>20</sup></p>\r\n<p>The screening for carbapenem resistance was performed in keeping with the EUCAST recommendations; however, this method was unable to ascertain clinical resistance to imipenem. Additionally, clinical resistance is not determined solely by the identification of the mechanism.<sup>21</sup> Given our observation that CRKP may be susceptible to imipenem, we determined that it would be appropriate to assess imipenem susceptibility separately. Although there is limited evidence supporting the use of a carbapenem for treating infections caused by carbapenem-resistant strains that exhibit susceptibility to specific carbapenems, we aimed to investigate this data because we still employ them, at least in combination with other antimicrobial agents, during our routine clinical practice.<sup>22</sup></p>\r\n<p>We employed <em>blaVIM, blaIMP, blaOXA-48, blaNDM</em>, and <em>blaKPC</em> primers to identify the resistance genes. The resistance genes were detected using colony polymerase chain reaction (PCR), and the extracted products were analyzed through gel electrophoresis. The presence of bands that corresponded to specific base pair lengths was used to identify resistance genes.<sup>23,24</sup> Details regarding the mixture prepared for performing colony PCR, the primers employed, and the protocol followed are outlined in <a href=\"https://balkanmedicaljournal.org/uploads/pdf/supplemantry-2024.2024-5-99.pdf\">Supplementary Tables 1-3</a>, which are available in the online supplementary material.</p>\r\n<p>The arbitrarily primed (AP) PCR method was implemented for clonality analysis, using the M13 primer (5&prime;-GAG GGT GGC GGT TCT-3&prime;) in accordance with the standardized protocol established by Menekşe et al.<sup>25</sup> <a href=\"https://balkanmedicaljournal.org/uploads/pdf/supplemantry-2024.2024-5-99.pdf\">Supplementary Tables 4-6</a>, which are available in the online supplementary material, contain comprehensive information on the thermal cycling profile and the formulation of amplification mixtures. The amplification products were subsequently subjected to gel electrophoresis, and the resulting band profiles for each isolate were compared. The original gel electrophoresis images have been uploaded as supplementary material. We utilized the GelCompar II software system (version 6.5; Applied Maths, Sint-Martens-Latem, Belgium) to compare the band profiles. The Dice Correlation coefficient was utilized to conduct similarity calculations for band analysis, and the Unweighted Pairwise Grouping Mathematical Averaging method was employed for clustering analysis.</p>\r\n<p><strong><em>Statistical analysis</em></strong></p>\r\n<p>The data were analyzed using the IBM SPSS version 25 (Chicago, Illinois, USA) software package. The Kolmogorov-Smirnov, Shapiro-Wilk, and Kurtosis-Skewness tests, as well as the box plot distribution, were employed to evaluate the normality of the data distribution.</p>\r\n<p>When conducting intragroup comparisons of continuous variables, units per group were considered. The Student&rsquo;s t-test, a parametric test for normally distributed data, was employed in addition to non-parametric tests like the Kruskal-Wallis and Mann-Whitney U tests. Chi-square analysis was implemented to compare categorical variables within groups, with a Bonferroni correction applied for multiple group evaluations.</p>\r\n<p>We considered 14- and 30-day mortalities to be dependent variables. Receiver operating characteristic (ROC) curves were generated for continuous data that were identified as significant predictors of 30-day mortality. The area under the curve (AUC), the cut-off points, and the sensitivity and specificity associated with these cut-off points were determined and eventually utilized in subsequent analyses. We applied the &ldquo;Concordance Probability Method&rdquo; proposed by Liu<sup>26</sup> which defines the optimum cut-off point as the point at which the product of sensitivity and specificity is highest.</p>\r\n<p>In the final phase of the study, we examined the independent parameters using the chi-square test, with &ldquo;30-day mortality&rdquo; being the dependent variable. Partial correlation analyses were conducted for independent variables whose effects we were particularly interested in exploring. Univariate and multivariate logistic regression analyses were conducted on those variables that were determined to be the most critical and influential, as indicated by a higher effect size (&eta;<sup>2</sup>) and a lower <em>p</em> value. Statistical significance was defined as a <em>p</em> value of &lt; 0.05.</p>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<!--\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t<a class=\"menu\" href=\"#summary_en\">Summary</a><a class=\"menu\" href=\"#introduction\">Introduction</a><a class=\"selected_menu\" href=\"#methods\">Methods</a><a class=\"menu\" href=\"#results\">Results</a><a class=\"menu\" href=\"#discussion\">Discussion</a><a class=\"menu\" href=\"#reference\">Reference</a>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t<div  style=\"width:100%; float:left;  padding-bottom:0px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:100%; float:left; min-height:45px; text-align:justify\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<a href=\"\" name=\"results\"></a>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<!--<h2>Results</h2>-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<h3>RESULTS</h3>\r\n<p>The study included two hundred and thirty patients who met the inclusion criteria. The mean age of the patients was 63.1 &plusmn; 15.9 years, and 135 (58.7%) of them were male. One hundred sixty (69.6%) patients were in ICU when bacteremia developed, while 70 (30.4%) were being monitored in the internal medicine and surgical wards. A hospitalization history was found in 222 (96.5%) patients. Antibiotic use in the preceding month was reported by 226 (98.3%) patients, and 170 (73.9%) were infected/colonized with CRKP prior to developing bacteremia. The most prevalent infection/colonization sites were the respiratory system (33%), followed by the gastrointestinal (21.3%) and urinary systems (15.7%), as well as decubitus ulcers/wounds (3.9%). <a href=\"javascript:openWin('uploads/grafik/table_BMJ_2636_0.jpg', 640, 480)\"><b>Table 1</b></a> presents the clinical and laboratory data, as well as the clinical scores that were determined on the day of the bacteremia.</p>\r\n<p>Monomicrobial bacteremia occurred in 205 (89.1%) patients, while polymicrobial growth was observed in 25 (10.9%) patients. Based on the analysis of polymicrobial bacteremias, it was revealed that <em>Acinetobacter baumannii</em> was isolated in 16 patients, <em>Pseudomonas aeruginosa</em> in five patients, <em>Candida</em> spp. in three patients, and <em>Stenotrophomonas maltophilia</em> in one patient.</p>\r\n<p>The distribution of carbapenemase resistance genes was as follows: OXA-48 in 172 (74.8%) patients, bla-New Delhi metallo-beta-lactamase (NDM) in 29 (12.6%) patients, OXA-48 + NDM in 18 (7.8%) patients, and KPC in 11 (4.8%) patients.</p>\r\n<p>In terms of mortality rates, the 14-day mortality stood at 57%, while the 30-day mortality reached 69.6%. The mortality rate during the COVID-19 pandemic was significantly higher than it was before the pandemic. The median duration of hospitalization before bacteremia was considerably shorter in the deceased patient group, and mortality was significantly higher in ICU patients. A history of continuous renal replacement therapy, ICU admission due to COVID-19, use of indwelling intravenous catheters, and a previous history of immunosuppression were significantly higher in the deceased group. Furthermore, the septic shock incidence and polymicrobial bacteremia rates, as well as the mean scores for the Charlson comorbidity index, Pitt bacteremia, and SOFA, were all significantly higher in the deceased group. Additionally, the deceased patient group exhibited substantially higher mean C-reactive protein (CRP) and procalcitonin values, whereas the mean platelet count was significantly lower (<a href=\"javascript:openWin('uploads/grafik/table_BMJ_2636_0.jpg', 640, 480)\"><b>Table 1</b></a>).</p>\r\n<p>To estimate significant cut-off values for the clinical scores and laboratory parameters for 30-day mortality, ROC curves were generated. The analysis revealed that the significant cut-off points ​​were as follows: 3 for the Charlson comorbidity index score, 5 for the Pitt bacteremia score, 8 for SOFA, 170 mg/L for CRP, 6 &micro;g/L for procalcitonin, and 140,000/&micro;l for platelets (<a href=\"javascript:openWin('uploads/grafik/table_BMJ_2636_1.jpg', 640, 480)\"><b>Table 2</b></a>).</p>\r\n<p>Several independent risk factors were identified in the multivariate logistic regression analysis for 30-day mortality. These consisted of a bacteremia developing in the ICU (<em>p</em> = 0.001) or during the COVID-19 pandemic period (<em>p</em> &lt; 0.001), polymicrobial bacteremia (<em>p</em> = 0.014), the presence of an indwelling intravenous catheter (<em>p</em> = 0.037), a platelet count &le;140,000/&micro;l (<em>p</em> &lt; 0.001), a procalcitonin level &ge; 6 &micro;g/l (<em>p</em> = 0.011), and a Charlson comorbidity score &ge; 3 (<em>p</em> = 0.001) (<a href=\"javascript:openWin('uploads/grafik/table_BMJ_2636_2.jpg', 640, 480)\"><b>Table 3</b></a>).</p>\r\n<p>All isolates exhibited meropenem resistance, with meropenem MIC levels of &le; 16 &micro;g/ml observed in 59 (25.7%) patients. The resistance rates for tigecycline, ceftazidime-avibactam, and colistin were 4.8%, 21.3%, and 57%, respectively (<a href=\"javascript:openWin('uploads/grafik/table_BMJ_2636_3.jpg', 640, 480)\"><b>Table 4</b></a>).</p>\r\n<p>When comparing the pre-COVID-19 pandemic data to the pandemic period data, it was noted that <em>OXA-48</em> and <em>KPC</em> genes substantially increased during the pandemic period, while <em>NDM</em> and <em>OXA-48 + NDM</em> gene group expression decreased significantly. Additionally, imipenem, amikacin, gentamicin, and tigecycline resistance showed an increase during the pandemic period, while ceftazidime-avibactam resistance decreased significantly. Furthermore, it was determined that the mortality rates for both 14- and 30-day periods significantly increased during the pandemic (<a href=\"javascript:openWin('uploads/grafik/table_BMJ_2636_3.jpg', 640, 480)\"><b>Table 4</b></a>).</p>\r\n<p>After excluding patients with polymicrobial bacteremia and those who passed away within 24 hours from the analysis, no significant difference was noted in the 14- or 30-day mortality rates of the remaining 180 patients who were receiving combination antibiotic therapy (<a href=\"javascript:openWin('uploads/grafik/table_BMJ_2636_4.jpg', 640, 480)\"><b>Table 5</b></a>).</p>\r\n<p>Clonality analysis was conducted using AP-PCR, which resulted in the identification of 92 distinct genotypes among the 230 isolates. These isolates were found to be distributed across 42 distinct clusters (with a tolerance of 1.0, an optimization of 1.0, and a limit value of 85%). Notably, 50 isolates did not belong to any cluster, resulting in a clustering rate of 78.3%. The clustering rate was 68% before the pandemic, which increased to 85.7% during the pandemic period (<em>p</em> = 0.001). No dominant epidemic strain was identified. <a href=\"javascript:openWin('uploads/grafik/figure_BMJ_2636_0.jpg', 640, 480)\"><b>Figures 1</b></a>, <a href=\"javascript:openWin('uploads/grafik/figure_BMJ_2636_1.jpg', 640, 480)\"><b>2</b></a> illustrate the dendrogram, which includes gel electrophoresis images.</p>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<!--\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t<a class=\"menu\" href=\"#summary_en\">Summary</a><a class=\"menu\" href=\"#introduction\">Introduction</a><a class=\"menu\" href=\"#methods\">Methods</a><a class=\"selected_menu\" href=\"#results\">Results</a><a class=\"menu\" href=\"#discussion\">Discussion</a><a class=\"menu\" href=\"#reference\">Reference</a>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t<div  style=\"width:100%; float:left;  padding-bottom:0px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:100%; float:left; min-height:45px; text-align:justify\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<a href=\"\" name=\"discussion\"></a>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<!--<h2>Discussion</h2>-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<h3>DISCUSSION</h3>\r\n<p>The prevalence of CRKP bacteremias has increased in recent years, presenting significant public health challenges due to their high mortality rates and limited treatment options.<sup>27</sup> Carbapenem resistance rates in <em>K. pneumoniae </em>isolates vary from 25% to 50% across the European continent, including ours.<sup>28</sup> Within this context, our study included 230 patients with CRKP bacteremia to assess the distribution of resistance genes and mortality rates. The mean age of the patients was 63.1 &plusmn; 15.9 years, and approximately 59% of them were male.</p>\r\n<p>The respiratory system, indwelling catheters, intra-abdominal/hepatobiliary system, urinary system, and soft tissue are the most common sources of CRKP bacteremias, although the rates may vary among studies. Primary bacteremias have also been reported to occur, but at varying rates.<sup>29-31</sup> In our investigation, indwelling catheters (35.7%) were the most frequently identified source of bacteremia, followed by the respiratory system (25.2%) and intra-abdominal infections (23.9%). The urinary system and soft tissues were involved at a lower rate, and primary bacteremia was detected in only 3% of the patients, in contrast to some other studies.<sup>30,31</sup></p>\r\n<p>In recent years, there have been significant changes in the global distribution of carbapenemase resistance genes, which is characterized by variability across continents and countries. In the USA, KPC-producing <em>K. pneumoniae</em> is prevalent, and it is also endemic in certain European countries, such as Greece and Italy.<sup>32</sup> In contrast, MBL-producing <em>K. pneumoniae </em>is more frequently observed in South and Southeast Asia, although the <em>MBL</em> gene may be the dominant gene in certain Eastern European countries.<sup>7</sup></p>\r\n<p>Regarding our own country, it is essential to acknowledge that while OXA-48 was previously considered the predominant resistance gene, in recent years a variety of other resistance genes have emerged.<sup>9,30</sup> In our study, the resistance genes were distributed as follows: OXA-48 at 74.8%, NDM at 12.6%, OXA-48 + NDM at 7.8%, and KPC at 4.8%. In contrast to the pre-pandemic period, the <em>OXA-48</em> and <em>KPC</em> gene groups experienced a substantial increase during the COVID-19 pandemic, while the <em>NDM</em> and <em>OXA-48 + NDM</em> gene groups experienced a significant decrease. Notably, our study highlighted a recent increase in <em>KPC</em> gene prevalence.</p>\r\n<p>In our study, all strains demonstrated meropenem resistance, with only 25.7% of them having a meropenem MIC level of &le; 16 &micro;g/ml. Tigecycline, ceftazidime-avibactam, and colistin resistance were observed in 4.8%, 21.3%, and 57% of cases, respectively. Isler et al.<sup>9</sup> conducted a multi-center study that included 187 isolates with dominant OXA-48-like genes, revealing resistance rates for colistin, tigecycline, and ceftazidime-avibactam of 72%, 57%, and 19%, respectively. Researchers proposed that the high-level colistin resistance may be linked to ST2096, resulting in OXA-232 carriage.<sup>9</sup></p>\r\n<p>Antibiotic resistance rates over time exhibited significant trends in our study. Specifically, we observed an increase in resistance to imipenem and tigecycline during the COVID-19 pandemic period, while resistance to ceftazidime-avibactam showed a significant decrease. The proliferation of resistant strains during the pandemic likely contributed to the increase in antibiotic resistance, as the clustering rate was higher during this period. This issue was potentially due to the decrease in healthcare staff adherence to infection control measures. Conversely, during the same period, a reduction in resistance gene groups that contained NDM was associated with a decrease in ceftazidime-avibactam resistance.</p>\r\n<p>The mortality rates associated with noscomial CRKP bacteremia are alarmingly high.<sup>12,13</sup> In the multi-center INCREMENT study, primarily in European countries, it was discovered that the 30-day mortality rate was 43.2% among 437 cases of carbapenem-resistant <em>Enterobacterales</em> bacteremia, while CRKP accounted for 86% of these cases.<sup>33</sup> Another multi-center prospective study, involving patients with CRKP bacteremia, reported a 30-day mortality rate of 34%. Upon regional analysis, this rate was 24% in China, 31% in the USA, and an alarming 56% in South America.<sup>34</sup></p>\r\n<p>Isler et al.<sup>9</sup> conducted a study spanning 13 centers between 2018 and 2019, involving 187 cases of CRKP bacteremia, and found a 30-day mortality rate of 44% in our country. A 30-day mortality rate of 52.4% was reported in a separate study conducted by Aslan et al.<sup>31</sup> from 2014 to 2018, which included 124 cases of CRKP bacteremia.<sup>9,30</sup> We observed 14-day and 30-day mortality rates of 57% and 69.6%, respectively, in our study, which included 230 patients with CRKP bacteremia. Following a comprehensive analysis, we discovered that the 14-day and 30-day mortality rates were 41.2% and 55.7%, respectively, in the period preceding the COVID-19 pandemic. However, during the COVID-19 pandemic, the 14-day mortality rate increased to 68.4%, while the 30-day mortality rate rose to 79.7%. The greater occurrence of polymicrobial bacteremia during COVID-19 period and the use of intensive immunosuppressive drugs due to cytokine storms in COVID-19 pneumonia may have contributed negatively to the elevated mortality rates.</p>\r\n<p>Numerous studies that evaluated CRKP infections identified statistically significant or insignificant cut-off values ​​in terms of mortality predictive scores and laboratory parameters. Despite a Pitt bacteremia score of &ge; 2 being associated with greater mortality in hematologic disorder patients with CRKP bacteremia,<sup>35</sup> Chen et al.<sup>36</sup> determined that a Charlson comorbidity index score of &ge; 3 was insignificant. Another study on CRKP infections detected that each one-point increase in the SOFA score and procalcitonin levels &ge; 5 &micro;g/l was significant in terms of mortality in multivariate analysis. Furthermore, a platelet count &le; 100,000/&micro;l was found to be significant in univariate analysis, while a CRP level &ge; 150 mg/l was not found to be significant.<sup>37</sup> Using ROC curve analysis, our study determined that the significant cut-off points were 3 for the Charlson comorbidity index score, 5 for the Pitt bacteremia score, 8 for SOFA, 170 mg/l for CRP, 6 &micro;g/l for procalcitonin, and 140,000/&micro;l for platelets.</p>\r\n<p>Procalcitonin and CRP exhibit relatively low AUC and sensitivity values. The specificity of the cut-off points for CRP and procalcitonin is 71% and 60%, respectively. The specificity and sensitivity of the inflammatory markers measured in these infections may differ due to the fact that inflammatory responses developing against infections caused by resistant microorganisms vary from patient to patient. Additionally, the predictive values ​​of these indicators may vary based on patient characteristics (age, gender, comorbidities, stage of infection, etc.). In this context, we believe that the predictive potential of these parameters may be affected by multiple factors.</p>\r\n<p>Although publications present diverse risk factors associated with CRKP bacteremia-related mortality, the most frequently identified ones include septic shock, admission to the ICU, mechanical ventilation, corticosteroid use prior to bacteremia, high comorbidity-mortality/bacteremia scores, immunosuppression, neutropenia, and colistin resistance.<sup>9-11</sup> In our study, multivariate logistic regression analysis demonstrated that the following factors were independent risk factors for 30-day mortality: the development of bacteremia in the ICU or during the COVID-19 pandemic, polymicrobial bacteremia, the use of indwelling intravenous catheters, a procalcitonin level &ge; 6 &micro;g/l, a platelet count &le; 140,000/&micro;l, and a Charlson comorbidity index score &ge; 3. Notably, there was no significant association found between colistin resistance and 30-day mortality.</p>\r\n<p>Contrary to some studies suggesting that different antibiotic combinations may impact mortality, there are numerous publications that report no significant differences in terms of mortality outcomes.<sup>9,11,29,31,33,38,39</sup> In a study by Tumbarello et al.,<sup>39</sup> which evaluated CRKP bacteremia patients, they found that the 30-day mortality was significantly lower in the group treated with ceftazidime-avibactam compared to patients who received other treatments. However, Falcone et al.,<sup>11</sup> in their evaluation of bacteremias caused by KPC-producing <em>K. pneumoniae</em> in 102 ICU patients, discovered no significant difference in 30-day mortality between treatment regimens containing colistin or ceftazidime-avibactam when compared to other treatment groups. Our study found no statistically significant differences between the antibiotic combination groups concerning both 14-day and 30-day mortality. This finding is consistent with the findings of numerous other studies.</p>\r\n<p>The AP-PCR method was employed in our study to investigate clonality among isolates, as opposed to the gold standard method, PFGE. AP-PCR has been reported as a rapid and cost-effective alternative to PFGE, particularly in the differentiation of genetically unrelated isolates. The AP-PCR method was chosen due to its technical convenience, cost-effectiveness, and ability to be successfully implemented in laboratories that have been appropriately optimized. Additionally, there are numerous studies in the literature that have demonstrated the compatibility of both methods.<sup>25,40</sup> The clonality analysis, performed using AP-PCR on 230 CRKP strains in our study, demonstrated the presence of 92 different genotypes, a clustering rate of 78.3%, and the lack of any dominant epidemic strain. In a separate study conducted in our country, which assessed CRKP bacteremias from 2014 to 2018, pulsed-field gel electrophoresis identified 22 distinct genotypes with a clustering rate of 60.5%.<sup>31</sup> Our study&rsquo;s increased clustering rate may be attributed to the inclusion of patients from the COVID-19 pandemic period and the difficulties encountered by healthcare workers in adhering to hospital infection control measures, glove usage, and proper hand hygiene during this critical period. Although the impact of infection control practices was not evaluated in this study, we have reported our data on this issue in another article published by our center that covers the same period and demonstrates that there were deficiencies in infection control measures.<sup>41</sup></p>\r\n<p>The relationship between resistance alleles and clinical course is not well documented in the literature. The evaluation of this relationship is one of the unique aspects of this study. Furthermore, it encompasses an exceptionally large volume of cases over the course of seven years, which encompassed the COVID-19 pandemic. Since this is a single-center study with a large number of cases, the impact of the pandemic can be readily assessed.</p>\r\n<p>In summary, it is important to observe that the emergence of the KPC gene in recent years is a disturbing trend, despite the fact that OXA-48 remains the most frequently detected gene in CRKP bacteremias at our center. Additionally, the observed increase in CRKP bacteremia-associated mortality rates during the COVID-19 pandemic, when infection control measures were disrupted, emphasizes the importance of hand hygiene, glove use, and stringent hospital infection control, especially given the limited treatment options for CRKP bacteremias.</p>\r\n<p><strong>Acknowledgements:</strong> We would like thank to PhD Hatice Yaşar Arsu, Muhammet Aydın and Şerife Kaya for their valuable contributions.</p>\r\n<p><strong>Ethics Committee Approval:</strong> This study was conducted with the approval of the Clinical Research Ethics Committee of İstanbul University-Cerrahpaşa, Cerrahpaşa Faculty of Medicine (approval number: E-83045809-604.01.01-329361, date: 04.03.2022).</p>\r\n<p><strong>Informed Consent:</strong> During hospitalization, informed consent form was signed by the patients themselves or, if not possible, by their legal relatives.</p>\r\n<p><strong>Data Sharing Statement:</strong> The datasets analyzed during the current study are available from the corresponding author upon reasonable request.</p>\r\n<p><strong>Authorship Contributions:</strong> Concept- A.F.K., B.M., G.A.; Design- A.F.K., F.T., N.S.; Supervision- S.Y.K., F.K.&Ccedil;., N.S.; Materials- A.F.K., E.S.T., M.Y., R.K., M.A.K., B.O., G.A.; Data Collection or Processing- A.F.K., E.S.T., M.Y., O.F.B.; Analysis or Interpretation- E.S.T., O.F.B., B.O., B.M., G.A., F.T., N.S.; Literature Search- A.F.K., S.Y.K., İ.İ.B., F.T., N.S.; Writing- A.F.K., O.F.B., R.K., M.A.K., B.M., G.A.; Critical Review- F.K.&Ccedil;., İ.İ.B., B.M., G.A., F.T., N.S.</p>\r\n<p><strong>Conflict of Interest:</strong> The authors declare that they have no conflict of interest.</p>\r\n<p><strong>Funding:</strong> This study was funded by Scientific Research Projects Coordination Unit of İstanbul University-Cerrahpaşa (project number TTU-2022-36607).</p>\r\n<p><strong>Supplementary:</strong> <a href=\"https://balkanmedicaljournal.org/uploads/pdf/supplemantry-2024.2024-5-99.pdf\">https://balkanmedicaljournal.org/uploads/pdf/supplemantry-2024.2024-5-99.pdf</a></p>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<!--\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t<a class=\"menu\" href=\"#summary_en\">Summary</a><a class=\"menu\" href=\"#introduction\">Introduction</a><a class=\"menu\" href=\"#methods\">Methods</a><a class=\"menu\" href=\"#results\">Results</a><a class=\"selected_menu\" href=\"#discussion\">Discussion</a><a class=\"menu\" href=\"#reference\">Reference</a>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t<div  style=\"width:100%; float:left;  padding-bottom:0px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:100%; float:left; min-height:45px; text-align:justify\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<a href=\"\" name=\"reference\"></a>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<!--<h2>Reference</h2>-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t<h3>REFERENCES</h3>\r\n<ol>\r\n<li>Chang D, Sharma L, Dela Cruz CS, Zhang D. Clinical Epidemiology, Risk Factors, and Control Strategies of Klebsiella pneumoniae Infection. <em>Front Microbiol. </em>2021;12:750662.</li>\r\n<li>Antibiotic Resistance Threats in the United States 2019. Centers for Disease Control and Prevention 2019. (cited 2024 May 12).</li>\r\n<li>National Antimicrobial Resistance Surveillance System, 2016 Annual Report. Turkish Public Health Institution, Ministry of Health Ankara 2016. (cited 2024 May 13.)</li>\r\n<li>Hekimoğlu C, Batir E, G&ouml;zel E et al. National Health Care-Associated Infections Surveillance Network (USHIESA) Summary Report 2021. (cited 2024 May 13).</li>\r\n<li>Papp-Wallace KM, Endimiani A, Taracila MA, Bonomo RA. Carbapenems: past, present, and future. <em>Antimicrob Agents Chemother.</em> 2011;55:4943-4960.</li>\r\n<li>Woodworth KR, Walters MS, Weiner LM, et al. Vital Signs: Containment of Novel Multidrug-Resistant Organisms and Resistance Mechanisms - United States, 2006-2017. <em>MMWR Morb Mortal Wkly Rep.</em> 2018;67:396-401.</li>\r\n<li>Boyd SE, Livermore DM, Hooper DC, Hope WW. Metallo-&beta;-Lactamases: Structure, Function, Epidemiology, Treatment Options, and the Development Pipeline. <em>Antimicrob Agents Chemother.</em> 2020;64:e00397-20.</li>\r\n<li>Van Duin D, Doi Y. The global epidemiology of carbapenemase-producing Enterobacteriaceae. <em>Virulence</em>. 2017;8:460-469.</li>\r\n<li>Isler B, &Ouml;zer B, &Ccedil;ınar G, et al. Characteristics and outcomes of carbapenemase harbouring carbapenem-resistant Klebsiella spp. bloodstream infections: a multicentre prospective cohort study in an OXA-48 endemic setting. <em>Eur J Clin Microbiol Infect Dis.</em> 2022;41:841-847.</li>\r\n<li>Balkan II, Alkan M, Ayg&uuml;n G, et al. Colistin resistance increases 28-day mortality in bloodstream infections due to carbapenem-resistant Klebsiella pneumoniae. <em>Eur J Clin Microbiol Infect Dis</em>. 2021;40:2161-2170.</li>\r\n<li>Falcone M, Bassetti M, Tiseo G, et al. Time to appropriate antibiotic therapy is a predictor of outcome in patients with bloodstream infection caused by KPC-producing Klebsiella pneumoniae. <em>Crit Care.</em> 2020;24:29.</li>\r\n<li>Xu M, Fu Y, Kong H, et al. Bloodstream infections caused by Klebsiella pneumoniae: prevalence of blaKPC, virulence factors and their impacts on clinical outcome. <em>BMC Infect Dis</em>. 2018;18:358.</li>\r\n<li>Kang CI, Kim SH, Bang JW, et al. Community-acquired versus nosocomial Klebsiella pneumoniae bacteremia: clinical features, treatment outcomes, and clinical implication of antimicrobial resistance. <em>J Korean Med Sci.</em> 2006;21:816-822.</li>\r\n<li>Singer M, Deutschman CS, Seymour CW, et al. The Third International Consensus Definitions for Sepsis and Septic Shock (Sepsis-3). <em>JAMA.</em> 2016;315:801-810.</li>\r\n<li>Bloodstream Infection Event (Central Line-Associated Bloodstream Infection and Non-central Line Associated Bloodstream Infection) 2024. (cited 2024 May 15).</li>\r\n<li>Roberts FJ. Definition of polymicrobial bacteremia. <em>Rev Infect Dis.</em> 1989;11:1029-1030.</li>\r\n<li>The European Committee on Antimicrobial Susceptibility Testing. Breakpoint tables for interpretation of MICs and zone diameters. Version 14.0, 2024. (cited 2024 May 15).</li>\r\n<li>Yaghoubi S, Zekiy AO, Krutova M, et al. Tigecycline antibacterial activity, clinical effectiveness, and mechanisms and epidemiology of resistance: narrative review. <em>Eur J Clin Microbiol Infect Dis.</em> 2022;41:1003-1022.</li>\r\n<li>FDA-Identified Interpretive Criteria, Tigecycline-Injection products 2023. (cited 2024 June 30).</li>\r\n<li>CLSI. Performance Standards for Antimicrobial Susceptibility Testing. 31st ed. CLSI supplement M100. Clinical and Laboratory Standards Institute; 2021.</li>\r\n<li>EUCAST guidelines for detection of resistance mechanisms and specific resistances of clinical and/or epidemiological importance, Version 2.0 2017. (cited 2024 June 30).</li>\r\n<li>Tamma PD, Aitken SL, Bonomo RA, Mathers AJ, van Duin D, Clancy CJ. Infectious Diseases Society of America 2023 Guidance on the Treatment of Antimicrobial Resistant Gram-Negative Infections. <em>Clin Infect Dis.</em> 2023;18:ciad428.</li>\r\n<li>K&uuml;&ccedil;&uuml;kbasmacı O, Midilli K, Issa G, G&uuml;ven O, G&ouml;n&uuml;ll&uuml; N. A New Multiplex PCR Method for Rapid Detection of Genes Encoding VIM and IMP Types of Metallo Beta Lactamases. <em>Turkiye Klinikleri J Med Sci.</em> 2010;30:1312-1316.</li>\r\n<li>Lee SH, Jeong SH, Lee KJ. Evolution of TEM beta--lactamase genes identified by PCR with newly designed primers in Korean clinical isolates. <em>Clin Microbiol Infect.</em> 2001;7:98-100.</li>\r\n<li>Menekşe Ş, Tanrıverdi ES, Altınay E, et al. A long-lasting Sphingomonas paucimobilis outbreak: A potential for pathogens to persist on environmental devices despite disinfection measures. <em>Am J Infect Control.</em> 2023;51:765-771.</li>\r\n<li>Liu X. Classification accuracy and cut point selection. <em>Stat Med.</em> 2012;31:2676-2686.</li>\r\n<li>Li D, Huang X, Rao H, et al. Klebsiella pneumoniae bacteremia mortality: a systematic review and meta-analysis. <em>Front Cell Infect Microbiol.</em> 2023;13:1157010.</li>\r\n<li>World Health Organization. Central Asian and European surveillance of antimicrobial resistance: annual report 2020. Copenhagen: Regional Office for Europe; 2020. (cited 2024 May 15).</li>\r\n<li>Xiao T, Zhu Y, Zhang S, et al. A Retrospective Analysis of Risk Factors and Outcomes of Carbapenem-Resistant Klebsiella pneumoniae Bacteremia in Nontransplant Patients. <em>J Infect Dis.</em> 2020;16:S174-S183.</li>\r\n<li>Zarkotou O, Pournaras S, Tselioti P, et al. Predictors of mortality in patients with bloodstream infections caused by KPC-producing Klebsiella pneumoniae and impact of appropriate antimicrobial treatment. <em>Clin Microbiol Infect.</em> 2011;17:1798-1803.</li>\r\n<li>Aslan AT, Kırbaş E, Sancak B, et al.; Study Group for Carbapenem Resistance (SCARE). A retrospective observational cohort study of the clinical epidemiology of bloodstream infections due to carbapenem-resistant Klebsiella pneumoniae in an OXA-48 endemic setting. <em>Int J Antimicrob Agents.</em> 2022;59:106554.</li>\r\n<li>Bonomo RA, Burd EM, Conly J, et al. Carbapenemase-Producing Organisms: A Global Scourge. <em>Clin Infect Dis.</em> 2018;66:1290-1297.</li>\r\n<li>Guti&eacute;rrez-Guti&eacute;rrez B, Salamanca E, de Cueto M, et al.; REIPI/ESGBIS/INCREMENT Investigators. Effect of appropriate combination therapy on mortality of patients with bloodstream infections due to carbapenemase-producing Enterobacteriaceae (INCREMENT): a retrospective cohort study. <em>Lancet Infect Dis.</em> 2017;17:726-734.</li>\r\n<li>Wang M, Earley M, Chen L, et al. Clinical outcomes and bacterial characteristics of carbapenem-resistant Klebsiella pneumoniae complex among patients from different global regions (CRACKLE-2): a prospective, multicentre, cohort study. <em>Lancet Infect Dis.</em> 2022;22:401-412.</li>\r\n<li>Meng H, Han L, Niu M, et al. Risk Factors for Mortality and Outcomes in Hematological Malignancy Patients with Carbapenem-Resistant Klebsiella pneumoniae Bloodstream Infections. <em>Infect Drug Resist.</em> 2022;15:4241-4251.</li>\r\n<li>Chen J, Ma H, Huang X, et al. Risk factors and mortality of carbapenem-resistant Klebsiella pneumoniae bloodstream infection in a tertiary-care hospital in China: an eight-year retrospective study. <em>Antimicrob Resist Infect Control.</em> 2022;11:161.</li>\r\n<li>Chen J, Yang Y, Yao H, et al. Prediction of Prognosis in Adult Patients With Carbapenem-Resistant Klebsiella pneumoniae Infection. <em>Front Cell Infect Microbiol.</em> 2022;11:818308.</li>\r\n<li>Fang Y, Zhong Q, Chen Y, et al. Ceftazidime/Avibactam, Polymyxin or Tigecycline as a Rescue Strategy for the Treatment of Carbapenem-Resistant Klebsiella pneumoniae in Bloodstream Infection: A Retrospective Cohort Study. <em>Infect Drug Resist.</em> 2023;16:2963-2971.</li>\r\n<li>Tumbarello M, Trecarichi EM, Corona A, et al. Efficacy of Ceftazidime-Avibactam Salvage Therapy in Patients with Infections Caused by Klebsiella pneumoniae Carbapenemase-producing K. pneumoniae. <em>Clin Infect Dis.</em> 2019;68:355-364.</li>\r\n<li>Fern&aacute;ndez-Cuenca F. Applications of PCR techniques for molecular epidemiology of infectious diseases. <em>Enferm Infecc Microbiol Clin.</em> 2004;22:355-360.</li>\r\n<li>Mete B, Kurt AF, Urkmez S, et al. The Bad Bug is Back: Acinetobacter Baumannii Bacteremia Outbreak during the COVID-19 Pandemic in an Intensive Care Unit. <em>Niger J Clin Pract</em>. 2022;25:702-709.</li>\r\n</ol>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<!--\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t<div\tstyle=\"width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;\">\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t<a class=\"menu\" href=\"#summary_en\">Summary</a><a class=\"menu\" href=\"#introduction\">Introduction</a><a class=\"menu\" href=\"#methods\">Methods</a><a class=\"menu\" href=\"#results\">Results</a><a class=\"menu\" href=\"#discussion\">Discussion</a><a class=\"selected_menu\" href=\"#reference\">Reference</a>\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t-->\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t</div>\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\n\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\t\t\t\t\t\t\t\t\t\t\t\t        \t\t\t\n\t\t\n\t\t\t\t\t\t\t\t\t\t\t        \t</div>\n\t\t\t</div>";
