<div class="col-lg-12 col-md-12 col-sm-12 col-xs-12">
																		        				
												        				<div  style="width:100%; float:left;  padding-bottom:0px;">
												        					<div	style="width:100%; float:left; min-height:45px; text-align:justify">
												        						<a href="" name="summary_en"></a>
												        						<!--<h2>Summary</h2>-->
												        						<p>Genome editing technologies have demonstrated significant potential for treating diseases caused by various molecular changes. These include cancer, cystic fibrosis, diabetes, sickle cell anemia, heart diseases, and rare genetic disorders.<sup>1,2</sup> Genome editing technologies also offer potential solutions for neurodegenerative diseases such as Alzheimer&rsquo;s and Huntington&rsquo;s disease and neuromuscular disorders such as Duchenne&rsquo;s muscular dystrophy, spinal muscular atrophy, and amyotrophic lateral sclerosis. The leading genome editing technologies are zinc-finger nucleases, transcription activatorlike effector nucleases, and clustered regularly interspaced short palindromic repeats (CRISPR)/CRISPR-associated protein 9 (Cas9). Among these technologies, CRISPR/Cas9 is the most widely used due to its versatility, effectiveness, efficiency, and ease of application.<sup>3</sup> This groundbreaking genome manipulation tool is revolutionizing scientific research, medical diagnostics, and therapeutic applications. Major advancements in editing efficacy have propelled genome editing strategies to phase 3 human clinical trials. However, the CRISPR technology is associated with some challenges such as high costs, ineffective delivery methods for specific targets, and poor efficiency of the genome editing processes, particularly due to offtarget effects.<sup>3,4</sup></p>
<p>The CRISPR-Cas9 system employs a single guide RNA (sgRNA) sequence and the Cas9 endonuclease. The sgRNA is formed by combining the CRISPR RNA (crRNA) and trans-activating CRISPR RNA (tracrRNA), which subsequently directs the sgRNA/Cas9 ribonucleoprotein complex to a specific DNA sequence. Once the target DNA is identified, Cas9 induces double-stranded breaks at the target site. These breaks can be repaired via insertions, deletions, additions or inversions. The DNA repair process can either rely on the cell&rsquo;s natural repair mechanisms or utilize custom-designed DNA sequences.<sup>2</sup> However, the system can be inefficient and imprecise, often resulting in small insertions or deletions, as well as unintended large deletions and complex rearrangements.<sup>5</sup> Artificial intelligence (AI) tools are increasingly being used across all areas of science and technology. The integration of AI with genome editing systems offers unparalleled precision and efficiency, which could potentially revolutionize the treatment of mutational diseases and oncology.<sup>6</sup></p>
<p>AI-based machine and deep learning tools have been employed to design sgRNAs for the CRISPR-Cas systems.<sup>7</sup> Off-target effects are primarily influenced by sgRNAs. Thus, the sgRNAs should be carefully designed to maximize the efficiency of the CRISPR-Cas9 gene-editing technology.<sup>2,8</sup> Tools such as DeepCRISPR, CRISTA, and DeepHF can predict the most effective sgRNAs for a given target sequence.<sup>3</sup> These methods use algorithms that are based on the globally reported growing gene-editing datasets, and they help predict CRISPR sgRNA activity and specificity scores.<sup>9,10</sup> AI models design and optimize genome editing techniques, which helps predict editing efficiency, specificity, and outcomes. The deep learning algorithm, which uses endogenous factors and sequence information from genomic datasets, can accurately predict base editing outcomes. It serves as a computational tool for selecting optimal sgRNAs for base editing applications.<sup>11</sup> For example, AI can help select the best editing method (base, prime, or epigenome) for specific target sequences by considering genomic context and potential off-target effects. Furthermore, it can optimize the distribution and expression of editing components such as Cas proteins, sgRNAs, and epigenetic modifiers.<sup>3</sup></p>
<p>AI can solve complex problems beyond human capability and is applicable in the screening, early diagnosis, treatment, and prognosis prediction of critical diseases such as cancer.<sup>12,13</sup> Precision oncology, a tailored approach to cancer treatment, relies on detailed tumor information, including biomarkers, to inform treatment decisions. This method frequently necessitates the analysis of vast datasets using advanced computational techniques. AI can accurately identify genomic profile mutations from multiple clones in heterogeneous tumor biopsies. This capability is essential because detecting cancer stem cells, which are linked to treatment resistance and recurrence, could significantly improve therapeutic strategies.<sup>14,15</sup> Therefore, AI could play a significant role in targeting these specific cells. Moreover, AI models can provide a quantitative framework for examining the relationship between network features and cancer, enabling the identification of potential anticancer targets and the discovery of new drug candidates. Despite its potential, there are still numerous challenges to overcome when applying AI to mutational diseases. These challenges include ensuring data privacy, obtaining diverse and representative datasets, and reducing algorithmic bias. Therefore, the ethical, reliable, and effective application of AI tools in healthcare requires collaboration among technologists, clinicians, and policymakers.</p>
<p>The ongoing evolution of technology, particularly the integration of AI with genome editing, could transform genetics, biomedicine, and healthcare. This could offer new opportunities for significant advancements in human health through enhanced data analysis and tailored treatments.</p>												        					</div>
												        					<!--
												        					<div	style="width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;">
												        									<a class="selected_menu" href="#summary_en">Summary</a><a class="menu" href="#reference">Reference</a>												        					</div>
												        					-->
												        				</div>
												        				
												        				
												        																        				
												        				<div  style="width:100%; float:left;  padding-bottom:0px;">
												        					<div	style="width:100%; float:left; min-height:45px; text-align:justify">
												        						<a href="" name="reference"></a>
												        						<!--<h2>Reference</h2>-->
												        						<h3>REFERENCES</h3>
<ol>
<li>Hayıroğlu Mİ, Altay S. The Role of Artificial Intelligence in Coronary Artery Disease and Atrial Fibrillation. <em>Balkan Med J.</em> 2023;40:151-152.</li>
<li>Sharma G, Sharma AR, Bhattacharya M, Lee SS, Chakraborty C. CRISPR-Cas9: A Preclinical and Clinical Perspective for the Treatment of Human Diseases. <em>Mol Ther. </em>2021;29:571-586.</li>
<li>Dixit S, Kumar A, Srinivasan K, Vincent PMDR, Ramu Krishnan N. Advancing genome editing with artificial intelligence: opportunities, challenges, and future directions. <em>Front Bioeng Biotechnol.</em> 2024;11:1335901.</li>
<li>Mahmoud M, Gobet N, Cruz-D&aacute;valos DI, Mounier N, Dessimoz C, Sedlazeck FJ. Structural variant calling: the long and the short of it. <em>Genome Biol.</em> 2019;20:246.</li>
<li>Choi J, Chen W, Suiter CC, et al. Precise genomic deletions using paired prime editing. <em>Nat Biotechnol.</em> 2022;40:218-226.</li>
<li>Tiwari PK, Ko TH, Dubey R, et al. CRISPR/Cas9 as a therapeutic tool for triple negative breast cancer: from bench to clinics. <em>Front Mol Biosci.</em> 2023;10:1214489.</li>
<li>Wessels HH, Stirn A, M&eacute;ndez-Mancilla A, et al. Prediction of on-target and off-target activity of CRISPR-Cas13d guide RNAs using deep learning. <em>Nat Biotechnol.</em> 2024;42:628-637.</li>
<li>Guo C, Ma X, Gao F, Guo Y. Off-target effects in CRISPR/Cas9 gene editing. <em>Front Bioeng Biotechnol.</em> 2023;11:1143157.</li>
<li>Bhat AA, Nisar S, Mukherjee S, et al. Integration of CRISPR/Cas9 with artificial intelligence for improved cancer therapeutics. <em>J Transl Med.</em> 2022;20:534.</li>
<li>Ko&ccedil;ak B, Cuocolo R, dos Santos DP, Stanzione A, Ugga L. Must-have Qualities of Clinical Research on Artificial Intelligence and Machine Learning. <em>Balkan Med J.</em> 2023;40:3-12.</li>
<li>Yuan T, Wu L, Li S, et al. Deep learning models incorporating endogenous factors beyond DNA sequences improve the prediction accuracy of base editing outcomes. <em>Cell Discov.</em> 2024;10:20.</li>
<li>Usta U, Taştekin E. Present and Future of Artificial Intelligence in Pathology. <em>Balkan Med J.</em> 2024;41:157-158.</li>
<li>Chen ZH, Lin L, Wu CF, Li CF, Xu RH, Sun Y. Artificial intelligence for assisting cancer diagnosis and treatment in the era of precision medicine. <em>Cancer Commun (Lond).</em> 2021;41:1100-1115.</li>
<li>Katti A, Diaz BJ, Caragine CM, Sanjana NE, Dow LE. CRISPR in cancer biology and therapy. <em>Nat Rev Cancer.</em> 2022;22:259-279.</li>
<li>Chen J, Xu L, Li X, Park S. Deep learning models for cancer stem cell detection: a brief review. <em>Front Immunol.</em> 2023;14:1214425.</li>
</ol>												        					</div>
												        					<!--
												        					<div	style="width:20%; float:left; min-height:150px; padding:10px; padding-top:40px;">
												        									<a class="menu" href="#summary_en">Summary</a><a class="selected_menu" href="#reference">Reference</a>												        					</div>
												        					-->
												        				</div>
												        				
												        				
												        															        			
		
											        	</div>
			<